<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="jamia"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3277617/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Use of diverse electronic medical record systems to identify genetic risk for type 2 diabetes within a genome-wide association study</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3277617/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of the American Medical Informatics Association : JAMIA"><meta name="citation_title" content="Use of diverse electronic medical record systems to identify genetic risk for type 2 diabetes within a genome-wide association study"><meta name="citation_authors" content="Abel N Kho, M Geoffrey Hayes, Laura Rasmussen-Torvik, Jennifer A Pacheco, William K Thompson, Loren L Armstrong, Joshua C Denny, Peggy L Peissig, Aaron W Miller, Wei-Qi Wei, Suzette J Bielinski, Christopher G Chute, Cynthia L Leibson, Gail P Jarvik, David R Crosslin, Christopher S Carlson, Katherine M Newton, Wendy A Wolf, Rex L Chisholm, William L Lowe"><meta name="citation_date" content="Mar-Apr 2012"><meta name="citation_issue" content="2"><meta name="citation_volume" content="19"><meta name="citation_firstpage" content="212"><meta name="citation_doi" content="10.1136/amiajnl-2011-000439"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3277617/?report=abstract"><meta name="citation_pmid" content="22101970"><meta name="DC.Title" content="Use of diverse electronic medical record systems to identify genetic risk for type 2 diabetes within a genome-wide association study"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Medical Informatics Association"><meta name="DC.Contributor" content="Abel N Kho"><meta name="DC.Contributor" content="M Geoffrey Hayes"><meta name="DC.Contributor" content="Laura Rasmussen-Torvik"><meta name="DC.Contributor" content="Jennifer A Pacheco"><meta name="DC.Contributor" content="William K Thompson"><meta name="DC.Contributor" content="Loren L Armstrong"><meta name="DC.Contributor" content="Joshua C Denny"><meta name="DC.Contributor" content="Peggy L Peissig"><meta name="DC.Contributor" content="Aaron W Miller"><meta name="DC.Contributor" content="Wei-Qi Wei"><meta name="DC.Contributor" content="Suzette J Bielinski"><meta name="DC.Contributor" content="Christopher G Chute"><meta name="DC.Contributor" content="Cynthia L Leibson"><meta name="DC.Contributor" content="Gail P Jarvik"><meta name="DC.Contributor" content="David R Crosslin"><meta name="DC.Contributor" content="Christopher S Carlson"><meta name="DC.Contributor" content="Katherine M Newton"><meta name="DC.Contributor" content="Wendy A Wolf"><meta name="DC.Contributor" content="Rex L Chisholm"><meta name="DC.Contributor" content="William L Lowe"><meta name="DC.Date" content="Mar-Apr 2012"><meta name="DC.Identifier" content="10.1136/amiajnl-2011-000439"><meta name="DC.Language" content="en"><meta property="og:title" content="Use of diverse electronic medical record systems to identify genetic risk for type 2 diabetes within a genome-wide association study"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association studies (GWAS) require high specificity and large numbers of subjects to identify genotype–phenotype correlations accurately. The aim of this study was to identify type 2 diabetes (T2D) cases and controls for a GWAS, using ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3277617/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-jamia.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3277617/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3277617/pdf/amiajnl-2011-000439.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8D817EDD537EB100000000024D01A6.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3277617%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="protein">Protein</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/76/">J Am Med Inform Assoc</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/205472/">v.19(2); Mar-Apr 2012</a></li><li class="accid">PMC3277617</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-jamia.gif" alt="Logo of jamia" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,0,43,57" alt="Alerts" title="Alerts" href="http://www.oxfordjournals.org/en/connect/email-alerts.html" ref="reftype=publisher&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="43,0,165,57" alt="Author Instructions" title="Author Instructions" href="http://jamia.oxfordjournals.org/for_authors/index.html" ref="reftype=publisher&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="165,0,220,57" alt="Submit" title="Submit" href="https://mc.manuscriptcentral.com/jamia" ref="reftype=publisher&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="220,0,268,57" alt="About" title="About" href="http://jamia.oxfordjournals.org/about" ref="reftype=publisher&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,500,75" alt="JAMIA - The Journal of the American Medical Informatics Association" title="JAMIA - The Journal of the American Medical Informatics Association" href="http://jamia.oxfordjournals.org/" ref="reftype=publisher&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">J Am Med Inform Assoc</a></span>. 2012 Mar-Apr; 19(2): 212–218. </span></div><div><span class="fm-vol-iss-date">Published online 2011 Nov 19. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1136%2Famiajnl-2011-000439" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1136/amiajnl-2011-000439</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3277617</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22101970">22101970</a></div></div></div></div><h1 class="content-title">Use of diverse electronic medical record systems to identify genetic risk for type 2 diabetes within a genome-wide association study</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Kho%20AN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206834442832" co-class="co-affbox">Abel N Kho</a>,<sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1</sup> <a href="/pubmed/?term=Hayes%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206828544528" co-class="co-affbox">M Geoffrey Hayes</a>,<sup>1</sup> <a href="/pubmed/?term=Rasmussen-Torvik%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206832944960" co-class="co-affbox">Laura Rasmussen-Torvik</a>,<sup>1</sup> <a href="/pubmed/?term=Pacheco%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206838137744" co-class="co-affbox">Jennifer A Pacheco</a>,<sup>1</sup> <a href="/pubmed/?term=Thompson%20WK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206875628672" co-class="co-affbox">William K Thompson</a>,<sup>1</sup> <a href="/pubmed/?term=Armstrong%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206875626688" co-class="co-affbox">Loren L Armstrong</a>,<sup>1</sup> <a href="/pubmed/?term=Denny%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206846282528" co-class="co-affbox">Joshua C Denny</a>,<sup>2</sup> <a href="/pubmed/?term=Peissig%20PL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206837501504" co-class="co-affbox">Peggy L Peissig</a>,<sup>3</sup> <a href="/pubmed/?term=Miller%20AW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206837499520" co-class="co-affbox">Aaron W Miller</a>,<sup>3</sup> <a href="/pubmed/?term=Wei%20WQ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206859595008" co-class="co-affbox">Wei-Qi Wei</a>,<sup>4</sup> <a href="/pubmed/?term=Bielinski%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206840420768" co-class="co-affbox">Suzette J Bielinski</a>,<sup>4</sup> <a href="/pubmed/?term=Chute%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206840418784" co-class="co-affbox">Christopher G Chute</a>,<sup>4</sup> <a href="/pubmed/?term=Leibson%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206913612272" co-class="co-affbox">Cynthia L Leibson</a>,<sup>4</sup> <a href="/pubmed/?term=Jarvik%20GP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206875618912" co-class="co-affbox">Gail P Jarvik</a>,<sup>5</sup> <a href="/pubmed/?term=Crosslin%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206875616928" co-class="co-affbox">David R Crosslin</a>,<sup>5</sup> <a href="/pubmed/?term=Carlson%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206875027776" co-class="co-affbox">Christopher S Carlson</a>,<sup>6</sup> <a href="/pubmed/?term=Newton%20KM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206871728224" co-class="co-affbox">Katherine M Newton</a>,<sup>7</sup> <a href="/pubmed/?term=Wolf%20WA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206871726240" co-class="co-affbox">Wendy A Wolf</a>,<sup>8</sup> <a href="/pubmed/?term=Chisholm%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206837074352" co-class="co-affbox">Rex L Chisholm</a>,<sup>1</sup> and  <a href="/pubmed/?term=Lowe%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970" class="affpopup" co-rid="_co_idm140206834399456" co-class="co-affbox">William L Lowe</a><sup>1</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140206834442832"><h3 class="no_margin">Abel N Kho</h3><p><sup>1</sup>Northwestern University, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Kho%20AN%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Abel N Kho</a></div></div><div id="_co_idm140206828544528"><h3 class="no_margin">M Geoffrey Hayes</h3><p><sup>1</sup>Northwestern University, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Hayes%20MG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">M Geoffrey Hayes</a></div></div><div id="_co_idm140206832944960"><h3 class="no_margin">Laura Rasmussen-Torvik</h3><p><sup>1</sup>Northwestern University, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Rasmussen-Torvik%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Laura Rasmussen-Torvik</a></div></div><div id="_co_idm140206838137744"><h3 class="no_margin">Jennifer A Pacheco</h3><p><sup>1</sup>Northwestern University, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Pacheco%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Jennifer A Pacheco</a></div></div><div id="_co_idm140206875628672"><h3 class="no_margin">William K Thompson</h3><p><sup>1</sup>Northwestern University, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Thompson%20WK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">William K Thompson</a></div></div><div id="_co_idm140206875626688"><h3 class="no_margin">Loren L Armstrong</h3><p><sup>1</sup>Northwestern University, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Armstrong%20LL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Loren L Armstrong</a></div></div><div id="_co_idm140206846282528"><h3 class="no_margin">Joshua C Denny</h3><p><sup>2</sup>Vanderbilt University, Nashville, Tennessee, USA</p><div>Find articles by <a href="/pubmed/?term=Denny%20JC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Joshua C Denny</a></div></div><div id="_co_idm140206837501504"><h3 class="no_margin">Peggy L Peissig</h3><p><sup>3</sup>Marshield Clinic, Marshfield, Wisconsin, USA</p><div>Find articles by <a href="/pubmed/?term=Peissig%20PL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Peggy L Peissig</a></div></div><div id="_co_idm140206837499520"><h3 class="no_margin">Aaron W Miller</h3><p><sup>3</sup>Marshield Clinic, Marshfield, Wisconsin, USA</p><div>Find articles by <a href="/pubmed/?term=Miller%20AW%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Aaron W Miller</a></div></div><div id="_co_idm140206859595008"><h3 class="no_margin">Wei-Qi Wei</h3><p><sup>4</sup>Mayo Clinic, Rochester, Minnesota, USA</p><div>Find articles by <a href="/pubmed/?term=Wei%20WQ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Wei-Qi Wei</a></div></div><div id="_co_idm140206840420768"><h3 class="no_margin">Suzette J Bielinski</h3><p><sup>4</sup>Mayo Clinic, Rochester, Minnesota, USA</p><div>Find articles by <a href="/pubmed/?term=Bielinski%20SJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Suzette J Bielinski</a></div></div><div id="_co_idm140206840418784"><h3 class="no_margin">Christopher G Chute</h3><p><sup>4</sup>Mayo Clinic, Rochester, Minnesota, USA</p><div>Find articles by <a href="/pubmed/?term=Chute%20CG%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Christopher G Chute</a></div></div><div id="_co_idm140206913612272"><h3 class="no_margin">Cynthia L Leibson</h3><p><sup>4</sup>Mayo Clinic, Rochester, Minnesota, USA</p><div>Find articles by <a href="/pubmed/?term=Leibson%20CL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Cynthia L Leibson</a></div></div><div id="_co_idm140206875618912"><h3 class="no_margin">Gail P Jarvik</h3><p><sup>5</sup>University of Washington Seattle, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Jarvik%20GP%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Gail P Jarvik</a></div></div><div id="_co_idm140206875616928"><h3 class="no_margin">David R Crosslin</h3><p><sup>5</sup>University of Washington Seattle, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Crosslin%20DR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">David R Crosslin</a></div></div><div id="_co_idm140206875027776"><h3 class="no_margin">Christopher S Carlson</h3><p><sup>6</sup>Fred Hutchinson Cancer Research Center, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Carlson%20CS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Christopher S Carlson</a></div></div><div id="_co_idm140206871728224"><h3 class="no_margin">Katherine M Newton</h3><p><sup>7</sup>Group Health Cooperative, Seattle, Washington, USA</p><div>Find articles by <a href="/pubmed/?term=Newton%20KM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Katherine M Newton</a></div></div><div id="_co_idm140206871726240"><h3 class="no_margin">Wendy A Wolf</h3><p><sup>8</sup>Division of Genetics, Children's Hospital Boston, Boston, Massachusetts, USA</p><div>Find articles by <a href="/pubmed/?term=Wolf%20WA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Wendy A Wolf</a></div></div><div id="_co_idm140206837074352"><h3 class="no_margin">Rex L Chisholm</h3><p><sup>1</sup>Northwestern University, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Chisholm%20RL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">Rex L Chisholm</a></div></div><div id="_co_idm140206834399456"><h3 class="no_margin">William L Lowe</h3><p><sup>1</sup>Northwestern University, Chicago, Illinois, USA</p><div>Find articles by <a href="/pubmed/?term=Lowe%20WL%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22101970">William L Lowe</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140206837596912_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140206837596912_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140206837596912_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140206837596912_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Northwestern University, Chicago, Illinois, USA</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Vanderbilt University, Nashville, Tennessee, USA</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Marshield Clinic, Marshfield, Wisconsin, USA</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Mayo Clinic, Rochester, Minnesota, USA</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>University of Washington Seattle, Seattle, Washington, USA</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Fred Hutchinson Cancer Research Center, Seattle, Washington, USA</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Group Health Cooperative, Seattle, Washington, USA</div><div class="fm-affl" id="aff8" lang="en"><sup>8</sup>Division of Genetics, Children's Hospital Boston, Boston, Massachusetts, USA</div><div><sup><img src="/corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div id="idm140206834064192"><strong>Correspondence to</strong> Abel N Kho, Northwestern University, Division of General Internal Medicine, 750 N. Lake Shore Drive, 10th Floor, Chicago, IL 60611, USA; <a href="mailto:dev@null" data-email="gro.ffmn@ohk.leba" class="oemail">gro.ffmn@ohk.leba</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140206837596912_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Jun 22; Accepted 2011 Oct 27.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140206837596912_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © 2012, Published by the BMJ Publishing Group Limited. For permission to use (where not already granted under a licence) please go to http://group.bmj.com/group/rights-licensing/permissions.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>See "<a href="/pmc/articles/PMC3277638/">Informatics research to enable clinically relevant, personalized genomic medicine</a>" in volume 19 on&nbsp;page&nbsp;149.</div><div style="border-top: 1px solid rgb(102, 102, 153); margin: 5px 10% 3px;"></div><div>This article has been <a href="/pmc/articles/PMC3277617/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3277617/bin/supp_19_2_212__index.html" data-ga-action="click_feat_suppl">supp_19_2_212__index.html</a><span style="color:gray"> (20K)</span></div><div class="small guid">GUID:&nbsp;55AAF428-BA17-459B-8752-46A94A76AE7D</div>
<div><a href="/pmc/articles/PMC3277617/bin/supp_amiajnl-2011-000439_Kho_Diabetes_Online_Appendix.pdf" data-ga-action="click_feat_suppl">supp_amiajnl-2011-000439_Kho_Diabetes_Online_Appendix.pdf</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID:&nbsp;32F9E46E-819A-469A-8742-E3EF3EBE4489</div>
<div><a href="/pmc/articles/PMC3277617/bin/supp_amiajnl-2011-000439_Kho.pdf" data-ga-action="click_feat_suppl">supp_amiajnl-2011-000439_Kho.pdf</a><span style="color:gray"> (12K)</span></div><div class="small guid">GUID:&nbsp;A40F6DF1-0C2B-4927-9AE1-34846E568FC2</div>
</div></dd></dl></div><div id="idm140206830481552" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140206830481552title">Abstract</h2><!--article-meta--><div><div id="__sec1" class="sec sec-first"><h3 id="__sec1title">Objective</h3><p id="__p1" class="p p-first-last">Genome-wide association studies (GWAS) require high specificity and large numbers of subjects to identify genotype–phenotype correlations accurately. The aim of this study was to identify type 2 diabetes (T2D) cases and controls for a GWAS, using data captured through routine clinical care across five institutions using different electronic medical record (EMR) systems.</p></div><div id="__sec2" class="sec"><h3 id="__sec2title">Materials and Methods</h3><p id="__p2" class="p p-first-last">An algorithm was developed to identify T2D cases and controls based on a combination of diagnoses, medications, and laboratory results. The performance of the algorithm was validated at three of the five participating institutions compared against clinician review. A GWAS was subsequently performed using cases and controls identified by the algorithm, with samples pooled across all five institutions.</p></div><div id="__sec3" class="sec"><h3 id="__sec3title">Results</h3><p id="__p3" class="p p-first-last">The algorithm achieved 98% and 100% positive predictive values for the identification of diabetic cases and controls, respectively, as compared against clinician review. By standardizing and applying the algorithm across institutions, 3353 cases and 3352 controls were identified. Subsequent GWAS using data from five institutions replicated the TCF7L2 gene variant (rs7903146) previously associated with T2D.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Discussion</h3><p id="__p4" class="p p-first-last">By applying stringent criteria to EMR data collected through routine clinical care, cases and controls for a GWAS were identified that subsequently replicated a known genetic variant. The use of standard terminologies to define data elements enabled pooling of subjects and data across five different institutions to achieve the robust numbers required for GWAS.</p></div><div id="__sec5" class="sec sec-last"><h3 id="__sec5title">Conclusions</h3><p id="__p5" class="p p-first-last">An algorithm using commonly available data from five different EMR can accurately identify T2D cases and controls for genetic study across multiple institutions.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Analytics, application of biological knowledge to clinical care, bioinformatics, biomedical informatics, clinical phenotyping, controlled terminologies and vocabularies, data mining, EHR, EMR secondary and meaningful use, genetic epidemiology, genetics, genome-wide association studies, genomics, HIT data standards, improving the education and skills training of health professionals, infection control, information retrieval, knowledge representations, linking the genotype and phenotype, medical informatics, modeling, natural-language processing, ontologies, pharmacogenomics, phenotyping, reuseability, translational research</span></div></div><div id="idm140206828779120" class="tsec sec headless whole_rhythm"><p id="__p6" class="p p-first">Type 2 diabetes (T2D) is an increasing public health problem.<sup><a href="#b1" rid="b1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709586">1</a></sup>
<sup><a href="#b2" rid="b2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709578">2</a></sup> Although environmental factors including diet and physical activity contribute to the etiology of T2D, a genetic contribution to T2D has been well documented.<sup><a href="#b3" rid="b3 b4 b5 b6" class=" bibr popnode">3–6</a></sup> Genome-wide association studies (GWAS) have been most effective in identifying T2D susceptibility genes, although susceptibility alleles identified through these approaches have typically conferred small increases in risk requiring large numbers of subjects to identify susceptibility genes.<sup><a href="#b7" rid="b7" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709588">7</a></sup></p><p id="__p7">To date, the traditional approach to case–control studies has been to recruit and phenotype study subjects prospectively. An alternative and more efficient approach would be to identify large numbers of cases and controls among patients receiving routine medical care. Until recently, it was unclear whether data collected through routine clinical care could achieve similar data quality compared with prospective study collection. The Electronic Medical Records and Genomics (eMERGE) consortium formed in 2007 to investigate if electronic medical record (EMR) linked DNA biorepositories can be leveraged for high-throughput genomic research.<sup><a href="#b8" rid="b8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709582">8</a></sup> One of eMERGE's main goals is to assess whether EMR used in routine clinical care provide suitable data to identify individuals with specific phenotypes for GWAS. To date, the eMERGE consortium has successfully developed and validated several algorithms to identify accurately individuals with specific phenotypes.<sup><a href="#b9" rid="b9 b10 b11" class=" bibr popnode">9–11</a></sup> With recent initiatives promoting the widespread adoption of EMR, algorithms leveraging EMR-derived data for the identification of phenotypes may take on increasing importance.<sup><a href="#b12" rid="b12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709577">12</a></sup>
<sup><a href="#b13" rid="b13" class=" bibr popnode">13</a></sup></p><p id="__p8" class="p">Candidate gene and GWAS have demonstrated a large number of genetic variants that contribute to T2D susceptibility providing known targets to test our hypothesis.<sup><a href="#b3" rid="b3 b4 b5 b6 b7" class=" bibr popnode">3–7</a></sup>
<sup><a href="#b14" rid="b14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709587">14</a></sup>
<sup><a href="#b15" rid="b15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_337219899">15</a></sup> Previous studies have documented that data captured through routine clinical care as part of the EMR can successfully identify patients with diabetes,<sup><a href="#b16" rid="b16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709583">16</a></sup>
<sup><a href="#b17" rid="b17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709574">17</a></sup> but T2D is a particularly challenging phenotype. First, it must be distinguished from type 1 diabetes (T1D), which has a similar phenotype of hyperglycemia and shares at least one treatment, insulin, with T2D. However, the genetic underpinnings of T1D and T2D differ.<sup><a href="#b18" rid="b18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709573">18</a></sup>
<sup><a href="#b19" rid="b19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709572">19</a></sup> Second, although T2D is increasingly common in youth and young adults, onset is typically later in life, complicating the identification of a control group at low risk of T2D.</p></div><div id="__sec6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec6title">Objective</h2><p id="__p9" class="p p-first-last">This study aimed to develop an algorithm, using commonly collected data across multiple EMR systems, to identify individuals with T2D and test the hypothesis that EMR-derived phenotypes can be used as an alternative approach to the prospective collection of disease cohorts to identify genetic variants associated with T2D. Described here is an algorithm that successfully identified a multi-ethnic cohort of T2D cases and controls across five institutions and permitted replication of the <em>TCF7L2</em> variant rs7903146, the polymorphisms most strongly genome-wide associated and replicated with T2D.</p></div><div id="__sec7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec7title">Research design and methods</h2><div id="__sec8" class="sec sec-first"><h3 id="__sec8title">eMERGE site overview</h3><p id="__p10" class="p p-first">Five institutions participated in this study. All institutions obtained appropriate approval from their respective institutional review boards, and made use of a common data use agreement to enable data sharing between institutions. Each institution used an EMR for documentation of routine clinical care linked to a research specimen biorepository. <a href="/pmc/articles/PMC3277617/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a> lists key features of each institution's EMR and biorepository. Additional details have been published previously.<sup><a href="#b8" rid="b8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709580">8</a></sup>
<sup><a href="#b20" rid="b20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_337219900">20</a></sup> Notably, each site obtained appropriate patient consent for all study participants, with one site, Vanderbilt University (VU) making use of an opt-out consent model.<sup><a href="#b21" rid="b21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709575">21</a></sup> Study participants represent the subset of patients who receive routine clinical care at study institutions, and also consented to participation within the institutional biorepository. Each eMERGE center selected a primary phenotype for investigation; T2D was led by Northwestern University (NU). The algorithm described below was used to identify all possible T2D case and control individuals at NU, and supplemented with all possible African ancestry cases and controls at VU. These samples were then supplemented with T2D cases and controls identified using the same algorithm from other eMERGE sites where individuals were selected for genotyping using independently derived algorithms for phenotypes of interest at that particular institution (eg, QRS duration at VU, cataracts at Marshfield, vascular disease at Mayo Clinic and dementia at Group Health Cooperative).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p11">Overview of participating institutions' EMR and biorepositories and recruitment models</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140206834414064" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1">Institution</td><td rowspan="1" colspan="1">Biorepository overview</td><td rowspan="1" colspan="1">Recruitment model</td><td rowspan="1" colspan="1">Repository size</td><td rowspan="1" colspan="1">EMR summary</td></tr></thead><tbody><tr><td rowspan="1" colspan="1">Marshfield Clinic Research Foundation (Marshfield, Wisconsin, USA)</td><td rowspan="1" colspan="1">Personalized medicine research project: Geographically defined cohort within an integrated regional healthcare system</td><td rowspan="1" colspan="1">Population based</td><td rowspan="1" colspan="1">20 000 98% Caucasian</td><td rowspan="1" colspan="1">Comprehensive internally developed EMR since 1985 75% participants have 20+ years medical history</td></tr><tr><td rowspan="1" colspan="1">Northwestern University (Chicago, Illinois, USA)</td><td rowspan="1" colspan="1">Nugene project: Northwestern affiliated hospitals and outpatient clinics</td><td rowspan="1" colspan="1">Population based</td><td rowspan="1" colspan="1">10 000 12% AA 8% Hispanic</td><td rowspan="1" colspan="1">Comprehensive vendor-based inpatient (Cerner, Kansas City, Missouri, USA) and outpatient (Epic Systems, Verona, Wisconsin, USA) EMR since 2000 20+ years ICD-9 data</td></tr><tr><td rowspan="1" colspan="1">Vanderbilt University (Nashville, Tennessee, USA)</td><td rowspan="1" colspan="1">BioVU: Vanderbilt Clinic, diverse outpatient clinic</td><td rowspan="1" colspan="1">Population based</td><td rowspan="1" colspan="1">100 000 11% AA</td><td rowspan="1" colspan="1">35+ Years medical history data Comprehensive internally developed EMR since 2000</td></tr><tr><td rowspan="1" colspan="1">Group Health Cooperative (Seattle, Washington, USA)</td><td rowspan="1" colspan="1">GHC Biobank Alzheimer's disease patient registry and adult changes in thought study</td><td rowspan="1" colspan="1">Disease-specific cohort</td><td rowspan="1" colspan="1">4000 (&gt;96% Caucasian)</td><td rowspan="1" colspan="1">Comprehensive vendor-based (Epic Systems) EMR since 2004 20+ years pharmacy data 15+ years ICD-9 data</td></tr><tr><td rowspan="1" colspan="1">Mayo Clinic (Rochester, Minnesota, USA)</td><td rowspan="1" colspan="1">Vascular diseases biorepository</td><td rowspan="1" colspan="1">Disease-specific cohort</td><td rowspan="1" colspan="1">3500 (&gt;96% Caucasian)</td><td rowspan="1" colspan="1">Comprehensive internally developed EMR since 1995 40-year history of data extraction</td></tr></tbody></table></div><div id="largeobj_idm140206834414064" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3277617/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140206839697744"><p id="__p12" class="p p-first-last">AA, African ancestry; EMR, electronic medical record; GHC, Group Health Cooperative; ICD-9, International Classification of Diseases, version 9.</p></div></div></div></div><div id="__sec9" class="sec"><h3 id="__sec9title">Algorithm development</h3><p id="__p13" class="p p-first">We used the existing clinical diagnostic criteria developed by the American Diabetes Association to develop an approach to identify T2D using commonly captured EMR data, including diagnostic codes, medications, and laboratory test results. T2D is typically diagnosed clinically by documenting hyperglycemia, a fasting glucose greater than or equal to 126 mg/dl or random glucose greater than or equal to 200 mg/dl. Our study focused on a non-pregnant adult population and did not utilize the results of oral glucose tolerance tests, which are most commonly used to screen for gestational diabetes. Each site used as many years of EMR data as were available for their study population.</p><p id="__p14">To ensure comparability across sites, we identified the appropriate national standards to define diagnoses, medications, and laboratory tests (see supplementary appendices 1–3, available online only). All sites utilized International Classification of Diseases, 9th revision, clinical modification (ICD-9-CM) diagnostic codes. We defined medications using unique RxNorm codes at an ingredient level and defined laboratory tests using the logical observations identifiers names and codes (LOINC) standard.<sup><a href="#b22" rid="b22" class=" bibr popnode">22</a></sup> We identified the ‘best fit’ LOINC codes by units of measurement and overall frequency of clinical use. We included all patients with ICD-9-CM codes of 250.x0 or 250.x2, except for codes 250.10 and 250.12 (indicative of T2D with ketoacidosis, a condition also closely associated with T1D), patients on T2D medications and/or insulin at any time, and all patients with abnormal glucose (&gt;200 mg/dl) or hemaglobin A1c (HbA1c; ≥6.5%) laboratory test results.</p><p id="__p15">The algorithm was originally developed at an institution with primarily structured data housed within a clinical data warehouse, easily accessed through Structured Query Language (SQL) queries. Each site adapted the algorithm to suit the clinical data stored within their institutional EMR, and to take advantage of local data extraction and analytical tools. All sites utilized a data warehouse, separate from their transactional EMR system to execute the algorithm and avoid impacting EMR performance. Subsequent sites required varying degrees of natural language processing to extract structured data from otherwise unstructured free text clinical notes. One site utilized all diabetes codes due to a noted consistent billing pattern of using ICD-9-CM code 250.00 for both T1D and T2D patients.</p><p id="__p16">In order to increase algorithm specificity and reduce the risk that an outlying observation might inadvertently capture a miscoded diagnosis, we required some redundancy in diagnostic criteria. We required cases with a T2D diagnosis code to have either an abnormal laboratory test or a prescription for a T2D medication. We required cases without a T2D diagnosis code in the EMR to have documentation of both a prescription for a T2D medication and an abnormal glucose (random glucose &gt;200 mg/dl, fasting glucose &gt;125 mg/dl) or a HbA1c laboratory test result of 6.5% or greater (<a href="/pmc/articles/PMC3277617/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>figure 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC3277617/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140206835746784" class="figure"><a href="/pmc/articles/PMC3277617/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3277617_amiajnl-2011-000439fig1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is amiajnl-2011-000439fig1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3277617/bin/amiajnl-2011-000439fig1.jpg"></a></div><div id="largeobj_idm140206835746784" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3277617/figure/fig1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC3277617/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p17">Algorithm for the identification of subjects with type 2 diabetes. *Random glucose &gt;200 mg/dl, fasting glucose &gt;125 mg/dl, HbA1c ≥6.5%. HbA1c, hemoglobin A1c; ICD-9, International Classification of Diseases, version 9; T1DM, type 1 diabetes mellitus; T2DM, type 2 diabetes mellitus.</p></div></div></div><p id="__p18">Patients with a diabetes diagnosis (ICD-9-CM 250.xx) and only documented as on insulin predictably proved the most difficult to categorize as T1D or T2D patients. To differentiate between T1D and T2D in subjects who were treated with insulin, we required that patients treated with insulin alone either have a past prescription for a T2D medication or meet the following criteria: no T1D diagnoses and greater than or equal to two T2D diagnoses entered by a clinician (ie, not billing coders) on different dates.</p><p id="__p19" class="p">We similarly developed an algorithm to identify control subjects without diabetes (<a href="/pmc/articles/PMC3277617/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>figure 2</span></a>). We excluded patients with any diabetes diagnosis (ICD-9-CM codes 250.xx) or any of the diagnoses listed in supplementary table 1 (available online only), patients on insulin or any of the medications listed in supplementary table 2 (available online only), patients who used any diabetic supplies (ie, insulin syringes, glucose monitors), and patients with any abnormal glucose (≥110 mg/dl) or HbA1c (≥6.0%) laboratory values. We also excluded patients with a family history of diabetes, either in the EMR or in questionnaire data if available. We also required controls to have had at least one normal glucose measurement and at least two in-person clinician encounters to ensure that patients had sufficient data in their EMR to determine confidently that they did not have diabetes. <a href="/pmc/articles/PMC3277617/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Figure 2</span></a> depicts the final algorithm for choosing control subjects.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC3277617/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140206837220704" class="figure"><a href="/pmc/articles/PMC3277617/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3277617_amiajnl-2011-000439fig2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is amiajnl-2011-000439fig2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3277617/bin/amiajnl-2011-000439fig2.jpg"></a></div><div id="largeobj_idm140206837220704" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3277617/figure/fig2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC3277617/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p20">Algorithm for the identification of type 2 diabetes controls. *Glucose ≥110 mg/dl, HbA1c ≥6.0%. HbA1c, hemoglobin A1c; ICD-9, International Classification of Diseases, version 9; T1DM, type 1 diabetes mellitus; T2DM, type 2 diabetes mellitus.</p></div></div></div></div><div id="__sec10" class="sec"><h3 id="__sec10title">Algorithm validation</h3><p id="__p21" class="p p-first-last">As an interim step to validate the results of our algorithms before genotyping, three sites conducted a blinded chart review of at least 100 total cases and controls, with one site conducting an additional 50 chart reviews for a related study. Two sites (NU, VU) utilized clinician reviewers, and one site (MCRF) used trained chart reviewers. We reviewed charts of patients identified as either cases or controls in order to assess completely the positive predictive value (PPV) of our algorithm to identify cases and controls accurately for subsequent GWAS. Using manual chart review as a comparison standard, we generated a PPV for both cases and controls, for both iterations of the automated algorithms. Statistical analyses were performed using R,<sup><a href="#b23" rid="b23" class=" bibr popnode">23</a></sup> specifically the epiR<sup><a href="#b24" rid="b24" class=" bibr popnode">24</a></sup> package.</p></div><div id="__sec11" class="sec sec-last"><h3 id="__sec11title">Genetic analysis</h3><p id="__p22" class="p p-first">Genotyping was performed at the Broad Institute and Center for Inherited Disease Research on the Illumina 660W and 1M Bead Chips (Illumina Inc, San Diego, CA, USA). Genotype cleaning and quality control was performed collaboratively by all five sites using a previously described approach.<sup><a href="#b25" rid="b25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_304776725">25</a></sup> Genotype data for rs7903146 in <em>TCF7L2</em> on chromosome 10 from individuals passing quality control, and identified as a T2D case or control was used for this analysis to test the validity of the algorithm. Demographic differences between cohorts was assessed by analysis of variance in R.<sup><a href="#b23" rid="b23" class=" bibr popnode">23</a></sup></p><p id="__p23" class="p p-last">Associations between genotype and T2D case–control status were assessed through linear regression assuming an additive model, and adjusting for site, age, sex, body nass index (BMI), and ancestry using PLINK.<sup><a href="#b26" rid="b26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709585">26</a></sup> We used age and median BMI at the time of earliest diabetes diagnosis or diabetes medication prescription for cases, and age and median BMI at the time of enrollment in the biobank (age when biospecimen was collected) for controls. We excluded BMI measures collected during pregnancy for both cases and controls. The OR, SE of the OR, and p value from each cohort from each of the five European ancestry cohorts were combined in a meta-analysis weighting each strata by the number of samples using default settings in PLINK. This was repeated for the two African ancestry cohorts, and all seven of the cohorts in total. We report p values and OR for a fixed-effect model.</p></div></div><div id="__sec12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">Results</h2><p id="__p24" class="p p-first">In combination, the five sites identified a total of 3353 cases and 3352 controls, of which 3266 cases and 3286 controls passed genomic quality control testing and were included in the genetic analysis. <a href="/pmc/articles/PMC3277617/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a> lists the demographics from each site for samples that passed genomic quality control testing. Each cohort was significantly different (p&lt;0.001) from each other with respect to age, sex, and BMI, although we adjusted for these differences in our subsequent genetic analysis. <a href="/pmc/articles/PMC3277617/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a> summarizes validation results at three participating sites that ranged from 98.2% to 100% PPV for case identification, and were 98–100% PPV for the identification of controls. The association results are presented in <a href="/pmc/articles/PMC3277617/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>table 4</span></a>. Three of the five sites (Group Health, Marshfield, and NU) produced moderate to strong associations between rs7903146 and T2D in European ancestry cohorts (p value range 0.002 to 9.27×10<sup>−5</sup>). At two sites, Mayo Clinic and VU, associations trended in the same direction and approached nominal significance (p=0.1177 and 0.0601, respectively). Allele frequencies for the VU African-American (AA) cohort were similar to the remaining European-American (EA) cohorts, but due to a much smaller sample size failed to reach significance. Among the two AA cohorts, NU and VU yielded similar case and control frequencies of the associated (T) allele at rs7903146, but the small NU AA cohort (N=294) yielded a p value of 0.0867, while the larger VU AA cohort achieved high significance (p=2.25×10<sup>−6</sup>), with the difference attributable to sample size. Our cross-site cohort meta-analyses produced very similar results with even smaller p values that are all highly significant: p=2.98×10<sup>−10</sup> for the EA, p=5.30×10<sup>−7</sup> for AA, and p=2.05×10<sup>−15</sup> for all subjects across all sites. The cross cohort OR for s7903146 was 1.46, which is similar to what has been found previously in other populations.<sup><a href="#b27" rid="b27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709590">27</a></sup></p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p25">Demographics of genotyped cases and controls</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140206914580736" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1">Cohort</td><td rowspan="1" colspan="1">#Samples</td><td rowspan="1" colspan="1">Sex %Male</td><td rowspan="1" colspan="1">Age<sup>*</sup> Mean</td><td rowspan="1" colspan="1">SE</td><td rowspan="1" colspan="1">BMI<sup>†</sup> Mean</td><td rowspan="1" colspan="1">SE</td></tr></thead><tbody><tr><td colspan="7" rowspan="1">Cases</td></tr><tr><td rowspan="1" colspan="1"> GHC EA</td><td rowspan="1" colspan="1" align="center">441</td><td rowspan="1" colspan="1" align="center">50.34</td><td rowspan="1" colspan="1" align="center">74.75</td><td rowspan="1" colspan="1" align="center">0.36</td><td rowspan="1" colspan="1" align="center">29.71</td><td rowspan="1" colspan="1" align="center">0.72</td></tr><tr><td rowspan="1" colspan="1"> Marshfield EA</td><td rowspan="1" colspan="1" align="center">562</td><td rowspan="1" colspan="1" align="center">52.49</td><td rowspan="1" colspan="1" align="center">63.75</td><td rowspan="1" colspan="1" align="center">0.46</td><td rowspan="1" colspan="1" align="center">33.25</td><td rowspan="1" colspan="1" align="center">0.31</td></tr><tr><td rowspan="1" colspan="1"> Mayo EA</td><td rowspan="1" colspan="1" align="center">518</td><td rowspan="1" colspan="1" align="center">70.46</td><td rowspan="1" colspan="1" align="center">64.03</td><td rowspan="1" colspan="1" align="center">0.40</td><td rowspan="1" colspan="1" align="center">32.28</td><td rowspan="1" colspan="1" align="center">0.41</td></tr><tr><td rowspan="1" colspan="1"> NU EA</td><td rowspan="1" colspan="1" align="center">561</td><td rowspan="1" colspan="1" align="center">55.26</td><td rowspan="1" colspan="1" align="center">57.36</td><td rowspan="1" colspan="1" align="center">0.49</td><td rowspan="1" colspan="1" align="center">33.44</td><td rowspan="1" colspan="1" align="center">0.34</td></tr><tr><td rowspan="1" colspan="1"> VU EA</td><td rowspan="1" colspan="1" align="center">331</td><td rowspan="1" colspan="1" align="center">53.78</td><td rowspan="1" colspan="1" align="center">57.77</td><td rowspan="1" colspan="1" align="center">0.70</td><td rowspan="1" colspan="1" align="center">33.29</td><td rowspan="1" colspan="1" align="center">0.47</td></tr><tr><td rowspan="1" colspan="1"> Cross cohort EA</td><td rowspan="1" colspan="1" align="center">2413</td><td rowspan="1" colspan="1" align="center">56.78<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">63.51<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">0.25</td><td rowspan="1" colspan="1" align="center">32.40<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">0.21</td></tr><tr><td rowspan="1" colspan="1"> NU AA</td><td rowspan="1" colspan="1" align="center">184</td><td rowspan="1" colspan="1" align="center">35.33</td><td rowspan="1" colspan="1" align="center">53.30</td><td rowspan="1" colspan="1" align="center">0.85</td><td rowspan="1" colspan="1" align="center">36.22</td><td rowspan="1" colspan="1" align="center">0.75</td></tr><tr><td rowspan="1" colspan="1"> VU AA</td><td rowspan="1" colspan="1" align="center">626</td><td rowspan="1" colspan="1" align="center">38.02</td><td rowspan="1" colspan="1" align="center">53.70</td><td rowspan="1" colspan="1" align="center">0.57</td><td rowspan="1" colspan="1" align="center">34.52</td><td rowspan="1" colspan="1" align="center">0.37</td></tr><tr><td rowspan="1" colspan="1"> Cross cohort AA</td><td rowspan="1" colspan="1" align="center">810</td><td rowspan="1" colspan="1" align="center">37.41<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">53.61<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">0.48</td><td rowspan="1" colspan="1" align="center">34.97<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">0.34</td></tr><tr><td colspan="7" rowspan="1">Controls</td></tr><tr><td rowspan="1" colspan="1"> GHC EA</td><td rowspan="1" colspan="1" align="center">379</td><td rowspan="1" colspan="1" align="center">41.69</td><td rowspan="1" colspan="1" align="center">74.64</td><td rowspan="1" colspan="1" align="center">0.32</td><td rowspan="1" colspan="1" align="center">25.02</td><td rowspan="1" colspan="1" align="center">0.22</td></tr><tr><td rowspan="1" colspan="1"> Marshfield EA</td><td rowspan="1" colspan="1" align="center">432</td><td rowspan="1" colspan="1" align="center">31.94</td><td rowspan="1" colspan="1" align="center">57.94</td><td rowspan="1" colspan="1" align="center">0.45</td><td rowspan="1" colspan="1" align="center">26.91</td><td rowspan="1" colspan="1" align="center">0.20</td></tr><tr><td rowspan="1" colspan="1"> Mayo EA</td><td rowspan="1" colspan="1" align="center">687</td><td rowspan="1" colspan="1" align="center">57.50</td><td rowspan="1" colspan="1" align="center">60.80</td><td rowspan="1" colspan="1" align="center">0.32</td><td rowspan="1" colspan="1" align="center">27.43</td><td rowspan="1" colspan="1" align="center">0.31</td></tr><tr><td rowspan="1" colspan="1"> NU EA</td><td rowspan="1" colspan="1" align="center">670</td><td rowspan="1" colspan="1" align="center">44.33</td><td rowspan="1" colspan="1" align="center">50.05</td><td rowspan="1" colspan="1" align="center">0.50</td><td rowspan="1" colspan="1" align="center">26.85</td><td rowspan="1" colspan="1" align="center">0.20</td></tr><tr><td rowspan="1" colspan="1"> VU EA</td><td rowspan="1" colspan="1" align="center">224</td><td rowspan="1" colspan="1" align="center">49.55</td><td rowspan="1" colspan="1" align="center">58.10</td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">28.04</td><td rowspan="1" colspan="1" align="center">0.48</td></tr><tr><td rowspan="1" colspan="1"> Cross cohort EA</td><td rowspan="1" colspan="1" align="center">2392</td><td rowspan="1" colspan="1" align="center">45.94<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">59.21<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">0.27</td><td rowspan="1" colspan="1" align="center">26.80<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">0.13</td></tr><tr><td rowspan="1" colspan="1"> NU AA</td><td rowspan="1" colspan="1" align="center">112</td><td rowspan="1" colspan="1" align="center">20.54</td><td rowspan="1" colspan="1" align="center">40.28</td><td rowspan="1" colspan="1" align="center">1.06</td><td rowspan="1" colspan="1" align="center">30.46</td><td rowspan="1" colspan="1" align="center">0.76</td></tr><tr><td rowspan="1" colspan="1"> VU AA</td><td rowspan="1" colspan="1" align="center">761</td><td rowspan="1" colspan="1" align="center">34.82</td><td rowspan="1" colspan="1" align="center">45.55</td><td rowspan="1" colspan="1" align="center">0.59</td><td rowspan="1" colspan="1" align="center">29.85</td><td rowspan="1" colspan="1" align="center">0.31</td></tr><tr><td rowspan="1" colspan="1"> Cross cohort AA</td><td rowspan="1" colspan="1" align="center">873</td><td rowspan="1" colspan="1" align="center">32.99<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">44.87<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">0.54</td><td rowspan="1" colspan="1" align="center">29.96<sup>‡</sup></td><td rowspan="1" colspan="1" align="center">0.29</td></tr></tbody></table></div><div id="largeobj_idm140206914580736" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3277617/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="table-fn1"><sup>*</sup>Age, age of onset for cases, and age of enrollment for controls.</div><div id="table-fn2"><sup>†</sup>BMI, body mass index, median BMI at age of onset for cases, and median BMI at last age collected for controls.</div><div id="table-fn3"><sup>‡</sup>Analysis of variance significant at p&lt;0.001.</div><div id="idm140206837331264"><p id="__p29" class="p p-first-last">AA, African ancestry; EA, European ancestry; GHC, Group Health Cooperative; NU, Northwestern University; VU, Vanderbilt University.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p30">Summary of chart review results at three participating sites</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140206822824816" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="3" colspan="1"></td><td colspan="9" rowspan="1">Manual chart review</td></tr><tr><td colspan="3" rowspan="1">Northwestern University<sup>*</sup></td><td colspan="3" rowspan="1">Vanderbilt University<sup>†</sup></td><td colspan="3" rowspan="1">Marshfield Clinic<sup>‡</sup></td></tr><tr><td rowspan="1" colspan="1">Case</td><td rowspan="1" colspan="1">Control</td><td rowspan="1" colspan="1">Total</td><td rowspan="1" colspan="1">Case</td><td rowspan="1" colspan="1">Control</td><td rowspan="1" colspan="1">Total</td><td rowspan="1" colspan="1">Case</td><td rowspan="1" colspan="1">Control</td><td rowspan="1" colspan="1">Total</td></tr></thead><tbody><tr><td colspan="10" rowspan="1">EMR prediction</td></tr><tr><td rowspan="1" colspan="1"> Case</td><td rowspan="1" colspan="1" align="center">56</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">57</td><td rowspan="1" colspan="1" align="center">50</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">50</td><td rowspan="1" colspan="1" align="center">99</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">100</td></tr><tr><td rowspan="1" colspan="1"> Control</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">43</td><td rowspan="1" colspan="1" align="center">43</td><td rowspan="1" colspan="1" align="center">0</td><td rowspan="1" colspan="1" align="center">50</td><td rowspan="1" colspan="1" align="center">50</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1" align="center">49</td><td rowspan="1" colspan="1" align="center">50</td></tr><tr><td rowspan="1" colspan="1"> Total</td><td rowspan="1" colspan="1" align="center">56</td><td rowspan="1" colspan="1" align="center">44</td><td rowspan="1" colspan="1" align="center">100</td><td rowspan="1" colspan="1" align="center">50</td><td rowspan="1" colspan="1" align="center">50</td><td rowspan="1" colspan="1" align="center">100</td><td rowspan="1" colspan="1" align="center">100</td><td rowspan="1" colspan="1" align="center">50</td><td rowspan="1" colspan="1" align="center">150</td></tr></tbody></table></div><div id="largeobj_idm140206822824816" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3277617/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="table-fn4"><sup>*</sup>Clinician reviewers, ANK, WLL.</div><div id="table-fn5"><sup>†</sup>Clinician reviewers not authors on this study.</div><div id="table-fn6"><sup>‡</sup>Trained chart reviewers.</div><div id="idm140206834371728"><p id="__p34" class="p p-first-last">EMR, electronic medical record.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p35">Association results for rs7903146 in TCF7L2 with type 2 diabetes</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140206914591824" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="2" colspan="1"></td><td rowspan="2" colspan="1">N</td><td colspan="2" rowspan="1">Allele frequencies (T)</td><td rowspan="2" colspan="1">OR</td><td rowspan="2" colspan="1">L95</td><td rowspan="2" colspan="1">U95</td><td rowspan="2" colspan="1">p Value</td></tr><tr><td rowspan="1" colspan="1">Cases</td><td rowspan="1" colspan="1">Controls</td></tr></thead><tbody><tr><td rowspan="1" colspan="1">GHC EA</td><td rowspan="1" colspan="1" align="center">813</td><td rowspan="1" colspan="1" align="center">0.327</td><td rowspan="1" colspan="1" align="center">0.248</td><td rowspan="1" colspan="1" align="center">1.59</td><td rowspan="1" colspan="1" align="center">1.25</td><td rowspan="1" colspan="1" align="center">2.02</td><td rowspan="1" colspan="1">0.0002</td></tr><tr><td rowspan="1" colspan="1">Marshfield EA</td><td rowspan="1" colspan="1" align="center">930</td><td rowspan="1" colspan="1" align="center">0.323</td><td rowspan="1" colspan="1" align="center">0.243</td><td rowspan="1" colspan="1" align="center">1.49</td><td rowspan="1" colspan="1" align="center">1.15</td><td rowspan="1" colspan="1" align="center">1.93</td><td rowspan="1" colspan="1">0.0025</td></tr><tr><td rowspan="1" colspan="1">Mayo EA</td><td rowspan="1" colspan="1" align="center">1159</td><td rowspan="1" colspan="1" align="center">0.303</td><td rowspan="1" colspan="1" align="center">0.281</td><td rowspan="1" colspan="1" align="center">1.17</td><td rowspan="1" colspan="1" align="center">0.96</td><td rowspan="1" colspan="1" align="center">1.43</td><td rowspan="1" colspan="1">0.1177</td></tr><tr><td rowspan="1" colspan="1">NU EA</td><td rowspan="1" colspan="1" align="center">1229</td><td rowspan="1" colspan="1" align="center">0.353</td><td rowspan="1" colspan="1" align="center">0.274</td><td rowspan="1" colspan="1" align="center">1.51</td><td rowspan="1" colspan="1" align="center">1.23</td><td rowspan="1" colspan="1" align="center">1.86</td><td rowspan="1" colspan="1">9.27×10<sup>−5</sup></td></tr><tr><td rowspan="1" colspan="1">VU EA</td><td rowspan="1" colspan="1" align="center">396</td><td rowspan="1" colspan="1" align="center">0.334</td><td rowspan="1" colspan="1" align="center">0.263</td><td rowspan="1" colspan="1" align="center">1.42</td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">2.04</td><td rowspan="1" colspan="1">0.0601</td></tr><tr><td rowspan="1" colspan="1">Cross cohort EA (Meta)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.328</td><td rowspan="1" colspan="1" align="center">0.265</td><td rowspan="1" colspan="1" align="center">1.41</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">2.98×10<sup>−10</sup></td></tr><tr><td rowspan="1" colspan="1">NU AA</td><td rowspan="1" colspan="1" align="center">294</td><td rowspan="1" colspan="1" align="center">0.353</td><td rowspan="1" colspan="1" align="center">0.241</td><td rowspan="1" colspan="1" align="center">0.94</td><td rowspan="1" colspan="1" align="center">0.24</td><td rowspan="1" colspan="1" align="center">2.39</td><td rowspan="1" colspan="1">0.08672</td></tr><tr><td rowspan="1" colspan="1">VU AA</td><td rowspan="1" colspan="1" align="center">1021</td><td rowspan="1" colspan="1" align="center">0.353</td><td rowspan="1" colspan="1" align="center">0.260</td><td rowspan="1" colspan="1" align="center">1.35</td><td rowspan="1" colspan="1" align="center">0.11</td><td rowspan="1" colspan="1" align="center">2.07</td><td rowspan="1" colspan="1">2.25×10<sup>−6</sup></td></tr><tr><td rowspan="1" colspan="1">Cross cohort AA (Meta)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.353</td><td rowspan="1" colspan="1" align="center">0.258</td><td rowspan="1" colspan="1" align="center">1.64</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">5.30×10<sup>−7</sup></td></tr><tr><td rowspan="1" colspan="1">Cross-cohort all (Meta)</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.334</td><td rowspan="1" colspan="1" align="center">0.263</td><td rowspan="1" colspan="1" align="center">1.46</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1">2.05×10<sup>−15</sup></td></tr></tbody></table></div><div id="largeobj_idm140206914591824" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3277617/table/tbl4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm140206837285408"><p id="__p36" class="p p-first-last">AA, African ancestry; EA, European ancestry; GHC, Group Health Cooperative; L95, 95% CI lower bound; NU, Northwestern University; U95, 95% CI upper bound; VU, Vanderbilt University.</p></div></div></div></div><div id="__sec13" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec13title">Discussion</h2><p id="__p37" class="p p-first">In this study, we developed and validated an algorithm to identify cases with T2D and controls using standardized data elements captured through routine clinical care across five different EMR systems. Despite variations in data capture and completeness across the different systems, by applying stringent minimum criteria, and clear definition of data elements through an iterative process, we developed a final algorithm with a 98% PPV for cases and a 100% PPV for controls. We subsequently used identified samples pooled across sites to perform a GWAS.</p><p id="__p38">The association tests between rs7903146 and T2D in the five EMR-derived cohorts yielded similar results to those from purposefully collected T2D case and control cohorts. In a recent meta-analysis of 29 195 T2D control subjects and 17 202 T2D case subjects from 27 populations spanning the globe Cauchi <em>et al</em><sup><a href="#b27" rid="b27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709579">27</a></sup> found the OR for developing T2D was 1.46 per copy of the rs7903146 T allele. We generated the exact same OR point estimate (1.46) for our EMR-derived samples using meta-analytical techniques in the pooled samples. Perhaps more importantly, our work demonstrates the power that can be achieved by combining samples across sites, evidenced by the highly significant p values from the cross-cohort analyses.</p><p id="__p39">Our work expands on earlier studies to identify patients with diabetes from EMR. Previously, Wilke <em>et al</em><sup><a href="#b17" rid="b17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709581">17</a></sup> at Marshfield Clinic developed an effective algorithm to identify diabetes mellitus patients, but did not specifically differentiate between T1D and T2D. Other studies have utilized laboratory values, or diagnoses, laboratory tests and natural language processing to achieve high specificity for the identification of T1D and T2D.<sup><a href="#b16" rid="b16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709576">16</a></sup>
<sup><a href="#b28" rid="b28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709591">28</a></sup> A related study used diagnoses and medications to identify patients with conditions that are risk factors for T2D, which were in turn used to identify patients with undiagnosed diabetes.<sup><a href="#b29" rid="b29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709589">29</a></sup></p><p id="__p40">We identified and addressed a number of specific challenges when developing the algorithms. We created specific definitions for cases and controls to avoid confounding by the inclusion of cases with T1D and, as much as possible, controls at risk of T2D, which has not, as yet, manifested itself. In EMR, fasting status at the time of blood draw for a patient was frequently not available. We therefore assumed that all glucose laboratory test results were not taken during the fasting state, so we used a lower glucose cut-off for controls, which resulted in lower sensitivity but higher specificity. In developing the final algorithm, a potential source of bias was recognized in that initially T2D subjects who were treated with insulin alone were excluded, although subjects with diabetes on insulin together with one of the diabetes medications listed above were eligible for inclusion. This approach would select against T2D subjects with significant pancreatic β-cell failure. Another problem presented by patients on insulin alone and an ICD-9-CM code for T2D is that some of these patients could represent patients with T1D, which was misclassified as T2D because of the age of onset or other issues. To address this, we identified patients on insulin alone as cases if they had been on a T2D medication in the past, or if they had at least two visits (on different dates) with a clinician who entered T2D diagnoses (ie, in the problem list or the encounter diagnosis).</p><p id="__p41">Identifying controls presented a challenge to ensure that the control group was not ‘contaminated’ with cases, which would negatively impact power in genetic studies. We operated on the principle that absence of a diagnosis, prescribed medications, laboratory results, or other data in the EMR did not necessarily correlate with true patient status, but may reflect the selective capture of data within the EMR. Particularly at tertiary care centers, some patients receive only a portion of their care at the center. To address this challenge, we required that controls have a minimal amount of data represented in the EMR. In particular, we required controls to have had glucose testing with normal results at least once and to have at least two in-person clinician encounters. Moreover, to eliminate younger patients at increased risk of T2D but in whom the disease was not manifest, potential controls with a family history of diabetes were excluded. Another potential confounder was patients with diet controlled diabetes, although our algorithm was developed with the assumption that these patients would either have an ICD-9-CM code for T2D or an abnormal laboratory test result, which would exclude them from the control group.</p><p id="__p42">Lack of standardization across EMR posed a challenge for the cross-site implementation and even within a given site where different EMR were in use. As a consortium, we identified the consolidated health informatics standards as the common lingua franca to achieve comparability of data across sites. For medications, we mapped medications to RxNORM codes at the generic name level as the common link between sites.<sup><a href="#b30" rid="b30" class=" bibr popnode">30</a></sup> For purposes of easier cross-institution sharing, we identified ingredient level RxNORM codes (included in the supplementary appendix, available online only) to reduce the total number of codes. We used LOINC codes specifically to define tests for glucose and HbA1C levels and ICD-9-CM codes for diagnoses. Despite these efforts the portability of algorithms across diverse sites poses a significant challenge and our future work is focused on developing methods to scale phenotyping more broadly. For example, we noted significant differences across sites in algorithm computing time, ranging from less than 10 s at a site using an optimized commercial data warehouse to 40 h at a site sequentially extracting categories of data using statistical software on their data warehouse. To this end, we include a link to our data dictionary, sample SQL code, and a data workflow built on an open source data mining tool for other investigators to explore: <a href="https://www.mc.vanderbilt.edu/victr/dcc/projects/acc/index.php/Library_of_Phenotype_Algorithms#Type_II_Diabetes" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.mc.vanderbilt.edu/victr/dcc/projects/acc/index.php/Library_of_Phenotype_Algorithms#Type_II_Diabetes</a>.</p><p id="__p43">Our study had a number of limitations. Study sites represent institutions with a significant research focus, and this may affect how data are routinely captured within the EMR. Study sites varied in the number of years of data available in the EMR and the degree of care fragmentation. Preliminary evidence suggests that the absence of longitudinal data and fragmentation of care across sites may decrease the specificity of our algorithm. Additional studies are under way to quantify these effects in greater detail. Rates of T2D varied across sites from 1.0% of the total available biorepository to 14.8% at Mayo, compared with an approximate rate of 8% for diabetes (all types) for the general population.<sup><a href="#b31" rid="b31" class=" bibr popnode">31</a></sup> Rate differences are likely to be due to bias in sample selection for genotyping, for which only NU selected all possible T2D cases and controls for genotyping. Other sites performed the T2D case and control algorithms on their already genotyped cohorts, which were selected for genotyping based on their suitability for other phenotypes (eg, QRS duration at VU, cataracts at Marshfield, vascular disease at Mayo Clinic and dementia at Group Health Cooperative/UW). Other sources of bias include variation in biorepository recruitment (eg, Mayo Clinic's biorepository focused on patients with vascular disease, strongly associated with T2D) and variation in local coding practices.<sup><a href="#b32" rid="b32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_297709584">32</a></sup></p><p id="__p44">While the Mayo EA, VU EA, and NU AA results do not reach nominal significance they do approach significance (p=0.11, 0.06, and 0.08, respectively) and all trend in the same direction as the remaining subcohorts. The most likely explanation for the VU EA and NU AA lack of significance is reduced power from relatively small sample size for a GWAS. The Mayo EA lack of significance may be due to the selection bias, as these samples were not selected for genotyping based on the T2D case and control algorithm, but rather an algorithm designed to identify cardiovascular disease phenotypes. As noted, 14.8% of this biased Mayo cohort were identified as a T2D case, significantly higher than the national population prevalence of this disease. We suspect increased co-occurrence of cardiovascular and metabolic diseases may contribute to the reduction in significance through an increased prevalence of undiagnosed T2D among the controls. Importantly, despite the failure to achieve significance for replication of TCF7L2 at individual sites, pooling samples across sites achieved highly significant results, supporting our collective approach.</p><p id="__p45" class="p p-last">In conclusion, we describe a practical approach to the identification of T2D cases and controls for GWAS using data captured in routine clinical care across five distinct EMR. To achieve the high specificity required for GWAS, we refined an algorithm over multiple iterations, and applied stringent criteria and nationally recognized coding standards to facilitate portability across different EMR. Although the overall number of cases and controls decreased with the increased specificity needed for GWAS, by generalizing the algorithm across diverse EMR we identified the large number of cases and controls needed for a well-powered GWAS, and generated the exact OR point estimate we expected from the literature. Applying this approach across a large number of institutions provides an alternative approach for generating a large cohort of T2D cases and controls to understand better the associations between genetics and expressions of disease.</p></div><div id="__sec14" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec14title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140206834451616"><a href="/pmc/articles/PMC3277617/bin/supp_19_2_212__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>


</div></div><div id="idm140206838752400" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140206838752400title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="idm140206838752272"></p><p id="__p46" class="p p-first-last"><strong>Funding:</strong>
The eMERGE Network was initiated and funded by NHGRI, with additional funding from NIGMS through the following grants: <span>U01-HG-004610</span> (Group Health Cooperative); <span>U01-HG-004608</span> (Marshfield Clinic); <span>U01-HG-04599</span> (Mayo Clinic); <span>U01HG004609</span> (Northwestern University); <span>U01-HG-04603</span> (Vanderbilt University, also serving as the Coordinating Center), and the State of Washington Life Sciences Discovery Fund award to the Northwest Institute of Medical Genetics. The Northwestern University Enterprise Data Warehouse was funded in part by a grant from the National Center for Research Resources, <span>UL1RR025741</span>. The genetic data are deposited in dbGaP (accession numbers phs000170, phs000188, phs000203, phs000234, phs000237).</p><p></p><p class="fn sec" id="idm140206835473888"> </p><p id="__p47" class="p p-first-last"><strong>Competing interests:</strong> None.</p><p></p><p class="fn sec" id="idm140206846619440"></p><p id="__p48" class="p p-first-last"><strong>Patient consent:</strong> Obtained.</p><p></p><p class="fn sec" id="idm140206846618320"></p><p id="__p49" class="p p-first-last"><strong>Ethics approval:</strong> Ethics approval was provided by institutional review boards from all participating sites (Northwestern University, Vanderbilt University, Group Health Cooperative, Marshfield Clinic, Mayo Clinic).</p><p></p><div>Contributed by <p id="__p50" class="p p-first-last"><strong>Contributors:</strong> ANK developed the algorithm, collected and analyzed data, performed chart reviews, and wrote the manuscript. MGH conducted genetic analyses, and wrote the manuscript. LRT conducted genetic analyses, and reviewed and edited the manuscript. JAP collected and analyzed data, and wrote the manuscript. WKT created the standardized data and workflow within KNIME. LLA conducted genetic analyses. JCD collected data, performed chart reviews and reviewed and edited the manuscript. PLP collected and analyzed data and reviewed and edited the manuscript. AWM collected and analyzed data. WQW collected and analyzed data and reviewed and edited the manuscript. SJB collected and analyzed data. CGC reviewed and edited the manuscript. CLL collected and analyzed data. GPJ collected and analyzed data. DRC performed genetic analyses. CSC collected and analyzed data. KMN reviewed and edited the manuscript. WAW reviewed and edited the manuscript, RLC reviewed and edited the manuscript. WLL developed the algorithm and wrote the manuscript.</p></div><p class="fn sec" id="idm140206915245456"></p><p id="__p51" class="p p-first-last"><strong>Provenance and peer review:</strong> Not commissioned; externally peer reviewed.</p><p></p></div></div><div id="idm140206915244448" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140206915244448title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="b1">1. <span class="mixed-citation">Fox CS, Pencina MJ, Meigs JB, et al. 
<span class="ref-title">Trends in the incidence of type 2 diabetes mellitus from the 1970s to the 1990s: the Framingham Heart Study</span>. <span class="ref-journal">Circulation</span>
2006;<span class="ref-vol">113</span>:2914–18 [<a href="/pubmed/16785337" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Trends+in+the+incidence+of+type+2+diabetes+mellitus+from+the+1970s+to+the+1990s:+the+Framingham+Heart+Study&amp;author=CS+Fox&amp;author=MJ+Pencina&amp;author=JB+Meigs&amp;volume=113&amp;publication_year=2006&amp;pages=2914-18&amp;pmid=16785337&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b2">2. <span class="mixed-citation">Sloan FA, Bethel MA, Ruiz D, Jr, et al. 
<span class="ref-title">The growing burden of diabetes mellitus in the US elderly population</span>. <span class="ref-journal">Arch Intern Med</span>
2008;<span class="ref-vol">168</span>:192–9; discussion 199. [<a href="/pubmed/18227367" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Intern+Med&amp;title=The+growing+burden+of+diabetes+mellitus+in+the+US+elderly+population&amp;author=FA+Sloan&amp;author=MA+Bethel&amp;author=D+Ruiz&amp;volume=168&amp;publication_year=2008&amp;pages=192-9&amp;pmid=18227367&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b3">3. <span class="mixed-citation">Scott LJ, Mohlke KL, Bonnycastle LL, et al. 
<span class="ref-title">A genome-wide association study of type 2 diabetes in Finns detects multiple susceptibility variants</span>. <span class="ref-journal">Science</span>
2007;<span class="ref-vol">316</span>:1341–5 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3214617/">PMC free article</a>]</span> [<a href="/pubmed/17463248" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+genome-wide+association+study+of+type+2+diabetes+in+Finns+detects+multiple+susceptibility+variants&amp;author=LJ+Scott&amp;author=KL+Mohlke&amp;author=LL+Bonnycastle&amp;volume=316&amp;publication_year=2007&amp;pages=1341-5&amp;pmid=17463248&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b4">4. <span class="mixed-citation">Sladek R, Rocheleau G, Rung J, et al. 
<span class="ref-title">A genome-wide association study identifies novel risk loci for type 2 diabetes</span>. <span class="ref-journal">Nature</span>
2007;<span class="ref-vol">445</span>:881–5 [<a href="/pubmed/17293876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+genome-wide+association+study+identifies+novel+risk+loci+for+type+2+diabetes&amp;author=R+Sladek&amp;author=G+Rocheleau&amp;author=J+Rung&amp;volume=445&amp;publication_year=2007&amp;pages=881-5&amp;pmid=17293876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b5">5. <span class="mixed-citation">Steinthorsdottir V, Thorleifsson G, Reynisdottir I, et al. 
<span class="ref-title">A variant in CDKAL1 influences insulin response and risk of type 2 diabetes</span>. <span class="ref-journal">Nat Genet</span>
2007;<span class="ref-vol">39</span>:770–5 [<a href="/pubmed/17460697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+variant+in+CDKAL1+influences+insulin+response+and+risk+of+type+2+diabetes&amp;author=V+Steinthorsdottir&amp;author=G+Thorleifsson&amp;author=I+Reynisdottir&amp;volume=39&amp;publication_year=2007&amp;pages=770-5&amp;pmid=17460697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b6">6. <span class="mixed-citation">Wellcome Trust Case Control Consortium
<span class="ref-title">Genome-wide association study of 14,000 cases of seven common diseases and 3,000 shared controls</span>. <span class="ref-journal">Nature</span>
2007;<span class="ref-vol">447</span>:661–78 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2719288/">PMC free article</a>]</span> [<a href="/pubmed/17554300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome-wide+association+study+of+14,000+cases+of+seven+common+diseases+and+3,000+shared+controls&amp;volume=447&amp;publication_year=2007&amp;pages=661-78&amp;pmid=17554300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b7">7. <span class="mixed-citation">Groves CJ, Zeggini E, Minton J, et al. 
<span class="ref-title">Association analysis of 6,736 U.K. subjects provides replication and confirms TCF7L2 as a type 2 diabetes susceptibility gene with a substantial effect on individual risk</span>. <span class="ref-journal">Diabetes</span>
2006;<span class="ref-vol">55</span>:2640–4 [<a href="/pubmed/16936215" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Association+analysis+of+6,736+U.K.+subjects+provides+replication+and+confirms+TCF7L2+as+a+type+2+diabetes+susceptibility+gene+with+a+substantial+effect+on+individual+risk&amp;author=CJ+Groves&amp;author=E+Zeggini&amp;author=J+Minton&amp;volume=55&amp;publication_year=2006&amp;pages=2640-4&amp;pmid=16936215&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b8">8. <span class="mixed-citation">McCarty CA, Chisholm RL, Chute CG, et al. ; eMERGE Team
<span class="ref-title">The eMERGE Network: a consortium of biorepositories linked to electronic medical records data for conducting genomic studies</span>. <span class="ref-journal">BMC Med Genomics</span>
2011;<span class="ref-vol">4</span>:13. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3038887/">PMC free article</a>]</span> [<a href="/pubmed/21269473" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Med+Genomics&amp;title=The+eMERGE+Network:+a+consortium+of+biorepositories+linked+to+electronic+medical+records+data+for+conducting+genomic+studies&amp;author=CA+McCarty&amp;author=RL+Chisholm&amp;author=CG+Chute&amp;volume=4&amp;publication_year=2011&amp;pages=13&amp;pmid=21269473&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b9">9. <span class="mixed-citation">Kullo IJ, Fan J, Pathak J, et al. 
<span class="ref-title">Leveraging informatics for genetic studies: use of the electronic medical record to enable a genome-wide association study of peripheral arterial disease</span>. <span class="ref-journal">J Am Med Inform Assoc</span>
2010;<span class="ref-vol">17</span>:568–74 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2995686/">PMC free article</a>]</span> [<a href="/pubmed/20819866" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Med+Inform+Assoc&amp;title=Leveraging+informatics+for+genetic+studies:+use+of+the+electronic+medical+record+to+enable+a+genome-wide+association+study+of+peripheral+arterial+disease&amp;author=IJ+Kullo&amp;author=J+Fan&amp;author=J+Pathak&amp;volume=17&amp;publication_year=2010&amp;pages=568-74&amp;pmid=20819866&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b10">10. <span class="mixed-citation">Denny JC, Ritchie MD, Crawford DC, et al. 
<span class="ref-title">Identification of genomic predictors of atrioventricular conduction: using electronic medical records as a tool for genome science</span>. <span class="ref-journal">Circulation</span>
2010;<span class="ref-vol">122</span>:2016–21 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2991609/">PMC free article</a>]</span> [<a href="/pubmed/21041692" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Identification+of+genomic+predictors+of+atrioventricular+conduction:+using+electronic+medical+records+as+a+tool+for+genome+science&amp;author=JC+Denny&amp;author=MD+Ritchie&amp;author=DC+Crawford&amp;volume=122&amp;publication_year=2010&amp;pages=2016-21&amp;pmid=21041692&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b11">11. <span class="mixed-citation">Ritchie MD, Denny JC, Crawford DC, et al. 
<span class="ref-title">Robust replication of genotype–phenotype associations across multiple diseases in an electronic medical record</span>. <span class="ref-journal">Am J Hum Genet</span>
2010;<span class="ref-vol">86</span>:560–72 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2850440/">PMC free article</a>]</span> [<a href="/pubmed/20362271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Robust+replication+of+genotype–phenotype+associations+across+multiple+diseases+in+an+electronic+medical+record&amp;author=MD+Ritchie&amp;author=JC+Denny&amp;author=DC+Crawford&amp;volume=86&amp;publication_year=2010&amp;pages=560-72&amp;pmid=20362271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b12">12. <span class="mixed-citation">Blumenthal D.
<span class="ref-title">Stimulating the adoption of health information technology</span>. <span class="ref-journal">N Engl J Med</span>
2009;<span class="ref-vol">360</span>:1477–9 [<a href="/pubmed/19321856" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Stimulating+the+adoption+of+health+information+technology&amp;author=D+Blumenthal&amp;volume=360&amp;publication_year=2009&amp;pages=1477-9&amp;pmid=19321856&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b13">13. <span class="mixed-citation">Congress US
<span class="ref-journal">American Recovery and Reinvestment Act</span>. Washington DC, USA: United States Congress, 2009 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=American+Recovery+and+Reinvestment+Act&amp;publication_year=2009&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b14">14. <span class="mixed-citation">Saxena R, Voight BF, Lyssenko V, et al. ; Diabetes Genetics Initiative of Broad Institute of Harvard and MIT, Lund University, and Novartis Institutes of BioMedical Research
<span class="ref-title">Genome-wide association analysis identifies loci for type 2 diabetes and triglyceride levels</span>. <span class="ref-journal">Science</span>
2007;<span class="ref-vol">316</span>:1331–6 [<a href="/pubmed/17463246" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Genome-wide+association+analysis+identifies+loci+for+type+2+diabetes+and+triglyceride+levels&amp;author=R+Saxena&amp;author=BF+Voight&amp;author=V+Lyssenko&amp;volume=316&amp;publication_year=2007&amp;pages=1331-6&amp;pmid=17463246&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b15">15. <span class="mixed-citation">Zeggini E, Weedon MN, Lindgren CM, et al. 
<span class="ref-title">Replication of genome-wide association signals in UK samples reveals risk loci for type 2 diabetes</span>. <span class="ref-journal">Science</span>
2007;<span class="ref-vol">316</span>:1336–41 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772310/">PMC free article</a>]</span> [<a href="/pubmed/17463249" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Replication+of+genome-wide+association+signals+in+UK+samples+reveals+risk+loci+for+type+2+diabetes&amp;author=E+Zeggini&amp;author=MN+Weedon&amp;author=CM+Lindgren&amp;volume=316&amp;publication_year=2007&amp;pages=1336-41&amp;pmid=17463249&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b16">16. <span class="mixed-citation">Turchin A, Kohane IS, Pendergrass ML.
<span class="ref-title">Identification of patients with diabetes from the text of physician notes in the electronic medical record</span>. <span class="ref-journal">Diabetes Care</span>
2005;<span class="ref-vol">28</span>:1794–5 [<a href="/pubmed/15983338" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes+Care&amp;title=Identification+of+patients+with+diabetes+from+the+text+of+physician+notes+in+the+electronic+medical+record&amp;author=A+Turchin&amp;author=IS+Kohane&amp;author=ML+Pendergrass&amp;volume=28&amp;publication_year=2005&amp;pages=1794-5&amp;pmid=15983338&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b17">17. <span class="mixed-citation">Wilke RA, Berg RL, Peissig P, et al. 
<span class="ref-title">Use of an electronic medical record for the identification of research subjects with diabetes mellitus</span>. <span class="ref-journal">Clin Med Res</span>
2007;<span class="ref-vol">5</span>:1–7 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1855339/">PMC free article</a>]</span> [<a href="/pubmed/17456828" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Med+Res&amp;title=Use+of+an+electronic+medical+record+for+the+identification+of+research+subjects+with+diabetes+mellitus&amp;author=RA+Wilke&amp;author=RL+Berg&amp;author=P+Peissig&amp;volume=5&amp;publication_year=2007&amp;pages=1-7&amp;pmid=17456828&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b18">18. <span class="mixed-citation">McCarthy MI.
<span class="ref-title">Genomics, type 2 diabetes, and obesity</span>. <span class="ref-journal">N Engl J Med</span>
2010;<span class="ref-vol">363</span>:2339–50 [<a href="/pubmed/21142536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Genomics,+type+2+diabetes,+and+obesity&amp;author=MI+McCarthy&amp;volume=363&amp;publication_year=2010&amp;pages=2339-50&amp;pmid=21142536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b19">19. <span class="mixed-citation">Pociot F, Akolkar B, Concannon P, et al. 
<span class="ref-title">Genetics of type 1 diabetes: what's next?</span>
<span class="ref-journal">Diabetes</span>
2010;<span class="ref-vol">59</span>:1561–71 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2889752/">PMC free article</a>]</span> [<a href="/pubmed/20587799" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Genetics+of+type+1+diabetes:+what's+next?&amp;author=F+Pociot&amp;author=B+Akolkar&amp;author=P+Concannon&amp;volume=59&amp;publication_year=2010&amp;pages=1561-71&amp;pmid=20587799&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b20">20. <span class="mixed-citation">Kho AN, Pacheco JA, Peissig PL, et al. 
<span class="ref-title">Electronic medical records for genetic research: results of the eMERGE consortium</span>. <span class="ref-journal">Sci Transl Med</span>
2011;<span class="ref-vol">3</span>:79re1 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3690272/">PMC free article</a>]</span> [<a href="/pubmed/21508311" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci+Transl+Med&amp;title=Electronic+medical+records+for+genetic+research:+results+of+the+eMERGE+consortium&amp;author=AN+Kho&amp;author=JA+Pacheco&amp;author=PL+Peissig&amp;volume=3&amp;publication_year=2011&amp;pages=79re1&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b21">21. <span class="mixed-citation">Pulley J, Clayton E, Bernard GR, et al. 
<span class="ref-title">Principles of human subjects protections applied in an opt-out, de-identified biobank</span>. <span class="ref-journal">Clin Transl Sci</span>
2010;<span class="ref-vol">3</span>:42–8 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3075971/">PMC free article</a>]</span> [<a href="/pubmed/20443953" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Transl+Sci&amp;title=Principles+of+human+subjects+protections+applied+in+an+opt-out,+de-identified+biobank&amp;author=J+Pulley&amp;author=E+Clayton&amp;author=GR+Bernard&amp;volume=3&amp;publication_year=2010&amp;pages=42-8&amp;pmid=20443953&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b22">22. <span class="mixed-citation">Regenstrief Institute I
<span class="ref-journal">Logical Observations Identifers Names and Codes (LOINC)</span>. 2010. 
<a href="http://loinc.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://loinc.org/</a> (accessed 26 May 2011). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Logical+Observations+Identifers+Names+and+Codes+(LOINC)&amp;publication_year=2010&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b23">23. <span class="mixed-citation"><span class="ref-journal">R: A Language and Environment for Statistical Computing</span>. Vienna, Austria: R Foundation for Statistical Computing, 2008 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=R:+A+Language+and+Environment+for+Statistical+Computing&amp;publication_year=2008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b24">24. <span class="mixed-citation">Stevenson M, Nunes T, Sanchez J.
<span class="ref-title">epiR: Functions for analysing epidemiological data. R package version 0.9–11 ed2008</span>. <a href="http://epicentre.massey.ac.nz/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://epicentre.massey.ac.nz/</a> <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Stevenson+M+Nunes+T+Sanchez+J+
+epiR:+Functions+for+analysing+epidemiological+data.+R+package+version+0.9–11+ed2008+.++http://epicentre.massey.ac.nz/+" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b25">25. <span class="mixed-citation">Turner S, Armstrong LL, Bradford Y, et al. 
<span class="ref-title">Quality control procedures for genome-wide association studies</span>. <span class="ref-journal">Curr Protoc Hum Genet</span>
2011;<span class="ref-vol">68</span>:1–19.1.18 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3066182/">PMC free article</a>]</span> [<a href="/pubmed/21234875" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Protoc+Hum+Genet&amp;title=Quality+control+procedures+for+genome-wide+association+studies&amp;author=S+Turner&amp;author=LL+Armstrong&amp;author=Y+Bradford&amp;volume=68&amp;publication_year=2011&amp;pages=1-19.1.18&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b26">26. <span class="mixed-citation">Purcell S, Neale B, Todd-Brown K, et al. 
<span class="ref-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <span class="ref-journal">Am J Hum Genet</span>
2007;<span class="ref-vol">81</span>:559–75 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b27">27. <span class="mixed-citation">Cauchi S, El Achhab Y, Choquet H, et al. 
<span class="ref-title">TCF7L2 is reproducibly associated with type 2 diabetes in various ethnic groups: a global meta-analysis</span>. <span class="ref-journal">J Mol Med (Berl)</span>
2007;<span class="ref-vol">85</span>:777–82 [<a href="/pubmed/17476472" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Mol+Med+(Berl)&amp;title=TCF7L2+is+reproducibly+associated+with+type+2+diabetes+in+various+ethnic+groups:+a+global+meta-analysis&amp;author=S+Cauchi&amp;author=Y+El+Achhab&amp;author=H+Choquet&amp;volume=85&amp;publication_year=2007&amp;pages=777-82&amp;pmid=17476472&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b28">28. <span class="mixed-citation">Manley SE, Sikaris KA, Lu ZX, et al. 
<span class="ref-title">Validation of an algorithm combining haemoglobin A1c and fasting plasma glucose for diagnosis of diabetes mellitus in UK and Australian populations</span>. <span class="ref-journal">Diabet Med</span>
2009;<span class="ref-vol">26</span>:115–21 [<a href="/pubmed/19236612" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabet+Med&amp;title=Validation+of+an+algorithm+combining+haemoglobin+A1c+and+fasting+plasma+glucose+for+diagnosis+of+diabetes+mellitus+in+UK+and+Australian+populations&amp;author=SE+Manley&amp;author=KA+Sikaris&amp;author=ZX+Lu&amp;volume=26&amp;publication_year=2009&amp;pages=115-21&amp;pmid=19236612&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b29">29. <span class="mixed-citation">Klein Woolthuis EP, de Grauw WJ, van Gerwen WH, et al. 
<span class="ref-title">Identifying people at risk for undiagnosed type 2 diabetes using the GP's electronic medical record</span>. <span class="ref-journal">Fam Pract</span>
2007;<span class="ref-vol">24</span>:230–6 [<a href="/pubmed/17510087" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Fam+Pract&amp;title=Identifying+people+at+risk+for+undiagnosed+type+2+diabetes+using+the+GP's+electronic+medical+record&amp;author=EP+Klein+Woolthuis&amp;author=WJ+de+Grauw&amp;author=WH+van+Gerwen&amp;volume=24&amp;publication_year=2007&amp;pages=230-6&amp;pmid=17510087&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b30">30. <span class="mixed-citation">Medicine NLo
<span class="ref-title">RxNorm</span>. 2010. 
<a href="http://www.nlm.nih.gov/research/umls/rxnorm/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.nlm.nih.gov/research/umls/rxnorm/</a> (accessed 26 May 2011). <span class="nowrap">[<a href="https://scholar.google.com/scholar?q=Medicine+NLo+
+RxNorm+.++2010+
+http://www.nlm.nih.gov/research/umls/rxnorm/++(accessed+26+May+2011).+" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b31">31. <span class="mixed-citation">CDC
<span class="ref-journal">National Diabetes Fact Sheet: National Estimates and General Information on Diabetes and Prediabetes in the United States, 2011</span>. Atlanta, GA: US Department of Health and Human Services, Centers for Disease Control and Prevention, 2011 <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=National+Diabetes+Fact+Sheet:+National+Estimates+and+General+Information+on+Diabetes+and+Prediabetes+in+the+United+States,+2011&amp;publication_year=2011&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b32">32. <span class="mixed-citation">Schildcrout JS, Basford MA, Pulley JM, et al. 
<span class="ref-title">An analytical approach to characterize morbidity profile dissimilarity between distinct cohorts using electronic medical records</span>. <span class="ref-journal">J Biomed Inform</span>
2010;<span class="ref-vol">43</span>:914–23 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2991387/">PMC free article</a>]</span> [<a href="/pubmed/20688191" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biomed+Inform&amp;title=An+analytical+approach+to+characterize+morbidity+profile+dissimilarity+between+distinct+cohorts+using+electronic+medical+records&amp;author=JS+Schildcrout&amp;author=MA+Basford&amp;author=JM+Pulley&amp;volume=43&amp;publication_year=2010&amp;pages=914-23&amp;pmid=20688191&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3277617&amp;issue-id=205472&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140206830481552title">Abstract</a></li><li><a href="#__sec6title">Objective</a></li><li><a href="#__sec7title">Research design and methods</a></li><li><a href="#__sec12title">Results</a></li><li><a href="#__sec13title">Discussion</a></li><li><a href="#__sec14title">Supplementary Material</a></li><li><a href="#idm140206838752400title">Footnotes</a></li><li><a href="#idm140206915244448title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Journal of the American Medical Informatics Association : JAMIA</span> are provided here courtesy of <strong>American Medical Informatics Association</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3277617/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3277617/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3277617/pdf/amiajnl-2011-000439.pdf">PDF (245K)</a></li> | <li><a href="#" data-citationid="PMC3277617" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3277617%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3277617%2F&amp;text=Use%20of%20diverse%20electronic%20medical%20record%20systems%20to%20identify%20genetic%20risk%20for%20type%202%20diabetes%20within%20a%20genome-wide%20association%20study" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3277617%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22101970" data-aiid="3277617" data-aid="3277617" data-iid="205472" data-domainid="76" data-domain="jamia" data-accid="PMC3277617" data-md5="284c1abd6cb68727b7bab1112ba2f469"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:11:36-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal106&amp;ncbi_phid=CE8D817EDD537EB100000000024D01A6&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3277617%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=jamia&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3277617/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal106 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>