<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4573240/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide association study of warfarin maintenance dose in a Brazilian sample</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4573240/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Pharmacogenomics"><meta name="citation_title" content="Genome-wide association study of warfarin maintenance dose in a Brazilian sample"><meta name="citation_authors" content="Esteban J Parra, Mariana R Botton, Jamila A Perini, S Krithika, Stephane Bourgeois, Todd A Johnson, Tatsuhiko Tsunoda, Munir Pirmohamed, Mia Wadelius, Nita A Limdi, Larisa H Cavallari, James K Burmester, Allan E Rettie, Teri E Klein, Julie A Johnson, Mara H Hutz, Guilherme Suarez-Kurtz"><meta name="citation_date" content="July 2015"><meta name="citation_issue" content="11"><meta name="citation_volume" content="16"><meta name="citation_firstpage" content="1"><meta name="citation_doi" content="10.2217/PGS.15.73"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4573240/?report=abstract"><meta name="citation_pmid" content="26265036"><meta name="DC.Title" content="Genome-wide association study of warfarin maintenance dose in a Brazilian sample"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Esteban J Parra"><meta name="DC.Contributor" content="Mariana R Botton"><meta name="DC.Contributor" content="Jamila A Perini"><meta name="DC.Contributor" content="S Krithika"><meta name="DC.Contributor" content="Stephane Bourgeois"><meta name="DC.Contributor" content="Todd A Johnson"><meta name="DC.Contributor" content="Tatsuhiko Tsunoda"><meta name="DC.Contributor" content="Munir Pirmohamed"><meta name="DC.Contributor" content="Mia Wadelius"><meta name="DC.Contributor" content="Nita A Limdi"><meta name="DC.Contributor" content="Larisa H Cavallari"><meta name="DC.Contributor" content="James K Burmester"><meta name="DC.Contributor" content="Allan E Rettie"><meta name="DC.Contributor" content="Teri E Klein"><meta name="DC.Contributor" content="Julie A Johnson"><meta name="DC.Contributor" content="Mara H Hutz"><meta name="DC.Contributor" content="Guilherme Suarez-Kurtz"><meta name="DC.Date" content="2015 Jul"><meta name="DC.Identifier" content="10.2217/PGS.15.73"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide association study of warfarin maintenance dose in a Brazilian sample"><meta property="og:type" content="article"><meta property="og:description" content="Extreme discordant phenotype and genome-wide association (GWA) approaches were combined to explore the role of genetic variants on warfarin dose requirement in Brazilians.Patients receiving low (≤20 mg/week; n = 180) or high stable warfarin doses ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4573240/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4573240/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4573240/pdf/nihms715170.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8981FFDD58258100000000069B03F8.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4573240%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="ipg">Identical Protein Groups</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC4573240</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Pharmacogenomics</a></span>. Author manuscript; available in PMC 2016 Jun 12.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26265036" target="pmc_ext" ref="reftype=publisher&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access"><span class="cit">Pharmacogenomics. 2015 Jul; 16(11): 1–11. </span></a></div><span class="fm-vol-iss-date">Published online 2015 Aug 12. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.2217%2FPGS.15.73" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.2217/PGS.15.73</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4573240</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS715170</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26265036">26265036</a></div></div></div></div><h1 class="content-title">Genome-wide association study of warfarin maintenance dose in a Brazilian sample</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Parra%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527252530288" co-class="co-affbox">Esteban J Parra</a>,<sup>*,</sup><sup>1</sup> <a href="/pubmed/?term=Botton%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527252977456" co-class="co-affbox">Mariana R Botton</a>,<sup>2</sup> <a href="/pubmed/?term=Perini%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527252914320" co-class="co-affbox">Jamila A Perini</a>,<sup>3</sup> <a href="/pubmed/?term=Krithika%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527251435472" co-class="co-affbox">S Krithika</a>,<sup>1</sup> <a href="/pubmed/?term=Bourgeois%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527251433488" co-class="co-affbox">Stephane Bourgeois</a>,<sup>4</sup> <a href="/pubmed/?term=Johnson%20TA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527253385536" co-class="co-affbox">Todd A Johnson</a>,<sup>5</sup> <a href="/pubmed/?term=Tsunoda%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527252582096" co-class="co-affbox">Tatsuhiko Tsunoda</a>,<sup>5</sup> <a href="/pubmed/?term=Pirmohamed%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527252580112" co-class="co-affbox">Munir Pirmohamed</a>,<sup>6</sup> <a href="/pubmed/?term=Wadelius%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527257398912" co-class="co-affbox">Mia Wadelius</a>,<sup>7</sup> <a href="/pubmed/?term=Limdi%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527257721904" co-class="co-affbox">Nita A Limdi</a>,<sup>8</sup> <a href="/pubmed/?term=Cavallari%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527257719920" co-class="co-affbox">Larisa H Cavallari</a>,<sup>9</sup> <a href="/pubmed/?term=Burmester%20JK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527257956736" co-class="co-affbox">James K Burmester</a>,<sup>10</sup> <a href="/pubmed/?term=Rettie%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527254531920" co-class="co-affbox">Allan E Rettie</a>,<sup>11</sup> <a href="/pubmed/?term=Klein%20TE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527254529936" co-class="co-affbox">Teri E Klein</a>,<sup>12</sup> <a href="/pubmed/?term=Johnson%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527255780256" co-class="co-affbox">Julie A Johnson</a>,<sup>9</sup> <a href="/pubmed/?term=Hutz%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527261071808" co-class="co-affbox">Mara H Hutz</a>,<sup>2</sup> and  <a href="/pubmed/?term=Suarez-Kurtz%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036" class="affpopup" co-rid="_co_idm140527261069824" co-class="co-affbox">Guilherme Suarez-Kurtz</a><sup>*,</sup><sup>3</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140527252530288"><h3 class="no_margin">Esteban J Parra</h3><p><sup>1</sup>Department of Anthropology, University of Toronto at Mississauga, ON, Canada</p><div>Find articles by <a href="/pubmed/?term=Parra%20EJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Esteban J Parra</a></div></div><div id="_co_idm140527252977456"><h3 class="no_margin">Mariana R Botton</h3><p><sup>2</sup>Department of Genetics, Universidade Federal do Rio Grande do Sul, Brazil</p><div>Find articles by <a href="/pubmed/?term=Botton%20MR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Mariana R Botton</a></div></div><div id="_co_idm140527252914320"><h3 class="no_margin">Jamila A Perini</h3><p><sup>3</sup>Pharmacology Division, Instituto Nacional de Câncer, Rio de Janeiro, Brazil</p><div>Find articles by <a href="/pubmed/?term=Perini%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Jamila A Perini</a></div></div><div id="_co_idm140527251435472"><h3 class="no_margin">S Krithika</h3><p><sup>1</sup>Department of Anthropology, University of Toronto at Mississauga, ON, Canada</p><div>Find articles by <a href="/pubmed/?term=Krithika%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">S Krithika</a></div></div><div id="_co_idm140527251433488"><h3 class="no_margin">Stephane Bourgeois</h3><p><sup>4</sup>William Harvey Research Institute, Barts &amp; The London School of Medicine &amp; Dentistry, Queen Mary University of London, UK</p><div>Find articles by <a href="/pubmed/?term=Bourgeois%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Stephane Bourgeois</a></div></div><div id="_co_idm140527253385536"><h3 class="no_margin">Todd A Johnson</h3><p><sup>5</sup>Laboratory for Medical Science Mathematics, RIKEN Center for Integrative Medical Sciences, Japan</p><div>Find articles by <a href="/pubmed/?term=Johnson%20TA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Todd A Johnson</a></div></div><div id="_co_idm140527252582096"><h3 class="no_margin">Tatsuhiko Tsunoda</h3><p><sup>5</sup>Laboratory for Medical Science Mathematics, RIKEN Center for Integrative Medical Sciences, Japan</p><div>Find articles by <a href="/pubmed/?term=Tsunoda%20T%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Tatsuhiko Tsunoda</a></div></div><div id="_co_idm140527252580112"><h3 class="no_margin">Munir Pirmohamed</h3><p><sup>6</sup>Department of Molecular &amp; Clinical Pharmacology, University of Liverpool, UK</p><div>Find articles by <a href="/pubmed/?term=Pirmohamed%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Munir Pirmohamed</a></div></div><div id="_co_idm140527257398912"><h3 class="no_margin">Mia Wadelius</h3><p><sup>7</sup>Department of Medical Sciences, Clinical Pharmacology &amp; Science for Life Laboratory, Uppsala University, Sweden</p><div>Find articles by <a href="/pubmed/?term=Wadelius%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Mia Wadelius</a></div></div><div id="_co_idm140527257721904"><h3 class="no_margin">Nita A Limdi</h3><p><sup>8</sup>Department of Neurology, University of Alabama at Birmingham, AL, USA</p><div>Find articles by <a href="/pubmed/?term=Limdi%20NA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Nita A Limdi</a></div></div><div id="_co_idm140527257719920"><h3 class="no_margin">Larisa H Cavallari</h3><p><sup>9</sup>University of Florida, Department of Pharmacotherapy &amp; Translational Research, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Cavallari%20LH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Larisa H Cavallari</a></div></div><div id="_co_idm140527257956736"><h3 class="no_margin">James K Burmester</h3><p><sup>10</sup>Clinical Research Center, Marshfield Clinic Research Foundation, WI, USA</p><div>Find articles by <a href="/pubmed/?term=Burmester%20JK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">James K Burmester</a></div></div><div id="_co_idm140527254531920"><h3 class="no_margin">Allan E Rettie</h3><p><sup>11</sup>Department Medicinal Chemistry, School of Pharmacy, University of Washington, WA, USA</p><div>Find articles by <a href="/pubmed/?term=Rettie%20AE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Allan E Rettie</a></div></div><div id="_co_idm140527254529936"><h3 class="no_margin">Teri E Klein</h3><p><sup>12</sup>Department of Genetics, Stanford University School of Medicine, CA, USA</p><div>Find articles by <a href="/pubmed/?term=Klein%20TE%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Teri E Klein</a></div></div><div id="_co_idm140527255780256"><h3 class="no_margin">Julie A Johnson</h3><p><sup>9</sup>University of Florida, Department of Pharmacotherapy &amp; Translational Research, FL, USA</p><div>Find articles by <a href="/pubmed/?term=Johnson%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Julie A Johnson</a></div></div><div id="_co_idm140527261071808"><h3 class="no_margin">Mara H Hutz</h3><p><sup>2</sup>Department of Genetics, Universidade Federal do Rio Grande do Sul, Brazil</p><div>Find articles by <a href="/pubmed/?term=Hutz%20MH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Mara H Hutz</a></div></div><div id="_co_idm140527261069824"><h3 class="no_margin">Guilherme Suarez-Kurtz</h3><p><sup>3</sup>Pharmacology Division, Instituto Nacional de Câncer, Rio de Janeiro, Brazil</p><div>Find articles by <a href="/pubmed/?term=Suarez-Kurtz%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26265036">Guilherme Suarez-Kurtz</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140527252057024_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140527252057024_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140527252057024_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Department of Anthropology, University of Toronto at Mississauga, ON, Canada</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Department of Genetics, Universidade Federal do Rio Grande do Sul, Brazil</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>Pharmacology Division, Instituto Nacional de Câncer, Rio de Janeiro, Brazil</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>William Harvey Research Institute, Barts &amp; The London School of Medicine &amp; Dentistry, Queen Mary University of London, UK</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Laboratory for Medical Science Mathematics, RIKEN Center for Integrative Medical Sciences, Japan</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>Department of Molecular &amp; Clinical Pharmacology, University of Liverpool, UK</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>Department of Medical Sciences, Clinical Pharmacology &amp; Science for Life Laboratory, Uppsala University, Sweden</div><div class="fm-affl" id="A8" lang="en"><sup>8</sup>Department of Neurology, University of Alabama at Birmingham, AL, USA</div><div class="fm-affl" id="A9" lang="en"><sup>9</sup>University of Florida, Department of Pharmacotherapy &amp; Translational Research, FL, USA</div><div class="fm-affl" id="A10" lang="en"><sup>10</sup>Clinical Research Center, Marshfield Clinic Research Foundation, WI, USA</div><div class="fm-affl" id="A11" lang="en"><sup>11</sup>Department Medicinal Chemistry, School of Pharmacy, University of Washington, WA, USA</div><div class="fm-affl" id="A12" lang="en"><sup>12</sup>Department of Genetics, Stanford University School of Medicine, CA, USA</div><div id="FN1"><sup>*</sup>Authors for correspondence: Tel.: +1 416 769 1646; <a href="mailto:dev@null" data-email="ac.otnorotu@arrap.nabetse" class="oemail">ac.otnorotu@arrap.nabetse</a> and, Tel.: +55 21 3207 6502; <a href="mailto:dev@null" data-email="rb.vog.acni@ztruk" class="oemail">rb.vog.acni@ztruk</a></div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140527252057024_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>The publisher's final edited version of this article is available  at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=26265036" target="pmc_ext" ref="reftype=publisher&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CRestricted%20Access">Pharmacogenomics</a></div><div>See other articles in PMC that <a href="/pmc/articles/PMC4573240/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="9" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp_Figures.</strong></div><div><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Supp_Figures.pptx" data-ga-action="click_feat_suppl">NIHMS715170-supplement-Supp_Figures.pptx</a><span style="color:gray"> (1.1M)</span></div><div class="small guid">GUID:&nbsp;B88D3511-6A45-4EB1-9232-B0800463C9BE</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp_Table_Legends.</strong></div><div><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Supp_Table_Legends.docx" data-ga-action="click_feat_suppl">NIHMS715170-supplement-Supp_Table_Legends.docx</a><span style="color:gray"> (15K)</span></div><div class="small guid">GUID:&nbsp;A6BC5426-C376-402F-8CE2-CE4D697BC76D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 1.</strong></div><div><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_1.xlsx" data-ga-action="click_feat_suppl">NIHMS715170-supplement-Table_1.xlsx</a><span style="color:gray"> (14K)</span></div><div class="small guid">GUID:&nbsp;AB912D90-98EC-482F-95F3-9F31D7B08EA0</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 2.</strong></div><div><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_2.xlsx" data-ga-action="click_feat_suppl">NIHMS715170-supplement-Table_2.xlsx</a><span style="color:gray"> (16K)</span></div><div class="small guid">GUID:&nbsp;5A05E97C-F219-426D-A312-8B9BF67D6EAC</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 3.</strong></div><div><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_3.xlsx" data-ga-action="click_feat_suppl">NIHMS715170-supplement-Table_3.xlsx</a><span style="color:gray"> (66K)</span></div><div class="small guid">GUID:&nbsp;491DEBFC-2AAE-444B-A690-F69D55000396</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 4.</strong></div><div><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_4.xlsx" data-ga-action="click_feat_suppl">NIHMS715170-supplement-Table_4.xlsx</a><span style="color:gray"> (52K)</span></div><div class="small guid">GUID:&nbsp;CD309EAB-619D-4F35-827F-21719554BD1D</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 5.</strong></div><div><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_5.xlsx" data-ga-action="click_feat_suppl">NIHMS715170-supplement-Table_5.xlsx</a><span style="color:gray"> (26K)</span></div><div class="small guid">GUID:&nbsp;BB5636B2-809B-4A45-A019-D100B432D757</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 6.</strong></div><div><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_6.xlsx" data-ga-action="click_feat_suppl">NIHMS715170-supplement-Table_6.xlsx</a><span style="color:gray"> (128K)</span></div><div class="small guid">GUID:&nbsp;3783BE49-184A-40ED-8255-EDC670A09D93</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Table 7.</strong></div><div><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_7.xlsx" data-ga-action="click_feat_suppl">NIHMS715170-supplement-Table_7.xlsx</a><span style="color:gray"> (15K)</span></div><div class="small guid">GUID:&nbsp;BDD2EA4D-9EE1-4FB6-A7DB-B5B04460DFEB</div></div></dd></dl></div><div id="idm140527239290960" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140527239290960title">Abstract</h2><!--article-meta--><div><div id="S1" class="sec sec-first"><h3 id="S1title">Aim</h3><p id="P1" class="p p-first-last">Extreme discordant phenotype and genome-wide association (GWA) approaches were combined to explore the role of genetic variants on warfarin dose requirement in Brazilians.</p></div><div id="S2" class="sec"><h3 id="S2title">Methods</h3><p id="P2" class="p p-first-last">Patients receiving low (≤20 mg/week; n = 180) or high stable warfarin doses (≥42.5 mg/week; n = 187) were genotyped with Affymetrix Axiom<sup>®</sup> Biobank arrays. Imputation was carried out using data from the combined 1000 Genomes project.</p></div><div id="S3" class="sec"><h3 id="S3title">Results</h3><p id="P3" class="p p-first-last">Genome-wide signals (p ≤5 × 10<sup>−8</sup>) were identified in the well-known <em>VKORC1</em> (lead SNP, rs749671; OR: 20.4; p = 1.08 × 10<sup>−33</sup>) and <em>CYP2C9</em> (lead SNP, rs9332238, OR: 6.8 and p = 4.4 × 10<sup>−13</sup>) regions. The rs9332238 polymorphism is in virtually perfect LD with <em>CYP2C9*2</em> (rs1799853) and <em>CYP2C9*3</em> (rs1057910). No other genome-wide significant regions were identified in the study.</p></div><div id="S4" class="sec sec-last"><h3 id="S4title">Conclusion</h3><p id="P4" class="p p-first-last">We confirmed the important role of <em>VKORC1</em> and <em>CYP2C9</em> polymorphisms in warfarin dose.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">1000 Genomes Project, Brazilians, <em>CYP2C9</em>, extreme discordant phenotypes, genome-wide association study, <em>VKORC1</em>, warfarin</span></div></div><div id="S5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S5title">Background</h2><p id="P5" class="p p-first-last">Warfarin, the most commonly used anticoagulant in the world, has been the target of several pharmacogenetic/genomic studies aimed at identifying the factors that account for the large interindividual variability in dose requirement. A number of warfarin dosing algorithms, comprising genetic and nongenetic covariates emerged from these studies, but their predictive power accounts for no more than 60% of the variance in warfarin dose requirements in ambulatory patients [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614098">1</a>–<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614130">10</a>]. Pharmacogenetic polymorphisms in <em>VKORC1</em> and <em>CYP2C9</em> affecting, respectively, warfarin’s pharmacodynamics and pharmacokinetics are the most informative covariates in most algorithms. Genome-wide association studies (GWAS) of warfarin dosing requirements in White/Caucasian [<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614109">11</a>,<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614128">12</a>], Japanese [<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614129">13</a>] and African–American patients [<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614114">14</a>] indicated that common SNPs with large effects on warfarin dose are unlikely to be discovered outside of the <em>CYP2C9</em> and <em>VKORC1</em> genes. However, the possibility that polymorphisms in other pharmacogenes may contribute in a smaller scale to the interindividual variance in warfarin dosing, may not be excluded. In this respect, conditional analyses and GWA or meta-analytic efforts including larger sample sizes may uncover novel common variants with smaller effect sizes. Indeed, a polymorphism in <em>CYP4F2</em> (rs2108622) reached genome-wide significant association with warfarin dose after conditioning for <em>VKORC1</em> and <em>CYP2C9</em> polymorphisms [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614133">12</a>,<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614124">13</a>]. Another study in African–Americans applied a stepwise conditional analysis and identified an association between a polymorphism in the <em>CYP2C</em> cluster on chromosome 10 (rs12777823) and warfarin dose [<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614113">14</a>]. In the present study, we applied the extreme discordant phenotype (EDP) methodology [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614145">15</a>] to select Brazilian patients under stable warfarin therapy for a genome-wide association study. The EDP approach contrasts the most sensitive and the most resistant phenotype groups, which in the case of a quantitative trait such as the individual warfarin dose requirement correspond to the lower and upper ends of the dose distribution histogram. The superior power of sampling extreme phenotype individuals compared with random cohorts for GWAS of drug phenotypes has been recently reviewed [<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614112">16</a>].</p></div><div id="S6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S6title">Materials &amp; methods</h2><div id="S7" class="sec sec-first"><h3 id="S7title">Study cohort</h3><p id="P6" class="p p-first-last">The present analyses are based on combined data from two cohorts of adult outpatients, enrolled in previously published retrospective studies designed to develop warfarin dosing algorithms for Brazilians [<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614122">5</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614115">17</a>]. The patients were attending anticoagulation clinics at two tertiary care institutions of the Brazilian Public Health System, namely the Instituto Nacional de Cardiologia Laranjeiras, a reference cardiology hospital located in Rio de Janeiro (n = 390) and the University Hospital of Universidade Federal do Rio Grande do Sul, in Porto Alegre (n = 488). Details of the original study design have been published [<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614111">5</a>,<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614143">17</a>]. The study protocols were approved by the respective institutional review boards and each patient provided a written informed consent. Patients were categorized according to the Brazilian census, which relies on self-perception of ‘race/color’, as White (<em>branco</em>), Brown (<em>pardo</em>) and Black (<em>preto</em>). The term Color and the color categories are capitalized to highlight their special meaning in the context of the Brazilian census. All patients were under stable warfarin dosing, defined as the prescribed weekly dose associated with the first three consecutive INR readings within the target range. The INR target range was 2–3 for all patients, except those with heart valve prosthesis, whose INR target was 2.5–3.5 [<a href="#R5" rid="R5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614119">5</a>].</p></div><div id="S8" class="sec"><h3 id="S8title">Extreme discordant phenotype</h3><p id="P7" class="p p-first-last">The target pharmacogenetic phenotype was the stable warfarin weekly dose, which ranged from 5 to 110 mg in the overall cohort. A frequency distribution histogram of the individual warfarin dose was constructed (<a href="/pmc/articles/PMC4573240/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>) and patients at opposite ends of the distribution were selected for genotyping with Affymetrix Axiom Biobank microarray. The cut-off points for the low-dose and high-dose groups were set at ≤20 mg/week (n = 180) and ≥42.5 mg/week (n = 187), respectively. Demographic and clinical characteristics of the patients in the two groups are shown in <a href="/pmc/articles/PMC4573240/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table 1</span></a>.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC4573240/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140527252446112" class="figure"><a href="/pmc/articles/PMC4573240/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4573240_nihms715170f1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms715170f1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4573240/bin/nihms715170f1.jpg"></a></div><div id="largeobj_idm140527252446112" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4573240/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC4573240/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Histogram showing the frequency distribution of weekly warfarin dose in the Brazilian sample</strong><p id="__p6">Patients at opposite ends of the distribution (gray bars) were selected for genotyping with the Affymetrix Axiom<sup>®</sup> Biobank</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p9">Demographics and clinical characteristics of the study cohort.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140527256802928" class="xtable"><table frame="box" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Variable</th><th rowspan="1" colspan="1" valign="top" align="left">Low dose (n = 180) (Mean [SD])</th><th rowspan="1" colspan="1" valign="top" align="left">High dose (n = 187) (Mean [SD])</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">Warfarin weekly dose (mg)</td><td rowspan="1" colspan="1" valign="top" align="left">16.2 (3.1)</td><td rowspan="1" colspan="1" valign="top" align="left">55.2 (2.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">INR</td><td rowspan="1" colspan="1" valign="top" align="left">2.5 (0.6)</td><td rowspan="1" colspan="1" valign="top" align="left">2.7 (0.6)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Age (years)</td><td rowspan="1" colspan="1" valign="top" align="left">65.8 (13.9)</td><td rowspan="1" colspan="1" valign="top" align="left">54.2 (13.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Weight (kg)</td><td rowspan="1" colspan="1" valign="top" align="left">67.5 (15.1)</td><td rowspan="1" colspan="1" valign="top" align="left">76.9 (16.0)</td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"></td><td rowspan="1" colspan="1" valign="top" align="left"><strong>n (%)</strong></td><td rowspan="1" colspan="1" valign="top" align="left"><strong>n (%)</strong></td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left">Gender</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">– Male</td><td rowspan="1" colspan="1" valign="top" align="left">87 (48.3)</td><td rowspan="1" colspan="1" valign="top" align="left">106 (56.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">– Female</td><td rowspan="1" colspan="1" valign="top" align="left">93 (51.7)</td><td rowspan="1" colspan="1" valign="top" align="left">81 (43.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">BMI</td><td rowspan="1" colspan="1" valign="top" align="left">25.7 (3.9)</td><td rowspan="1" colspan="1" valign="top" align="left">27.3 (4.8)</td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left"><strong>Cohort</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Rio de Janeiro</td><td rowspan="1" colspan="1" valign="top" align="left">119 (66.1)</td><td rowspan="1" colspan="1" valign="top" align="left">126 (67.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Porto Alegre</td><td rowspan="1" colspan="1" valign="top" align="left">61 (33.9)</td><td rowspan="1" colspan="1" valign="top" align="left">61 (32.6)</td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left"><strong>Race/color</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">White</td><td rowspan="1" colspan="1" valign="top" align="left">153 (85.0)</td><td rowspan="1" colspan="1" valign="top" align="left">139 (74.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Brown</td><td rowspan="1" colspan="1" valign="top" align="left">18 (10.0)</td><td rowspan="1" colspan="1" valign="top" align="left">27 (14.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Black</td><td rowspan="1" colspan="1" valign="top" align="left">9 (5.0)</td><td rowspan="1" colspan="1" valign="top" align="left">10 (120.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Yellow</td><td rowspan="1" colspan="1" valign="top" align="left">0</td><td rowspan="1" colspan="1" valign="top" align="left">1 (0.5)</td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left"><strong>Admixture proportions (%)</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">European</td><td rowspan="1" colspan="1" valign="top" align="left">81.7 (17.8)</td><td rowspan="1" colspan="1" valign="top" align="left">72.4 (23.2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">African</td><td rowspan="1" colspan="1" valign="top" align="left">11.7 (15.9)</td><td rowspan="1" colspan="1" valign="top" align="left">18.2 (21.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Native American</td><td rowspan="1" colspan="1" valign="top" align="left">6.6 (7.1)</td><td rowspan="1" colspan="1" valign="top" align="left">9.4 (8.5)</td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left"><strong>Indication for warfarin</strong></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Atrial fibrillation</td><td rowspan="1" colspan="1" valign="top" align="left">97 (53.9)</td><td rowspan="1" colspan="1" valign="top" align="left">55 (29.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Heart valve prosthesis</td><td rowspan="1" colspan="1" valign="top" align="left">28 (15.6)</td><td rowspan="1" colspan="1" valign="top" align="left">88 (47.1)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Atrial fibrillation and heart valve prosthesis</td><td rowspan="1" colspan="1" valign="top" align="left">18 (10.0)</td><td rowspan="1" colspan="1" valign="top" align="left">12 (6.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Thromboembolic disease</td><td rowspan="1" colspan="1" valign="top" align="left">17 (9.4)</td><td rowspan="1" colspan="1" valign="top" align="left">18 (9.6)</td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left"><strong>Comorbidities</strong><sup>†</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Systemic hypertension</td><td rowspan="1" colspan="1" valign="top" align="left">119 (66.1)</td><td rowspan="1" colspan="1" valign="top" align="left">102 (54.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Rheumatic fever</td><td rowspan="1" colspan="1" valign="top" align="left">35 (19.4)</td><td rowspan="1" colspan="1" valign="top" align="left">27 (14.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Dislipidemia</td><td rowspan="1" colspan="1" valign="top" align="left">30 (16.7)</td><td rowspan="1" colspan="1" valign="top" align="left">35 (18.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Diabetes mellitus</td><td rowspan="1" colspan="1" valign="top" align="left">20 (11.1)</td><td rowspan="1" colspan="1" valign="top" align="left">27 (14.4)</td></tr><tr><td colspan="3" rowspan="1" valign="bottom" align="left">
<hr></td></tr><tr><td colspan="3" rowspan="1" valign="top" align="left"><strong>Comedication</strong><sup>†</sup></td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">ACE inhibitors</td><td rowspan="1" colspan="1" valign="top" align="left">96 (53.3)</td><td rowspan="1" colspan="1" valign="top" align="left">91 (48.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Aspirin</td><td rowspan="1" colspan="1" valign="top" align="left">39 (21.7)</td><td rowspan="1" colspan="1" valign="top" align="left">41 (21.9)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Amiodarone</td><td rowspan="1" colspan="1" valign="top" align="left">31 (17.2)</td><td rowspan="1" colspan="1" valign="top" align="left">10 (5.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Beta blockers</td><td rowspan="1" colspan="1" valign="top" align="left">79 (43.9)</td><td rowspan="1" colspan="1" valign="top" align="left">59 (31.6)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Ca-entry blockers</td><td rowspan="1" colspan="1" valign="top" align="left">25 (13.9)</td><td rowspan="1" colspan="1" valign="top" align="left">13 (7.0)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Digoxin</td><td rowspan="1" colspan="1" valign="top" align="left">56 (31.1)</td><td rowspan="1" colspan="1" valign="top" align="left">38 (20.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Isosorbide</td><td rowspan="1" colspan="1" valign="top" align="left">20 (11.1)</td><td rowspan="1" colspan="1" valign="top" align="left">10 (5.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Loop diuretics</td><td rowspan="1" colspan="1" valign="top" align="left">103 (57.2)</td><td rowspan="1" colspan="1" valign="top" align="left">77 (41.2)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Losartan</td><td rowspan="1" colspan="1" valign="top" align="left">22 (12.2)</td><td rowspan="1" colspan="1" valign="top" align="left">13 (7.0)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Metformin</td><td rowspan="1" colspan="1" valign="top" align="left">10 (5.6)</td><td rowspan="1" colspan="1" valign="top" align="left">20 (10.7)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Omeprazol</td><td rowspan="1" colspan="1" valign="top" align="left">35 (19.4)</td><td rowspan="1" colspan="1" valign="top" align="left">23 (12.3)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Simvastatin</td><td rowspan="1" colspan="1" valign="top" align="left">51 (28.3)</td><td rowspan="1" colspan="1" valign="top" align="left">40 (21.4)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Spironolactone</td><td rowspan="1" colspan="1" valign="top" align="left">20 (11.1)</td><td rowspan="1" colspan="1" valign="top" align="left">17 (9.1)</td></tr></tbody></table></div><div id="largeobj_idm140527256802928" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4573240/table/T1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN1"><sup>†</sup>Frequency &gt;5% in the combined groups.</div></div></div></div><div id="S9" class="sec"><h3 id="S9title">Genotyping</h3><p id="P8" class="p p-first-last">The sample was genotyped with the Affymetrix Axiom<sup>®</sup> Biobank array (Affymetrix, CA, USA) following standard protocols. Genotype calling was done with the Affymetrix PowerTools (APT) software package, using the AxiomGT1/brlmm-p algorithm and the manufacturer recommended calling pipeline. The Biobank array has more than 650,000 variants, including 265,000 exome coding snps and indels, 70,000 novel loss-of-function SNPs and indels, 23,000 eQTLs, 2000 markers of pharmacogenomic relevance and 246,000 genome-wide association markers designed to ensure good genome-wide coverage in major populations, which make it possible to carry out imputation in order to increase the number of markers in the statistical analyses.</p></div><div id="S10" class="sec"><h3 id="S10title">Quality control prior to imputation</h3><p id="P9" class="p p-first-last">Prior to imputation, SNPs were removed from the initial list of autosomal markers based on the following criteria: 1/Markers classified as CallRateBelowThreshold, OffTargetVariants or Other by the program SNPolisher, 2/minor allele frequency &lt;1%, 3/Hardy–Weinberg p-values &lt;10–5 in the low-dose or high-dose sample and 4/missing genotyping rate per SNP &gt; 5%. All the QC steps were carried out with the program PLINK [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614117">18</a>].</p></div><div id="S11" class="sec"><h3 id="S11title">Population stratification &amp; admixture estimates</h3><p id="P10" class="p p-first-last">In order to evaluate population stratification we used the program EIGENSOFT [<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614146">19</a>] to perform a principal component (PC) analysis of the Mexico City sample, after pruning markers showing linkage disequilibrium. The samples from Brazil were plotted jointly with samples from other population groups, including Europeans (CEU), Africans (YRI), East Asians (CHB), Native Americans (Nahua and Maya from Mexico, and Andeans from South America) and other admixed groups from the Americas (Mexican Americans from LA, Puerto Ricans and Colombians). In addition to the PC analyses, we also estimated admixture proportions based on genome-wide data using the program ADMIXTURE [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614136">20</a>].</p></div><div id="S12" class="sec"><h3 id="S12title">Imputation</h3><p id="P11" class="p p-first-last">Imputation was carried out with the program IMPUTE v2 [<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614131">21</a>], using the combined 1000 Genomes project samples as the reference sample (1000 Genomes Phase I integrated haplotypes, NCBI build b37, release date December 2013, no singletons). Imputations were carried out using a two-step strategy: 1/prephasing using the program SHAPEIT [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614106">22</a>] and 2/Imputation from the reference panel into the estimated haplotypes with IMPUTE v2 [<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614116">21</a>,<a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614134">23</a>].</p></div><div id="S13" class="sec"><h3 id="S13title">Exclusion of individuals prior to association tests</h3><p id="P12" class="p p-first-last">Prior to the association tests, four individuals were excluded from the dataset: three individuals due to cryptic relatedness (pi-Hat &gt; 0.2, based on an analysis with PLINK; [<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614120">18</a>]) and one individual showing evidence of substantial East Asian ancestry based on the PC plots. For the pairs of individuals showing cryptic relatedness, we removed the individual with the highest genotype missingness rate.</p></div><div id="S14" class="sec"><h3 id="S14title">Association tests</h3><p id="P13" class="p p-first-last">In order to evaluate the association of genetic markers with warfarin dose (low- vs high-dose groups), we used the program SNPTEST v2 [<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614140">24</a>], including as covariates sex, age, BMI, treatment with amiodarone and the first two PCA scores. In order to control for genotype uncertainty, we used the <em>ml</em> method, which uses multiple Newton–Raphson iterations to estimate the parameters in the missing data likelihood for the model. In addition to this initial analysis, we also carried out a conditional analysis including in the models the genotypes of <em>VKORC1</em> (rs749671) and CYP2C9 (<em>CYP2C9*2</em> and <em>CYP2C9*3</em>).</p></div><div id="S15" class="sec"><h3 id="S15title">Identification of markers to follow-up</h3><p id="P14" class="p p-first-last">Based on the results of the conditional analyses, we identified a list of highly suggestive regions (p &lt; 10<sup>−5</sup>), and the lead SNPs for each region were followed up in 798 samples of European ancestry, from five previously published cohorts [<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614102">6</a>,<a href="#R25" rid="R25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614144">25</a>–<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614125">28</a>], 1611 samples of Japanese origin [<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614121">13</a>] and 191 samples of African–American ancestry, from two cohorts [<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614101">29</a>–<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614105">31</a>].</p></div><div id="S16" class="sec"><h3 id="S16title">Gene-based &amp; pathway association tests</h3><p id="P15" class="p p-first-last">We carried out gene-based and pathway association tests using the program KGG3 [<a href="#R32" rid="R32" class=" bibr popnode">32</a>]. For the gene-based association tests, we used the GATES approach [<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614110">33</a>], using a window of 10Kb for the extended gene region length. This test takes into account the LD patterns between the SNPs, which in this case were defined by the patterns observed in the Puerto Rican 1000 Genomes sample, which is the closest sample to the Brazilian sample in terms of admixture proportions (e.g., European, African and Native American contributions). For the pathway association tests. we used the HYST [<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614096">34</a>] test implemented in the KGG3 program. We only report pathways that are significant using the HYST test and also a hypergeometric test enrolling genes with a p-value of 0.001. We also performed a pathway analysis using an alternative program, GSA-SNP [<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614127">35</a>], using the PAGE (parametric analysis of gene set enrichment) method [<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614100">36</a>]. We used the second best SNP to define the gene p-values, and used a padding window of 10 kb. The program was run using both Gene Ontology (GO) and KEGG gene sets.</p></div><div id="S17" class="sec sec-last"><h3 id="S17title">eQTL analyses</h3><p id="P16" class="p p-first-last">We used the eQTL browser [<a href="#R37" rid="R37" class=" bibr popnode">37</a>] to download eQTL datasets generated in lymphoblastoid cell lines (LCLs) [<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614108">38</a>,<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614139">39</a>] and human liver [<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614126">40</a>]. We then evaluated if any of the eQTLs in the aforementioned tissues (here defined as markers associated with gene expression levels with p-values ≤10<sup>−5</sup>) are associated with warfarin dose in our study, after Bonferroni correction to take into account multiple tests (e.g., total number of eQTLS relevant for each tissue).</p></div></div><div id="S18" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S18title">Results</h2><div id="S19" class="sec sec-first"><h3 id="S19title">Characterization of admixture in the Brazilian sample</h3><p id="P17" class="p p-first-last">The PC analysis shows that the Brazilian sample is primarily the result of admixture between European and African populations (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>). The Brazilian individuals are located between the relatively tight clusters corresponding to European and West African individuals. However, there is also evidence of Native American ancestry in the sample, based on the presence of individuals that are located toward the cluster defined by the Native American populations. The placement of the Brazilian individuals in the plot, primarily between the European and African clusters, contrasts with the position of the Mexican Americans from LA, which are primarily located between the Native American and European clusters. The samples from Colombia and Puerto Rico also show clear evidence of admixture. We estimated the admixture proportions in the Brazilian sample using the program ADMIXTURE [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614135">20</a>]. The average European, African and Native American ancestry in the Brazilian sample were estimated as 76.8, 15.1 and 8.1%, respectively. The estimates of individual ancestry obtained with the program ADMIXTURE are highly correlated with the scores of the first two PC-axes (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a>). The scores of PC1 are highly correlated with the African contribution (R<sup>2</sup> = 0.9921), and the scores of PC2 are highly correlated with the Native American contribution (R<sup>2</sup> = 0.9802).</p></div><div id="S20" class="sec"><h3 id="S20title">GWAS of warfarin maintenance dose</h3><p id="P18" class="p p-first-last">We evaluated the association of directly genotyped and imputed genetic markers with warfarin dose (low vs high dose) including as covariates sex, age, BMI, treatment with amiodarone and the first two PC scores. The Manhattan and QQ plots corresponding to this analysis are shown in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 3 &amp; 4</a>, respectively. There was little evidence of inflation of p-values (lambda = 1.03). We observed genome-wide significant signals (p ≤ 5 × 10<sup>−8</sup>) in the well-known <em>VKORC1</em> and <em>CYP2C9</em> regions. <a href="/pmc/articles/PMC4573240/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figures 2</span></a> and <a href="/pmc/articles/PMC4573240/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">​<span class="figpopup-sensitive-area" style="left: -1.5em;">and3</span></span><span>3</span></a> show the regional plots for the <em>VKORC1</em> and <em>CYP2C9</em> regions, respectively. In the <em>VKORC1</em> region, the marker showing the strongest effect was rs749671. For this SNP, the G allele was strongly associated with high warfarin dose (G allele, OR: 20.4 [14.3–29.0]; p = 1.08 × 10<sup>−33</sup>). In the <em>CYP2C9</em> region, the strongest effect was observed for rs9332238. For this marker, the G allele was associated with high warfarin dose (G allele, OR: 6.8 [5.0–9.1]; p = 4.4 × 10<sup>−13</sup>), although the nearby marker rs4918798 had the lowest p-value (p = 2.3 × 10<sup>−13</sup>). A detailed analysis of the linkage disequilibrium (LD) patterns in the <em>CYP2C9</em> region indicated that rs9332238 (and to a lesser extent rs4918798) is in almost perfect LD with two known functional SNPs, <em>CYP2C9*2</em> (rs1799853) and <em>CYP2C9*3</em> (rs1057910) (<a href="/pmc/articles/PMC4573240/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). With the exception of a single haplotype, the minor allele of rs9332238 (allele A) is always associated with <em>CYP2C9*2</em> (rs1799853 T allele) or <em>*3</em> (rs1057910 C allele). The ORs [CI] and p-values observed in the meta-analysis for <em>CYP2C9*2</em> were 0.21 [0.16–0.30]; p = 2.1 × 10<sup>−7</sup> and for CYP2C9*3 0.16 [0.10–0.26]; p = 1.8 × 10<sup>−5</sup>, respectively. Aside from <em>VKORC1</em> and <em>CYP2C9</em>, no other regions surpassed genome-wide significance in our analysis. <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 1</a> shows additional information about the main signals identified in the <em>VKORC1</em> and <em>CYP2C9</em> regions.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC4573240/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140527253829008" class="figure"><a href="/pmc/articles/PMC4573240/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4573240_nihms715170f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms715170f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4573240/bin/nihms715170f2.jpg"></a></div><div id="largeobj_idm140527253829008" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4573240/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC4573240/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p7">Regional plot of the <em>VKORC1</em> region.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC4573240/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140527256805904" class="figure"><a href="/pmc/articles/PMC4573240/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4573240_nihms715170f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms715170f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4573240/bin/nihms715170f3.jpg"></a></div><div id="largeobj_idm140527256805904" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4573240/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC4573240/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p8">Regional plot of the <em>CYP2C9</em> region.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p11">Linkage disequilibrium patterns observed between <em>CYP2C9*2</em> (rs1799853), <em>CYP2C9*3</em> (rs1057910) and rs9332238.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140527254202400" class="xtable"><table frame="box" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Haplotype (rs1799853, rs1057910, rs9332338)</th><th rowspan="1" colspan="1" valign="bottom" align="left">Expected</th><th rowspan="1" colspan="1" valign="bottom" align="left">Estimated</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">C A G</td><td rowspan="1" colspan="1" valign="top" align="left">0.6463</td><td rowspan="1" colspan="1" valign="top" align="left">0.7980</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">C A A</td><td rowspan="1" colspan="1" valign="top" align="left">0.1621</td><td rowspan="1" colspan="1" valign="top" align="left">0.0014</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">C C G</td><td rowspan="1" colspan="1" valign="top" align="left">0.0463</td><td rowspan="1" colspan="1" valign="top" align="left">0.0014</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">C C A</td><td rowspan="1" colspan="1" valign="top" align="left">0.0116</td><td rowspan="1" colspan="1" valign="top" align="left">0.0655</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">T A G</td><td rowspan="1" colspan="1" valign="top" align="left">0.0997</td><td rowspan="1" colspan="1" valign="top" align="left">0.0000</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">T A A</td><td rowspan="1" colspan="1" valign="top" align="left">0.0250</td><td rowspan="1" colspan="1" valign="top" align="left">0.1337</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">T C G</td><td rowspan="1" colspan="1" valign="top" align="left">0.0071</td><td rowspan="1" colspan="1" valign="top" align="left">0.0000</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">T C A</td><td rowspan="1" colspan="1" valign="top" align="left">0.0018</td><td rowspan="1" colspan="1" valign="top" align="left">0.0000</td></tr></tbody></table></div><div id="largeobj_idm140527254202400" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4573240/table/T2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TFN2"><p id="__p12" class="p p-first-last">The total number of haplotypes in the sample is 718. Note that, with the exception of a single haplotype, the minor allele of rs9332238 (A allele) is always associated with either <em>CYP2C9*2</em> (rs1799853 T allele) or <em>CYP2C9*3</em> (rs1057910 C allele).</p></div></div></div></div><div id="S21" class="sec"><h3 id="S21title">GWAS conditioning on VKORC1 &amp; CYP2C9 &amp; follow-up of suggestive signals</h3><p id="P19" class="p p-first-last">We carried out conditional analysis including as covariates the <em>VKORC1</em> rs749671 SNP and <em>CYP2C9*2</em> and <em>CYP2C9*3</em>, in addition to the covariates used in the initial analysis. The Manhattan and QQ plots are depicted in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 5 and 6</a>. No genome-wide significant regions were identified in this analysis, but there were several regions with suggestive signals (p &lt; 10<sup>−5</sup>). We analyzed in more detail these regions and, based on the regional plots and the imputation scores, we identified a number of markers to follow-up in three available datasets corresponding to GWA studies in European, African–American and Japanese samples. The European and African–American GWA studies used stable warfarin dose as a quantitative trait, and did not follow the EDP approach that we used in our study. In order to make these two studies directly comparable to our Brazilian study, the European and African–American samples were classified in two categories, following the same approach used in our study: a low-dose category including individuals with stable warfarin weekly dose less than 20 mg and a high-dose category including individuals with weekly dose greater than 42.5 mg/week. A logistic regression was then used to evaluate the association of the lead SNPs with warfarin dose, conditioning on the same markers used as covariates in our study. The Japanese GWA study used a similar EDP approach to that employed in our study, classifying individuals into low and high warfarin categories. However, Japanese typically require lower warfarin dosage than individuals from other populations, and the weekly warfarin thresholds used in the Japanese study were lower than the thresholds defined in our study: 7 mg/week for the low dose category and 28 mg/week for the high-dose category. <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a> shows information about the suggestive regions identified in our conditional analysis. The Table provides information about the lead SNP in each region, and the p-values and ORs observed in the Brazilian sample, as well as the European, African–American and Japanese samples. We could not replicate any of the suggestive signals observed in our conditional analysis. Two of the markers (rs10916661 and rs6897106) were nominally significant in the African–American sample, but the direction of effect is discordant in both studies.</p></div><div id="S22" class="sec"><h3 id="S22title">Gene-based &amp; pathway-based analyses</h3><p id="P20" class="p p-first-last">We carried out gene-based and pathway-based analyses using the programs KGG3 and GSA-SNP (for more details, see Materials &amp; methods section). As expected, the KGG3 program identified genome-wide significant results for genes in the <em>VKORC1</em> and <em>CYP2C9</em> regions. In the analysis taking into account LD patterns (based on the Puerto Rican 1000 Genomes sample), we identified three additional genome-wide significant genes: <em>LINC00511</em> and <em>NDEL1</em>, located on different regions of chromosome 17, and <em>POU4F3</em>, located on chromosome 5. Similar results were obtained when assuming independence between markers. These three genes show clusters of markers showing highly suggestive signals in our SNP analysis. The gene-based results are depicted in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 3</a>. The pathway analysis using the program KGG3, based on 1320 gene sets extracted from several pathway databases, primarily highlighted gene sets related to the Cytochrome P450 pathway. The complete list of pathways that were both significant using the HYST and hypergeometric tests are depicted in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 4</a>. The lists of pathways identified by the program GSA-SNP, based on KEGG and Gene Ontology (GO) gene sets are provided in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Tables 5 and 6</a>.</p></div><div id="S23" class="sec sec-last"><h3 id="S23title">eQTL analyses</h3><p id="P21" class="p p-first-last">We evaluated if eQTLs reported for LCL and human liver are associated with warfarin dose in our study, after Bonferroni correction for multiple testing. For LCL, we used two different datasets: one based on RNA-seq [<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614141">31</a>], and the other based on gene expression arrays [<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614137">33</a>]. For human liver, we used the array-based dataset generated by Schadt [<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614138">34</a>]. For LCL, the only eQTL reaching significance was rs36063822 (p = 4.4 × 10<sup>−7</sup>), which is associated with the expression of the <em>SRCAP</em> gene on chromosome 16. For liver, two eQTLs reached Bonferroni corrected significance. The first eQTL is rs4889606, which is one of the top signals of our GWA study (p = 1.0 × 10<sup>−29</sup>). This SNP is located on the <em>STX1B</em> gene on chromosome 16 and is a cis-eQTL strongly associated with the expression of the nearby <em>VKORC1</em> gene (1.6 × 10<sup>−23</sup>). The second eQTL is rs2289442 (3.2 × 10<sup>−7</sup>), which is associated with the expression of the <em>RNF40</em> gene on chromosome 16.</p></div></div><div id="S24" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S24title">Discussion</h2><p id="P22" class="p p-first">We carried out a GWA study of warfarin maintenance dose in a Brazilian sample. Given our limitations regarding the number of samples that could be analyzed in the study, we applied the EDP approach in order to maximize the statistical power to identify loci with significant effects on warfarin dose [<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614123">16</a>]. The cutoffs for inclusion in the low- and high-dose warfarin dose were defined as at ≤20 mg/week and ≥42.5 mg/week, respectively. We identified genome-wide significant associations in the <em>VKORC1</em> and <em>CYP2C9</em> regions. In the <em>VKORC1</em> region, there were multiple markers with p-values &lt;10<sup>−30</sup> (<a href="/pmc/articles/PMC4573240/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2</span></a>). The top signal was rs749671, with a p-value of 1.08 × 10<sup>−33</sup> and an odds ratio estimate of 20.4 for the G allele. The SNP rs9923231, which has been associated with warfarin dose in previous studies [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614132">12</a>,<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614142">13</a>], showed a similar p-value (p = 3.38 × 10<sup>−33</sup>). These two markers are in very strong LD in the Brazilian samples (r<sup>2</sup> = 0.976) as well as in the European (r<sup>2</sup> = 1.0) and East Asian samples (r<sup>2</sup> = 0.901). Therefore, we seem to be capturing the same signal that has been observed in these populations. Given that our study is based on very dense imputed data generated on 1000 Genomes reference files, we carried out annotations of the top variants (p-value &lt; 10<sup>−28</sup>) identified in the <em>VKORC1</em> region using SNP-Nexus [<a href="#R41" rid="R41" class=" bibr popnode">41</a>] and SNVrap [<a href="#R42" rid="R42" class=" bibr popnode">42</a>]. These results are summarized in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 7</a>. A number of variants are worth mentioning. In our analyses with SNP-Nexus, our top signal, rs749671 was identified as a conserved site based on PHAST and GERP++. Another of our top signals, rs749670 (p = 5 × 10<sup>−32</sup>), is a nonsynonymous polymorphism and the program Polyphen indicates, based on the analysis of one of the transcripts, that it is possibly damaging. This variant was also estimated to be a conserved site based on PHAST and GERP++. The SNP rs14235 (p = 4.25 × 10<sup>−31</sup>) was identified as a conserved transcription binding site, and labeled as evolutionary conserved by PHAST and GERP++. The annotation portal SNVrap provides functional prediction scores that are based on multiple annotation sources. Our top variant, rs749671, as well as rs749670 and rs2303223, were classified as deleterious by SNVrap. Finally, our eQTL analysis identified another SNP, rs4889606 (p = 1.02 × 10<sup>−29</sup>), which is strongly associated with the expression of the <em>VKORC1</em> gene in human liver.</p><p id="P23">In the <em>CYP2C9</em> region, the marker showing the highest OR was rs9332238 (G allele, OR: 6.8 and p = 4.4 × 10<sup>−13</sup>), although there were a few other markers with slightly lower p-values (<a href="/pmc/articles/PMC4573240/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>). Interestingly, the two major functional variants reported in this region, <em>CYP2C9*2</em> and <em>CYP2C9*3</em>, showed substantially lower p-values (p = 2.1 × 10<sup>−7</sup> and p = 1.8 × 10<sup>−5</sup>, respectively) than rs9332238. A detailed analysis of LD between the three markers indicates that rs9332238 (as well as other polymorphisms, such as rs4918798, our top marker in terms of p-value) may be capturing the effects of both <em>CYP2C9*2</em> and <em>CYP2C9*3</em>. The minor allele of rs9332238 is always associated with <em>CYP2C9*2</em> or <em>CYP2C9*3</em> (but never with both, see <a href="/pmc/articles/PMC4573240/table/T2/" target="table" class="fig-table-link figpopup" rid-figpopup="T2" rid-ob="ob-T2" co-legend-rid=""><span>Table 2</span></a>). A similar situation was reported by Takeuchi <em>et al.</em> [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614107">12</a>], who described in a European sample that their top signal in the <em>CYP2C9</em> region, rs4917639, was almost perfectly associated with the ‘composite’ <em>CYP2C9*2</em> and <em>*3</em> allele. In our sample, rs9332238 (minor allele frequency 19.6%) shows a stronger association with the ‘composite’ of <em>CYP2C9*2</em> (frequency 13.1%) and <em>*3</em> (frequency 6.6%) than rs4917639 (frequency 22.2%) or rs4918798 (frequency 21.9%).</p><p id="P24">In our initial analysis, aside from <em>VKORC1</em> and <em>CYP2C9</em>, no other regions reached genome-wide significance. In previous GWA studies of warfarin dose, the application of conditional analysis including <em>VKORC1</em> and <em>CYP2C9</em> in the statistical models uncovered novel variants in <em>CYP4F2</em> and the <em>CYP2C</em> cluster on chromosome 10 [<a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614103">12</a>–<a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614099">14</a>]. We carried out statistical analysis conditioning on <em>VKORC1</em> rs749671, <em>CYP2C9*2</em> and <em>CYP2C9*3</em>. We did not observe any genome-wide significant signals in this analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figures 5 &amp; 6</a>). Based on the regional plots and imputation scores, we selected a number of suggestive SNPs (p &lt; 10<sup>−5</sup>) for follow-up in European and African–American samples, which were analyzed using the same EDP approach used in our study. We could not replicate any of the suggestive signals (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Table 2</a>). The ideal approach for replication would have been to analyze the markers in another Brazilian sample, but no such sample was available to us. The results of the replication do not have a straightforward interpretation, given that the patterns of LD may differ between these populations and the Brazilian admixed groups. The lack of replication could be due to a number of factors, including: 1) the signals identified are false positives, 2) lack of statistical power, particularly considering that the effect sizes observed in our study may be inflated (winner’s curse), and 3) discordant patterns of LD in samples of different ancestry [<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614104">43</a>]. In our gene-based association tests using the GATES approach<sup>32</sup>, in addition to <em>VKORC1</em> and <em>CYP2C9</em>, three other genes, <em>LINC00511</em>, <em>NDEL1</em> and <em>POU4F3</em>, reached genome-wide significance. To our knowledge, these genes have not been associated with warfarin dose in any previous studies. We evaluated the association of the lead SNPs in these genes in the European and African–American samples, but none of the SNPs was nominally significant in these samples (data not shown). As expected, our pathway analyses using KEGG annotations identified genesets involved in the metabolism of xenobiotics, driven by genome-wide signals observed on chromosome 10, where <em>CYP2C9</em> and other genes of the cytochrome P450 CYP2C cluster are located. The pathway analyses based on GO annotations also highlighted genesets involved in drug metabolism process, including <em>VKORC1</em> and <em>CYP2C9</em>.</p><p id="P25" class="p p-last">In conclusion, here we present the first GWA study of warfarin dose in the Brazilian population. In agreement with previous studies in other populations, the strongest effects on warfarin dose were observed for <em>VKORC1</em> and <em>CYP2C9</em>. No other genome-wide significant regions were identified in the study, and we could not replicate a number of suggestive signals in European and African–American samples. This may be related to two important limitations of this study: the sample size of the discovery sample was relatively small, in spite of the fact that we used an EDP approach to maximize our power to detect significant effects, and no Brazilian samples were available for replication, therefore complicating the interpretation of the replication results. Further GWA studies or meta-analytic efforts with larger samples sizes will be necessary to identify other potential regions with smaller effects on warfarin dose in this population.</p></div><div id="S25" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S25title">Future perspective</h2><p id="P26" class="p p-first-last">The complex structure of the Brazilian population, highly heterogeneous and admixed, implies that extrapolation of pharmacogenomic data from candidate gene or GWA studies in well-defined ethnic groups (e.g., European and North American Caucasians) may not be appropriate to the majority of Brazilians [<a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614097">44</a>]. This prompted the present study, which extended to Brazilians the GWAS evidence that common SNPs with large effects on warfarin dose appear to be restricted to the <em>CYP2C9</em> and <em>VKORC1</em> regions [<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_474614118">11</a>]. One of the major strengths of this study was the availability of imputed data based on the dense 1000 Genomes reference dataset, which made it possible to evaluate in detail the effect of common variants in these regions. Based on our annotations of the top signals, several SNPs within the <em>VKORC1</em> region could have putative functional effects, and it would be important to explore through functional assays or studies in animal models the specific role of these variants in warfarin dose. In <em>CYP2C9</em>, we identified a polymorphism, rs749671, that is virtually in perfect association with the <em>CYP2C9*2</em> and <em>*3</em> alleles. It would be interesting to examine the impact of including rs749671 – rather than <em>CYP2C9*2</em> and <em>*3</em> – as a covariate in warfarin dosing algorithms for Brazilians. Finally, further GWA studies or meta-analytic efforts with independent cohorts of Brazilian patients will be necessary to validate the suggestive signals with smaller effects on warfarin dose in this population.</p></div><div id="unreferenced-ft" class="tsec sec headless whole_rhythm"><div class="unreferenced">​
                    <div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="idm140527260622672title">Executive summary</h3></div><div class="sec" id="S26"><h3 id="S26title">Background</h3><ul class="unordered" style="list-style-type:disc" id="L1"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><p id="P28">Warfarin is the most commonly used anticoagulant in the world, and several genome-wide association (GWA) studies in diverse populations have explored the role of genetic variants on warfarin dose.</p></li><li><p id="P29">Here we report the first GWA study in Brazilian patients, from two cohorts who had been previously enrolled in warfarin pharmacogenomic trials using candidate gene approaches.</p></li><li><p id="P30">The extreme discordant phenotype (EDP) methodology was applied to select patients receiving low (≤20 mg/week, n = 180) or high stable warfarin doses (≥42.5 mg/week, n = 187). Sampling extreme phenotype individuals confers higher power for GWAS of drug phenotypes, compared with random cohorts.</p></li></ul></div><div class="sec" id="S27"><h3 id="S27title">Genome-wide signals are identified in the VKORC1 &amp; CYP2C9 regions</h3><ul class="unordered" style="list-style-type:disc" id="L2"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><p id="P32">In the <em>VKORC1</em> region, the lead SNP was rs749671 (OR: 20.4; p = 1.08 × 10<sup>−33</sup>), whereas in the <em>CYP2C9</em> region, the strongest effect was observed for rs9332238 (OR: 6.8; p = 4.4 × 10<sup>−13</sup>).</p></li><li><p id="P33">The rs9332238 SNP is in virtually perfect LD with two known functional SNPs, namely <em>CYP2C9*2</em> (rs1799853) and <em>CYP2C9*3</em> (rs1057910).</p></li><li><p id="P34">No other genome-wide significant regions were identified in the initial GWA analysis.</p></li></ul></div><div class="sec" id="S28"><h3 id="S28title">GWAS analyses conditioning for VKORC1 rs749671, CYP2C9*2 &amp; CYP2C9*3 disclosed suggestive signals</h3><ul class="unordered" style="list-style-type:disc" id="L3"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><p id="P36">Conditional analyses identified a list of highly suggestive regions (p &lt; 10<sup>−5</sup>).</p></li><li><p id="P37">Because no samples from Brazilian patients were available to us for validation, the lead SNPs for each region were followed up in European, East Asian and African–American samples, which were analyzed using the EDP approach.</p></li><li><p id="P38">We could not replicate any of the suggestive signals in these samples.</p></li></ul></div><div class="sec" id="S29"><h3 id="S29title">Conclusion</h3><ul class="unordered" style="list-style-type:disc" id="L4"><!--
list-behavior=unordered
prefix-word=
mark-type=disc
max-label-size=0
--><li><p id="P40">In agreement with previous studies in other populations, the strongest effects on warfarin dose were observed for <em>VKORC1</em> and <em>CYP2C9</em>. No other genome-wide significant regions were identified in the study.</p></li><li><p id="P41">Suggestive signals observed in conditional analyses could not be replicated in European, East Asian and African–American samples. This may be related to two important limitations of this study: the sample size of the discovery sample was relatively small, in spite of the fact that we used an EDP approach to maximize our power to detect significant effects, and no Brazilian samples were available for replication, which hinders the interpretation of the replication results.</p></li><li><p id="P42">Further GWA studies or meta-analytic efforts with larger sample sizes will be necessary to identify other potential regions with smaller effects on warfarin dose in this population.</p></li></ul></div></div></div></div><div id="S31" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S31title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supp_Figures</h4><div class="sup-box half_rhythm" id="d36e1054"><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Supp_Figures.pptx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.1M, pptx)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supp_Table_Legends</h4><div class="sup-box half_rhythm" id="d36e1058"><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Supp_Table_Legends.docx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(15K, docx)</sup></div></div><div class="sec suppmat" id="SD3"><h4>Table 1</h4><div class="sup-box half_rhythm" id="d36e1062"><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_1.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(14K, xlsx)</sup></div></div><div class="sec suppmat" id="SD4"><h4>Table 2</h4><div class="sup-box half_rhythm" id="d36e1066"><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(16K, xlsx)</sup></div></div><div class="sec suppmat" id="SD5"><h4>Table 3</h4><div class="sup-box half_rhythm" id="d36e1070"><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_3.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(66K, xlsx)</sup></div></div><div class="sec suppmat" id="SD6"><h4>Table 4</h4><div class="sup-box half_rhythm" id="d36e1074"><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_4.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(52K, xlsx)</sup></div></div><div class="sec suppmat" id="SD7"><h4>Table 5</h4><div class="sup-box half_rhythm" id="d36e1078"><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_5.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(26K, xlsx)</sup></div></div><div class="sec suppmat" id="SD8"><h4>Table 6</h4><div class="sup-box half_rhythm" id="d36e1082"><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_6.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(128K, xlsx)</sup></div></div><div class="sec suppmat" id="SD9"><h4>Table 7</h4><div class="sup-box half_rhythm" id="d36e1086"><a href="/pmc/articles/PMC4573240/bin/NIHMS715170-supplement-Table_7.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(15K, xlsx)</sup></div></div></div><div id="idm140527261714176" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140527261714176title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"></p><p id="__p1" class="p p-first"><strong>Ethical conduct of research</strong></p><p id="__p2" class="p p-last">The authors state that they have obtained appropriate institutional review board approval or have followed the principles outlined in the Declaration of Helsinki for all human or animal experimental investigations. In addition, for investigations involving human subjects, informed consent has been obtained from the participants involved.</p><p></p><p class="fn sec" id="FN3"></p><p id="__p3" class="p p-first"><strong>Financial &amp; competing interests disclosure</strong></p><p id="__p4" class="p">The research was supported in part by grants from Financiadora de Estudos e Projetos (Finep 01.08.01230.00), Conselho Nacional de Desenvolvimento Científico e Tecnológico (CNPq) and Fundação de Amparo à Pesquisa do Estado do Rio de Janeiro (Faperj) to GS-K. TK was supported by NIH NIGMS R24 GM61374. The authors have no other relevant affiliations or financial involvement with any organization or entity with a financial interest in or financial conflict with the subject matter or materials discussed in the manuscript. This includes employment, consultancies, honoraria, stock ownership or options, expert testimony, grants or patents received or pending, or royalties.</p><p id="__p5" class="p p-last">No writing assistance was utilized in the production of this manuscript.</p><p></p></div></div><div id="idm140527252240016" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140527252240016title">References</h2><div class="ref-list-sec sec" id="reference-list"><p id="P57">Papers of special note have been highlighted as:</p><p id="P58">• of interest</p><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Sconce EA, Khan TI, Wynne HA, et al.  The impact of <em>CYP2C9</em> and <em>VKORC1</em> genetic polymorphism and patient characteristics upon warfarin dose requirements: proposal for a new dosing regimen. <span><span class="ref-journal">Blood. </span>2005;<span class="ref-vol">106</span>:2329–2333.</span> [<a href="/pubmed/15947090" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=The+impact+of+CYP2C9+and+VKORC1+genetic+polymorphism+and+patient+characteristics+upon+warfarin+dose+requirements:+proposal+for+a+new+dosing+regimen&amp;author=EA+Sconce&amp;author=TI+Khan&amp;author=HA+Wynne&amp;volume=106&amp;publication_year=2005&amp;pages=2329-2333&amp;pmid=15947090&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Anderson JL, Horne BD, Stevens SM, et al.  Randomized trial of genotype-guided versus standard warfarin dosing in patients initiating oral anticoagulation. <span><span class="ref-journal">Circulation. </span>2007;<span class="ref-vol">116</span>:2563–2570.</span> [<a href="/pubmed/17989110" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Randomized+trial+of+genotype-guided+versus+standard+warfarin+dosing+in+patients+initiating+oral+anticoagulation&amp;author=JL+Anderson&amp;author=BD+Horne&amp;author=SM+Stevens&amp;volume=116&amp;publication_year=2007&amp;pages=2563-2570&amp;pmid=17989110&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Wen MS, Lee M, Chen JJ, et al.  Prospective study of warfarin dosage requirements based on <em>CYP2C9</em> and <em>VKORC1</em> genotypes. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2008;<span class="ref-vol">84</span>:83–89.</span> [<a href="/pubmed/18183038" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Prospective+study+of+warfarin+dosage+requirements+based+on+CYP2C9+and+VKORC1+genotypes&amp;author=MS+Wen&amp;author=M+Lee&amp;author=JJ+Chen&amp;volume=84&amp;publication_year=2008&amp;pages=83-89&amp;pmid=18183038&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Gage BF, Eby C, Johnson JA, et al.  Use of pharmacogenetic and clinical factors to predict the therapeutic dose of warfarin. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2008;<span class="ref-vol">84</span>:326–331.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683977/">PMC free article</a>]</span> [<a href="/pubmed/18305455" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Use+of+pharmacogenetic+and+clinical+factors+to+predict+the+therapeutic+dose+of+warfarin&amp;author=BF+Gage&amp;author=C+Eby&amp;author=JA+Johnson&amp;volume=84&amp;publication_year=2008&amp;pages=326-331&amp;pmid=18305455&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5•. <span class="element-citation">Perini JA, Struchiner CJ, Silva-Assunção E, et al.  Pharmacogenetics of warfarin: development of a dosing algorithm for brazilian patients. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2008;<span class="ref-vol">84</span>:722–728.</span> First study of warfarin pharmacogenetics in Brazilian patients. The present study enrolled some of these patients. [<a href="/pubmed/18754001" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Pharmacogenetics+of+warfarin:+development+of+a+dosing+algorithm+for+brazilian+patients&amp;author=JA+Perini&amp;author=CJ+Struchiner&amp;author=E+Silva-Assunção&amp;volume=84&amp;publication_year=2008&amp;pages=722-728&amp;pmid=18754001&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">International Warfarin Pharmacogenetics Consortium. Klein TE, Altman RB, et al.  Estimation of the warfarin dose with clinical and pharmacogenetic data. <span><span class="ref-journal">N Engl J Med. </span>2009;<span class="ref-vol">360</span>:753–764.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2722908/">PMC free article</a>]</span> [<a href="/pubmed/19228618" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Estimation+of+the+warfarin+dose+with+clinical+and+pharmacogenetic+data&amp;author=TE+Klein&amp;author=RB+Altman&amp;volume=360&amp;publication_year=2009&amp;pages=753-764&amp;pmid=19228618&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Lenzini P, Wadelius M, Kimmel S, et al.  Integration of genetic, clinical, and INR data to refine warfarin dosing. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2010;<span class="ref-vol">87</span>:572–578.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2858245/">PMC free article</a>]</span> [<a href="/pubmed/20375999" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Integration+of+genetic,+clinical,+and+INR+data+to+refine+warfarin+dosing&amp;author=P+Lenzini&amp;author=M+Wadelius&amp;author=S+Kimmel&amp;volume=87&amp;publication_year=2010&amp;pages=572-578&amp;pmid=20375999&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Limdi NA, Wadelius M, Cavallari L, et al.  Warfarin pharmacogenetics: a single <em>VKORC1</em> polymorphism is predictive of dose across 3 racial groups. <span><span class="ref-journal">Blood. </span>2010;<span class="ref-vol">115</span>:3827–3834.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2865873/">PMC free article</a>]</span> [<a href="/pubmed/20203262" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Warfarin+pharmacogenetics:+a+single+VKORC1+polymorphism+is+predictive+of+dose+across+3+racial+groups&amp;author=NA+Limdi&amp;author=M+Wadelius&amp;author=L+Cavallari&amp;volume=115&amp;publication_year=2010&amp;pages=3827-3834&amp;pmid=20203262&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Kimmel SE, French B, Kasner SE, et al.  A pharmacogenetic versus a clinical algorithm for warfarin dosing. <span><span class="ref-journal">N Engl J Med. </span>2013;<span class="ref-vol">369</span>:2283–2293.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3942158/">PMC free article</a>]</span> [<a href="/pubmed/24251361" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=A+pharmacogenetic+versus+a+clinical+algorithm+for+warfarin+dosing&amp;author=SE+Kimmel&amp;author=B+French&amp;author=SE+Kasner&amp;volume=369&amp;publication_year=2013&amp;pages=2283-2293&amp;pmid=24251361&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Pirmohamed M, Burnside G, Eriksson N, et al.  A randomized trial of genotype-guided dosing of warfarin. <span><span class="ref-journal">N Engl J Med. </span>2013;<span class="ref-vol">369</span>:2294–2303.</span> [<a href="/pubmed/24251363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=A+randomized+trial+of+genotype-guided+dosing+of+warfarin&amp;author=M+Pirmohamed&amp;author=G+Burnside&amp;author=N+Eriksson&amp;volume=369&amp;publication_year=2013&amp;pages=2294-2303&amp;pmid=24251363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11•. <span class="element-citation">Cooper GM, Johnson JA, Langaee TY, et al.  A genome-wide scan for common genetic variants with a large influence on warfarin maintenance dose. <span><span class="ref-journal">Blood. </span>2008;<span class="ref-vol">112</span>:1022–1027.</span> The first warfarin GWA study, in which it was suggested that common SNPs with large effects on warfarin dose were unlikely to be discovered outside of the <em>CYP2C9</em> and <em>VKORC1</em> genes. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2515139/">PMC free article</a>]</span> [<a href="/pubmed/18535201" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=A+genome-wide+scan+for+common+genetic+variants+with+a+large+influence+on+warfarin+maintenance+dose&amp;author=GM+Cooper&amp;author=JA+Johnson&amp;author=TY+Langaee&amp;volume=112&amp;publication_year=2008&amp;pages=1022-1027&amp;pmid=18535201&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12•. <span class="element-citation">Takeuchi F, McGinnis R, Bourgeois, et al.  A genome-wide association study confirms <em>VKORC1</em>, <em>CYP2C9</em>, and <em>CYP4F2</em> as principal genetic determinants of warfarin dose. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000433.</span> Conditional analysis revealed a genome-wide association study (GWA) signal in <em>CYP4F2</em> (rs2108622) <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2652833/">PMC free article</a>]</span> [<a href="/pubmed/19300499" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+genome-wide+association+study+confirms+VKORC1,+CYP2C9,+and+CYP4F2+as+principal+genetic+determinants+of+warfarin+dose&amp;author=F+Takeuchi&amp;author=R+McGinnis&amp;author=+Bourgeois&amp;volume=5&amp;publication_year=2009&amp;pages=e1000433&amp;pmid=19300499&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Cha PC, Mushiroda T, Takahashi A, et al.  Genome-wide association study identifies genetic determinants of warfarin responsiveness for Japanese. <span><span class="ref-journal">Hum Mol Genet. </span>2010;<span class="ref-vol">19</span>:4735–4744.</span> [<a href="/pubmed/20833655" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=Genome-wide+association+study+identifies+genetic+determinants+of+warfarin+responsiveness+for+Japanese&amp;author=PC+Cha&amp;author=T+Mushiroda&amp;author=A+Takahashi&amp;volume=19&amp;publication_year=2010&amp;pages=4735-4744&amp;pmid=20833655&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Perera MA, Cavallari LH, Limdi NA, et al.  Genetic variants associated with warfarin dose in African–American individuals: a genome-wide association study. <span><span class="ref-journal">Lancet. </span>2013;<span class="ref-vol">382</span>:790–796.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3759580/">PMC free article</a>]</span> [<a href="/pubmed/23755828" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Genetic+variants+associated+with+warfarin+dose+in+African–American+individuals:+a+genome-wide+association+study&amp;author=MA+Perera&amp;author=LH+Cavallari&amp;author=NA+Limdi&amp;volume=382&amp;publication_year=2013&amp;pages=790-796&amp;pmid=23755828&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15•. <span class="element-citation">Nebert DW. Extreme discordant phenotype methodology: an intuitive approach to clinical pharmacogenetics. <span><span class="ref-journal">Eur J Pharmacol. </span>2000;<span class="ref-vol">410</span>:107–120.</span> Description of the extreme discordant phenotype methodology. [<a href="/pubmed/11134663" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Pharmacol&amp;title=Extreme+discordant+phenotype+methodology:+an+intuitive+approach+to+clinical+pharmacogenetics&amp;author=DW+Nebert&amp;volume=410&amp;publication_year=2000&amp;pages=107-120&amp;pmid=11134663&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Gurwitz D, McLeod HL. Genome-wide studies in pharmacogenomics: harnessing the power of extreme phenotypes. <span><span class="ref-journal">Pharmacogenomics. </span>2013;<span class="ref-vol">14</span>:337–339.</span> [<a href="/pubmed/23438876" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&amp;title=Genome-wide+studies+in+pharmacogenomics:+harnessing+the+power+of+extreme+phenotypes&amp;author=D+Gurwitz&amp;author=HL+McLeod&amp;volume=14&amp;publication_year=2013&amp;pages=337-339&amp;pmid=23438876&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17•. <span class="element-citation">Botton MR, Bandinelli E, Rohde LE, Amon LC, Hutz MH. Influence of genetic, biological and pharmacological factors on warfarin dose in a Southern Brazilian population of European ancestry. <span><span class="ref-journal">Br J Clin Pharmacol. </span>2011;<span class="ref-vol">72</span>:442–450.</span> Patients from this cohort were enrolled in the present GWA study. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3175513/">PMC free article</a>]</span> [<a href="/pubmed/21320153" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Clin+Pharmacol&amp;title=Influence+of+genetic,+biological+and+pharmacological+factors+on+warfarin+dose+in+a+Southern+Brazilian+population+of+European+ancestry&amp;author=MR+Botton&amp;author=E+Bandinelli&amp;author=LE+Rohde&amp;author=LC+Amon&amp;author=MH+Hutz&amp;volume=72&amp;publication_year=2011&amp;pages=442-450&amp;pmid=21320153&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Purcell S, Neale B, Todd-Brown K, et al.  PLINK: a toolset for whole-genome association and population-based linkage analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+toolset+for+whole-genome+association+and+population-based+linkage+analysis&amp;author=S+Purcell&amp;author=B+Neale&amp;author=K+Todd-Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">Patterson N, Price AL, Reich D. Population structure and eigenanalysis. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e190.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1713260/">PMC free article</a>]</span> [<a href="/pubmed/17194218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=Population+structure+and+eigenanalysis&amp;author=N+Patterson&amp;author=AL+Price&amp;author=D+Reich&amp;volume=2&amp;publication_year=2006&amp;pages=e190&amp;pmid=17194218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Alexander DH, Novembre J, Lange K. Fast model-based estimation of ancestry in unrelated individuals. <span><span class="ref-journal">Genome Res. </span>2009;<span class="ref-vol">19</span>:1655–1664.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2752134/">PMC free article</a>]</span> [<a href="/pubmed/19648217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Fast+model-based+estimation+of+ancestry+in+unrelated+individuals&amp;author=DH+Alexander&amp;author=J+Novembre&amp;author=K+Lange&amp;volume=19&amp;publication_year=2009&amp;pages=1655-1664&amp;pmid=19648217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Marchini J, Howie B, Myers S, McVean G, Donnelly A new multipoint method for genome-wide association studies via imputation of genotypes. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+via+imputation+of+genotypes&amp;author=J+Marchini&amp;author=B+Howie&amp;author=S+Myers&amp;author=G+McVean&amp;author=+Donnelly&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Delaneau O, Marchini J, Zagury JF. A linear complexity phasing method for thousands of genomes. <span><span class="ref-journal">Nat Methods. </span>2012;<span class="ref-vol">9</span>:179–181.</span> [<a href="/pubmed/22138821" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Methods&amp;title=A+linear+complexity+phasing+method+for+thousands+of+genomes&amp;author=O+Delaneau&amp;author=J+Marchini&amp;author=JF+Zagury&amp;volume=9&amp;publication_year=2012&amp;pages=179-181&amp;pmid=22138821&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:955–959.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3696580/">PMC free article</a>]</span> [<a href="/pubmed/22820512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Fast+and+accurate+genotype+imputation+in+genome-wide+association+studies+through+pre-phasing&amp;author=B+Howie&amp;author=C+Fuchsberger&amp;author=M+Stephens&amp;author=J+Marchini&amp;author=GR+Abecasis&amp;volume=44&amp;publication_year=2012&amp;pages=955-959&amp;pmid=22820512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Marchini J, Howie B. Genotype imputation for genome-wide association studies. <span><span class="ref-journal">Nat Rev Genet. </span>2010;<span class="ref-vol">11</span>:499–511.</span> [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Rev+Genet&amp;title=Genotype+imputation+for+genome-wide+association+studies&amp;author=J+Marchini&amp;author=B+Howie&amp;volume=11&amp;publication_year=2010&amp;pages=499-511&amp;pmid=20517342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Jorgensen AL, Al-Zubiedi S, Zhang JE, et al.  Genetic and environmental factors determining clinical outcomes and cost of warfarin therapy, a prospective study. <span><span class="ref-journal">Pharmacogenet Genomics. </span>2009;<span class="ref-vol">19</span>:800–812.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3330749/">PMC free article</a>]</span> [<a href="/pubmed/19752777" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet+Genomics&amp;title=Genetic+and+environmental+factors+determining+clinical+outcomes+and+cost+of+warfarin+therapy,+a+prospective+study&amp;author=AL+Jorgensen&amp;author=S+Al-Zubiedi&amp;author=JE+Zhang&amp;volume=19&amp;publication_year=2009&amp;pages=800-812&amp;pmid=19752777&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Wadelius M, Chen LY, Lindh JD, et al.  The largest prospective warfarin-treated cohort supports genetic forecasting. <span><span class="ref-journal">Blood. </span>2009;<span class="ref-vol">113</span>:784–792.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2630264/">PMC free article</a>]</span> [<a href="/pubmed/18574025" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=The+largest+prospective+warfarin-treated+cohort+supports+genetic+forecasting&amp;author=M+Wadelius&amp;author=LY+Chen&amp;author=JD+Lindh&amp;volume=113&amp;publication_year=2009&amp;pages=784-792&amp;pmid=18574025&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Hillman MA, Wilke RA, Caldwell MD, Berg RL, Glurich I, Burmester JK. Relative impact of covariates in prescribing warfarin according to <em>CYP2C9</em> genotype. <span><span class="ref-journal">Pharmacogenetics. </span>2004;<span class="ref-vol">14</span>:539–547.</span> [<a href="/pubmed/15284536" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&amp;title=Relative+impact+of+covariates+in+prescribing+warfarin+according+to+CYP2C9+genotype&amp;author=MA+Hillman&amp;author=RA+Wilke&amp;author=MD+Caldwell&amp;author=RL+Berg&amp;author=I+Glurich&amp;volume=14&amp;publication_year=2004&amp;pages=539-547&amp;pmid=15284536&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Higashi MK, Veenstra DL, Kondo LM, et al.  Association between <em>CYP2C9</em> genetic variants and anticoagulation-related outcomes during warfarin therapy. <span><span class="ref-journal">JAMA. </span>2002;<span class="ref-vol">287</span>:1690–1698.</span> [<a href="/pubmed/11926893" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Association+between+CYP2C9+genetic+variants+and+anticoagulation-related+outcomes+during+warfarin+therapy&amp;author=MK+Higashi&amp;author=DL+Veenstra&amp;author=LM+Kondo&amp;volume=287&amp;publication_year=2002&amp;pages=1690-1698&amp;pmid=11926893&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Limdi NA, McGwin G, Goldstein JA, et al.  Influence of <em>CYP2C9</em> and <em>VKORC1</em> 1173C/T genotype on the risk of hemorrhagic complications in African–American and European-American patients on warfarin. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2008;<span class="ref-vol">83</span>:312–321.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683398/">PMC free article</a>]</span> [<a href="/pubmed/17653141" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Influence+of+CYP2C9+and+VKORC1+1173C/T+genotype+on+the+risk+of+hemorrhagic+complications+in+African–American+and+European-American+patients+on+warfarin&amp;author=NA+Limdi&amp;author=G+McGwin&amp;author=JA+Goldstein&amp;volume=83&amp;publication_year=2008&amp;pages=312-321&amp;pmid=17653141&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Limdi NA, Beasley TM, Baird MF, et al.  Kidney function influences warfarin responsiveness and hemorrhagic complications. <span><span class="ref-journal">J Am Soc Nephrol. </span>2009;<span class="ref-vol">20</span>:912–921.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2663833/">PMC free article</a>]</span> [<a href="/pubmed/19225037" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Soc+Nephrol&amp;title=Kidney+function+influences+warfarin+responsiveness+and+hemorrhagic+complications&amp;author=NA+Limdi&amp;author=TM+Beasley&amp;author=MF+Baird&amp;volume=20&amp;publication_year=2009&amp;pages=912-921&amp;pmid=19225037&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Cavallari LH, Langaee TY, Momary KM, et al.  Genetic and clinical predictors of warfarin dose requirements in African Americans. <span><span class="ref-journal">Clin Pharmacol Ther. </span>2010;<span class="ref-vol">87</span>:459–464.</span> [<a href="/pubmed/20072124" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Pharmacol+Ther&amp;title=Genetic+and+clinical+predictors+of+warfarin+dose+requirements+in+African+Americans&amp;author=LH+Cavallari&amp;author=TY+Langaee&amp;author=KM+Momary&amp;volume=87&amp;publication_year=2010&amp;pages=459-464&amp;pmid=20072124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation"><span><span class="ref-journal">KGG: A systematic biological knowledge-based mining system for genome-wide association studies. </span></span>  <a href="http://statgenpro.psychiatry.hku.hk/limx/kgg/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://statgenpro.psychiatry.hku.hk/limx/kgg/</a></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Li MX, Gui HS, Kwan JS, Sham PC. GATES, a rapid and powerful gene-based association test using extended Simes procedure. <span><span class="ref-journal">Am J Hum Genet. </span>2011;<span class="ref-vol">88</span>:283–293.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3059433/">PMC free article</a>]</span> [<a href="/pubmed/21397060" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=GATES,+a+rapid+and+powerful+gene-based+association+test+using+extended+Simes+procedure&amp;author=MX+Li&amp;author=HS+Gui&amp;author=JS+Kwan&amp;author=PC+Sham&amp;volume=88&amp;publication_year=2011&amp;pages=283-293&amp;pmid=21397060&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Li MX, Kwan JS, Sham PC. HYST, A hybrid set-based test for genome-wide association studies, with application to protein-protein interaction-based association analysis. <span><span class="ref-journal">Am J Hum Genet. </span>2012;<span class="ref-vol">91</span>:478–488.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3511992/">PMC free article</a>]</span> [<a href="/pubmed/22958900" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=HYST,+A+hybrid+set-based+test+for+genome-wide+association+studies,+with+application+to+protein-protein+interaction-based+association+analysis&amp;author=MX+Li&amp;author=JS+Kwan&amp;author=PC+Sham&amp;volume=91&amp;publication_year=2012&amp;pages=478-488&amp;pmid=22958900&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Nam D, Kim J, Kim SY, Kim S. GSA-SNP, a general approach for gene set analysis of polymorphisms. <span><span class="ref-journal">Nucleic Acids Res. </span>2010;<span class="ref-vol">28</span>:W749–W754.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2896081/">PMC free article</a>]</span> [<a href="/pubmed/20501604" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=GSA-SNP,+a+general+approach+for+gene+set+analysis+of+polymorphisms&amp;author=D+Nam&amp;author=J+Kim&amp;author=SY+Kim&amp;author=S+Kim&amp;volume=28&amp;publication_year=2010&amp;pages=W749-W754&amp;pmid=20501604&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Kim S-Y, Volsky DJ. PAGE, parametric analysis of gene set enrichment. <span><span class="ref-journal">BMC Bioinformatics. </span>2005;<span class="ref-vol">6</span>:144.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1183189/">PMC free article</a>]</span> [<a href="/pubmed/15941488" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinformatics&amp;title=PAGE,+parametric+analysis+of+gene+set+enrichment&amp;author=S-Y+Kim&amp;author=DJ+Volsky&amp;volume=6&amp;publication_year=2005&amp;pages=144&amp;pmid=15941488&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">NCBI eQTL browser. <span></span>  <a href="http://www.ncbi.nlm.nih.gov/projects/gap/eqtl/index.cgi#" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.ncbi.nlm.nih.gov/projects/gap/eqtl/index.cgi#</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="element-citation">Montgomery SB, Sammeth M, Gutierrez-Arcelus M, et al.  Transcriptome genetics using second generation sequencing in a Caucasian population. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">464</span>:773–777.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3836232/">PMC free article</a>]</span> [<a href="/pubmed/20220756" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Transcriptome+genetics+using+second+generation+sequencing+in+a+Caucasian+population&amp;author=SB+Montgomery&amp;author=M+Sammeth&amp;author=M+Gutierrez-Arcelus&amp;volume=464&amp;publication_year=2010&amp;pages=773-777&amp;pmid=20220756&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="element-citation">Stranger BE, Nica AC, Forrest MS, et al.  Population genomics of human gene expression. <span><span class="ref-journal">Nat Genet. </span>2007;<span class="ref-vol">39</span>:1217–1224.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2683249/">PMC free article</a>]</span> [<a href="/pubmed/17873874" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Population+genomics+of+human+gene+expression&amp;author=BE+Stranger&amp;author=AC+Nica&amp;author=MS+Forrest&amp;volume=39&amp;publication_year=2007&amp;pages=1217-1224&amp;pmid=17873874&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="element-citation">Schadt EE, Molony C, Chudin E, et al.  Mapping the genetic architecture of gene expression in human liver. <span><span class="ref-journal">PLoS Biol. </span>2008;<span class="ref-vol">6</span>:e107.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2365981/">PMC free article</a>]</span> [<a href="/pubmed/18462017" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Biol&amp;title=Mapping+the+genetic+architecture+of+gene+expression+in+human+liver&amp;author=EE+Schadt&amp;author=C+Molony&amp;author=E+Chudin&amp;volume=6&amp;publication_year=2008&amp;pages=e107&amp;pmid=18462017&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="element-citation">SNP nexus. <span></span>  <a href="http://www.snp-nexus.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://www.snp-nexus.org/</a></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="element-citation"><span><span class="ref-journal">SNVrap: Quick annotation web portal. </span></span>  <a href="http://jjwanglab.org/snvrap" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">http://jjwanglab.org/snvrap</a>.</span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="element-citation">Kraft P, Zeggini E, Ioannidis JP. Replication in genome-wide association studies. <span><span class="ref-journal">Stat Sci. </span>2009;<span class="ref-vol">24</span>:561–573.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2865141/">PMC free article</a>]</span> [<a href="/pubmed/20454541" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat+Sci&amp;title=Replication+in+genome-wide+association+studies&amp;author=P+Kraft&amp;author=E+Zeggini&amp;author=JP+Ioannidis&amp;volume=24&amp;publication_year=2009&amp;pages=561-573&amp;pmid=20454541&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44•. <span class="element-citation">Suarez-Kurtz G. Pharmacogenetics in the Brazilian population. <span><span class="ref-journal">Front Pharmacol. </span>2010;<span class="ref-vol">1</span>:118.</span> An analysis of the pharmacogenomic implications of the Brazilian population structure. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3153000/">PMC free article</a>]</span> [<a href="/pubmed/21833165" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front+Pharmacol&amp;title=Pharmacogenetics+in+the+Brazilian+population&amp;author=G+Suarez-Kurtz&amp;volume=1&amp;publication_year=2010&amp;pages=118&amp;pmid=21833165&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4573240&amp;issue-id=257959&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140527239290960title">Abstract</a></li><li><a href="#S5title">Background</a></li><li><a href="#S6title">Materials &amp; methods</a></li><li><a href="#S18title">Results</a></li><li><a href="#S24title">Discussion</a></li><li><a href="#S25title">Future perspective</a></li><li><a href="#S31title">Supplementary Material</a></li><li><a href="#idm140527261714176title">Footnotes</a></li><li><a href="#idm140527252240016title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4573240/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4573240/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4573240/pdf/nihms715170.pdf">PDF (727K)</a></li> | <li><a href="#" data-citationid="PMC4573240" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4573240%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4573240%2F&amp;text=Genome-wide%20association%20study%20of%20warfarin%20maintenance%20dose%20in%20a%20Brazilian%20sample" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4573240%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26265036" data-aiid="4573240" data-aid="4573240" data-iid="257959" data-domainid="319" data-domain="nihpa" data-accid="PMC4573240" data-md5="755d19b3fb5fc3b2e058a0671283f792"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T13:56:19-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE8981FFDD58258100000000069B03F8&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4573240%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4573240/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>