<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="carcin"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3291863/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>A genome-wide search for loci interacting with known prostate cancer risk-associated genetic variants</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3291863/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Carcinogenesis"><meta name="citation_title" content="A genome-wide search for loci interacting with known prostate cancer risk-associated genetic variants"><meta name="citation_authors" content="Sha Tao, Zhong Wang, Junjie Feng, Fang-Chi Hsu, Guangfu Jin, Seong-Tae Kim, Zheng Zhang, Henrik Gronberg, Lilly S. Zheng, William B. Isaacs, Jianfeng Xu, Jielin Sun"><meta name="citation_date" content="March 2012"><meta name="citation_issue" content="3"><meta name="citation_volume" content="33"><meta name="citation_firstpage" content="598"><meta name="citation_doi" content="10.1093/carcin/bgr316"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3291863/?report=abstract"><meta name="citation_pmid" content="22219177"><meta name="DC.Title" content="A genome-wide search for loci interacting with known prostate cancer risk-associated genetic variants"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Sha Tao"><meta name="DC.Contributor" content="Zhong Wang"><meta name="DC.Contributor" content="Junjie Feng"><meta name="DC.Contributor" content="Fang-Chi Hsu"><meta name="DC.Contributor" content="Guangfu Jin"><meta name="DC.Contributor" content="Seong-Tae Kim"><meta name="DC.Contributor" content="Zheng Zhang"><meta name="DC.Contributor" content="Henrik Gronberg"><meta name="DC.Contributor" content="Lilly S. Zheng"><meta name="DC.Contributor" content="William B. Isaacs"><meta name="DC.Contributor" content="Jianfeng Xu"><meta name="DC.Contributor" content="Jielin Sun"><meta name="DC.Date" content="2012 Mar"><meta name="DC.Identifier" content="10.1093/carcin/bgr316"><meta name="DC.Language" content="en"><meta property="og:title" content="A genome-wide search for loci interacting with known prostate cancer risk-associated genetic variants"><meta property="og:type" content="article"><meta property="og:description" content="Genome-wide association studies (GWAS) have identified ∼30 single-nucleotide polymorphisms (SNPs) consistently associated with prostate cancer (PCa) risk. To test the hypothesis that other sequence variants in the genome may interact with those ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3291863/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-carcin.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3291863/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3291863/pdf/bgr316.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8B6441DD52E0A1000000000A770722.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3291863%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="pubmed">PubMed</option><option value="books" class="last">Books</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Carcinogenesis%22[journal]" class="navlink">Carcinogenesis</a></li><li class="accid">PMC3291863</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-carcin.png" alt="Logo of carcin" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/carcin" target="pmc_ext" ref="reftype=publisher&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Carcinogenesis</a></span>. 2012 Mar; 33(3): 598–603. </span></div><div><span class="fm-vol-iss-date">Published online 2012 Jan 4. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fcarcin%2Fbgr316" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/carcin/bgr316</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3291863</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22219177">22219177</a></div></div></div></div><h1 class="content-title">A genome-wide search for loci interacting with known prostate cancer risk-associated genetic variants</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Tao%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862577967824" co-class="co-affbox">Sha Tao</a>,<sup>1</sup> <a href="/pubmed/?term=Wang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862512280416" co-class="co-affbox">Zhong Wang</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Feng%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862580555904" co-class="co-affbox">Junjie Feng</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Hsu%20FC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862578013664" co-class="co-affbox">Fang-Chi Hsu</a>,<sup>2,</sup><sup>4</sup> <a href="/pubmed/?term=Jin%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862583841696" co-class="co-affbox">Guangfu Jin</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Kim%20ST%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862506084432" co-class="co-affbox">Seong-Tae Kim</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Zhang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862511975488" co-class="co-affbox">Zheng Zhang</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Gronberg%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862503911216" co-class="co-affbox">Henrik Gronberg</a>,<sup>5</sup> <a href="/pubmed/?term=Zheng%20LS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862506245472" co-class="co-affbox">Lilly S. Zheng</a>,<sup>2,</sup><sup>3</sup> <a href="/pubmed/?term=Isaacs%20WB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862587167136" co-class="co-affbox">William B. Isaacs</a>,<sup>6</sup> <a href="/pubmed/?term=Xu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862508484288" co-class="co-affbox">Jianfeng Xu</a>,<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>7</sup> and  <a href="/pubmed/?term=Sun%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177" class="affpopup" co-rid="_co_idm139862579660976" co-class="co-affbox">Jielin Sun</a><sup>2,</sup><sup>3,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm139862577967824"><h3 class="no_margin">Sha Tao</h3><p><sup>1</sup>Center for Genetic Epidemiology and Prevention, Van Andel Research Institute, Grand Rapids, MI, USA</p><div>Find articles by <a href="/pubmed/?term=Tao%20S%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">Sha Tao</a></div></div><div id="_co_idm139862512280416"><h3 class="no_margin">Zhong Wang</h3><p><sup>2</sup>Center for Cancer Genomics, Medical Center Boulevard</p><p><sup>3</sup>Center for Genomics and Personalized Medicine Research</p><div>Find articles by <a href="/pubmed/?term=Wang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">Zhong Wang</a></div></div><div id="_co_idm139862580555904"><h3 class="no_margin">Junjie Feng</h3><p><sup>2</sup>Center for Cancer Genomics, Medical Center Boulevard</p><p><sup>3</sup>Center for Genomics and Personalized Medicine Research</p><div>Find articles by <a href="/pubmed/?term=Feng%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">Junjie Feng</a></div></div><div id="_co_idm139862578013664"><h3 class="no_margin">Fang-Chi Hsu</h3><p><sup>2</sup>Center for Cancer Genomics, Medical Center Boulevard</p><p><sup>4</sup>Department of Biostatistical Sciences, Wake Forest University School of Medicine, Winston-Salem, NC 27157, USA</p><div>Find articles by <a href="/pubmed/?term=Hsu%20FC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">Fang-Chi Hsu</a></div></div><div id="_co_idm139862583841696"><h3 class="no_margin">Guangfu Jin</h3><p><sup>2</sup>Center for Cancer Genomics, Medical Center Boulevard</p><p><sup>3</sup>Center for Genomics and Personalized Medicine Research</p><div>Find articles by <a href="/pubmed/?term=Jin%20G%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">Guangfu Jin</a></div></div><div id="_co_idm139862506084432"><h3 class="no_margin">Seong-Tae Kim</h3><p><sup>2</sup>Center for Cancer Genomics, Medical Center Boulevard</p><p><sup>3</sup>Center for Genomics and Personalized Medicine Research</p><div>Find articles by <a href="/pubmed/?term=Kim%20ST%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">Seong-Tae Kim</a></div></div><div id="_co_idm139862511975488"><h3 class="no_margin">Zheng Zhang</h3><p><sup>2</sup>Center for Cancer Genomics, Medical Center Boulevard</p><p><sup>3</sup>Center for Genomics and Personalized Medicine Research</p><div>Find articles by <a href="/pubmed/?term=Zhang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">Zheng Zhang</a></div></div><div id="_co_idm139862503911216"><h3 class="no_margin">Henrik Gronberg</h3><p><sup>5</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institute, Stockholm 17177, Sweden</p><div>Find articles by <a href="/pubmed/?term=Gronberg%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">Henrik Gronberg</a></div></div><div id="_co_idm139862506245472"><h3 class="no_margin">Lilly S. Zheng</h3><p><sup>2</sup>Center for Cancer Genomics, Medical Center Boulevard</p><p><sup>3</sup>Center for Genomics and Personalized Medicine Research</p><div>Find articles by <a href="/pubmed/?term=Zheng%20LS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">Lilly S. Zheng</a></div></div><div id="_co_idm139862587167136"><h3 class="no_margin">William B. Isaacs</h3><p><sup>6</sup>Department of Urology, Johns Hopkins Medical Institutions, Baltimore, MD 21201, USA</p><div>Find articles by <a href="/pubmed/?term=Isaacs%20WB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">William B. Isaacs</a></div></div><div id="_co_idm139862508484288"><h3 class="no_margin">Jianfeng Xu</h3><p><sup>1</sup>Center for Genetic Epidemiology and Prevention, Van Andel Research Institute, Grand Rapids, MI, USA</p><p><sup>2</sup>Center for Cancer Genomics, Medical Center Boulevard</p><p><sup>3</sup>Center for Genomics and Personalized Medicine Research</p><p><sup>7</sup>Department of Urology, Wake Forest University School of Medicine, Winston-Salem, NC 27157, USA</p><div>Find articles by <a href="/pubmed/?term=Xu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">Jianfeng Xu</a></div></div><div id="_co_idm139862579660976"><h3 class="no_margin">Jielin Sun</h3><p><sup>2</sup>Center for Cancer Genomics, Medical Center Boulevard</p><p><sup>3</sup>Center for Genomics and Personalized Medicine Research</p><div>Find articles by <a href="/pubmed/?term=Sun%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22219177">Jielin Sun</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm139862545050848_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139862545050848_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139862545050848_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm139862545050848_ai" style="display:none"><div class="fm-affl" id="aff1" lang="en"><sup>1</sup>Center for Genetic Epidemiology and Prevention, Van Andel Research Institute, Grand Rapids, MI, USA</div><div class="fm-affl" id="aff2" lang="en"><sup>2</sup>Center for Cancer Genomics, Medical Center Boulevard</div><div class="fm-affl" id="aff3" lang="en"><sup>3</sup>Center for Genomics and Personalized Medicine Research</div><div class="fm-affl" id="aff4" lang="en"><sup>4</sup>Department of Biostatistical Sciences, Wake Forest University School of Medicine, Winston-Salem, NC 27157, USA</div><div class="fm-affl" id="aff5" lang="en"><sup>5</sup>Department of Medical Epidemiology and Biostatistics, Karolinska Institute, Stockholm 17177, Sweden</div><div class="fm-affl" id="aff6" lang="en"><sup>6</sup>Department of Urology, Johns Hopkins Medical Institutions, Baltimore, MD 21201, USA</div><div class="fm-affl" id="aff7" lang="en"><sup>7</sup>Department of Urology, Wake Forest University School of Medicine, Winston-Salem, NC 27157, USA</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed. Tel: +1 (336) 713 7500; Fax: +1 (336) 713-7566; Email: <a href="mailto:dev@null" data-email="ude.cmbufw@nusij" class="oemail">ude.cmbufw@nusij</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm139862545050848_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2011 Oct 26; Revised 2011 Dec 13; Accepted 2011 Dec 15.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm139862545050848_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2012. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC3291863/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC3291863/bin/supp_33_3_598__index.html" data-ga-action="click_feat_suppl">supp_33_3_598__index.html</a><span style="color:gray"> (1.0K)</span></div><div class="small guid">GUID:&nbsp;3E1415F5-C977-4F60-ABFF-DAC0EF7A5629</div>
<div><a href="/pmc/articles/PMC3291863/bin/supp_bgr316_BGR316_Supplementary_Material_r1.docx" data-ga-action="click_feat_suppl">supp_bgr316_BGR316_Supplementary_Material_r1.docx</a><span style="color:gray"> (77K)</span></div><div class="small guid">GUID:&nbsp;04AFE075-0056-4EF7-A278-CC1F38A7909F</div>
</div></dd></dl></div><div id="idm139862509194896" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139862509194896title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Genome-wide association studies (GWAS) have identified ∼30 single-nucleotide polymorphisms (SNPs) consistently associated with prostate cancer (PCa) risk. To test the hypothesis that other sequence variants in the genome may interact with those 32 known PCa risk-associated SNPs identified from GWAS to affect PCa risk, we performed a systematic evaluation among three existing PCa GWAS populations: CAncer of the Prostate in Sweden population, a Johns Hopkins Hospital population, and the Cancer Genetic Markers of Susceptibility population, with a total sample size of 4723 PCa cases and 4792 control subjects. Meta-analysis of the interaction term between each of those 32 SNPs and SNPs in the genome was performed in three PCa GWAS populations. The most significant interaction detected was between rs12418451 in <em>MYEOV</em> and rs784411 in <em>CEP152</em>, with a <em>P</em><sub>interaction</sub> of 1.15 × 10<sup>−7</sup> in the meta-analysis. In addition, we emphasized two pairs of interactions with potential biological implication, including an interaction between rs7127900 near insulin-like growth factor-2 (<em>IGF2</em>)<em>/IGF2AS</em> and rs12628051 in <em>TNRC6B</em>, with a <em>P</em><sub>interaction</sub> of 3.39 × 10<sup>−6</sup> and an interaction between rs7679763 near <em>TET2</em> and rs290258 in <em>SYK</em>, with a <em>P</em><sub>interaction</sub> of 1.49 × 10<sup>−6</sup>. Those results show statistical evidence for novel loci interacting with known risk-associated SNPs to modify PCa risk. The interacting loci identified provide hints on the underlying molecular mechanism of the associations with PCa risk for the known risk-associated SNPs. Additional studies are warranted to further confirm the interaction effects detected in this study.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Introduction</h2><p id="__p2" class="p p-first">Prostate cancer (PCa) is the most common non-skin cancer affecting men in western countries. Inherited genetic variants play an important role in contributing to familial aggregation of PCa. Since 2007, genome-wide association studies (GWAS) successfully identified at least 33 PCa risk-associated single-nucleotide polymorphisms (SNPs) (<a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568040">1</a>–<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568063">15</a>).</p><p id="__p3">Although those risk-associated SNPs are well replicated in multiple studies (<a href="#bib16" rid="bib16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568071">16</a>–<a href="#bib20" rid="bib20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568064">20</a>), very few studies assess the potential epistasis or gene–gene interaction between those SNPs and the rest of SNPs that reside in the genome. In fact, epistatic effect is the norm rather than exception for complex diseases, such as PCa. Inference from tumorigenesis and results from genetic modeling studies suggest that multiple susceptibility genes, either additively or multiplicatively, determine individual risk to PCa. The importance of epistasis is also supported by empirical evidence from model organisms and human studies (<a href="#bib21" rid="bib21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568068">21</a>–<a href="#bib23" rid="bib23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_762477116">23</a>).</p><p id="__p4">The evidence of epistatic effect from the empirical data suggests that gene–gene interactions need to be examined in GWAS when searching for PCa risk variants. Actually, assessment of gene–gene interaction may reveal additional PCa risk variants, especially in the situation where multiple risk-associated variants have been identified. It is computationally possible to use a logistic regression model to search the genome and to identify additional variants that interact with these known risk variants to modify the risk of developing PCa.</p><p id="__p5" class="p p-last">Recently, Ciampa <em>et al.</em> (<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568055">24</a>) reported a two-stage GWAS of epistasis between 13 known PCa risk-associated SNPs and SNPs across the genome in the National Cancer Institute Cancer Genetic Markers of Susceptibility (CGEMS) Stage I population with 523 841 SNPs and Stage II population with 27 383 SNPs, which were selected based on the main effects in Stage I. No SNP–SNP interaction was identified that reached a genome-wide significance level in Stage I or Stage II data, and a list of top interactions were suggested and warranted replication in other studies. The lack of replication data in Ciampa’s study emphasized the importance of evaluating gene–gene interaction in multiple GWAS populations. More importantly, combining individual level data of multiple GWAS can improve the power to identify SNPs that interact with the known risk-associated SNPs to impact PCa risk. To this end, we performed a combined genome-wide search for SNPs that interact with 32 PCa risk-associated variants identified from GWAS in three case–control populations of European descents, including 1583 PCa cases and 519 control subjects from the CAncer Prostate in Sweden (CAPS), 1964 PCa cases and 3172 control subjects from a Johns Hopkins Hospital (JHH) PCa and iControl database and 1176 PCa cases and 1101 control subjects in the National Cancer Institute CGEMS study. We also evaluated the list of SNP–SNP interactions suggested by Ciampa’s study in the two independent GWAS populations (CAPS and JHH).</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">Materials and methods</h2><div id="__sec3" class="sec sec-first"><h3 id="__sec3title">Study populations</h3><p id="__p6" class="p p-first">The first GWAS population included 1583 PCa patients and 519 control subjects that matched the age distribution of case subjects from CAPS, a population-based PCa case–control study from Sweden (CAPS) (<a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568052">6</a>). Briefly, the CAPS population was recruited from four regional cancer registries in Sweden and diagnosed between July 2001 and October 2003. The clinical characteristics of these patients are presented in <a href="http://www.carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgr316/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 1</a>, available at <em>Carcinogenesis</em> Online.</p><p id="__p7">The second population was from a JHH PCa GWAS, which included 1964 PCa cases and 3172 control subjects. The cases are Caucasian PCa patients who underwent radical prostatectomy for the treatment of PCa at JHH from 1 January 1999 through 31 December 2008 (<a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568059">25</a>). The clinical characteristics of these patients are presented in <a href="http://www.carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgr316/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 2</a>, available at <em>Carcinogenesis</em> Online. The control subjects for this population were an independent group of Caucasian individuals from the Illumina iControlDB (iControls) dataset (<a href="https://www.illumina.com/science/icontrodb.ilmn" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.illumina.com/science/icontrodb.ilmn</a>).</p><p id="__p8" class="p p-last">The third population was obtained from Stage I of the National Cancer Institute CGEMS study. It included 1176 PCa cases and 1101 control subjects, selected from the Prostate, Lung, Colon and Ovarian Cancer Screening Trial (<a href="#bib6" rid="bib6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568066">6</a>,<a href="#bib9" rid="bib9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_277709388">9</a>). The genotype and phenotype data of the study are publicly available and our use of the data was approved by CGEMS.</p></div><div id="__sec4" class="sec"><h3 id="__sec4title">Genotype data, imputation and quality control</h3><p id="__p9" class="p p-first">GWAS of the CAPS population was performed using Affymetrix 5.0 chip. GWAS of the JHH case population was performed using the Illumina 610K chip (<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568058">24</a>). GWAS of the iControls population (<a href="#bib25" rid="bib25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568054">25</a>) was performed using Illumina Hap300 and Hap550 chips. GWAS of the CGEMS population was performed using HumanHap300 and HumanHap240 assays from Illumina Corp.</p><p id="__p10" class="p p-last">For each GWAS population, we imputed all the known SNPs that are catalogued in HapMap Phase II (<a href="http://www.hapmap.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.hapmap.org</a>) using the IMPUTE computer program (<a href="#bib26" rid="bib26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568053">26</a>) with a posterior probability of 0.9 as a threshold to call genotypes. Individuals with a call rate &lt;0.95 were removed from GWAS analysis. The following quality control criteria were used to filter SNPs: Minor Allele Frequency &lt; 0.01, Hardy-Weinberg Equilibrium &lt; 0.001 and call rate &lt;0.95.</p></div><div id="__sec5" class="sec"><h3 id="__sec5title">PCa-known risk SNPs identified from GWAS</h3><p id="__p11" class="p p-first">The 33 PCa-known risk-associated SNPs were discovered by GWAS and the following fine-mapping studies, with <em>P</em>-values ≤ 10 × 10<sup>−7</sup> (<a href="#bib1" rid="bib1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568069">1</a>–<a href="#bib15" rid="bib15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568045">15</a>). The detailed information for the 33 risk SNPs are presented in <a href="/pmc/articles/PMC3291863/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table I</span></a>. The SNP rs16901979 was not evaluated in the following interaction analysis due to the unavailability of imputation of this SNP since it was not catalogued in the HapMap database.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl1"><h3>Table I.</h3><!--caption a7--><div class="caption"><p id="__p12">Reported SNPs associated with PCa</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139862512234576" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="1" colspan="1">Chr</td><td rowspan="1" colspan="1">SNPs</td><td rowspan="1" colspan="1">Cytogenetic bands</td><td rowspan="1" colspan="1">Position</td><td rowspan="1" colspan="1">Known genes</td><td rowspan="1" colspan="1">m/M allele</td><td rowspan="1" colspan="1">Risk allele</td></tr></thead><tbody><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs1465618</td><td rowspan="1" colspan="1">2p21</td><td rowspan="1" colspan="1" align="center">43,407,453</td><td rowspan="1" colspan="1"><em>THADA</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs721048</td><td rowspan="1" colspan="1">2p15</td><td rowspan="1" colspan="1" align="center">62,985,235</td><td rowspan="1" colspan="1"><em>EHBP1</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">2</td><td rowspan="1" colspan="1">rs12621278</td><td rowspan="1" colspan="1">2q31.1</td><td rowspan="1" colspan="1" align="center">173,019,799</td><td rowspan="1" colspan="1"><em>ITGA6</em></td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">rs2660753</td><td rowspan="1" colspan="1">3p12</td><td rowspan="1" colspan="1" align="center">87,193,364</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1">3</td><td rowspan="1" colspan="1">rs10934853</td><td rowspan="1" colspan="1">3q21.3</td><td rowspan="1" colspan="1" align="center">129,521,063</td><td rowspan="1" colspan="1"><em>EEFSEC</em></td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">rs17021918</td><td rowspan="1" colspan="1">4q22.3</td><td rowspan="1" colspan="1" align="center">95,781,900</td><td rowspan="1" colspan="1"><em>PDLIM5</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">C</td></tr><tr><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">rs7679673</td><td rowspan="1" colspan="1">4q24</td><td rowspan="1" colspan="1" align="center">106,280,983</td><td rowspan="1" colspan="1"><em>TET2</em></td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">C</td></tr><tr><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">rs9364554</td><td rowspan="1" colspan="1">6q25</td><td rowspan="1" colspan="1" align="center">160,753,654</td><td rowspan="1" colspan="1"><em>SLC22A3</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">rs10486567</td><td rowspan="1" colspan="1">7p15</td><td rowspan="1" colspan="1" align="center">27,943,088</td><td rowspan="1" colspan="1"><em>JAZF1</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">rs6465657</td><td rowspan="1" colspan="1">7q21</td><td rowspan="1" colspan="1" align="center">97,654,263</td><td rowspan="1" colspan="1"><em>LMTK2</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">C</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs2928679</td><td rowspan="1" colspan="1">8p21.2</td><td rowspan="1" colspan="1" align="center">23,494,920</td><td rowspan="1" colspan="1"><em>SLC25A37</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs1512268</td><td rowspan="1" colspan="1">8p21.2</td><td rowspan="1" colspan="1" align="center">23,582,408</td><td rowspan="1" colspan="1"><em>NKX3.1</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs10086908</td><td rowspan="1" colspan="1">8q24 (5)</td><td rowspan="1" colspan="1" align="center">128,081,119</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs16901979</td><td rowspan="1" colspan="1">8q24 (2)</td><td rowspan="1" colspan="1" align="center">128,194,098</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs16902094</td><td rowspan="1" colspan="1">8q24.21</td><td rowspan="1" colspan="1" align="center">128,389,528</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">N/A</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs620861</td><td rowspan="1" colspan="1">8q24 (4)</td><td rowspan="1" colspan="1" align="center">128,404,855</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs6983267</td><td rowspan="1" colspan="1">8q24 (3)</td><td rowspan="1" colspan="1" align="center">128,482,487</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs1447295</td><td rowspan="1" colspan="1">8q24 (1)</td><td rowspan="1" colspan="1" align="center">128,554,220</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">A/C</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">9</td><td rowspan="1" colspan="1">rs1571801</td><td rowspan="1" colspan="1">9q33</td><td rowspan="1" colspan="1" align="center">123,467,194</td><td rowspan="1" colspan="1"><em>DAB2IC</em></td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs10993994</td><td rowspan="1" colspan="1">10q11</td><td rowspan="1" colspan="1" align="center">51,219,502</td><td rowspan="1" colspan="1"><em>MSMB</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs4962416</td><td rowspan="1" colspan="1">10q26</td><td rowspan="1" colspan="1" align="center">126,686,862</td><td rowspan="1" colspan="1"><em>CTBP2</em></td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">C</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs7127900</td><td rowspan="1" colspan="1">11p15.5</td><td rowspan="1" colspan="1" align="center">2,190,150</td><td rowspan="1" colspan="1"><em>IGF2</em>, <em>IGF2AS</em>, <em>INS</em>, <em>TH</em></td><td rowspan="1" colspan="1">G/A</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs12418451</td><td rowspan="1" colspan="1">11q13 (2)</td><td rowspan="1" colspan="1" align="center">68,691,995</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs10896449</td><td rowspan="1" colspan="1">11q13 (1)</td><td rowspan="1" colspan="1" align="center">68,751,243</td><td rowspan="1" colspan="1"><em>MYEOV</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs11649743</td><td rowspan="1" colspan="1">17q12 (2)</td><td rowspan="1" colspan="1" align="center">33,149,092</td><td rowspan="1" colspan="1"><em>HNF1B</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs4430796</td><td rowspan="1" colspan="1">17q12 (1)</td><td rowspan="1" colspan="1" align="center">33,172,153</td><td rowspan="1" colspan="1"><em>HNF1B</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">A</td></tr><tr><td rowspan="1" colspan="1">17</td><td rowspan="1" colspan="1">rs1859962</td><td rowspan="1" colspan="1">17q24.3</td><td rowspan="1" colspan="1" align="center">66,620,348</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">G/T</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs8102476</td><td rowspan="1" colspan="1">19q13.2</td><td rowspan="1" colspan="1" align="center">43,427,453</td><td rowspan="1" colspan="1"><em>PPP1R14A</em></td><td rowspan="1" colspan="1">T/C</td><td rowspan="1" colspan="1">C</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs887391</td><td rowspan="1" colspan="1">19q13</td><td rowspan="1" colspan="1" align="center">46,677,464</td><td rowspan="1" colspan="1">—</td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">T</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs2735839</td><td rowspan="1" colspan="1">19q13</td><td rowspan="1" colspan="1" align="center">56,056,435</td><td rowspan="1" colspan="1"><em>KLK3</em></td><td rowspan="1" colspan="1">A/G</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">rs9623117</td><td rowspan="1" colspan="1">New 22q13</td><td rowspan="1" colspan="1" align="center">38,782,065</td><td rowspan="1" colspan="1"><em>TNRC6B</em></td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">C</td></tr><tr><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">rs5759167</td><td rowspan="1" colspan="1">New 22q13.2</td><td rowspan="1" colspan="1" align="center">41,830,156</td><td rowspan="1" colspan="1"><em>TTLL1,</em>
<em>BIK, MCAT, PACSIN2</em></td><td rowspan="1" colspan="1">T/G</td><td rowspan="1" colspan="1">G</td></tr><tr><td rowspan="1" colspan="1">23</td><td rowspan="1" colspan="1">rs5945619</td><td rowspan="1" colspan="1">Xp11</td><td rowspan="1" colspan="1" align="center">51,258,412</td><td rowspan="1" colspan="1"><em>NUDT10,</em>
<em>NUDT11, LOC340602</em></td><td rowspan="1" colspan="1">C/T</td><td rowspan="1" colspan="1">C</td></tr></tbody></table></div><div id="largeobj_idm139862512234576" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3291863/table/tbl1/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139862503633440"><p id="__p13" class="p p-first-last">Base pair position is based on NCBI build 36; Chr, chromosome; m/M, minor allele/major allele.</p></div></div></div></div><div id="__sec6" class="sec sec-last"><h3 id="__sec6title">Statistical analysis</h3><p id="__p14" class="p p-first-last">Multiplicative interactions between each one of the 32 known PCa risk variants and each SNP in the genome were systematically tested by including both SNPs and an interaction term (product of two SNPs), as implemented in the computer program PLINK (<a href="#bib27" rid="bib27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568047">27</a>). Ancestral proportions obtained based on EIGENSOFT software (<a href="#bib28" rid="bib28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568062">28</a>) were included as covariates to minimize the impact of potential population stratification in the JHH population. An additive genetic model was used, where the genotypes were coded as 0, 1 and 2 and each SNP was treated as a continuous variable. The interaction term was tested using a Wald test with degree of freedom of 1. A meta-analysis of the interaction term for the three study populations was performed using the method developed by Manning <em>et al.</em> (<a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568049">29</a>). Briefly, the meta-odds ratio (OR<sub>M</sub>) of the interaction term across the three populations was estimated using an inverse variance weighted meta-analysis, where <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="n1" overflow="scroll"><mrow><mi>ln</mi><mo>(</mo><msub><mtext>OR</mtext><mtext>M</mtext></msub><mo>)</mo><mo>=</mo><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></msubsup><mrow><mo>(</mo><msub><mi>w</mi><mi>i</mi></msub><mi>ln</mi><mo>(</mo><msub><mtext>OR</mtext><mi>i</mi></msub><mo>)</mo><mo>/</mo><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></msubsup><mrow><msub><mi>w</mi><mi>i</mi></msub></mrow></mstyle><mo>)</mo></mrow></mstyle></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-1-Frame" tabindex="0" style=""></span><script type="math/mml" id="MathJax-Element-1"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="n1" overflow="scroll"><mrow><mi>ln</mi><mo>(</mo><msub><mtext>OR</mtext><mtext>M</mtext></msub><mo>)</mo><mo>=</mo><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></msubsup><mrow><mo>(</mo><msub><mi>w</mi><mi>i</mi></msub><mi>ln</mi><mo>(</mo><msub><mtext>OR</mtext><mi>i</mi></msub><mo>)</mo><mo>/</mo><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></msubsup><mrow><msub><mi>w</mi><mi>i</mi></msub></mrow></mstyle><mo>)</mo></mrow></mstyle></mrow></math></script>, <em>w</em><sub><em>i</em></sub> =&nbsp;1/var(ln(OR<sub>i</sub>)) and <span class="MathJax_Preview"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="n3" overflow="scroll"><mrow><mi>s</mi><mi>e</mi><mo>(</mo><mi>ln</mi><mo>(</mo><msub><mrow><mtext>OR</mtext></mrow><mtext>M</mtext></msub><mo>)</mo><mo>)</mo><mo>=</mo><msup><mrow><mo>[</mo><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></msubsup><mrow><mo>(</mo><mn>1</mn><mo>/</mo><mi>var</mi><mo>(</mo><mi>ln</mi><mo>(</mo><msub><mrow><mtext>OR</mtext></mrow><mi>i</mi></msub><mo>)</mo><mo>)</mo><mo>)</mo></mrow></mstyle><mo>]</mo></mrow><mrow><mo>−</mo><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup></mrow></math></span><span class="MathJax MathJax_Processing" id="MathJax-Element-2-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-2"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="n3" overflow="scroll"><mrow><mi>s</mi><mi>e</mi><mo>(</mo><mi>ln</mi><mo>(</mo><msub><mrow><mtext>OR</mtext></mrow><mtext>M</mtext></msub><mo>)</mo><mo>)</mo><mo>=</mo><msup><mrow><mo>[</mo><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mn>3</mn></msubsup><mrow><mo>(</mo><mn>1</mn><mo>/</mo><mi>var</mi><mo>(</mo><mi>ln</mi><mo>(</mo><msub><mrow><mtext>OR</mtext></mrow><mi>i</mi></msub><mo>)</mo><mo>)</mo><mo>)</mo></mrow></mstyle><mo>]</mo></mrow><mrow><mo>−</mo><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup></mrow></math></script>(<a href="#bib29" rid="bib29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568051">29</a>).</p></div></div><div id="__sec7" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec7title">Results</h2><p id="__p15" class="p p-first">After imputation and applying quality control criteria, 1 314 700, 1 646 196 and 1 757 946 SNPs remained for CAPS, JHH and CGEMS studies, respectively. A total of 1 117 531 common SNPs for those three populations were used in the interaction analysis.</p><p id="__p16">We examined the inflation factor and the quantile–quantile plots for interaction tests in the combined analysis of three populations. No systematic bias was observed as the inflation factors for the 32 GWAS scans for SNP–SNP interactions ranged from 0.98 to 1.03 (<a href="http://www.carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgr316/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 1</a> is available at <em>Carcinogenesis</em> Online).</p><p id="__p17">The results for the top-ranked SNPs that interacted with each of the 32 known PCa risk SNPs (<em>P</em><sub>interaction</sub> &lt; 1.0 × 10<sup>−5</sup> in the meta-analysis) were presented in <a href="http://www.carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgr316/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 2</a>, available at <em>Carcinogenesis</em> Online. For SNPs in linkage disequilibrium (as defined by <em>r</em><sup>2</sup> &gt; 0.5), only the one with the smallest <em>P</em>-value based on meta-analysis was included in the <a href="http://www.carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgr316/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Table 2</a>, available at <em>Carcinogenesis</em> Online. We then further examined the interaction effects for the top-ranked SNPs (<em>P</em><sub>interaction</sub> &lt; 1 × 10<sup>−5</sup><em>)</em> in each of the three populations. SNPs that significantly interacted with the 32 SNPs in all three populations at a nominal <em>P</em><sub>interaction</sub> of 0.05 were presented in <a href="/pmc/articles/PMC3291863/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table II</span></a>. No SNP–SNP interaction reached a genome-wide significant level of 1.5 × 10<sup>−9</sup> [0.05/(1 × 10<sup>−6</sup> × 32)]. The most significant interaction was observed between rs12418451 in the <em>MYEOV</em> gene region and rs784411 in the intron of <em>CEP152</em>, with a <em>P</em><sub>interaction</sub> of 1.15 × 10<sup>−7</sup> [OR<sub>interaction</sub> = 1.42; 95% confidence interval (CI): 1.25–1.61] in the meta-analysis. This interaction pair was significant in all three populations and the effects of the interaction were in the same direction [<em>P</em><sub>interaction</sub> = 0.008, OR<sub>interaction</sub> = 1.55 (95% CI: 1.12–2.16) for CAPS; <em>P</em><sub>interaction</sub> = 0.005, OR<sub>interaction</sub> = 1.34 (95% CI: 1.14–1.58) for JHH and <em>P</em><sub>interaction</sub> = 0.001, OR<sub>interaction</sub> = 1.53 (95% CI = 1.18–1.99) in CGEMS, respectively] (<a href="/pmc/articles/PMC3291863/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table II</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl2"><h3>Table II.</h3><!--caption a7--><div class="caption"><p id="__p18">Results for top SNPs that interact with the known PCa risk SNPs (<em>P</em><sub>interaction</sub> &lt; 1.0 × 10<sup>−5</sup> in the meta-analysis and <em>P</em><sub>interaction</sub> &lt; 0.05 in each of the three populations)</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139862577920512" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td colspan="3" rowspan="1">SNP 1<hr></td><td colspan="7" rowspan="1">SNP 2<hr></td><td colspan="2" rowspan="1">Meta-results<hr></td><td colspan="2" rowspan="1">CAPS<hr></td><td colspan="2" rowspan="1">JHH<hr></td><td colspan="2" rowspan="1">CGEMS<hr></td></tr><tr><td rowspan="1" colspan="1">Chr</td><td rowspan="1" colspan="1">SNP</td><td rowspan="1" colspan="1">Gene</td><td rowspan="1" colspan="1">Chr</td><td rowspan="1" colspan="1">SNP</td><td rowspan="1" colspan="1">Position</td><td rowspan="1" colspan="1">Minor allele</td><td rowspan="1" colspan="1">Gene</td><td rowspan="1" colspan="1">Location</td><td rowspan="1" colspan="1">Relative position</td><td rowspan="1" colspan="1"><em>P</em></td><td rowspan="1" colspan="1">OR (95% CI)</td><td rowspan="1" colspan="1"><em>P</em></td><td rowspan="1" colspan="1">OR (95% CI)</td><td rowspan="1" colspan="1"><em>P</em></td><td rowspan="1" colspan="1">OR (95% CI)</td><td rowspan="1" colspan="1"><em>P</em></td><td rowspan="1" colspan="1">OR (95% CI)</td></tr></thead><tbody><tr><td rowspan="2" colspan="1">2</td><td rowspan="2" colspan="1">rs1465618</td><td rowspan="2" colspan="1"><em>THADA</em></td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1">rs11735008</td><td rowspan="1" colspan="1" align="center">393,303</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>ABCA11P</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">15,921</td><td rowspan="1" colspan="1" align="center">6.65 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.76 (0.68–0.86)</td><td rowspan="1" colspan="1" align="center">5.04 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.69 (0.54–0.90)</td><td rowspan="1" colspan="1" align="center">3.11 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.83 (0.71–0.98)</td><td rowspan="1" colspan="1" align="center">1.35 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.69 (0.55–0.86)</td></tr><tr><td rowspan="1" colspan="1" align="center">13</td><td rowspan="1" colspan="1">rs9567349</td><td rowspan="1" colspan="1" align="center">43,535,405</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>NCRNA00284</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">32,806</td><td rowspan="1" colspan="1" align="center">3.93 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.61 (0.49–0.75)</td><td rowspan="1" colspan="1" align="center">4.57 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">0.41 (0.25–0.67)</td><td rowspan="1" colspan="1" align="center">3.24 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.72 (0.54–0.97)</td><td rowspan="1" colspan="1" align="center">3.97 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.57 (0.39–0.84)</td></tr><tr><td rowspan="2" colspan="1">3</td><td rowspan="2" colspan="1">rs10934853</td><td rowspan="2" colspan="1"><em>EEFSEC</em></td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1">rs7847271</td><td rowspan="1" colspan="1" align="center">116,870,633</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>TNC</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">3.85 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.67 (0.56–0.79)</td><td rowspan="1" colspan="1" align="center">7.73 × 10<sup>−</sup>3</td><td rowspan="1" colspan="1" align="center">0.60 (0.41–0.87)</td><td rowspan="1" colspan="1" align="center">2.56 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.68 (0.54–0.88)</td><td rowspan="1" colspan="1" align="center">1.87 × 10<sup>−</sup>2</td><td rowspan="1" colspan="1" align="center">0.69 (0.50–0.94)</td></tr><tr><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1">rs998124</td><td rowspan="1" colspan="1" align="center">40,979,660</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>MIR4319</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">175,531</td><td rowspan="1" colspan="1" align="center">5.21 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.33 (1.18–1.51)</td><td rowspan="1" colspan="1" align="center">3.56 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.39 (1.02–1.88)</td><td rowspan="1" colspan="1" align="center">3.64 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.28 (1.08–1.51)</td><td rowspan="1" colspan="1" align="center">3.56 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.42 (1.12–1.80)</td></tr><tr><td rowspan="1" colspan="1">4</td><td rowspan="1" colspan="1">rs17021918</td><td rowspan="1" colspan="1"><em>PDLIM5</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">rs9757252</td><td rowspan="1" colspan="1" align="center">86,977,168</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>VGLL3</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">92,645</td><td rowspan="1" colspan="1" align="center">4.73 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.25 (1.13–1.37)</td><td rowspan="1" colspan="1" align="center">8.45 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.35 (1.08–1.69)</td><td rowspan="1" colspan="1" align="center">2.23 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.22 (1.07–1.38)</td><td rowspan="1" colspan="1" align="center">2.16 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.24 (1.03–1.50)</td></tr><tr><td rowspan="2" colspan="1">4</td><td rowspan="2" colspan="1">rs7679673</td><td rowspan="2" colspan="1"><em>TET2</em></td><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1">rs290258</td><td rowspan="1" colspan="1" align="center">92,595,560</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>SYK</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">−8,273</td><td rowspan="1" colspan="1" align="center">1.49 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.75 (0.67–0.84)</td><td rowspan="1" colspan="1" align="center">2.11 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.66 (0.51–0.86)</td><td rowspan="1" colspan="1" align="center">3.01 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.78 (0.67–0.92)</td><td rowspan="1" colspan="1" align="center">1.39 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.75 (0.59–0.94)</td></tr><tr><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1">rs5751168</td><td rowspan="1" colspan="1" align="center">21,175,240</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>ZNF280B</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">4.11 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.44 (1.23–1.67)</td><td rowspan="1" colspan="1" align="center">4.75 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">2.19 (1.50–3.19)</td><td rowspan="1" colspan="1" align="center">3.38E × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.25 (1.02–1.53)</td><td rowspan="1" colspan="1" align="center">9.09 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.48 (1.10–1.99)</td></tr><tr><td rowspan="2" colspan="1">7</td><td rowspan="2" colspan="1">rs10486567</td><td rowspan="2" colspan="1"><em>JAZF1</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">rs1795355</td><td rowspan="1" colspan="1" align="center">41,574,530</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>ULK4</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">9.46 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.79 (0.71–0.88)</td><td rowspan="1" colspan="1" align="center">3.37 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.77 (0.60–0.98)</td><td rowspan="1" colspan="1" align="center">9.11 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.83 (0.72–0.95)</td><td rowspan="1" colspan="1" align="center">2.14 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.73 (0.60–0.89)</td></tr><tr><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">rs11720607</td><td rowspan="1" colspan="1" align="center">174,325,971</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>SPATA16</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">4.87 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.73 (0.63–0.83)</td><td rowspan="1" colspan="1" align="center">2.45 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.62 (0.45–0.84)</td><td rowspan="1" colspan="1" align="center">2.34 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.75 (0.62–0.90)</td><td rowspan="1" colspan="1" align="center">5.43 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.77 (0.58–1.00)</td></tr><tr><td rowspan="1" colspan="1">7</td><td rowspan="1" colspan="1">rs6465657</td><td rowspan="1" colspan="1"><em>LMTK2</em></td><td rowspan="1" colspan="1" align="center">16</td><td rowspan="1" colspan="1">rs8057939</td><td rowspan="1" colspan="1" align="center">47,951,777</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>C16orf78</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">−13,532</td><td rowspan="1" colspan="1" align="center">4.71 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.37 (1.20–1.57)</td><td rowspan="1" colspan="1" align="center">3.31 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.43 (1.03–1.98)</td><td rowspan="1" colspan="1" align="center">1.03 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.36 (1.13–1.63)</td><td rowspan="1" colspan="1" align="center">1.70 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.36 (1.06–1.75)</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs10086908</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">rs10456809</td><td rowspan="1" colspan="1" align="center">17,921,804</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>KIF13A</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">4.83 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.25 (1.14–1.38)</td><td rowspan="1" colspan="1" align="center">1.30 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.31 (1.06–1.62)</td><td rowspan="1" colspan="1" align="center">2.28 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.23 (1.08–1.40)</td><td rowspan="1" colspan="1" align="center">1.69 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.26 (1.04–1.52)</td></tr><tr><td rowspan="3" colspan="1">8</td><td rowspan="3" colspan="1">rs1447295</td><td rowspan="3" colspan="1"><em>NA</em></td><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1">rs7789197</td><td rowspan="1" colspan="1" align="center">40,931,652</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>INHBA</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">763,474</td><td rowspan="1" colspan="1" align="center">3.36 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.66 (0.56–0.79)</td><td rowspan="1" colspan="1" align="center">2.57 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.55 (0.38–0.81)</td><td rowspan="1" colspan="1" align="center">7.27 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.72 (0.56–0.91)</td><td rowspan="1" colspan="1" align="center">1.06 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.66 (0.48–0.91)</td></tr><tr><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1">rs12682851</td><td rowspan="1" colspan="1" align="center">8,002,418</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>C9orf123</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">212,619</td><td rowspan="1" colspan="1" align="center">1.53 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.72 (0.63–0.82)</td><td rowspan="1" colspan="1" align="center">9.66 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.67 (0.50–0.91)</td><td rowspan="1" colspan="1" align="center">2.15 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.75 (0.62–0.90)</td><td rowspan="1" colspan="1" align="center">6.56 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.70 (0.54–0.90)</td></tr><tr><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1">rs10885582</td><td rowspan="1" colspan="1" align="center">116,317,540</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>ABLIM1</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">3.70 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.73 (0.63–0.83)</td><td rowspan="1" colspan="1" align="center">9.33 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">0.54 (0.39–0.73)</td><td rowspan="1" colspan="1" align="center">1.27 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.79 (0.66–0.95)</td><td rowspan="1" colspan="1" align="center">3.46 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.75 (0.57–0.98)</td></tr><tr><td rowspan="1" colspan="1">8</td><td rowspan="1" colspan="1">rs6983267</td><td rowspan="1" colspan="1"><em>NA</em></td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1">rs1011119</td><td rowspan="1" colspan="1" align="center">19,972,144</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>ID4</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">23,250</td><td rowspan="1" colspan="1" align="center">7.20 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.81 (0.74–0.89)</td><td rowspan="1" colspan="1" align="center">1.45 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.76 (0.61–0.95)</td><td rowspan="1" colspan="1" align="center">2.85 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.83 (0.74–0.94)</td><td rowspan="1" colspan="1" align="center">1.58 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.80 (0.67–0.96)</td></tr><tr><td rowspan="3" colspan="1">9</td><td rowspan="3" colspan="1">rs1571801</td><td rowspan="3" colspan="1"><em>DAB2IC</em></td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1">rs2219968</td><td rowspan="1" colspan="1" align="center">79,119,213</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>PKIA</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">471,678</td><td rowspan="1" colspan="1" align="center">6.07 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.30 (1.17–1.43)</td><td rowspan="1" colspan="1" align="center">1.18 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.35 (1.07–1.70)</td><td rowspan="1" colspan="1" align="center">1.64 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.30 (1.14–1.50)</td><td rowspan="1" colspan="1" align="center">3.17 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.24 (1.02–1.52)</td></tr><tr><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1">rs13264970</td><td rowspan="1" colspan="1" align="center">83,236,384</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>SNX16</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">319,308</td><td rowspan="1" colspan="1" align="center">3.53 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.77 (0.69–0.86)</td><td rowspan="1" colspan="1" align="center">1.50 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.74 (0.59–0.94)</td><td rowspan="1" colspan="1" align="center">5.02 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.80 (0.68–0.93)</td><td rowspan="1" colspan="1" align="center">3.92 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.72 (0.58–0.90)</td></tr><tr><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1">rs1547851</td><td rowspan="1" colspan="1" align="center">92,364,806</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>HTR7</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">125,750</td><td rowspan="1" colspan="1" align="center">7.45 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.59 (1.30–1.95)</td><td rowspan="1" colspan="1" align="center">2.72 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.98 (1.27–3.09)</td><td rowspan="1" colspan="1" align="center">7.35 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.49 (1.11–1.99)</td><td rowspan="1" colspan="1" align="center">2.42 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.52 (1.06–2.20)</td></tr><tr><td rowspan="1" colspan="1">10</td><td rowspan="1" colspan="1">rs4962416</td><td rowspan="1" colspan="1"><em>CTBP2</em></td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1">rs10940579</td><td rowspan="1" colspan="1" align="center">57,166,575</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>ACTBL2</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">352,182</td><td rowspan="1" colspan="1" align="center">3.81 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.32 (1.18–1.49)</td><td rowspan="1" colspan="1" align="center">4.85 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.36 (1.00–1.84)</td><td rowspan="1" colspan="1" align="center">1.77 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.29 (1.10–1.51)</td><td rowspan="1" colspan="1" align="center">4.74 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.39 (1.10–1.74)</td></tr><tr><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1">rs10896449</td><td rowspan="1" colspan="1"><em>MYEOV</em></td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1">rs17354197</td><td rowspan="1" colspan="1" align="center">88,169,501</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>DUSP6</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">96,467</td><td rowspan="1" colspan="1" align="center">8.82 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.41 (1.21–1.64)</td><td rowspan="1" colspan="1" align="center">3.97 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.49 (1.02–2.18)</td><td rowspan="1" colspan="1" align="center">2.35 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.37 (1.12–1.67)</td><td rowspan="1" colspan="1" align="center">1.10 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.45 (1.09–1.92)</td></tr><tr><td rowspan="3" colspan="1">11</td><td rowspan="3" colspan="1">rs12418451</td><td rowspan="3" colspan="1"><em>MYEOV</em></td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1">rs10513723</td><td rowspan="1" colspan="1" align="center">176,062,702</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>NAALADL2</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">5.61 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.41 (1.21–1.63)</td><td rowspan="1" colspan="1" align="center">7.22 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.58 (1.13–2.21)</td><td rowspan="1" colspan="1" align="center">1.61 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.27 (1.05–1.54)</td><td rowspan="1" colspan="1" align="center">1.46 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.64 (1.21–2.22)</td></tr><tr><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1">rs7829048</td><td rowspan="1" colspan="1" align="center">4,689,690</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>CSMD1</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">9.76 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.74 (0.65–0.85)</td><td rowspan="1" colspan="1" align="center">1.22 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.68 (0.50–0.92)</td><td rowspan="1" colspan="1" align="center">4.52 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.84 (0.71–1.00)</td><td rowspan="1" colspan="1" align="center">1.81 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">0.60 (0.46–0.78)</td></tr><tr><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1">rs784411</td><td rowspan="1" colspan="1" align="center">46,827,089</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>CEP152</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.15 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" align="center">1.42 (1.25–1.61)</td><td rowspan="1" colspan="1" align="center">8.83 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.55 (1.12–2.16)</td><td rowspan="1" colspan="1" align="center">5.28 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.34 (1.14–1.58)</td><td rowspan="1" colspan="1" align="center">1.32 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.53 (1.18–1.99)</td></tr><tr><td rowspan="2" colspan="1">11</td><td rowspan="2" colspan="1">rs7127900</td><td rowspan="1" colspan="1">IGF2, IGF2AS</td><td rowspan="1" colspan="1" align="center">8</td><td rowspan="1" colspan="1">rs13258681</td><td rowspan="1" colspan="1" align="center">124,783,903</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>ANXA13</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">3.65 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.32 (1.17–1.48)</td><td rowspan="1" colspan="1" align="center">4.58 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.32 (1.01–1.73)</td><td rowspan="1" colspan="1" align="center">5.43 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.33 (1.13–1.56)</td><td rowspan="1" colspan="1" align="center">1.90 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.31 (1.05–1.64)</td></tr><tr><td rowspan="1" colspan="1"><em>INS,</em>
<em>TH</em></td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1">rs12628051</td><td rowspan="1" colspan="1" align="center">38,984,222</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>TNRC6B</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">3.39 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.30 (1.17–1.46)</td><td rowspan="1" colspan="1" align="center">1.82 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.50 (1.16–1.93)</td><td rowspan="1" colspan="1" align="center">6.14 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.24 (1.06–1.44)</td><td rowspan="1" colspan="1" align="center">1.44 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.32 (1.06–1.65)</td></tr><tr><td rowspan="3" colspan="1">17</td><td rowspan="3" colspan="1">rs4430796</td><td rowspan="3" colspan="1"><em>HNF1B</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1">rs731174</td><td rowspan="1" colspan="1" align="center">37,969,428</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>EPHA10</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">4.55 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.27 (1.15–1.40)</td><td rowspan="1" colspan="1" align="center">5.03 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.24 (1.00–1.54)</td><td rowspan="1" colspan="1" align="center">3.20 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.18 (1.01–1.38)</td><td rowspan="1" colspan="1" align="center">1.13 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" align="center">1.41 (1.19–1.68)</td></tr><tr><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1">rs12694942</td><td rowspan="1" colspan="1" align="center">158,518,681</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>UPP2</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">41,256</td><td rowspan="1" colspan="1" align="center">7.84 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.80 (0.73–0.88)</td><td rowspan="1" colspan="1" align="center">5.22 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.75 (0.61–0.92)</td><td rowspan="1" colspan="1" align="center">1.87 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.79 (0.68–0.92)</td><td rowspan="1" colspan="1" align="center">6.36 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.85 (0.72–1.01)</td></tr><tr><td rowspan="1" colspan="1" align="center">9</td><td rowspan="1" colspan="1">rs10812303</td><td rowspan="1" colspan="1" align="center">25,712,117</td><td rowspan="1" colspan="1">T</td><td rowspan="1" colspan="1"><em>TUSC1</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">43,261</td><td rowspan="1" colspan="1" align="center">5.59 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.34 (1.18–1.52)</td><td rowspan="1" colspan="1" align="center">6.63 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.46 (1.11–1.91)</td><td rowspan="1" colspan="1" align="center">7.57 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.27 (1.07–1.52)</td><td rowspan="1" colspan="1" align="center">9.01 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.37 (1.08–1.73)</td></tr><tr><td rowspan="2" colspan="1">17</td><td rowspan="2" colspan="1">rs1859962</td><td rowspan="2" colspan="1"><em>NA</em></td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1">rs16867225</td><td rowspan="1" colspan="1" align="center">180,749,531</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>CWC22</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">169,506</td><td rowspan="1" colspan="1" align="center">3.12 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.64 (0.53–0.77)</td><td rowspan="1" colspan="1" align="center">1.33 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.48 (0.30–0.75)</td><td rowspan="1" colspan="1" align="center">4.80 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.70 (0.54–0.90)</td><td rowspan="1" colspan="1" align="center">1.84 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.65 (0.45–0.93)</td></tr><tr><td rowspan="1" colspan="1" align="center">7</td><td rowspan="1" colspan="1">rs10277209</td><td rowspan="1" colspan="1" align="center">108,790,810</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>C7orf66</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">478,937</td><td rowspan="1" colspan="1" align="center">3.81 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.36 (1.19–1.55)</td><td rowspan="1" colspan="1" align="center">4.08 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.39 (1.01–1.89)</td><td rowspan="1" colspan="1" align="center">4.04 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.29 (1.08–1.53)</td><td rowspan="1" colspan="1" align="center">1.55 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.52 (1.17–1.97)</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs2735839</td><td rowspan="1" colspan="1"><em>KLK3</em></td><td rowspan="1" colspan="1" align="center">20</td><td rowspan="1" colspan="1">rs6089829</td><td rowspan="1" colspan="1" align="center">61,139,481</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>LOC63930</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">3.21 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.74 (0.65–0.84)</td><td rowspan="1" colspan="1" align="center">2.53 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.71 (0.52–0.96)</td><td rowspan="1" colspan="1" align="center">1.26 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.75 (0.64–0.90)</td><td rowspan="1" colspan="1" align="center">1.10 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.72 (0.56–0.93)</td></tr><tr><td rowspan="1" colspan="1">19</td><td rowspan="1" colspan="1">rs8102476</td><td rowspan="1" colspan="1"><em>PPP1R14A</em></td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1">rs1866967</td><td rowspan="1" colspan="1" align="center">29,958,249</td><td rowspan="1" colspan="1">G</td><td rowspan="1" colspan="1"><em>PTPRU</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">432,337</td><td rowspan="1" colspan="1" align="center">5.16 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.82 (0.75–0.89)</td><td rowspan="1" colspan="1" align="center">2.70 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.79 (0.64–0.97)</td><td rowspan="1" colspan="1" align="center">5.10 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.85 (0.75–0.95)</td><td rowspan="1" colspan="1" align="center">2.97 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">0.78 (0.66–0.92)</td></tr><tr><td rowspan="2" colspan="1">19</td><td rowspan="2" colspan="1">rs887391</td><td rowspan="2" colspan="1"><em>NA</em></td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1">rs735172</td><td rowspan="1" colspan="1" align="center">5,809,770</td><td rowspan="1" colspan="1">C</td><td rowspan="1" colspan="1"><em>EVC</em></td><td rowspan="1" colspan="1">Intron</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">2.03 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.31 (1.17–1.46)</td><td rowspan="1" colspan="1" align="center">6.58 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.43 (1.10–1.85)</td><td rowspan="1" colspan="1" align="center">2.37 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.27 (1.09–1.47)</td><td rowspan="1" colspan="1" align="center">1.02 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.32 (1.07–1.63)</td></tr><tr><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1">rs4463179</td><td rowspan="1" colspan="1" align="center">13,558,432</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>DNAH5</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">185,005</td><td rowspan="1" colspan="1" align="center">2.22 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">0.64 (0.53–0.77)</td><td rowspan="1" colspan="1" align="center">3.00 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.64 (0.42–0.96)</td><td rowspan="1" colspan="1" align="center">8.35 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" align="center">0.59 (0.46–0.77)</td><td rowspan="1" colspan="1" align="center">8.28 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">0.73 (0.51–1.04)</td></tr><tr><td rowspan="1" colspan="1">22</td><td rowspan="1" colspan="1">rs9623117</td><td rowspan="1" colspan="1"><em>TNRC6B</em></td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1">rs1713511</td><td rowspan="1" colspan="1" align="center">43,472,127</td><td rowspan="1" colspan="1">A</td><td rowspan="1" colspan="1"><em>KCTD8</em></td><td rowspan="1" colspan="1">Intergenic</td><td rowspan="1" colspan="1" align="center">398,550</td><td rowspan="1" colspan="1" align="center">7.87 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" align="center">1.31 (1.16–1.47)</td><td rowspan="1" colspan="1" align="center">4.94 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.54 (1.14–2.09)</td><td rowspan="1" colspan="1" align="center">6.71 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" align="center">1.23 (1.06–1.44)</td><td rowspan="1" colspan="1" align="center">1.06 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" align="center">1.36 (1.07–1.72)</td></tr></tbody></table></div><div id="largeobj_idm139862577920512" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3291863/table/tbl2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139862587594848"><p id="__p19" class="p p-first-last">SNP1 indicate the 32 known PCa-risk SNPs. SNP 2 indicates the interacting SNPs; Chr, chromosome; Position is based on NVBI Build 36: relative position is the distance of SNP2 relative to the nearest gene if SNP2 is located in the intergenic region; P and OR are for the multiplicative interaction term. P and OR for the meta-analysis are calculated based on a Cochran–Mantel–Haenszel test.</p></div></div></div><p id="__p20">Among the other 34 pairs of interactions that were significant at a <em>P</em><sub>interaction</sub> cutoff of 1 × 10<sup>−5</sup> in the meta-analysis, two pairs were noteworthy to be emphasized when considering possible biological function. The first pair involved an interaction between rs7127900 at insulin-like growth factor (<em>IGF2</em>)<em>/IGF2AS</em> region and rs12628051 in the intron of <em>TNRC6B</em>, with a <em>P</em><sub>interaction</sub> of 3.39 × 10<sup>−6</sup> (OR<sub>interaction</sub> = 1.30; 95% CI = 1.17–1.46) (<a href="/pmc/articles/PMC3291863/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table II</span></a>). The interaction was significant in all three populations and the effects of the interaction were in the same direction (<em>P</em><sub>interaction</sub> = 0.002, OR<sub>interaction</sub> = 1.50, 95% CI = 1.16–1.93 in CAPS; <em>P</em><sub>interaction</sub> = 0.006, OR<sub>interaction</sub> = 1.24, 95% CI = 1.06–1.44 in JHH and <em>P</em><sub>interaction</sub> = 0.014, OR<sub>interaction</sub> = 1.32, 95% CI = 1.06–1.65 in CGEMS). The second pair of interaction was between rs7679763 in <em>TET2</em> gene region and rs290258 in the promoter region of SYK, with a <em>P</em><sub>interaction</sub> of 1.49 × 10<sup>−6</sup> (OR = 0.75; 95% CI = 0.67–0.84) (<a href="/pmc/articles/PMC3291863/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table II</span></a>). Similarly, the interaction effect was consistently observed in all three populations with the same direction of interaction effect (<em>P</em><sub>interaction</sub> = 0.002, OR<sub>interaction</sub> = 0.66, 95% CI = 0.51–0.86 in CAPS; <em>P</em><sub>interaction</sub> = 0.003, OR<sub>interaction</sub> = 0.78, 95% CI = 0.67–0.92 in JHH and <em>P</em><sub>interaction</sub> = 0.014, OR<sub>interaction</sub> = 0.75, 95% CI = 0.59–0.94 in CGEMS).</p><p id="__p21" class="p">We then carefully examined the significant pairs of SNP–SNP interactions reported by Ciampa <em>et al.</em> (<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568065">24</a>) in CAPS and JHH population. Among the 25 pairs reported in the previous study, 16 pairs were also evaluated in our data. Three pairs of SNP–SNP interaction reached nominal <em>P</em><sub>interaction</sub> of 0.05 in CAPS population (<a href="/pmc/articles/PMC3291863/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table III</span></a>). The most significant interaction replicated in CAPS was between rs6983267 and rs4953347 (<em>P</em><sub>interaction</sub> = 0.001, OR<sub>interaction</sub> = 1.42). However, this interaction was not significant in the JHH population (<em>P</em>= 0.69). The other two pairs of SNPs were replicated in CAPS at a nominal <em>P</em><sub>interaction</sub> of 0.05, including the interaction between rs2735839 and rs12196677 (<em>P</em><sub>interaction</sub> = 0.017, same direction of interaction effect) and the interaction between rs10934853 and rs10458466 (<em>P</em><sub>interaction</sub> = 0.02 but with opposite direction of interaction effect). The interaction effect of the two pairs of SNP–SNP interactions that were significant in JHH population at a <em>P</em><sub>interaction</sub> cutoff of 0.05 were in the opposite direction compared with the previous study (<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568034">24</a>) (<a href="/pmc/articles/PMC3291863/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table III</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="tbl3"><h3>Table III.</h3><!--caption a7--><div class="caption"><p id="__p22">Results for the top pairs of SNP–SNP interactions reported by the previous study (<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568036">24</a>) in JHH and CAPS population</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm139862509236304" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><td rowspan="2" colspan="1">Gene</td><td rowspan="2" colspan="1">Conditioning SNP</td><td rowspan="2" colspan="1">Interacting SNP</td><td rowspan="2" colspan="1">Chr</td><td rowspan="2" colspan="1">Gene</td><td colspan="3" rowspan="1">CAPS<hr></td><td colspan="3" rowspan="1">JHH<hr></td></tr><tr><td rowspan="1" colspan="1">OR<sup>a</sup></td><td rowspan="1" colspan="1">95% CI</td><td rowspan="1" colspan="1"><em>P</em><sup>a</sup></td><td rowspan="1" colspan="1">OR<sup>a</sup></td><td rowspan="1" colspan="1">95% CI</td><td rowspan="1" colspan="1"><em>P</em><sup>a</sup></td></tr></thead><tbody><tr><td rowspan="1" colspan="1"><em>DAB2IP</em></td><td rowspan="1" colspan="1">rs1571801</td><td rowspan="1" colspan="1">rs4242</td><td rowspan="1" colspan="1" align="center">22</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.14</td><td rowspan="1" colspan="1" align="center">0.63–2.05</td><td rowspan="1" colspan="1" align="center">0.662</td><td rowspan="1" colspan="1" align="center">0.80</td><td rowspan="1" colspan="1" align="center">0.7–0.93</td><td rowspan="1" colspan="1" align="center"><strong>0.003</strong></td></tr><tr><td rowspan="2" colspan="1"><em>EEFSEC</em></td><td rowspan="1" colspan="1">rs10934853</td><td rowspan="1" colspan="1">rs12489404</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1"><em>GRM7</em></td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">0.76–1.29</td><td rowspan="1" colspan="1" align="center">0.941</td><td rowspan="1" colspan="1" align="center">1.18</td><td rowspan="1" colspan="1" align="center">1.03–1.34</td><td rowspan="1" colspan="1" align="center"><strong>0.016</strong></td></tr><tr><td rowspan="1" colspan="1">rs10934853</td><td rowspan="1" colspan="1">rs10458466</td><td rowspan="1" colspan="1" align="center">1</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.77</td><td rowspan="1" colspan="1" align="center">0.62–0.96</td><td rowspan="1" colspan="1" align="center"><strong>0.020</strong></td><td rowspan="1" colspan="1" align="center">0.93</td><td rowspan="1" colspan="1" align="center">0.81–1.06</td><td rowspan="1" colspan="1" align="center">0.259</td></tr><tr><td rowspan="3" colspan="1"><em>EHBP1</em></td><td rowspan="1" colspan="1">rs721048</td><td rowspan="1" colspan="1">rs10514124</td><td rowspan="1" colspan="1" align="center">5</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">0.98</td><td rowspan="1" colspan="1" align="center">0.66–1.46</td><td rowspan="1" colspan="1" align="center">0.921</td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.78–1.05</td><td rowspan="1" colspan="1" align="center">0.194</td></tr><tr><td rowspan="1" colspan="1">rs721048</td><td rowspan="1" colspan="1">rs704638</td><td rowspan="1" colspan="1" align="center">4</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1" align="center">0.83–1.52</td><td rowspan="1" colspan="1" align="center">0.462</td><td rowspan="1" colspan="1" align="center">1.10</td><td rowspan="1" colspan="1" align="center">0.94–1.29</td><td rowspan="1" colspan="1" align="center">0.240</td></tr><tr><td rowspan="1" colspan="1">rs721048</td><td rowspan="1" colspan="1">rs7604809</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1"><em>GTF3C3</em></td><td rowspan="1" colspan="1" align="center">0.91</td><td rowspan="1" colspan="1" align="center">0.60–1.38</td><td rowspan="1" colspan="1" align="center">0.656</td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">0.75–1.14</td><td rowspan="1" colspan="1" align="center">0.455</td></tr><tr><td rowspan="1" colspan="1"><em>JAZF1</em></td><td rowspan="1" colspan="1">rs10486567</td><td rowspan="1" colspan="1">rs2899748</td><td rowspan="1" colspan="1" align="center">15</td><td rowspan="1" colspan="1"><em>GLCE</em></td><td rowspan="1" colspan="1" align="center">0.99</td><td rowspan="1" colspan="1" align="center">0.73–1.34</td><td rowspan="1" colspan="1" align="center">0.948</td><td rowspan="1" colspan="1" align="center">0.96</td><td rowspan="1" colspan="1" align="center">0.83–1.11</td><td rowspan="1" colspan="1" align="center">0.573</td></tr><tr><td rowspan="1" colspan="1"><em>KLK2-KLK3</em></td><td rowspan="1" colspan="1">rs2735839</td><td rowspan="1" colspan="1">rs12196677</td><td rowspan="1" colspan="1" align="center">6</td><td rowspan="1" colspan="1"><em>PNPLA1</em></td><td rowspan="1" colspan="1" align="center">3.90</td><td rowspan="1" colspan="1" align="center">1.27–11.95</td><td rowspan="1" colspan="1" align="center"><strong>0.017</strong></td><td rowspan="1" colspan="1" align="center">1.06</td><td rowspan="1" colspan="1" align="center">0.83–1.36</td><td rowspan="1" colspan="1" align="center">0.616</td></tr><tr><td rowspan="4" colspan="1"><em>MSMB</em></td><td rowspan="1" colspan="1">rs10993994</td><td rowspan="1" colspan="1">rs12605415</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.07</td><td rowspan="1" colspan="1" align="center">0.87–1.32</td><td rowspan="1" colspan="1" align="center">0.525</td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">0.96–1.22</td><td rowspan="1" colspan="1" align="center">0.179</td></tr><tr><td rowspan="1" colspan="1">rs10993994</td><td rowspan="1" colspan="1">rs11083271</td><td rowspan="1" colspan="1" align="center">18</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.12</td><td rowspan="1" colspan="1" align="center">0.89–1.41</td><td rowspan="1" colspan="1" align="center">0.329</td><td rowspan="1" colspan="1" align="center">1.04</td><td rowspan="1" colspan="1" align="center">0.93–1.18</td><td rowspan="1" colspan="1" align="center">0.486</td></tr><tr><td rowspan="1" colspan="1">rs10993994</td><td rowspan="1" colspan="1">rs9880831</td><td rowspan="1" colspan="1" align="center">3</td><td rowspan="1" colspan="1"><em>LOC391524</em></td><td rowspan="1" colspan="1" align="center">1.00</td><td rowspan="1" colspan="1" align="center">1.00–1.00</td><td rowspan="1" colspan="1" align="center">0.975</td><td rowspan="1" colspan="1" align="center">0.95</td><td rowspan="1" colspan="1" align="center">0.84–1.07</td><td rowspan="1" colspan="1" align="center">0.372</td></tr><tr><td rowspan="1" colspan="1">rs10993994</td><td rowspan="1" colspan="1">rs7921651</td><td rowspan="1" colspan="1" align="center">10</td><td rowspan="1" colspan="1"><em>FAM107B</em></td><td rowspan="1" colspan="1" align="center">1.18</td><td rowspan="1" colspan="1" align="center">0.93–1.49</td><td rowspan="1" colspan="1" align="center">0.164</td><td rowspan="1" colspan="1" align="center">1.07</td><td rowspan="1" colspan="1" align="center">0.93–1.22</td><td rowspan="1" colspan="1" align="center">0.365</td></tr><tr><td rowspan="1" colspan="1"><em>MYEOV</em></td><td rowspan="1" colspan="1">rs10896449</td><td rowspan="1" colspan="1">rs1240224</td><td rowspan="1" colspan="1" align="center">12</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.08</td><td rowspan="1" colspan="1" align="center">0.86–1.36</td><td rowspan="1" colspan="1" align="center">0.506</td><td rowspan="1" colspan="1" align="center">1.01</td><td rowspan="1" colspan="1" align="center">0.89–1.15</td><td rowspan="1" colspan="1" align="center">0.851</td></tr><tr><td rowspan="2" colspan="1"><em>8q24</em></td><td rowspan="1" colspan="1">rs16901979</td><td rowspan="1" colspan="1">rs991000</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1"></td><td rowspan="1" colspan="1" align="center">1.14</td><td rowspan="1" colspan="1" align="center">0.64–2.05</td><td rowspan="1" colspan="1" align="center">0.661</td><td rowspan="1" colspan="1" align="center">1.00</td><td rowspan="1" colspan="1" align="center">0.70–1.43</td><td rowspan="1" colspan="1" align="center">0.997</td></tr><tr><td rowspan="1" colspan="1">rs16901979</td><td rowspan="1" colspan="1">rs943323</td><td rowspan="1" colspan="1" align="center">14</td><td rowspan="1" colspan="1"><em>KIAA1409</em></td><td rowspan="1" colspan="1" align="center">0.87</td><td rowspan="1" colspan="1" align="center">0.50–1.52</td><td rowspan="1" colspan="1" align="center">0.623</td><td rowspan="1" colspan="1" align="center">0.92</td><td rowspan="1" colspan="1" align="center">0.65–1.30</td><td rowspan="1" colspan="1" align="center">0.628</td></tr><tr><td rowspan="1" colspan="1"><em>8q24</em></td><td rowspan="1" colspan="1">rs6983267</td><td rowspan="1" colspan="1">rs4953347</td><td rowspan="1" colspan="1" align="center">2</td><td rowspan="1" colspan="1"><em>EPAS1</em></td><td rowspan="1" colspan="1" align="center">1.42</td><td rowspan="1" colspan="1" align="center">1.15–1.76</td><td rowspan="1" colspan="1" align="center"><strong>0.001</strong></td><td rowspan="1" colspan="1" align="center">0.98</td><td rowspan="1" colspan="1" align="center">0.87–1.10</td><td rowspan="1" colspan="1" align="center">0.688</td></tr></tbody></table></div><div id="largeobj_idm139862509236304" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3291863/table/tbl3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="idm139862504450336"><p id="__p23" class="p p-first-last">Bold indicates P- values with a nominal significant level of 0.05. Chr, chromosome.</p></div><div id="tblfn1"><sup>a</sup><em>P</em> values, OR and 95% CI are calculated for the multiplicative interaction term.</div></div></div></div><div id="__sec8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec8title">Discussion</h2><p id="__p25" class="p p-first">To our knowledge, our study represents one of the first comprehensive gene–gene interaction scans in three PCa GWAS populations. Specifically, we performed a genome-wide gene–gene interaction scan for each of the 32 known PCa risk-associated variants identified from GWAS in three case–control populations of European descents, which includes a total of 4723 PCa cases and 4792 controls. In the meta-analysis, we found 35 pairs of SNP–SNP interactions that were significantly associated with PCa risk (<em>P</em><sub>interaction</sub> &lt; 1 × 10<sup>−5</sup>). In addition, the interactions for those 35 pairs were significant in all three populations (all <em>P</em><sub>interaction</sub> &lt; 0.05). Among those 35 pairs of statistically significant interactions, we emphasized three pairs of interactions with potential biological implication, including an interaction between rs12418451 in <em>MYEOV</em> and rs16961635 in <em>CEP152</em>, with a <em>P</em><sub>interaction</sub> of 1.15 × 10<sup>−7</sup> (OR = 1.42, 95% CI = 1.25–1.61), an interaction between rs7127900 at <em>IGF2/IGF2AS</em> region and rs12628051 in the intron of <em>TNRC6B</em>, with a <em>P</em><sub>interaction</sub> of 3.39 × 10<sup>−6</sup> (OR = 1.30, 95% CI = 1.17–1.46) and an interaction between rs7679763 in <em>TET2</em> gene region and rs290258 in the promoter region of SYK, with a <em>P</em><sub>interaction</sub> of 1.49 × 10<sup>−6</sup> (OR = 0.75, 95% CI = 0.67–0.84).</p><p id="__p26">The discovery of approximately three dozen PCa risk variants using single SNP analysis suggests that it is possible to detect individual risk variants. However, when the underlying genetic model involves interaction of multiple genes, a single gene approach is less effective and may not be able to explain the complex etiology of the disease. Therefore, evaluation of the joint effect (epistasis) of multiple genetic variants is critical to understand the underlying causes of complex diseases (<a href="#bib30" rid="bib30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568056">30</a>), especially in the situation where several individual risk variants have been identified. The next question is to explore whether other SNPs interact with those SNPs to modify risk to PCa. The identified loci that interact with the known PCa risk-associated SNPs may help to elucidate the underlying molecular mechanisms of the associations of those risk SNPs.</p><p id="__p27">The most significant interaction was seen between the PCa risk-associated SNPs rs12418451 and rs784411. The SNP rs12418451 is located at the 11q13.2 that is ∼77 kb upstream of <em>TPCN2</em>, a putative cation-selective ion channel gene and ∼126 kb upstream of <em>MYEOV</em>, an oncogene that has been implicated in multiple cancers (<a href="#bib31" rid="bib31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568037">31</a>–<a href="#bib35" rid="bib35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568057">35</a>). The SNP rs784411 resides in the intron of CEP152, a centrosomal protein that was recently shown to function as a regulator of genomic integrity (<a href="#bib36" rid="bib36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568060">36</a>) and cellular response to DNA damage (<a href="#bib37" rid="bib37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_277709389">37</a>). Given the limited information, we speculate that observed interaction may reflect the close collaboration of MYEOV (or TPCN2, even though it is less likely) and CEP152 in the same or different oncogenic pathways that drive the tumorigenesis of prostatic epithelial cells.</p><p id="__p28">Among the two SNPs that were shown to consistently interact with the PCa risk-associated SNP rs7127900 at 11p15.5, one SNP (rs12628051) is located within TNRC6B, which encodes an RNA interference (RNAi) machinery component protein crucial for the microRNA/small interfering RNA-dependent translational repression or degradation of target messenger RNAs. It is worthy to mention that this gene also contains a GWAS-identified PCa risk-associated SNP (rs9623117). Several mechanisms may potentially explain for these interactions. Firstly, we noticed that at ∼70 kb telomeric to rs7127900 reside the PCa-implicated <em>IGF2</em> gene and its antisense transcript-encoding <em>IGF2AS</em>. IGF2 encodes a member of the insulin family of polypeptide growth factors that promotes cell proliferation during fetal development but becomes less active in healthy adults due to genomic imprinting. Dysregulated overexpression of IGF2 caused by loss of imprinting has been associated with a variety of human cancers, including PCa (<a href="#bib38" rid="bib38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568061">38</a>–<a href="#bib41" rid="bib41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_342836232">41</a>). IGF2AS encodes a predictably non-coding RNA that is antisense to IGF2 and thus may potentially regulate IGF2 expression through RNAi in a similar manner as some other natural antisense transcripts. Thus, one plausible scenario is that TNRC6B may affect the RNAi-mediated transcriptional regulation of IGF2AS on IGF2, which may underlie the observed interaction between genetic variants within these two loci. Secondly, there are two microRNA genes located at 11p15.5, miR-4686 (∼40 kb from the PCa-risk SNP rs7127900) and miR-483 (∼80 kb from rs7127900). Although the role of miR-4686 remains to be determined, miR-483 has been demonstrated to act as an oncogene to suppress proapoptotic BBC3 (PUMA) or tumor suppressive DPC4 (Smad4) in a variety of human cancers (<a href="#bib42" rid="bib42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_481090293">42</a>,<a href="#bib43" rid="bib43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568039">43</a>). Thus, an alternative mechanism for the observed interaction between the 11p15.5 locus and the TNRC6B locus is that genetic variants in TNRC6B may affect the miR-483 (or miR-4686)-mediated RNAi toward its/their target tumor suppressor genes.</p><p id="__p29">Another pair of interacting SNPs was found between rs7679673 (∼6 kb upstream of <em>TET2</em>) and rs290258 (∼8 kb upstream of <em>SYK</em>). TET2 encodes an enzyme hydroxylating methylcytosine and is implicated in epigenetic programming that involves DNA methylation and demethylation (reviewed in ref. <a href="#bib44" rid="bib44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568042">44</a>). The critical role of TET2 in cancer is suggested by the observation that loss-of-function mutations of TET2 are frequently identified in various hematological malignancies (<a href="#bib45" rid="bib45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568038">45</a>,<a href="#bib46" rid="bib46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568044">46</a>). As a non-receptor tyrosine protein kinase that mediates cellular proliferation and differentiation, SYK is believed to function as a potential tumor suppressive gene (reviewed in ref. <a href="#bib47" rid="bib47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568070">47</a>). It is noteworthy that hypermethylation of SYK gene promoter has been frequently found in and widely associated with lung, gastric and breast cancer (<a href="#bib48" rid="bib48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568048">48</a>,<a href="#bib49" rid="bib49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568072">49</a>). Thus, although it remains to be determined whether SYK promoter in prostatic tumors also undergoes silencing via DNA methylation, the observed interaction between TET2 and SYK suggests that it is a plausible hypothesis.</p><p id="__p30">Two SNPs (rs731174 and rs10812303) were found to interact with the GWAS-identified PCa risk-associated SNP rs4430796, residing within <em>HNF1B</em>, a homeodomain-containing transcription factor whose expression alteration has been widely implicated in various human cancers, including PCa. The SNP rs731174 is located within the intron of <em>EPHA10</em>, a member of the EPH subfamily of receptor tyrosine kinases. This family of receptor tyrosine kinases play an important role in cell–cell communication regulating cell attachment, shape and mobility in epithelial cells and are believed to be implicated in carcinogenesis (reviewed in ref. <a href="#bib50" rid="bib50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_277709391">50</a>). It is possible that HNF1Ba and EPHA10 collaborate in the signaling network that is crucial for the well-being of prostatic cells, whereas the genetic variants located within these two genes may synergistically contribute to the oncogenesis of PCa. The other SNP rs10812303 is ∼40 kb upstream of <em>TUSC1</em>, an intronless gene that has been suggested to serve as a tumor suppressor in lung tumorigenesis (<a href="#bib51" rid="bib51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_277709390">51</a>). Thus, the interaction between genetic variants in TUSC1 and HNF1B may also suggest a plausible collaboration of these two genes.</p><p id="__p31">Besides our novel findings, we also replicated the most notable finding reported by the study of Ciampa <em>et al.</em> (<a href="#bib24" rid="bib24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_264568050">24</a>) in our CAPS population. The interaction involves 8q24 region 3 (rs6983267) and <em>EPAS1</em> (rs4953347). We also observed the interaction between the <em>KLK2–</em><em>KLK3</em> (rs2735839) and <em>PNPLA1</em> in the CAPS population. However, none of those two pairs of interaction were significant in the JHH dataset. Therefore, further statistical evidence supports from additional replication studies are needed to reach a more robust conclusion.</p><p id="__p32">Our results need to be interpreted with caution. The most significant SNP–SNP interaction pair detected in the meta-analysis had a <em>P</em><sub>interaction</sub> of 1.15 × 10<sup>−7</sup>. It did not reach a genome-wide significance level of 1.5 × 10<sup>−9</sup> as we performed multiple tests in this study (1 million × 32 = 32 million tests). One possible reason for not achieving genome-wide significance may be due to limited statistical power to detect small to modest interaction effects. In the meta-analysis of 4723 PCa cases and 4759 controls, we had 80% power to detect relatively large interaction effects (OR &gt; 1.7), using the stringent Bonferonni-corrected <em>P</em>-value cutoff of 1.5 × 10<sup>−9</sup> to claim a genome-wide significant level. Therefore, additional samples were needed to detect modest interaction effects at a genome-wide significant level, which was the case in our study (OR approximately ranged from 1.2 to 1.5). However, the interaction effect detected in our study was consistently implicated in all three populations and with the same direction of interaction effects. In addition, similar pattern of integrations were observed (quantitative interactions) among the three populations. This may represent statistically meaningful SNP–SNP interactions with a modest magnitude of interaction effect. However, the pairs of SNP–SNP interactions identified by our study still warrant follow-up in other populations to further exclude the possibilities of false positive findings. More importantly, functional studies are needed to better understand the underlying molecular mechanisms of the interactions implicated.</p><p id="__p33">The limitations of our study include the use of a pairwise gene–gene interaction approach and limiting the search of interactions on SNPs that confer main effects only. Firstly, the search for gene–gene interactions based on a pairwise approach leads to reduced or no power when high-order interactions are present. However, the search for high-order interactions in a GWAS scale still represents a computational challenge and warrants novel statistical approaches to handle this difficult task. Secondly, we did not evaluate gene–gene interactions among SNPs that didn’t confer a main effect. We can’t exclude the possibilities that the epistasis between other pairs of SNPs without strong main effect may also affect risk of PCa. However, the exhaustive search for gene–gene interaction is beyond the scope of the current study as we focused on identifying genes that interact with known risk-associated SNPs. We would like to continue our research on this field in future studies. In addition, the current analysis of the imputed data was based on the most likely genotype with the posterior probability of ≥0.90. Alternatively, using the imputed dosage data would be more accurate and may improve the power of this study. Future epistasis analysis based on the imputed dosage data would be explored once the statistical and computation methods are available.</p><p id="__p34" class="p p-last">In summary, our systematic evaluation of gene–gene interactions in three GWAS populations suggested a list of loci interacting with known PCa risk-associated SNPs that may warrant follow-up in other study populations. Three pairs of interactions are worthwhile to be emphasized, including an interaction between rs12418451 in the <em>MYEOV</em> gene region and rs784411 in the intron of <em>CEP152</em>, an interaction between rs7127900 in the <em>IGF2/IGF2AS</em> gene region and rs12628051 in the intron of <em>TNRC6B</em> and an interaction between rs7679673 in the <em>TET2</em> gene region and rs290258 in the intron of <em>SYK.</em> Those results showed statistical evidence for genes interacting with known risk-associated SNPs on PCa risk. The interacting loci identified also provide more hints on the underlying molecular mechanism of the associations with PCa risk for the known risk-associated SNPs. Additional studies are warranted to further confirm the gene–gene interaction effects detected in this study.</p></div><div id="__sec9" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec9title">Supplementary material</h2><p id="__p35" class="p p-first-last"><a href="http://www.carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgr316/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Tables 1</a> and <a href="http://www.carcin.oxfordjournals.org/lookup/suppl/doi:10.1093/carcin/bgr316/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">2</a> can be found at <a href="http://carcin.oxfordjournals.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://carcin.oxfordjournals.org/</a>.</p></div><div id="__sec10" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec10title">Funding</h2><p id="__p36" class="p p-first-last">This work was supported by the Department of Defense (<span>W81XWH-09-1-0488</span> to J.S.); an intramural funding from the Van Andel Research Institute to J.X. and the National Cancer Institute (<span>CA129684</span> J.X.).</p></div><div id="__sec11" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec11title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm139862504663936"><a href="/pmc/articles/PMC3291863/bin/supp_33_3_598__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm139862503355216" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139862503355216title">Acknowledgments</h2><div class="sec"><p id="__p37">We thank all of the study subjects who participated in the CAncer of the Prostate in Sweden study and the urologists who provided their patients to the CAncer of the Prostate in Sweden and Johns Hopkins Hospital studies. We acknowledge the contribution of multiple physicians and researchers in designing and recruiting study subjects, including Dr H.-O.Adami. We also acknowledge the National Cancer Institute Cancer Genetic Markers of Susceptibility Initiative for making the data publicly available.</p><p id="__p38"><em>Conflict of Interest Statement:</em> None declared.</p></div></div><div id="idm139862578646336" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139862578646336title">Glossary</h2><div class="bk-sec"><div><h4>Abbreviations</h4><table class="default-table glossary"><tbody><tr><td>CAPS</td><td>CAncer of the Prostate in Sweden</td></tr><tr><td>CGEMS</td><td>Cancer Genetic Markers of Susceptibility</td></tr><tr><td>CI</td><td>confidence interval</td></tr><tr><td>GWAS</td><td>genome-wide association studies</td></tr><tr><td>IGF</td><td>insulin-like growth factor</td></tr><tr><td>JHH</td><td>Johns Hopkins Hospital</td></tr><tr><td>OR</td><td>odds ratio</td></tr><tr><td>PCa</td><td>prostate cancer</td></tr><tr><td>RNAi</td><td>RNA interference</td></tr><tr><td>SNP</td><td>single-nucleotide polymorphism</td></tr></tbody></table></div></div></div><div id="idm139862508861504" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm139862508861504title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation">Amundadottir LT, et al.  A common variant associated with prostate cancer in European and African populations. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:652–658.</span> [<a href="/pubmed/16682969" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+common+variant+associated+with+prostate+cancer+in+European+and+African+populations&amp;author=LT+Amundadottir&amp;volume=38&amp;publication_year=2006&amp;pages=652-658&amp;pmid=16682969&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation">Gudmundsson J, et al.  Genome-wide association study identifies a second prostate cancer susceptibility variant at 8q24. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:631–637.</span> [<a href="/pubmed/17401366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+identifies+a+second+prostate+cancer+susceptibility+variant+at+8q24&amp;author=J+Gudmundsson&amp;volume=39&amp;publication_year=2007&amp;pages=631-637&amp;pmid=17401366&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation">Yeager M, et al.  Genome-wide association study of prostate cancer identifies a second risk locus at 8q24. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:645–649.</span> [<a href="/pubmed/17401363" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+study+of+prostate+cancer+identifies+a+second+risk+locus+at+8q24&amp;author=M+Yeager&amp;volume=39&amp;publication_year=2007&amp;pages=645-649&amp;pmid=17401363&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation">Gudmundsson J, et al.  Two variants on chromosome 17 confer prostate cancer risk, and the one in TCF2 protects against type 2 diabetes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:977–983.</span> [<a href="/pubmed/17603485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Two+variants+on+chromosome+17+confer+prostate+cancer+risk,+and+the+one+in+TCF2+protects+against+type+2+diabetes&amp;author=J+Gudmundsson&amp;volume=39&amp;publication_year=2007&amp;pages=977-983&amp;pmid=17603485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation">Duggan D, et al.  Two genome-wide association studies of aggressive prostate cancer implicate putative prostate tumor suppressor gene DAB2IP. <span><span class="ref-journal">J. Natl Cancer Inst. </span>2007;<span class="ref-vol">99</span>:1836–1844.</span> [<a href="/pubmed/18073375" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl+Cancer+Inst.&amp;title=Two+genome-wide+association+studies+of+aggressive+prostate+cancer+implicate+putative+prostate+tumor+suppressor+gene+DAB2IP&amp;author=D+Duggan&amp;volume=99&amp;publication_year=2007&amp;pages=1836-1844&amp;pmid=18073375&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation">Thomas G, et al.  Multiple loci identified in a genome-wide association study of prostate cancer. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:310–315.</span> [<a href="/pubmed/18264096" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Multiple+loci+identified+in+a+genome-wide+association+study+of+prostate+cancer&amp;author=G+Thomas&amp;volume=40&amp;publication_year=2008&amp;pages=310-315&amp;pmid=18264096&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation">Gudmundsson J, et al.  Common sequence variants on 2p15 and Xp11.22 confer susceptibility to prostate cancer. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:281–283.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3598012/">PMC free article</a>]</span> [<a href="/pubmed/18264098" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Common+sequence+variants+on+2p15+and+Xp11.22+confer+susceptibility+to+prostate+cancer&amp;author=J+Gudmundsson&amp;volume=40&amp;publication_year=2008&amp;pages=281-283&amp;pmid=18264098&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation">Eeles RA, et al.  Multiple newly identified loci associated with prostate cancer susceptibility. <span><span class="ref-journal">Nat. Genet. </span>2008;<span class="ref-vol">40</span>:316–321.</span> [<a href="/pubmed/18264097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Multiple+newly+identified+loci+associated+with+prostate+cancer+susceptibility&amp;author=RA+Eeles&amp;volume=40&amp;publication_year=2008&amp;pages=316-321&amp;pmid=18264097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation">Yeager M, et al.  Identification of a new prostate cancer susceptibility locus on chromosome 8q24. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:1055–1057.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3430510/">PMC free article</a>]</span> [<a href="/pubmed/19767755" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Identification+of+a+new+prostate+cancer+susceptibility+locus+on+chromosome+8q24&amp;author=M+Yeager&amp;volume=41&amp;publication_year=2009&amp;pages=1055-1057&amp;pmid=19767755&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation">Gudmundsson J, et al.  Genome-wide association and replication studies identify four variants associated with prostate cancer susceptibility. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:1122–1126.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3562712/">PMC free article</a>]</span> [<a href="/pubmed/19767754" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+association+and+replication+studies+identify+four+variants+associated+with+prostate+cancer+susceptibility&amp;author=J+Gudmundsson&amp;volume=41&amp;publication_year=2009&amp;pages=1122-1126&amp;pmid=19767754&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation">Eeles RA, et al.  Identification of seven new prostate cancer susceptibility loci through a genome-wide association study. <span><span class="ref-journal">Nat. Genet. </span>2009;<span class="ref-vol">41</span>:1116–1121.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2846760/">PMC free article</a>]</span> [<a href="/pubmed/19767753" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Identification+of+seven+new+prostate+cancer+susceptibility+loci+through+a+genome-wide+association+study&amp;author=RA+Eeles&amp;volume=41&amp;publication_year=2009&amp;pages=1116-1121&amp;pmid=19767753&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation">Sun J, et al.  Association between sequence variants at 17q12 and 17q24.3 and prostate cancer risk in European and African Americans. <span><span class="ref-journal">Prostate. </span>2008;<span class="ref-vol">68</span>:691–697.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3176499/">PMC free article</a>]</span> [<a href="/pubmed/18361410" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prostate&amp;title=Association+between+sequence+variants+at+17q12+and+17q24.3+and+prostate+cancer+risk+in+European+and+African+Americans&amp;author=J+Sun&amp;volume=68&amp;publication_year=2008&amp;pages=691-697&amp;pmid=18361410&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation">Hsu FC, et al.  A novel prostate cancer susceptibility locus at 19q13. <span><span class="ref-journal">Cancer Res. </span>2009;<span class="ref-vol">69</span>:2720–2723.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2803342/">PMC free article</a>]</span> [<a href="/pubmed/19318570" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=A+novel+prostate+cancer+susceptibility+locus+at+19q13&amp;author=FC+Hsu&amp;volume=69&amp;publication_year=2009&amp;pages=2720-2723&amp;pmid=19318570&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation">Sun J, et al.  Sequence variants at 22q13 are associated with prostate cancer risk. <span><span class="ref-journal">Cancer Res. </span>2009;<span class="ref-vol">69</span>:10–15.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2705898/">PMC free article</a>]</span> [<a href="/pubmed/19117981" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Sequence+variants+at+22q13+are+associated+with+prostate+cancer+risk&amp;author=J+Sun&amp;volume=69&amp;publication_year=2009&amp;pages=10-15&amp;pmid=19117981&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation">Zheng SL, et al.  Two independent prostate cancer risk-associated loci at 11q13. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>2009;<span class="ref-vol">18</span>:1815–1820.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2802212/">PMC free article</a>]</span> [<a href="/pubmed/19505914" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=Two+independent+prostate+cancer+risk-associated+loci+at+11q13&amp;author=SL+Zheng&amp;volume=18&amp;publication_year=2009&amp;pages=1815-1820&amp;pmid=19505914&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation">Kote-Jarai Z, et al.  Multiple novel prostate cancer predisposition loci confirmed by an international study: the PRACTICAL Consortium. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>2008;<span class="ref-vol">17</span>:2052–2061.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2776652/">PMC free article</a>]</span> [<a href="/pubmed/18708398" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=Multiple+novel+prostate+cancer+predisposition+loci+confirmed+by+an+international+study:+the+PRACTICAL+Consortium&amp;author=Z+Kote-Jarai&amp;volume=17&amp;publication_year=2008&amp;pages=2052-2061&amp;pmid=18708398&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation">Chang BL, et al.  Validation of genome-wide prostate cancer associations in men of African descent. <span><span class="ref-journal">Cancer Epidemiol. Biomarkers Prev. </span>2011;<span class="ref-vol">20</span>:23–32.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3110616/">PMC free article</a>]</span> [<a href="/pubmed/21071540" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Epidemiol.+Biomarkers+Prev.&amp;title=Validation+of+genome-wide+prostate+cancer+associations+in+men+of+African+descent&amp;author=BL+Chang&amp;volume=20&amp;publication_year=2011&amp;pages=23-32&amp;pmid=21071540&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation">Zheng SL, et al.  Association of 17 prostate cancer susceptibility loci with prostate cancer risk in Chinese men. <span><span class="ref-journal">Prostate. </span>2010;<span class="ref-vol">70</span>:425–432.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3078699/">PMC free article</a>]</span> [<a href="/pubmed/19866473" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prostate&amp;title=Association+of+17+prostate+cancer+susceptibility+loci+with+prostate+cancer+risk+in+Chinese+men&amp;author=SL+Zheng&amp;volume=70&amp;publication_year=2010&amp;pages=425-432&amp;pmid=19866473&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation">Yamada H, et al.  Replication of prostate cancer risk loci in a Japanese case-control association study. <span><span class="ref-journal">J. Natl Cancer Inst. </span>2009;<span class="ref-vol">101</span>:1330–1336.</span> [<a href="/pubmed/19726753" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Natl+Cancer+Inst.&amp;title=Replication+of+prostate+cancer+risk+loci+in+a+Japanese+case-control+association+study&amp;author=H+Yamada&amp;volume=101&amp;publication_year=2009&amp;pages=1330-1336&amp;pmid=19726753&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation">Lindstrom S, et al.  Characterizing associations and SNP-environment interactions for GWAS-identified prostate cancer risk markers—results from BPC3. <span><span class="ref-journal">PLoS One. </span>2011;<span class="ref-vol">6</span>:e17142.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3044744/">PMC free article</a>]</span> [<a href="/pubmed/21390317" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Characterizing+associations+and+SNP-environment+interactions+for+GWAS-identified+prostate+cancer+risk+markers—results+from+BPC3&amp;author=S+Lindstrom&amp;volume=6&amp;publication_year=2011&amp;pages=e17142&amp;pmid=21390317&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation">Mackay TF. Quantitative trait loci in Drosophila. <span><span class="ref-journal">Nat. Rev. Genet. </span>2001;<span class="ref-vol">2</span>:11–20.</span> [<a href="/pubmed/11253063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Quantitative+trait+loci+in+Drosophila&amp;author=TF+Mackay&amp;volume=2&amp;publication_year=2001&amp;pages=11-20&amp;pmid=11253063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation">Williams SM, et al.  The use of animal models in the study of complex disease: all else is never equal or why do so many human studies fail to replicate animal findings? <span><span class="ref-journal">Bioessays. </span>2004;<span class="ref-vol">26</span>:170–179.</span> [<a href="/pubmed/14745835" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioessays&amp;title=The+use+of+animal+models+in+the+study+of+complex+disease:+all+else+is+never+equal+or+why+do+so+many+human+studies+fail+to+replicate+animal+findings?&amp;author=SM+Williams&amp;volume=26&amp;publication_year=2004&amp;pages=170-179&amp;pmid=14745835&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation">Routman EJC, et al.  Gene effects on a quantitative trait: two-locus epistatic effects measured at microsatellite markers and at estimated QTL. <span><span class="ref-journal">Evolution. </span>1997;<span class="ref-vol">51</span>:1654–1662.</span> [<a href="/pubmed/28568634" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Gene+effects+on+a+quantitative+trait:+two-locus+epistatic+effects+measured+at+microsatellite+markers+and+at+estimated+QTL&amp;author=EJC+Routman&amp;volume=51&amp;publication_year=1997&amp;pages=1654-1662&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation">Ciampa J, et al.  Large-scale exploration of gene-gene interactions in prostate cancer using a multistage genome-wide association study. <span><span class="ref-journal">Cancer Res. </span>2011;<span class="ref-vol">71</span>:3287–3295.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3085580/">PMC free article</a>]</span> [<a href="/pubmed/21372204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Large-scale+exploration+of+gene-gene+interactions+in+prostate+cancer+using+a+multistage+genome-wide+association+study&amp;author=J+Ciampa&amp;volume=71&amp;publication_year=2011&amp;pages=3287-3295&amp;pmid=21372204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation">Xu J, et al.  Inherited genetic variant predisposes to aggressive but not indolent prostate cancer. <span><span class="ref-journal">Proc. Natl Acad. Sci. USA. </span>2010;<span class="ref-vol">107</span>:2136–2140.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2836698/">PMC free article</a>]</span> [<a href="/pubmed/20080650" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl+Acad.+Sci.+USA&amp;title=Inherited+genetic+variant+predisposes+to+aggressive+but+not+indolent+prostate+cancer&amp;author=J+Xu&amp;volume=107&amp;publication_year=2010&amp;pages=2136-2140&amp;pmid=20080650&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation">Marchini J, et al.  A new multipoint method for genome-wide association studies by imputation of genotypes. <span><span class="ref-journal">Nat. Genet. </span>2007;<span class="ref-vol">39</span>:906–913.</span> [<a href="/pubmed/17572673" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=A+new+multipoint+method+for+genome-wide+association+studies+by+imputation+of+genotypes&amp;author=J+Marchini&amp;volume=39&amp;publication_year=2007&amp;pages=906-913&amp;pmid=17572673&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am. J. Hum. Genet. </span>2007;<span class="ref-vol">81</span>:559–575.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat. Genet. </span>2006;<span class="ref-vol">38</span>:904–909.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-909&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation">Manning AK, et al.  Meta-analysis of gene-environment interaction: joint estimation of SNP and SNP x environment regression coefficients. <span><span class="ref-journal">Genet. Epidemiol. </span>2011;<span class="ref-vol">35</span>:11–18.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3312394/">PMC free article</a>]</span> [<a href="/pubmed/21181894" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Meta-analysis+of+gene-environment+interaction:+joint+estimation+of+SNP+and+SNP+x+environment+regression+coefficients&amp;author=AK+Manning&amp;volume=35&amp;publication_year=2011&amp;pages=11-18&amp;pmid=21181894&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation">Cordell HJ. Detecting gene-gene interactions that underlie human diseases. <span><span class="ref-journal">Nat. Rev. Genet. </span>2009;<span class="ref-vol">10</span>:392–404.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2872761/">PMC free article</a>]</span> [<a href="/pubmed/19434077" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Detecting+gene-gene+interactions+that+underlie+human+diseases&amp;author=HJ+Cordell&amp;volume=10&amp;publication_year=2009&amp;pages=392-404&amp;pmid=19434077&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation">Janssen JW, et al.  Concurrent activation of a novel putative transforming gene, myeov, and cyclin D1 in a subset of multiple myeloma cell lines with t(11;14)(q13;q32) <span><span class="ref-journal">Blood. </span>2000;<span class="ref-vol">95</span>:2691–2698.</span> [<a href="/pubmed/10753852" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Concurrent+activation+of+a+novel+putative+transforming+gene,+myeov,+and+cyclin+D1+in+a+subset+of+multiple+myeloma+cell+lines+with+t(11;14)(q13;q32)&amp;author=JW+Janssen&amp;volume=95&amp;publication_year=2000&amp;pages=2691-2698&amp;pmid=10753852&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation">Janssen JW, et al.  MYEOV, a gene at 11q13, is coamplified with CCND1, but epigenetically inactivated in a subset of esophageal squamous cell carcinomas. <span><span class="ref-journal">J. Hum. Genet. </span>2002;<span class="ref-vol">47</span>:460–464.</span> [<a href="/pubmed/12202983" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hum.+Genet.&amp;title=MYEOV,+a+gene+at+11q13,+is+coamplified+with+CCND1,+but+epigenetically+inactivated+in+a+subset+of+esophageal+squamous+cell+carcinomas&amp;author=JW+Janssen&amp;volume=47&amp;publication_year=2002&amp;pages=460-464&amp;pmid=12202983&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation">Janssen JW, et al.  MYEOV: a candidate gene for DNA amplification events occurring centromeric to CCND1 in breast cancer. <span><span class="ref-journal">Int. J. Cancer. </span>2002;<span class="ref-vol">102</span>:608–614.</span> [<a href="/pubmed/12448002" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=MYEOV:+a+candidate+gene+for+DNA+amplification+events+occurring+centromeric+to+CCND1+in+breast+cancer&amp;author=JW+Janssen&amp;volume=102&amp;publication_year=2002&amp;pages=608-614&amp;pmid=12448002&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation">Specht K, et al.  Different mechanisms of cyclin D1 overexpression in multiple myeloma revealed by fluorescence <em>in situ</em> hybridization and quantitative analysis of mRNA levels. <span><span class="ref-journal">Blood. </span>2004;<span class="ref-vol">104</span>:1120–1126.</span> [<a href="/pubmed/15090460" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Different+mechanisms+of+cyclin+D1+overexpression+in+multiple+myeloma+revealed+by+fluorescence+in+situ+hybridization+and+quantitative+analysis+of+mRNA+levels&amp;author=K+Specht&amp;volume=104&amp;publication_year=2004&amp;pages=1120-1126&amp;pmid=15090460&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation">Moss AC, et al.  ETV4 and Myeov knockdown impairs colon cancer cell line proliferation and invasion. <span><span class="ref-journal">Biochem. Biophys. Res. Commun. </span>2006;<span class="ref-vol">345</span>:216–221.</span> [<a href="/pubmed/16678123" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+Biophys.+Res.+Commun.&amp;title=ETV4+and+Myeov+knockdown+impairs+colon+cancer+cell+line+proliferation+and+invasion&amp;author=AC+Moss&amp;volume=345&amp;publication_year=2006&amp;pages=216-221&amp;pmid=16678123&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation">Hatch EM, et al.  Cep152 interacts with Plk4 and is required for centriole duplication. <span><span class="ref-journal">J. Cell Biol. </span>2010;<span class="ref-vol">191</span>:721–729.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2983069/">PMC free article</a>]</span> [<a href="/pubmed/21059850" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell+Biol.&amp;title=Cep152+interacts+with+Plk4+and+is+required+for+centriole+duplication&amp;author=EM+Hatch&amp;volume=191&amp;publication_year=2010&amp;pages=721-729&amp;pmid=21059850&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation">Kalay E, et al.  CEP152 is a genome maintenance protein disrupted in Seckel syndrome. <span><span class="ref-journal">Nat. Genet. </span>2011;<span class="ref-vol">43</span>:23–26.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3430850/">PMC free article</a>]</span> [<a href="/pubmed/21131973" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=CEP152+is+a+genome+maintenance+protein+disrupted+in+Seckel+syndrome&amp;author=E+Kalay&amp;volume=43&amp;publication_year=2011&amp;pages=23-26&amp;pmid=21131973&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation">Cui H, et al.  Loss of IGF2 imprinting: a potential marker of colorectal cancer risk. <span><span class="ref-journal">Science. </span>2003;<span class="ref-vol">299</span>:1753–1755.</span> [<a href="/pubmed/12637750" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Loss+of+IGF2+imprinting:+a+potential+marker+of+colorectal+cancer+risk&amp;author=H+Cui&amp;volume=299&amp;publication_year=2003&amp;pages=1753-1755&amp;pmid=12637750&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation">Zhao R, et al.  Loss of imprinting of the insulin-like growth factor II (IGF2) gene in esophageal normal and adenocarcinoma tissues. <span><span class="ref-journal">Carcinogenesis. </span>2009;<span class="ref-vol">30</span>:2117–2122.</span> [<a href="/pubmed/19843644" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Loss+of+imprinting+of+the+insulin-like+growth+factor+II+(IGF2)+gene+in+esophageal+normal+and+adenocarcinoma+tissues&amp;author=R+Zhao&amp;volume=30&amp;publication_year=2009&amp;pages=2117-2122&amp;pmid=19843644&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation">Vorwerk P, et al.  Loss of imprinting of IGF-II gene in children with acute lymphoblastic leukemia. <span><span class="ref-journal">Leuk. Res. </span>2003;<span class="ref-vol">27</span>:807–812.</span> [<a href="/pubmed/12804639" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leuk.+Res.&amp;title=Loss+of+imprinting+of+IGF-II+gene+in+children+with+acute+lymphoblastic+leukemia&amp;author=P+Vorwerk&amp;volume=27&amp;publication_year=2003&amp;pages=807-812&amp;pmid=12804639&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation">Jarrard DF, et al.  Regional loss of imprinting of the insulin-like growth factor II gene occurs in human prostate tissues. <span><span class="ref-journal">Clin. Cancer Res. </span>1995;<span class="ref-vol">1</span>:8.</span> [<a href="/pubmed/9815946" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;title=Regional+loss+of+imprinting+of+the+insulin-like+growth+factor+II+gene+occurs+in+human+prostate+tissues&amp;author=DF+Jarrard&amp;volume=1&amp;publication_year=1995&amp;pages=8&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation">Veronese A, et al.  Oncogenic role of miR-483-3p at the IGF2/483 locus. <span><span class="ref-journal">Cancer Res. </span>2010;<span class="ref-vol">70</span>:3140–3149.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4303586/">PMC free article</a>]</span> [<a href="/pubmed/20388800" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Oncogenic+role+of+miR-483-3p+at+the+IGF2/483+locus&amp;author=A+Veronese&amp;volume=70&amp;publication_year=2010&amp;pages=3140-3149&amp;pmid=20388800&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation">Hao J, et al.  MicroRNA 483-3p suppresses the expression of DPC4/Smad4 in pancreatic cancer. <span><span class="ref-journal">FEBS Lett. </span>2011;<span class="ref-vol">585</span>:207–213.</span> [<a href="/pubmed/21112326" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FEBS+Lett.&amp;title=MicroRNA+483-3p+suppresses+the+expression+of+DPC4/Smad4+in+pancreatic+cancer&amp;author=J+Hao&amp;volume=585&amp;publication_year=2011&amp;pages=207-213&amp;pmid=21112326&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation">Mohr F, et al.  TET genes: new players in DNA demethylation and important determinants for stemness. <span><span class="ref-journal">Exp. Hematol. </span>2011;<span class="ref-vol">39</span>:272–281.</span> [<a href="/pubmed/21168469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Hematol.&amp;title=TET+genes:+new+players+in+DNA+demethylation+and+important+determinants+for+stemness&amp;author=F+Mohr&amp;volume=39&amp;publication_year=2011&amp;pages=272-281&amp;pmid=21168469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">45. <span class="element-citation">Lorsbach RB, et al.  TET1, a member of a novel protein family, is fused to MLL in acute myeloid leukemia containing the t(10;11)(q22;q23) <span><span class="ref-journal">Leukemia. </span>2003;<span class="ref-vol">17</span>:637–641.</span> [<a href="/pubmed/12646957" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=TET1,+a+member+of+a+novel+protein+family,+is+fused+to+MLL+in+acute+myeloid+leukemia+containing+the+t(10;11)(q22;q23)&amp;author=RB+Lorsbach&amp;volume=17&amp;publication_year=2003&amp;pages=637-641&amp;pmid=12646957&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">46. <span class="element-citation">Abdel-Wahab O, et al.  Genetic characterization of TET1, TET2, and TET3 alterations in myeloid malignancies. <span><span class="ref-journal">Blood. </span>2009;<span class="ref-vol">114</span>:144–147.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2710942/">PMC free article</a>]</span> [<a href="/pubmed/19420352" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Genetic+characterization+of+TET1,+TET2,+and+TET3+alterations+in+myeloid+malignancies&amp;author=O+Abdel-Wahab&amp;volume=114&amp;publication_year=2009&amp;pages=144-147&amp;pmid=19420352&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">47. <span class="element-citation">Coopman PJ, et al.  The Syk tyrosine kinase: a new negative regulator in tumor growth and progression. <span><span class="ref-journal">Cancer Lett. </span>2006;<span class="ref-vol">241</span>:159–173.</span> [<a href="/pubmed/16442709" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Lett.&amp;title=The+Syk+tyrosine+kinase:+a+new+negative+regulator+in+tumor+growth+and+progression&amp;author=PJ+Coopman&amp;volume=241&amp;publication_year=2006&amp;pages=159-173&amp;pmid=16442709&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">48. <span class="element-citation">Ma L, et al.  The relationship between methylation of the Syk gene in the promoter region and the genesis of lung cancer. <span><span class="ref-journal">Clin. Lab. </span>2010;<span class="ref-vol">56</span>:407–416.</span> [<a href="/pubmed/21086786" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Lab.&amp;title=The+relationship+between+methylation+of+the+Syk+gene+in+the+promoter+region+and+the+genesis+of+lung+cancer&amp;author=L+Ma&amp;volume=56&amp;publication_year=2010&amp;pages=407-416&amp;pmid=21086786&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">49. <span class="element-citation">Yuan Y, et al.  Hypermethylation leads to silencing of the SYK gene in human breast cancer. <span><span class="ref-journal">Cancer Res. </span>2001;<span class="ref-vol">61</span>:5558–5561.</span> [<a href="/pubmed/11454707" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Hypermethylation+leads+to+silencing+of+the+SYK+gene+in+human+breast+cancer&amp;author=Y+Yuan&amp;volume=61&amp;publication_year=2001&amp;pages=5558-5561&amp;pmid=11454707&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">50. <span class="element-citation">Wang B. Cancer cells exploit the Eph-ephrin system to promote invasion and metastasis: tales of unwitting partners. <span><span class="ref-journal">Sci. Signal. </span>2011;<span class="ref-vol">175</span>:pe28–pe30.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3431919/">PMC free article</a>]</span> [<a href="/pubmed/21632467" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Signal.&amp;title=Cancer+cells+exploit+the+Eph-ephrin+system+to+promote+invasion+and+metastasis:+tales+of+unwitting+partners&amp;author=B+Wang&amp;volume=175&amp;publication_year=2011&amp;pages=pe28-pe30&amp;pmid=21632467&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">51. <span class="element-citation">Shan Z, et al.  Identifying novel homozygous deletions by microsatellite analysis and characterization of tumor suppressor candidate 1 gene, TUSC1, on chromosome 9p in human lung cancer. <span><span class="ref-journal">Oncogene. </span>2004;<span class="ref-vol">23</span>:6612–6620.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3449089/">PMC free article</a>]</span> [<a href="/pubmed/15208665" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Identifying+novel+homozygous+deletions+by+microsatellite+analysis+and+characterization+of+tumor+suppressor+candidate+1+gene,+TUSC1,+on+chromosome+9p+in+human+lung+cancer&amp;author=Z+Shan&amp;volume=23&amp;publication_year=2004&amp;pages=6612-6620&amp;pmid=15208665&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3291863&amp;issue-id=206138&amp;journal-id=724&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm139862509194896title">Abstract</a></li><li><a href="#__sec1title">Introduction</a></li><li><a href="#__sec2title">Materials and methods</a></li><li><a href="#__sec7title">Results</a></li><li><a href="#__sec8title">Discussion</a></li><li><a href="#__sec9title">Supplementary material</a></li><li><a href="#__sec10title">Funding</a></li><li><a href="#__sec11title">Supplementary Material</a></li><li><a href="#idm139862503355216title">Acknowledgments</a></li><li><a href="#idm139862578646336title">Glossary</a></li><li><a href="#idm139862508861504title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Carcinogenesis</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3291863/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3291863/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3291863/pdf/bgr316.pdf">PDF (109K)</a></li> | <li><a href="#" data-citationid="PMC3291863" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3291863%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3291863%2F&amp;text=A%20genome-wide%20search%20for%20loci%20interacting%20with%20known%20prostate%20cancer%20risk-associated%20genetic%20variants" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3291863%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22219177" data-aiid="3291863" data-aid="3291863" data-iid="206138" data-domainid="724" data-domain="carcin" data-accid="PMC3291863" data-md5="09571442ca75594abe47f3a9941472fb"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:22:41-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal104&amp;ncbi_phid=CE8B6441DD52E0A1000000000A770722&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3291863%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=carcin&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3291863/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal104 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size2, monospace;"></div></div></body></html>