<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="nihpa"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC3684025/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-wide Association Study of N370S Homozygous Gaucher Disease Reveals the Candidacy of CLN8 gene as a Genetic Modifier Contributing to Extreme Phenotypic Variation</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684025/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American journal of hematology"><meta name="citation_title" content="Genome-wide Association Study of N370S Homozygous Gaucher Disease Reveals the Candidacy of CLN8 gene as a Genetic Modifier Contributing to Extreme Phenotypic Variation"><meta name="citation_authors" content="Clarence K. Zhang, Philip B. Stein, Jun Liu, Gregory M. Pastores, Zuoheng Wang, Ruhua Yang, Judy H. Cho, Peter K. Gregersen, Johannes M. F. G. Aerts, Hongyu Zhao, Pramod K. Mistry"><meta name="citation_date" content="April 2012"><meta name="citation_issue" content="4"><meta name="citation_volume" content="87"><meta name="citation_firstpage" content="377"><meta name="citation_doi" content="10.1002/ajh.23118"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC3684025/?report=abstract"><meta name="citation_pmid" content="22388998"><meta name="DC.Title" content="Genome-wide Association Study of N370S Homozygous Gaucher Disease Reveals the Candidacy of CLN8 gene as a Genetic Modifier Contributing to Extreme Phenotypic Variation"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="NIH Public Access"><meta name="DC.Contributor" content="Clarence K. Zhang"><meta name="DC.Contributor" content="Philip B. Stein"><meta name="DC.Contributor" content="Jun Liu"><meta name="DC.Contributor" content="Gregory M. Pastores"><meta name="DC.Contributor" content="Zuoheng Wang"><meta name="DC.Contributor" content="Ruhua Yang"><meta name="DC.Contributor" content="Judy H. Cho"><meta name="DC.Contributor" content="Peter K. Gregersen"><meta name="DC.Contributor" content="Johannes M. F. G. Aerts"><meta name="DC.Contributor" content="Hongyu Zhao"><meta name="DC.Contributor" content="Pramod K. Mistry"><meta name="DC.Date" content="2012 Apr"><meta name="DC.Identifier" content="10.1002/ajh.23118"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-wide Association Study of N370S Homozygous Gaucher Disease Reveals the Candidacy of CLN8 gene as a Genetic Modifier Contributing to Extreme Phenotypic Variation"><meta property="og:type" content="article"><meta property="og:description" content="Mutations in GBA1 gene result in defective acid β-glucosidase and the complex phenotype of Gaucher disease (GD) related to the accumulation of glucosylceramide-laden macrophages. The phenotype is highly variable even among patients harboring identical ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3684025/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hhspa.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC3684025/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC3684025/pdf/nihms467231.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A3183DD53C4D10000000003A80285.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathEvents.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathZoom.js?rev=2.6.1"></script><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/extensions/MathMenu.js?rev=2.6.1"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3684025%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a></li><li class="accid">PMC3684025</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hhspa.png" alt="Logo of nihpa" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,57,255,75" alt="About Author manuscripts" title="About Author manuscripts" href="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html" ref="https://www.ncbi.nlm.nih.gov/pmc/about/authorms.html"><area shape="rect" coords="256,57,500,75" alt="Submit a manuscript" title="Submit a manuscript" href="https://www.nihms.nih.gov/" ref="reftype=publisher&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"><area shape="rect" coords="0,0,499,74" alt="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" title="HHS Public Access; Author Manuscript; Accepted for publication in peer reviewed journal;" href="https://www.ncbi.nlm.nih.gov/pmc/about/public-access/" ref="reftype=publisher&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" target="pmc_ext"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Am J Hematol</a></span>. Author manuscript; available in PMC 2013 Jun 17.</div><div></div><div>Published in final edited form as:</div><div style="margin-left:1em"><div class="fm-vol-iss-date"><span class="cit">Am J Hematol. 2012 Apr; 87(4): 377–383. </span></div><span class="fm-vol-iss-date">Published online 2012 Mar 3. </span> <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1002%2Fajh.23118" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1002/ajh.23118</a></span></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC3684025</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS467231</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/22388998">22388998</a></div></div></div></div><h1 class="content-title">Genome-wide Association Study of N370S Homozygous Gaucher Disease Reveals the Candidacy of <em>CLN8</em> gene as a Genetic Modifier Contributing to Extreme Phenotypic Variation</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Zhang%20CK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998" class="affpopup" co-rid="_co_idm140408529342512" co-class="co-affbox">Clarence K. Zhang</a>,<sup>1,</sup><sup>*</sup> <a href="/pubmed/?term=Stein%20PB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998" class="affpopup" co-rid="_co_idm140408489377024" co-class="co-affbox">Philip B. Stein</a>,<sup>2,</sup><sup>*</sup> <a href="/pubmed/?term=Liu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998" class="affpopup" co-rid="_co_idm140408494544128" co-class="co-affbox">Jun Liu</a>,<sup>2</sup> <a href="/pubmed/?term=Pastores%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998" class="affpopup" co-rid="_co_idm140408494541792" co-class="co-affbox">Gregory M. Pastores</a>,<sup>3</sup> <a href="/pubmed/?term=Wang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998" class="affpopup" co-rid="_co_idm140408494539408" co-class="co-affbox">Zuoheng Wang</a>,<sup>4</sup> <a href="/pubmed/?term=Yang%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998" class="affpopup" co-rid="_co_idm140408494537072" co-class="co-affbox">Ruhua Yang</a>,<sup>2</sup> <a href="/pubmed/?term=Cho%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998" class="affpopup" co-rid="_co_idm140408494534784" co-class="co-affbox">Judy H. Cho</a>,<sup>5</sup> <a href="/pubmed/?term=Gregersen%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998" class="affpopup" co-rid="_co_idm140408529849888" co-class="co-affbox">Peter K. Gregersen</a>,<sup>6</sup> <a href="/pubmed/?term=Aerts%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998" class="affpopup" co-rid="_co_idm140408529847552" co-class="co-affbox">Johannes M. F. G. Aerts</a>,<sup>7</sup> <a href="/pubmed/?term=Zhao%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998" class="affpopup" co-rid="_co_idm140408529845216" co-class="co-affbox">Hongyu Zhao</a>,<sup>1,</sup><sup>4</sup> and  <a href="/pubmed/?term=Mistry%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998" class="affpopup" co-rid="_co_idm140408529842144" co-class="co-affbox">Pramod K. Mistry</a><sup>2,</sup><sup>5</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140408529342512"><h3 class="no_margin">Clarence K. Zhang</h3><p><sup>1</sup>Keck Biotechnology Laboratory Biostatistics Resource, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</p><div>Find articles by <a href="/pubmed/?term=Zhang%20CK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998">Clarence K. Zhang</a></div></div><div id="_co_idm140408489377024"><h3 class="no_margin">Philip B. Stein</h3><p><sup>2</sup>Pediatrics, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</p><div>Find articles by <a href="/pubmed/?term=Stein%20PB%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998">Philip B. Stein</a></div></div><div id="_co_idm140408494544128"><h3 class="no_margin">Jun Liu</h3><p><sup>2</sup>Pediatrics, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</p><div>Find articles by <a href="/pubmed/?term=Liu%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998">Jun Liu</a></div></div><div id="_co_idm140408494541792"><h3 class="no_margin">Gregory M. Pastores</h3><p><sup>3</sup>NYU Department of Neurology, Neurogenetics Unit, 403 East 34th Street New York, NY 10016</p><div>Find articles by <a href="/pubmed/?term=Pastores%20GM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998">Gregory M. Pastores</a></div></div><div id="_co_idm140408494539408"><h3 class="no_margin">Zuoheng Wang</h3><p><sup>4</sup>Epidemiology and Public Health, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</p><div>Find articles by <a href="/pubmed/?term=Wang%20Z%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998">Zuoheng Wang</a></div></div><div id="_co_idm140408494537072"><h3 class="no_margin">Ruhua Yang</h3><p><sup>2</sup>Pediatrics, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</p><div>Find articles by <a href="/pubmed/?term=Yang%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998">Ruhua Yang</a></div></div><div id="_co_idm140408494534784"><h3 class="no_margin">Judy H. Cho</h3><p><sup>5</sup>Department of Internal Medicine, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</p><div>Find articles by <a href="/pubmed/?term=Cho%20JH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998">Judy H. Cho</a></div></div><div id="_co_idm140408529849888"><h3 class="no_margin">Peter K. Gregersen</h3><p><sup>6</sup>North Shore University Hospital, Center for Genomics and Human Genetics, Community Drive, Manhasset, NY 11030</p><div>Find articles by <a href="/pubmed/?term=Gregersen%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998">Peter K. Gregersen</a></div></div><div id="_co_idm140408529847552"><h3 class="no_margin">Johannes M. F. G. Aerts</h3><p><sup>7</sup>University of Amsterdam, Department of Biochemistry, Academic Medical Centre, Amsterdam, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Aerts%20JM%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998">Johannes M. F. G. Aerts</a></div></div><div id="_co_idm140408529845216"><h3 class="no_margin">Hongyu Zhao</h3><p><sup>1</sup>Keck Biotechnology Laboratory Biostatistics Resource, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</p><p><sup>4</sup>Epidemiology and Public Health, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</p><div>Find articles by <a href="/pubmed/?term=Zhao%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998">Hongyu Zhao</a></div></div><div id="_co_idm140408529842144"><h3 class="no_margin">Pramod K. Mistry</h3><p><sup>2</sup>Pediatrics, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</p><p><sup>5</sup>Department of Internal Medicine, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</p><div>Find articles by <a href="/pubmed/?term=Mistry%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=22388998">Pramod K. Mistry</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140408493850976_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140408493850976_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140408493850976_ai" style="display:none"><div class="fm-affl" id="A1" lang="en"><sup>1</sup>Keck Biotechnology Laboratory Biostatistics Resource, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</div><div class="fm-affl" id="A2" lang="en"><sup>2</sup>Pediatrics, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</div><div class="fm-affl" id="A3" lang="en"><sup>3</sup>NYU Department of Neurology, Neurogenetics Unit, 403 East 34th Street New York, NY 10016</div><div class="fm-affl" id="A4" lang="en"><sup>4</sup>Epidemiology and Public Health, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</div><div class="fm-affl" id="A5" lang="en"><sup>5</sup>Department of Internal Medicine, Yale University School of Medicine 333 Cedar Street, LMP 4093, New Haven, Connecticut. 06562</div><div class="fm-affl" id="A6" lang="en"><sup>6</sup>North Shore University Hospital, Center for Genomics and Human Genetics, Community Drive, Manhasset, NY 11030</div><div class="fm-affl" id="A7" lang="en"><sup>7</sup>University of Amsterdam, Department of Biochemistry, Academic Medical Centre, Amsterdam, the Netherlands</div><div><span class="contrib-email" id="idm140408529342512">Clarence K. Zhang: <a href="mailto:dev@null" data-email="ude.elay@gnahz.k.ecneralc" class="oemail">ude.elay@gnahz.k.ecneralc</a>; </span> <span class="contrib-email" id="idm140408489377024">Philip B. Stein: <a href="mailto:dev@null" data-email="ude.elay@niets.pilihp" class="oemail">ude.elay@niets.pilihp</a>; </span> <span class="contrib-email" id="idm140408494544128">Jun Liu: <a href="mailto:dev@null" data-email="ude.elay@uil.nuj" class="oemail">ude.elay@uil.nuj</a>; </span> <span class="contrib-email" id="idm140408494541792">Gregory M. Pastores: <a href="mailto:dev@null" data-email="gro.cmuyn@serotsap.yrogerG" class="oemail">gro.cmuyn@serotsap.yrogerG</a>; </span> <span class="contrib-email" id="idm140408494539408">Zuoheng Wang: <a href="mailto:dev@null" data-email="udelay@gnaw.gnehouz" class="oemail">udelay@gnaw.gnehouz</a>; </span> <span class="contrib-email" id="idm140408494537072">Ruhua Yang: <a href="mailto:dev@null" data-email="ude.elay@gnay.auhur" class="oemail">ude.elay@gnay.auhur</a>; </span> <span class="contrib-email" id="idm140408494534784">Judy H. Cho: <a href="mailto:dev@null" data-email="ude.elay@ohc.yduj" class="oemail">ude.elay@ohc.yduj</a>; </span> <span class="contrib-email" id="idm140408529849888">Peter K. Gregersen: <a href="mailto:dev@null" data-email="ude.shsn@gretep" class="oemail">ude.shsn@gretep</a>; </span> <span class="contrib-email" id="idm140408529847552">Johannes M. F. G. Aerts: <a href="mailto:dev@null" data-email="ln.avu.cma@strea.m.j" class="oemail">ln.avu.cma@strea.m.j</a>; </span> <span class="contrib-email" id="idm140408529845216">Hongyu Zhao: <a href="mailto:dev@null" data-email="ude.elay@oahz.uygnoh" class="oemail">ude.elay@oahz.uygnoh</a></span> </div><div id="FN1">Corresponding author: Pramod K. Mistry, MD, PhD, FRCP, Professor of Pediatrics and Internal Medicine, Chief, Pediatric Gastroenterology and Hepatology, Yale University School of Medicine, PO Box 208064, 333 Cedar Street; LMP 4093,New Haven, CT 06520, Telephone: 203-785-4649, Fax: 203-785-3365, <a href="mailto:dev@null" data-email="ude.elay@yrtsim.domarp" class="oemail">ude.elay@yrtsim.domarp</a></div><div id="FN2"><sup>*</sup>These authors contributed equally</div></div><div class="fm-article-notes fm-panel half_rhythm"></div><div class="permissions fm-panel half_rhythm hide" id="idm140408493850976_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright notice</a> </div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>See other articles in PMC that <a href="/pmc/articles/PMC3684025/citedby/">cite</a> the published article.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="2" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Fig S1-S3.</strong></div><p><strong>Supplementary Figure 1 Timelines of N370S Homozygous GD patients with severe disease compared to patient with mild disease.</strong> The presentation in Gaucher disease is very variable. This timeline shows significant events in the natural history of two N370S homozygous patients enrolled in the current GWAS study. These contrasting patterns illustrate mild vs. severe disease in our patient population. Patient A with severe disease had early onset osteomyelitis/AVN. He had severe splenomegaly and hepatomegaly at an early age and went on to have a splenic rupture and splenectomy. Patient B with mild disease was diagnosed based on maternal diagnosis and has been asymptomatic. She has virtually no disease manifestations and does not require ERT at 41 years of age.</p><p><strong>Supplementary Figure 2:</strong> QTL association results in CLN8 region. This screenshot from the eQTL browser displays eQTLs near the CLN8 region. The chromosome 8 and CLN8/C8orf61 genes are displayed. And each SNP associated with CLN8 or other genes are displayed below, together with related references.</p><p><strong>Supplementary Figure 3: Tissue specific expression profile for CLN8</strong> This figure displays the human tissue specific expression profile from U133plus2 Affymetrics microarray (CLN8, probe ID 219340_s_at). The expression value is a z-score derived from the barcode function of the R package "frma". A z-score &gt; 5 suggests that the gene is expressed in that tissue. Color codes highlight neuronal tissues (blue), immune cells/organs (red), viscera (purple) and miscellaneous (orange).</p><div><a href="/pmc/articles/PMC3684025/bin/NIHMS467231-supplement-Supp_Fig_S1-S3.pdf" data-ga-action="click_feat_suppl">NIHMS467231-supplement-Supp_Fig_S1-S3.pdf</a><span style="color:gray"> (287K)</span></div><div class="small guid">GUID:&nbsp;5C97455F-3D34-49D4-9513-09F83756FC77</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supp Table S1-S4.</strong></div><p><strong>Supplementary Table I-IV Table S I Pairwise IBD coefficients and inferred family relationships.</strong> A. Top 50 sample pairs with largest kinship coefficients. Columns ID1 and 2 are IDs from two individuals, D7, D8 and D9 are the last three IBD coefficients inferred assuming no inbreeding, KinshipCoef is the kinship coefficients calculated from 9 IBD coefficients. B. Reconstructed samples potentially within a family, 17 families were inferred.</p><p><strong>Table S II Top associated SNPs for the three traits, and their nearest gene neighborhood and distance to gene start and gene end.</strong> SNPs are listed in the order of their genomic location. The Symbol column shows the nearest gene neighborhood. The distance to the start of the gene (diststart) is calculated by the gene start bp minus the snp location; the distance to the end of the gene (distend) is calculated by the gene end bp minus the snp location. The chromosome location is from NCBI genome assembly 36.3. A. Overall top SNPs. B. Visceral top SNPs. C. Bone top SNPs</p><p><strong>Table S III Association of SNPs in CLN8 region with overall disease severity.</strong> In the annotation column, flanking refers to the flanking region of CLN8. The eight strongly associated SNPs within the region are colored in red.</p><p><strong>Table S IV. The genotype and allelic configuration for SNP rs11986414 in CLN8 region, allelic pairwise odds ratio and genotypic odds ratio.</strong> The genotype configuration for the top SNP rs11986414 associated with CLN8 with a p value of 1.26 × 10<sup>-6</sup> for overall disease severity is depicted. Allele A is the risk allele associated with more severe phenotype. The A allele is observed more frequently in moderate and severe disease categories compared to the mild category.</p><p>A. genotype and allelic configuration for SNP rs11986414.</p><p>B. Allelic pairwise odds ratio.</p><p>C. Genotypic odds ratio for combined severe and moderate phenotype vs mild phenotype.</p><div><a href="/pmc/articles/PMC3684025/bin/NIHMS467231-supplement-Supp_Table_S1-S4.doc" data-ga-action="click_feat_suppl">NIHMS467231-supplement-Supp_Table_S1-S4.doc</a><span style="color:gray"> (225K)</span></div><div class="small guid">GUID:&nbsp;CE274F74-AB2E-41B7-A9CE-349A23E9DB03</div></div></dd></dl></div><div id="idm140408489726608" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140408489726608title">Abstract</h2><!--article-meta--><div><p id="P1" class="p p-first-last">Mutations in <em>GBA1</em> gene result in defective acid β-glucosidase and the complex phenotype of Gaucher disease (GD) related to the accumulation of glucosylceramide-laden macrophages. The phenotype is highly variable even among patients harboring identical <em>GBA1</em> mutations. We hypothesized that modifier gene(s) underlie phenotypic diversity in GD and performed a GWAS study in Ashkenazi Jewish patients with type 1 GD (GD1), homozygous for N370S mutation. Patients were assigned to mild, moderate or severe disease category using composite disease severity scoring systems. Whole-genome genotyping for &gt;500,000 SNPs was performed to search for associations using OQLS algorithm in 139 eligible patients. Several SNPs in linkage disequilibrium within the <em>CLN8</em> gene locus were associated with the GD1 severity: SNP rs11986414 was associated with GD1 severity at p value 1.26 × 10<sup>−6</sup>. Compared to mild disease, risk allele A at rs11986414 conferred an odds ratio of 3.72 for moderate/severe disease. Loss of function mutations in <em>CLN8</em> causes neuronal ceroid-lipofuscinosis but our results indicate that its <em>increased</em> expression may protect against severe GD1. In cultured skin fibroblasts, the relative expression of CLN8 was higher in mild GD compared to severely affected patients in whom CLN8 risk alleles were over-represented. In an <em>in vitro</em> cell model of GD, CLN8 expression was increased which was further enhanced in the presence of bioactive substrate, glucosylsphingosine. Taken together, CLN8 is a candidate modifier gene for GD1 that may function as a protective sphingolipid sensor and/or in glycosphingolipid trafficking. Future studies should explore the role of CLN8 in pathophysiology of GD.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">Gaucher disease, GWAS, genotype/phenotype correlations, phenotypic diversity, modifier genes, CLN8, N370S, GBA mutations</span></div></div><div id="S1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S1title">Introduction</h2><p id="P2" class="p p-first">In Gaucher disease (GD), the defective activity of acid β-glucosidase due to mutations in the <em>GBA1</em> gene leads to progressive accumulation of the substrate, glucocerebroside in the lysosomes of mononuclear phagocytes [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713965">1</a>]. The result is widespread accumulation of glucocerebroside-laden macrophages, systemic macrophage activation and a complex phenotype involving the liver, the spleen, the bones and the bone marrow [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713994">1</a>]. Non-neuronopathic Type 1 Gaucher disease (GD1) represents the most common form accounting for &gt;90% of currently known patients. Acute and chronic neuronopathic forms, type 2 and type 3 GD respectively, manifest neurodegenerative disease in addition to the systemic features [<a href="#R2" rid="R2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714010">2</a>]. Among lysosomal storage diseases, GD is the most common, with overall frequency of ∼ 1 in 40,000 individuals but in the Ashkenazi Jews, as many as ∼ 1 in 500 individuals are affected [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713973">1</a>]. GD1 has served as an excellent prototype for genotype/phenotype delineation and therapeutic innovation for lysosomal storage diseases [<a href="#R3" rid="R3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714012">3</a>]. However, the pathways from lysosomal accumulation of glucocerebrosides to clinical manifestations of GD are not known and this has hindered the delineation of basis of phenotypic variation [<a href="#R4" rid="R4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713953">4</a>-<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713964">6</a>].</p><p id="P3">The gene encoding acid β-glucosidase (<em>GBA1</em>) and its highly homologous pseudogene (96% identity) are closely linked on human chromosome 1q21 [<a href="#R7" rid="R7" class=" bibr popnode">7</a>]. More than 200 mutations in the active <em>GBA1</em> gene have been catalogued, but only 4 account for &gt;90% of disease alleles in patients of Ashkenazi Jewish ancestry and ∼ 50% of disease alleles in non-Jewish patients [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713962">1</a>, <a href="#R7" rid="R7" class=" bibr popnode">7</a>]. In studies from North America and Europe, N370S comprise up to 70% of all disease alleles; N370S mutation occurs in homozygous form typically in patients of Ashkenazi Jewish ancestry and it is estimated ∼1 in 15 individuals in this population are carriers of this mutation [<a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714014">8</a>]. A disease mutation comprising 55bp deletion in exon 9 spans the site of N370S mutation, which results in erroneous genotype assignment (N370S homozygous in N370S/55bp deletion) when allele-specific PCR assays are employed [<a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713986">9</a>].</p><p id="P4">There is some genotype/phenotype correlation so that the presence of at least one N370S allele absolutely precludes the development of neuronopathic disease [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713980">1</a>]. As a group, N370S homozygous patients tend to have less severe disease compared to compound heterozygous N370S/L444P or N370S/84GG [<a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R8" rid="R8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713955">8</a>, <a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713961">10</a>, <a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713976">11</a>]. However, within each <em>GBA1</em> genotype category and even among affected siblings there is extreme variation of disease severity [<a href="#R11" rid="R11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714008">11</a>, <a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713966">15</a>,<a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713975">16</a>]. In fact, it has been suggested that homozygosity for N370S mutation results in such mild disease that a significant number never come to medical attention; yet others develop incapacitating disease at young age [<a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R12" rid="R12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713987">12</a>].</p><p id="P5">The basis of such extraordinary phenotypic variation in this single gene/single enzyme disorder and widely regarded as involving a single cell type (the mononuclear phagocyte), is not understood. Studies have demonstrated lack of correlation between the residual acid β-glucosidase activity and disease severity, however these studies have been questioned due to failure to measure <em>in situ</em> lysosomal enzyme activity in intact cells [<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713985">13</a>, <a href="#R14" rid="R14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714011">14</a>]. Sib-pairs with GD have been reported to exhibit different pattern of disease manifestations as well differ in overall disease severity [<a href="#R15" rid="R15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713978">15</a>, <a href="#R16" rid="R16" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713998">16</a>]. Therefore, modifier gene(s) have been proposed to underlie phenotypic variation [<a href="#R13" rid="R13" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714001">13</a>]. It should be kept in mind, that two pairs of identical twins have been reported with divergent phenotypes, suggesting that the basis of phenotypic diversity may be more complex [<a href="#R17" rid="R17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714020">17</a>,<a href="#R18" rid="R18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713990">18</a>].</p><p id="P6">A number of modulators of residual enzyme activity have been proposed as determinants of disease severity, i.e. lysosomal pH [<a href="#R19" rid="R19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713957">19</a>], saposin [<a href="#R20" rid="R20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713995">20</a>] and intracellular trafficking [<a href="#R21" rid="R21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714003">21</a>]. <em>A priori</em>, the determinants of phenotype variation in GD among patients with the same <em>GBA1</em> genotype could arise from environmental factors or through role of genetic modifier(s). Anecdotal evidence suggests that environmental factors through viral infections, i.e., EBV, may accelerate disease course [<a href="#R22" rid="R22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714002">22</a>]. With respect to potential genetic modifier(s) in the gene dense <em>GBA1</em> locus there are eleven widely distributed polymorphic sites that are in linkage disequilibrium. All N370S chromosomes appear to harbor an identical haplotype, regardless of geographic origin of the patient [<a href="#R7" rid="R7" class=" bibr popnode">7</a>]. Therefore variation in disease severity cannot be explained on the basis of other mutations or polymorphic variations in the <em>GBA1</em> gene locus, the promoter or enhancer sequences—at least in this ethnic group [<a href="#R7" rid="R7" class=" bibr popnode">7</a>, <a href="#R23" rid="R23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713974">23</a>]. Studies have explored SNPs in genes encoding glucosylceramide synthase gene, <em>GBA2</em>, <em>GBA3</em>, vitamin D receptor, IL6 and TNF alpha, but there was no consistent correlation with disease severity [<a href="#R24" rid="R24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713968">24</a>-<a href="#R28" rid="R28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713996">28</a>]. No other plausible candidate genes have emerged based on current understanding of pathophysiology to explain phenotypic variation in GD…</p><p id="P7" class="p p-last">We conducted a hypothesis-generating GWAS through unbiased exploration for association of SNPs with GD1 severity. The study was focused on N370S homozygous GD1 patients of Ashkenazi Jewish ancestry who exhibit extreme phenotypic variation with the rationale of conducting the study in genetically homogenous population to minimize population stratification.</p></div><div id="S2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S2title">Methods</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Determination of Disease Severity</h3><p id="P8" class="p p-first">Patients were enrolled at the Gaucher Disease Treatment Centers of Yale School of Medicine and NYU School of Medicine. The study was approved by the Human Investigation Committee of the Yale University of Medicine and the Institutional Review Board of NYU School of Medicine.</p><p id="P9">Eligible patients represented consecutive N370S homozygous GD patients in whom comprehensive phenotype data prior to enzyme replacement therapy (ERT) was available. Patients had confirmed diagnosis of GD1 based on low (i.e., &lt;10% of normal) acid β-glucosidase activity in peripheral blood leucocytes and <em>GBA1</em> mutation analysis. <em>GBA1</em> genotyping was performed as described previously to ensure absence of 55bp deletion in exon 9 [<a href="#R9" rid="R9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713991">9</a>]. Database included longitudinal phenotype data regarding bone, hematology, visceral involvement, age of onset, severity score index and whether prior splenectomy had been performed. Liver and spleen volumes were determined by volumetric MRI and depicted as fold-enlargement compared to normal (x N); normal liver volume is 2.5% body weight and spleen volume 0.2% body weight as described previously [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713989">10</a>]. Hemoglobin, platelet count, MRI, Hermann Score [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714000">10</a>], DXA scan, Bone Marrow Burden (BMB) score [<a href="#R29" rid="R29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714009">29</a>] were determined prior to ERT. Overall disease severity was determined according to Severity Score Index (SSI) [<a href="#R30" rid="R30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713969">30</a>] and the recently validated DS3 score [<a href="#R31" rid="R31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714015">31</a>]. Age of onset of symptoms, age at splenectomy and age at initiation of ERT were also recorded in the database. We aimed to assess association of SNPs based on ordinal ranking of disease severity, and therefore, the patients were assigned categories of mild, moderate and severe disease (<a href="/pmc/articles/PMC3684025/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>). After initial scoring via DS3 and SSI, patients were ranked by disease severity from most mild to severe. Additionally, experts in Gaucher disease (PKM and GMP) who have followed patients for &gt;10 yrs assigned a global assessment of disease severity of the patient. Using the DS3, SSI score and global assessments, patients were assigned to severity categories of mild, moderate and severe. Illustrative natural history of mildly affected GD1 patient compared to severely affected patient is depicted in <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 1</a>. In addition to variation in overall disease severity, GD is recognized to exhibit different patterns of disease among affected patients, i.e., visceral vs skeletal disease. Therefore, in addition to overall disease severity, we assessed severity of individual disease compartments. A category designation was created for bone disease severity as well as severity of visceral disease. For bone disease, severity was based on pre-ERT Hermann score as described previously [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713982">10</a>] with scores 1-2 classified as mild, 3 as moderate, and 4-5 as severe. Patients were also assessed based on visceral disease with spleen size: 1-5 × N as 0 point, 5-10 × N as 1 point, 10-15 × N as 2, and greater than 15 or splenectomy as 3 points. Severity scoring for hepatomegaly was as follows: less than 1.25 × N as 0 point, 1.25-1.5 × N as 1 point, 1.5-1.75 as 2 point and greater than 1.75 as 3 points; presence of cirrhosis/portal hypertension or hepatopulmonary syndrome as 4 points. Aggregate cores for liver and spleen involvement assigned severity of visceral disease: 0-1 classified as mild; 2-3 points, moderate and ≥4 points, severe.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="T1"><h3>Table I</h3><!--caption a7--><div class="caption"><!--caption a8--><strong>Demographics of Study Population of N370S Homozygous Gaucher disease</strong></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408529571664" class="xtable"><table frame="hsides" rules="rows" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="top" align="left">Severity</th><th rowspan="1" colspan="1" valign="top" align="left">Age</th><th rowspan="1" colspan="1" valign="top" align="left">SSI</th><th rowspan="1" colspan="1" valign="top" align="left">DS3</th><th rowspan="1" colspan="1" valign="top" align="left">Liver Size</th><th rowspan="1" colspan="1" valign="top" align="left">Spleen</th><th rowspan="1" colspan="1" valign="top" align="left">% Female</th><th rowspan="1" colspan="1" valign="top" align="left">% Splenectomy</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Mild</strong></td><td rowspan="1" colspan="1" valign="top" align="left">47.7±1.9 (n=80)</td><td rowspan="1" colspan="1" valign="top" align="left">4.2±0.2 (n=80)</td><td rowspan="1" colspan="1" valign="top" align="left">1.7±0.1 (n=80)</td><td rowspan="1" colspan="1" valign="top" align="left">1.1±0.0 (n=80)</td><td rowspan="1" colspan="1" valign="top" align="left">4.9±0.3 (n=80)</td><td rowspan="1" colspan="1" valign="top" align="left">55%</td><td rowspan="1" colspan="1" valign="top" align="left">0%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Moderate</strong></td><td rowspan="1" colspan="1" valign="top" align="left">51.8±3.3 (n=35)</td><td rowspan="1" colspan="1" valign="top" align="left">7.4±0.3 (n=35)</td><td rowspan="1" colspan="1" valign="top" align="left">4.3±0.1 (n=35)</td><td rowspan="1" colspan="1" valign="top" align="left">1.3±0.1 (n=35)</td><td rowspan="1" colspan="1" valign="top" align="left">7.7±1.1 (n=32)</td><td rowspan="1" colspan="1" valign="top" align="left">55%</td><td rowspan="1" colspan="1" valign="top" align="left">9.6%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Severe</strong></td><td rowspan="1" colspan="1" valign="top" align="left">57.5±3.5 (n=24)</td><td rowspan="1" colspan="1" valign="top" align="left">12.1±0.4 (n=24)</td><td rowspan="1" colspan="1" valign="top" align="left">7.0±0.3 (n=24)</td><td rowspan="1" colspan="1" valign="top" align="left">1.6±0.2 (n=24)</td><td rowspan="1" colspan="1" valign="top" align="left">13.5±2.4 (n=17)</td><td rowspan="1" colspan="1" valign="top" align="left">50%</td><td rowspan="1" colspan="1" valign="top" align="left">29.2%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><strong>Total</strong></td><td rowspan="1" colspan="1" valign="top" align="left">50.4±1.5 (n=139)</td><td rowspan="1" colspan="1" valign="top" align="left">6.4±0.3 (n=139)</td><td rowspan="1" colspan="1" valign="top" align="left">3.2±0.2 (n=139)</td><td rowspan="1" colspan="1" valign="top" align="left">1.2±0.0 (n=139)</td><td rowspan="1" colspan="1" valign="top" align="left">6.8±0.5 (n=129)</td><td rowspan="1" colspan="1" valign="top" align="left">55%</td><td rowspan="1" colspan="1" valign="top" align="left">7.2%</td></tr></tbody></table></div><div id="largeobj_idm140408529571664" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3684025/table/T1/?report=objectonly">Open in a separate window</a></div></div><p id="P10" class="p p-last">Demographic information of the study population is depicted in <a href="/pmc/articles/PMC3684025/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>.</p></div><div id="S4" class="sec"><h3 id="S4title">Genotyping and Patient Exclusion Criteria</h3><p id="P11" class="p p-first">A total of 186 GD1 patients confirmed to be homozygous for N370S mutation were genotyped on Illumina 650 SNP array, which contains 620,901 SNPs. The GenTrain 2.0 calling algorithm implemented in GenomeStudio from Illumina was used to infer the genotype. Quality control measures included: excluding samples with &lt;95% call rate, excluding SNPs with call rates &lt;94%, MAF less than 0.5% and HWE p-value &lt;1×10<sup>−5</sup>. The above quality control measurements were implemented in PLINK package [<a href="#R32" rid="R32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713956">32</a>].</p><p id="P12" class="p p-last">Additionally, we excluded patients under age 18 yrs who were diagnosed through genetic screening because these patients tend to be asymptomatic and exhibit minimal or even no manifestations of the disease. Patients were also excluded if there were insufficient records for determination of SSI and DS3 scores prior to ERT. After these exclusions, we were left with 139 patients for the final analysis of SNP association with disease severity.</p></div><div id="S5" class="sec"><h3 id="S5title">Family Structure and Association Analysis</h3><p id="P13" class="p p-first">The Kinship program [<a href="#R33" rid="R33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714005">33</a>] was used to identify cryptic family relationship directly from genotypes. It estimates the IBD (identical by descent) coefficients and calculates kinship coefficients. These coefficients were used to infer duplicated samples, siblings and parent-offspring relationships.</p><p id="P14">After family relationship among individuals was established, a newly developed method, OQLS was used to compare the allele frequency of SNPs among different phenotype categories while accounting for the ordinal nature of trait information. OQLS (Wang, Z, unpublished) is an extension of the single-marker testing method for a binary trait, MQLS [<a href="#R34" rid="R34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713960">34</a>]. The advantage of OQLS/MQLS is that it is applicable to completely general combinations of family and case-control designs. OQLS/MQLS explicitly incorporates the phenotyped relatives which provide additional information on the enrichment of predisposing variants, and uses the dependence structure to derive optimal weights to increase the power of association analysis.</p><p id="P15" class="p p-last">For top association regions, LD pruning was performed as described in the PLINK package. SNPs were pruned if their r<sup>2</sup> were larger than 0.8. The HaploView [<a href="#R35" rid="R35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714018">35</a>] was used to calculate and visualize the fine scale LD structure and infer haplotype blocks. The SNAP tool [<a href="#R36" rid="R36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713959">36</a>] was used to plot the regional association result and LD values. The eQTL browser (<a href="http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://eqtl.uchicago.edu/cgi-bin/gbrowse/eqtl/</a>) was used to search for SNPs associated with the expression of CLN8 [<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713954">37</a>].</p></div><div id="S6" class="sec"><h3 id="S6title">Skin Fibroblast Cell Lines</h3><p id="P16" class="p p-first-last">Two millimeter punch skin biopsy was obtained from 6 patients (3 with severe GD and 3 with mild GD), and 2 healthy controls using the AcuPunch kit (Acuderm Inc, Lauderdale, FL). A sterile 12mm circular cover glass was place on each of the skin biopsy fragments and 6ml of primary growth culture (DMEM supplemented with 15% fetal calf serum, 1% penicillin/streptomycin/gentamicin, 1% NEAA) was added. A fibroblast cell line was established once the outgrowth of fibroblasts skin biopsy covered 75% - 90% of the dish. The cells were digested with 0.05% typsin/EDTA and split at a ratio of 1:3 to 1:4. Cell lines between 4 - 7 generations were used for the experiments.</p></div><div id="S7" class="sec sec-last"><h3 id="S7title"><em>In vitro</em> studies with fibroblast cell lines</h3><p id="P17" class="p p-first">The inhibitor of glucocerebrosidase, conduritol B epoxide (CBE) was purchased from Calbiochem (La Jolla, CA); Glucocerebroside (GL1) and Glucosylsphingosine (Lyso GL1) were purchased from Matreya (Pleasant Gap, PA). Stock solutions of the drug were prepared according to the manufacturer's instructions. Final concentrations of 50μM of CBE, 40μM of GL1, and 20μM of Lyso GL1 were used to treat the cells as described previously [37a]. CBE alone, CBE with GL1 or Lyso GL1 were separately added to the two normal (healthy) cell lines with the same amount of DMSO as control. After incubation for the specified periods, RNA was extracted for real time PCR.</p><p id="P18">Total RNA from each cell line was prepared using the TRIzol Plus RNA Purification kit (Invitrogen, Carlsbad CA). Briefly, adherent fibroblast cells were scratched and concentrated to an Eppendorf micro-centrifuge tube to which 1ml of TRIzol reagent was added. Following phase separation, the solutions containing RNA were transferred to a spin cartridge. After binding and washing, RNA was eluted into a fresh microtube and RNA concentration was determined.</p><p id="P19">First strand cDNA synthesis was performed using the AffinityScript Multiple Temperature cDNA Synthesis kit (Agilent Tech, Santa Clara, CA) according to the manufacturer's protocols. Oligo dT was used as primer for the reverse transcription. RNA-Oligo dT mixture was preheated at 70°C for 5min before gradual cooling at room temperature for 10min, then followed by addition of the reaction mixture on ice and and at 42°C for 5min and 50°C for 55min.</p><p id="P20">Quantitative real-time PCR was performed using a 7500 Real-time PCR system (Applied Biosystems, Carlebad, CA). Primer and probe mixtures for CLN8 and Actin-β (used as internal reference control) were ordered from the inventory of Applied Biosystems and pre-labeled with the reporter dye 6-carboxyfluorescein (6′-FAM) at the 5′ end and with the quencher dye 6-carboxytetramethylrodamine (TAMRA) at the 3′ end. Quantitative PCR was performed with 2μl of 1:3 diluted cDNA, 10μl of the FastStart universal probe master (Roche, Mannheim, Germany), 0.5μl of 900nM primers, and 0.5μl of 250nM probe in a final volume of 20μl. Each reaction was set up in triplicate and the mean values were used for analysis.</p><p id="P21" class="p p-last">The study was approved by the Human Investigation Committee of the Yale University of Medicine and IRB of NYU School of Medicine.</p></div></div><div id="S8" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S8title">Results</h2><div id="S9" class="sec sec-first"><h3 id="S9title">Genotype Quality Control and Family Structure</h3><p id="P22" class="p p-first">Five out of 186 samples had genotyping call rates 95% and these samples were removed from further analysis. Among the remaining 181 samples, 2 samples had call rates between 96 and 97%. The remaining 179 samples had call rates &gt; 99%. After SNP cleaning using the quality control criteria described in methods section, 554,340 SNPs were left. The total genotyping rate in post-QC samples is 99.8%. There were 540,902 autosomal SNPs retained for final association analysis.</p><p id="P23" class="p p-last">From pair-wise estimated distribution of IBD coefficients and kinship coefficients, we inferred 14 nuclear families. Among these 14 families, 3 were parent-offspring, 10 were sibling pairs and one family had 3 siblings (<a href="#SD2" rid="SD2" class=" supplementary-material">supplementary table IB</a>). We found two categories of kinship coefficients, 31 pairs near 0.25 representing parental-child or sibling relationship (<a href="#SD2" rid="SD2" class=" supplementary-material">supplementary table IA</a>). For a parent offspring pair, they are anticipated to share 1 allele IBD, whereas for a sib pair, the probabilities of sharing 2, 1 or 0 alleles IBD are 0.25, 0.5 and 0.25, respectively. Other sample pairs had kinship coefficients less than 0.05, and therefore less likely to be related with each other.</p></div><div id="S10" class="sec"><h3 id="S10title">Association Results</h3><p id="P24" class="p p-first">After applying the OQLS algorithm to our ordinal valued phenotype in a combined sample of family and unrelated cases/controls, the genetic association of 540,902 autosomal SNPs was identified in the three categories of disease severity (overall disease severity, severity of visceral disease and severity of bone disease). <a href="/pmc/articles/PMC3684025/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a> shows the Manhattan plot for the genome wide association results. No markers achieved genome wide significance based on the stringent Bonferroni correction (p ≤ 9 × 10<sup>−8</sup>). However, the Bonferroni correction assumes all markers behave independently. In practice, a significant proportion of markers do not segregate independently because they are in LD with each other. Therefore, the top signals displayed in the Manhattan plot are worthy of further investigation. Consequently, we analyzed SNPs with lowest P values for associations with disease severity. In addition, cluster of top associations displayed in the Manhattan plot (spikes) were examined in detail.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC3684025/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408490989744" class="figure"><a href="/pmc/articles/PMC3684025/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3684025_nihms467231f1a.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms467231f1a.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3684025/bin/nihms467231f1a.jpg"></a></div><div id="largeobj_idm140408490989744" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3684025/figure/F1/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408490988896" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3684025_nihms467231f1b.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms467231f1b.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3684025/bin/nihms467231f1b.jpg"></a></div><div id="largeobj_idm140408490988896" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3684025/figure/F1/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408529583280" class="figure"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3684025_nihms467231f1c.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms467231f1c.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3684025/bin/nihms467231f1c.jpg"></a></div><div id="largeobj_idm140408529583280" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC3684025/figure/F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC3684025/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Manhattan plots for genome wide association results in three disease severity categories</strong><p id="__p4">The x-axis shows the chromosomes arranged according to their genome location. The y-axis shows the –log p value of the association strength. Each chromosome is in different colors. A. Overall disease severity. B. Severity of visceral disease. C. Severity of skeletal disease.</p></div></div></div><p id="P25"><a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table II</a> lists the top associated markers with each of the three disease severity categories. If not within the known gene region, their nearest gene neighborhoods and distances were displayed. The distance to the start of the gene (disstart) was calculated by the gene start position minus the SNP location; the distance to the end of the gene (distend) was calculated by the gene end position minus the SNP location.</p><p id="P26">When assessing SNP association with overall disease severity, the region harboring <em>CLN8</em> gene achieved the strongest association signal. In addition, the <em>CLN8</em> gene region also demonstrated strong association with the severity of visceral disease (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary table II</a>). Association signals for SNPs within CLN8 and its 100 kb flanking regions with the overall disease severity are depicted in <a href="/pmc/articles/PMC3684025/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2a</span></a>. LD structure within 100 kb flanking region is displayed in <a href="/pmc/articles/PMC3684025/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2b</span></a>. Closer resolution of LD structure and haplotype block within the 25 kb flanking region is displayed in <a href="/pmc/articles/PMC3684025/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2c</span></a>. Several SNPs within or near CLN8 regions are in strong LD with each other and majority of these SNPS are strongly associated with the overall disease severity. <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary table S III</a> shows the detailed association results for each SNP within the CLN8 region for the overall disease category. Interestingly, among these SNPs two intronic SNPs (rs11136424 and rs4875958, pvalues 1.59E<sup>-05</sup> and 2.69E<sup>-05</sup>) are strongly associated with CLN8 expression as depicted in the eQTL browser. SNP rs11136424 is associated with CLN8 expression in lymphoblastoid cell lines and SNP rs4875958 is associated with CLN8 expression in liver tissue (<a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 2</a>). Both eQTL data are derived from people of European ancestry [<a href="#R38" rid="R38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714016">37</a>].</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC3684025/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408529578608" class="figure"><a href="/pmc/articles/PMC3684025/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3684025_nihms467231f2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms467231f2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3684025/bin/nihms467231f2.jpg"></a></div><div id="largeobj_idm140408529578608" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3684025/figure/F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC3684025/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Association plots for the CLN8 and its 100kb flanking region with the overall disease severity</strong><p id="__p5">A. Regional Manhattan plot and LD structure with the top SNP rs11986414. The x-axis shows the chromosome position, the left y-axis shows the –log (p) for the association strength, and the right y-axis shows the recombination rate calculated from Hapmap3 CEU reference population. The color for each marker represents the LD r<sup>2</sup> value with the topmost snp rs11986414. B. LD structure. The pair-wise LD pattern is shown for all markers within the 100kb flanking region. The chromosomal location is shown above, the red, blue and white colors represent different LD measured by D′. The blocks bordered with thick black box represent haplotype blocks inferred. C. LD structure and haplotype block for the 25kb flanking region. The value inside each cell displays the LD measurement r<sup>2</sup>, the color scheme is based on D′. The inferred haplotype block and haplotype frequency are shown too.</p></div></div></div><p id="P27">Among SNPs within the CLN8 region, the topmost SNP rs11986414 achieved a p value of 1.26 × 10<sup>−6</sup>. The two alleles of this SNP are A and G; A is the risk allele associated with more severe disease. As shown in <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary Table IVa</a>, there was over-representation of A allele in patients with moderate and severe disease compared to the patients with mild disease. Compared to patients with mild disease, A allele conferred odds ratio (OR) of 3.62 (95% CI 1.96 - 6.70) for moderate disease and 3.85 (95% CI 1.96 - 7.55) for severe disease (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S IVb</a>). The OR between moderate and severe disease categories is 1.06 (95% CI 0.51 - 2.22). Therefore genetic risk conferred by rs11986414 appears to be similar for both moderate and severe disease. The OR with combined severe and moderate vs mild disease category is 3.72 (95% CI 2.19 - 6.31). To understand the genetic inheritance model, the genotypic odds ratios for combined severe and moderate category vs mild disease were further evaluated. <a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary table IVc</a> shows the OR for AA vs AG, AG vs GG and AA vs GG: 2.38 (95% CI 0.76 -9.04), 4.52 (95% CI 2.05 – 9.97) and 11.84 (95% CI 3.33 - 42.0), respectively. The large confidence interval for odds ratios with genotype AA are due to the fact that there are only 4 patients with genotype AA in the mild category. These results are consistent with the notion that there is a gene dosage effect, where each addition of allele A increases the risk of severe disease.</p><p id="P28" class="p p-last">For SNPs association with the severity of bone disease, the top SNP is rs510933 on chromosome 6 (p value 7.87E<sup>-07</sup>) (<a href="#SD2" rid="SD2" class=" supplementary-material">Supplementary table IIc</a>). There are 4 other SNPs from this region. However, this region is devoid of genes within 200kb. The most proximal genes near this region are around 500kb away and include STXBP5<em>, SAMD5, SASH1</em>. A noteworthy candidate modifier gene for severity of bone disease is indicated by SNP rs7920091, a SNP in <em>ADAM12</em> (p=1.11E<sup>-05</sup>). <em>ADAM12</em> is member of the ADAM (a disintegrin and metalloprotease) protein family. Polymorphisms in this gene have been associated with osteoarthritis and its progression [<a href="#R39" rid="R39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714006">38</a>]. The ADAM family is also thought to regulate human osteoclast differentiation and activity [<a href="#R40" rid="R40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713992">39</a>]. Other noteworthy associations were with SNPs rs6129532 and rs6124225 on chromosome 20 linked with HSPE pseudogene [HSPE1P1] (p value 1.39E<sup>-05</sup> and 2.12E<sup>-05</sup> respectively) and rs11087800, intronic SNP in <em>ANGPT4</em> (p value 1.45E<sup>-05</sup>)<em>. ANGPT4</em> is involved in angiogenesis. This may be relevant since avascular osteonecrosis in GD is considered a primarily vascular complication reflecting endothelial cell involvement in GD and may involve impaired angiogenic support due to osteoblastic dysfunction [<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714007">6</a>, <a href="#R41" rid="R41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714017">40</a>, <a href="#R42" rid="R42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713999">41</a>].</p></div><div id="S11" class="sec sec-last"><h3 id="S11title">Expression of CLN8 in skin fibroblasts and macrophages</h3><p id="P29" class="p p-first-last">Using Real Time PCR, CLN8 relative expression was higher in GD patients compared to controls (<a href="/pmc/articles/PMC3684025/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3b</span></a>). There was also a trend of GD patients with mild disease having higher expression of CLN8 compared to GD patients with severe disease. To assess the relationship between relative expression of CLN8 and the number of risk alleles in <em>CLN8</em> gene region, two SNPs (rs4595147 and rs11986414) with the lowest LD were selected from the associated SNPs within the CLN8 region. Of the 6 samples analyzed (3 severe and 3 mild patients), the total number of risk alleles appear to be marginally correlated with relative expression (correlation coefficient -0.796, p value 0.058). In control fibroblasts cultured with CBE, an <em>in vitro</em> chemically-induced cellular model of GD, there was slight increase of CLN8 expression compared to untreated cells (<a href="/pmc/articles/PMC3684025/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3a</span></a>). There was more striking increase (2 - 2.5 fold) of CLN8 expression in CBE-treated cells exposed to potent bioactive lipid substrate of acid β-glucosidase, Lyso GL1. Of the two lipid substrates of acid β-glucosidase, Lyso-GL1 was the more potent inducer of CLN8 expression compared to GL1 (<a href="/pmc/articles/PMC3684025/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3a</span></a>). We performed RT PCR for CLN8 expression in primary monocyte-derived macrophages of GD1 patients, and the results were similar to the findings in skin fibroblasts (data not shown). To further assess the expression pattern of CLN8 in different tissues, we interrogated Novartis expression atlas (<a href="http://www.biogps.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.biogps.org</a>) to determine whether it is expressed preferentially in immune cells [<a href="#R43" rid="R43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713997">42</a>, <a href="#R44" rid="R44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713967">43</a>]. <a href="#SD1" rid="SD1" class=" supplementary-material">Supplementary Figure 3</a> depicts the tissue specific expression pattern of CLN8. Interestingly, highest expression is observed in immune cells including macrophages and monocytes. Curiously, lowest expression was observed in brain tissues, which is most vulnerable to loss of function mutations in this gene.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC3684025/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140408529574080" class="figure"><a href="/pmc/articles/PMC3684025/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=3684025_nihms467231f3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is nihms467231f3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC3684025/bin/nihms467231f3.jpg"></a></div><div id="largeobj_idm140408529574080" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC3684025/figure/F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC3684025/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>CLN8 expression in a) healthy control fibroblasts at baseline and after addition of GL1 and Lyso-GL1 b) in GD fibroblasts stratified according to disease severity</strong><p id="__p6"><strong>A.</strong> CBE alone, CBE with GL1 or Lyso GL1 were separately added to the two normal (healthy) cell lines with replacement of medium and drugs every 5-7days. RNA was then extracted and real time PCR was performed for CLN8 expression. Actin-β was used as internal reference control. Concentration of culture solutions were: 50 μM CBE, 40 μM GL1, 20 μM Lyso GL1. <strong>B.</strong> CLN8 expression at baseline in mild and severe GD patient fibroblasts compared to control cells by real time PCR. Results are expressed as times control. The cycle threshold for actin was 19.8 and the average control cycle threshold was 27.8. Each patient's risk alleles are displayed on top for the 2 SNPs (rs11986414 and rs4875960) associated with CLN8. Allele C for the first SNP and allele A for second SNP are associated with a higher risk of disease severity.</p></div></div></div></div></div><div id="S12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S12title">Discussion</h2><p id="P30" class="p p-first">The basis of remarkable variation of severity of GD1 and its heterogeneity among patients with identical <em>GBA1</em> gene mutations is not understood [<a href="#R1" rid="R1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714019">1</a>]. Therefore, it is not possible to identify patients who will benefit from pre-emptive intervention with expensive ERT or small molecule therapy before onset of incapacitating disease manifestations. Identification of genetic modifier(s) will enhance the ability to accurately predict natural history in individual patients in order to develop a personalized monitoring and therapeutic strategy as well as refine genetic counseling.</p><p id="P31">There are significant inherent obstacles to conduct robust GWAS studies for discovery of modifier gene(s) in a Mendelian disorder such as GD. The chronicity and the heterogeneity of the disease as well as the lack of sufficiently well-characterized patients limit the ability to conduct such studies. The greatest hurdle is that the new generation of patients tends to be treated early before the full natural history unfolds [<a href="#R53" rid="R53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713958">52</a>,<a href="#R54" rid="R54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713993">53</a>]. Therefore the accuracy of ascertainment of true disease phenotype with respect to severity is severely compromised. Moreover, the instruments to depict disease severity have limitations to adequately capture the full scope of heterogeneity of disease manifestations. The latter is exemplified most strikingly by some GD patients who predominantly exhibit bone manifestations with only mild visceral or hematological disease and vice versa; these patterns are in contrast to occasional patients who manifest uniformly severe disease in all disease compartments [<a href="#R10" rid="R10" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713971">10</a>]. These patterns of disease, may in fact be driven by modifier gene(s) that underlie monocyte trafficking and/or macrophage-mediated determinants of individual organ injury. Therefore, in our study, we elected to depict disease severity by multiple approaches, including SSI, validated DS3 as well as global physician assessment by two investigators with extensive experience of GD. Moreover, we also examined individual disease compartments, including the viscera and the bone. Thus the final dataset of mild, moderate and severe disease are remarkably consistent with these classifications in the context of overall disease severity as well as severity of individual disease compartments (<a href="/pmc/articles/PMC3684025/table/T1/" target="table" class="fig-table-link figpopup" rid-figpopup="T1" rid-ob="ob-T1" co-legend-rid=""><span>Table I</span></a>).</p><p id="P32">Our study revealed several candidate modifier genes, not previously considered on the basis of current understanding of GD pathophysiology: CLN8, ADAM12 and ANGPT4. The lead candidate modifier gene was <em>CLN8</em>. The physiological function of <em>CLN8</em> is not known but a loss of function mutation is responsible for a subtype of neuronal ceroid-lipofuscinosis (NCL), the Northern epilepsy mental retardation (EPMR) syndrome, characterized by seizures in childhood and progressive mental retardation [<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714004">44</a>]. The <em>CLN8</em> gene encodes a predicted 286–amino acid transmembrane protein containing a C-terminal ER-retrieval signal (KKRP) [<a href="#R46" rid="R46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713988">45</a>] Consistent with this structure, <em>CLN8</em> protein, a nonglycosylated membrane protein, is located in the ER and shuttles between the ER and the ER-Golgi intermediate compartment. Highest expression of CLN8 is surprisingly in the liver and the spleen (compared to the brain) and it has been suggested that it plays an important role as a sensor of sphingolipids and sphingolipid trafficking in all cell types [<a href="#R47" rid="R47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713970">46</a>]. <em>CLN8</em> is a member of a family of homologues that includes ER-resident proteins: yeast Lag1p and human TRAM (translocating chain-associating membrane protein) [<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713983">47</a>]. A C-terminal motif of 200 amino acid residues designated the Tram- Lag-CLN8 (TLC) domain in these proteins has been proposed to play a role in ceramide synthesis and glycosphingolipid trafficking [<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714021">47</a>].</p><p id="P33">In an induced <em>in vitro</em> cellular model of GD, we found there was an induction of CLN8 expression and the level of expression was further increased in the presence of potent bioactive lipid substrate of acid β-glucosidase, Lyso-GL1. In cells derived from, GD1 patients, there was increased basal expression of CLN8 relative to control cells; moreover there was a trend towards increased expression in patients with mild disease compared to patients with moderate/severe disease. We found the SNP alleles in CLN8 region associated with severe GD correlated with relatively lower expression of CLN8 compared to the lower risk alleles (<a href="/pmc/articles/PMC3684025/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figure 3</span></a>). Interestingly, CLN8 and CLN6, two integral ER trans-membrane proteins and cathepsin D, are increased in expression (3-4-fold) in the spleens of mice with conditional deletion of <em>GBA1</em>[<a href="#R6" rid="R6" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713963">6</a>]; loss of function mutations in these three genes result in NCLs [<a href="#R49" rid="R49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713972">48</a>]. Moreover, MYC, a regulator of <em>CLN8</em> was marginally associated with GD1 severity (top SNP p value 4.96 × 10<sup>−5</sup>). Finally interrogation of Novartis gene expression pattern highlighted prominent relative expression of CLN8 in immune cells compared to other cell types and organs. Taken together, CLN8 is a plausible candidate modifier gene of GD and its induction likely protects cells from the damaging effect of accumulating lipids in GD1.</p><p id="P34">Interestingly loss of function mutation in CLN8 results in lysosomal accumulation of autofluorescent material but with lower levels of plasmalogen [<a href="#R48" rid="R48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713984">47</a>] and the latter has been suggested to promote lipid peroxidation. In fact, in GD1, serum plasmalogen levels are reduced, the baseline levels correlate negatively with chitotriosidase and furthermore increase after ERT [<a href="#R51" rid="R51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713977">50</a>]. This observation is also consistent with our notion that induction of CLN8 may be protective towards damaging cellular effects of glycosphingolpid accumulation in GD1.</p><p id="P35"><em>CLN8</em> and the other potential candidate modifier genes (<em>ADAM12</em> and <em>ANGPT4</em>) identified in our GWAS do not achieve the threshold for genome-wide significance after stringent Bonferroni correction [<a href="#R52" rid="R52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713979">51</a>]. Given the potential impediments to conduct a replication study, an alternative approach to define the potential role of CLN8 as a modifier of GD will derive from studies in transgenic mice since CLN8 KO as well as conditional KO mouse models of GD are available [<a href="#R45" rid="R45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319713981">44</a>, <a href="#R55" rid="R55" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_319714013">54</a>].</p><p id="P36" class="p p-last">In conclusion, <em>CLN8</em> is a plausible candidate modifier gene for Gaucher disease based on our GWAS results and <em>in vitro</em> studies in chemically-induced cellular model of GD. Further studies are warranted to define the physiologic role of CLN8 in health and its role in GD pathophysiology. The availability of mouse models should facilitate these studies.</p></div><div id="S13" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S13title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>Supp Fig S1-S3</h4><p id="P37"><strong>Supplementary Figure 1 Timelines of N370S Homozygous GD patients with severe disease compared to patient with mild disease.</strong> The presentation in Gaucher disease is very variable. This timeline shows significant events in the natural history of two N370S homozygous patients enrolled in the current GWAS study. These contrasting patterns illustrate mild vs. severe disease in our patient population. Patient A with severe disease had early onset osteomyelitis/AVN. He had severe splenomegaly and hepatomegaly at an early age and went on to have a splenic rupture and splenectomy. Patient B with mild disease was diagnosed based on maternal diagnosis and has been asymptomatic. She has virtually no disease manifestations and does not require ERT at 41 years of age.</p><p id="P38"><strong>Supplementary Figure 2:</strong> QTL association results in CLN8 region. This screenshot from the eQTL browser displays eQTLs near the CLN8 region. The chromosome 8 and CLN8/C8orf61 genes are displayed. And each SNP associated with CLN8 or other genes are displayed below, together with related references.</p><p id="P39"><strong>Supplementary Figure 3: Tissue specific expression profile for CLN8</strong> This figure displays the human tissue specific expression profile from U133plus2 Affymetrics microarray (CLN8, probe ID 219340_s_at). The expression value is a z-score derived from the barcode function of the R package "frma". A z-score &gt; 5 suggests that the gene is expressed in that tissue. Color codes highlight neuronal tissues (blue), immune cells/organs (red), viscera (purple) and miscellaneous (orange).</p><div class="sup-box half_rhythm" id="d36e845"><a href="/pmc/articles/PMC3684025/bin/NIHMS467231-supplement-Supp_Fig_S1-S3.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(287K, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>Supp Table S1-S4</h4><p id="P40"><strong>Supplementary Table I-IV Table S I Pairwise IBD coefficients and inferred family relationships.</strong> A. Top 50 sample pairs with largest kinship coefficients. Columns ID1 and 2 are IDs from two individuals, D7, D8 and D9 are the last three IBD coefficients inferred assuming no inbreeding, KinshipCoef is the kinship coefficients calculated from 9 IBD coefficients. B. Reconstructed samples potentially within a family, 17 families were inferred.</p><p id="P41"><strong>Table S II Top associated SNPs for the three traits, and their nearest gene neighborhood and distance to gene start and gene end.</strong> SNPs are listed in the order of their genomic location. The Symbol column shows the nearest gene neighborhood. The distance to the start of the gene (diststart) is calculated by the gene start bp minus the snp location; the distance to the end of the gene (distend) is calculated by the gene end bp minus the snp location. The chromosome location is from NCBI genome assembly 36.3. A. Overall top SNPs. B. Visceral top SNPs. C. Bone top SNPs</p><p id="P42"><strong>Table S III Association of SNPs in CLN8 region with overall disease severity.</strong> In the annotation column, flanking refers to the flanking region of CLN8. The eight strongly associated SNPs within the region are colored in red.</p><p id="P43"><strong>Table S IV. The genotype and allelic configuration for SNP rs11986414 in CLN8 region, allelic pairwise odds ratio and genotypic odds ratio.</strong> The genotype configuration for the top SNP rs11986414 associated with CLN8 with a p value of 1.26 × 10<sup>-6</sup> for overall disease severity is depicted. Allele A is the risk allele associated with more severe phenotype. The A allele is observed more frequently in moderate and severe disease categories compared to the mild category.</p><p id="P44">A. genotype and allelic configuration for SNP rs11986414.</p><p id="P45">B. Allelic pairwise odds ratio.</p><p id="P46">C. Genotypic odds ratio for combined severe and moderate phenotype vs mild phenotype.</p><div class="sup-box half_rhythm" id="d36e874"><a href="/pmc/articles/PMC3684025/bin/NIHMS467231-supplement-Supp_Table_S1-S4.doc" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(225K, doc)</sup></div></div></div><div id="S14" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="S14title">Acknowledgments</h2><div class="sec"><p id="__p2">We thank the patients at Yale and NYU Gaucher Disease Treatment Centers for participation in this study. The study was supported by the National Gaucher Foundation and by Lysosomal Fellowship Training Grant from Genzyme-Sanofi to PS. PKM thanks Dr Jeffrey Gruen and Karen K. Weisbord for advice.</p><p id="__p3">Contract grant sponsor: PKM is supported by NIDDK K24DK066306 mid-career clinical investigator award, the National Gaucher Foundation and a grant from Genzyme Corporation for investigator-initiated study on biomarker discovery and validation. PS was supported by a Lysosomal Storage Disease Fellowship Training Grant from Genzyme-Sanofi.</p></div></div><div id="idm140408488501648" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140408488501648title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="element-citation">Grabowski GA. Phenotype, diagnosis, and treatment of Gaucher's disease. <span><span class="ref-journal">Lancet. </span>2008;<span class="ref-vol">372</span>(9645):1263–71.</span> [<a href="/pubmed/19094956" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Phenotype,+diagnosis,+and+treatment+of+Gaucher's+disease&amp;author=GA+Grabowski&amp;volume=372&amp;issue=9645&amp;publication_year=2008&amp;pages=1263-71&amp;pmid=19094956&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="element-citation">Brady RO, Barton NW, Grabowski GA. The role of neurogenetics in Gaucher disease. <span><span class="ref-journal">Arch Neurol. </span>1993;<span class="ref-vol">50</span>(11):1212–24.</span> [<a href="/pubmed/8215980" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch+Neurol&amp;title=The+role+of+neurogenetics+in+Gaucher+disease&amp;author=RO+Brady&amp;author=NW+Barton&amp;author=GA+Grabowski&amp;volume=50&amp;issue=11&amp;publication_year=1993&amp;pages=1212-24&amp;pmid=8215980&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="element-citation">Cox TM. Gaucher disease: clinical profile and therapeutic developments. <span><span class="ref-journal">Biologics. </span><span class="ref-vol">4</span>:299–313.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3010821/">PMC free article</a>]</span> [<a href="/pubmed/21209725" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biologics&amp;title=Gaucher+disease:+clinical+profile+and+therapeutic+developments&amp;author=TM+Cox&amp;volume=4&amp;pages=299-313&amp;pmid=21209725&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="element-citation">Jmoudiak M, Futerman AH. Gaucher disease: pathological mechanisms and modern management. <span><span class="ref-journal">Br J Haematol. </span>2005;<span class="ref-vol">129</span>(2):178–88.</span> [<a href="/pubmed/15813845" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=Gaucher+disease:+pathological+mechanisms+and+modern+management&amp;author=M+Jmoudiak&amp;author=AH+Futerman&amp;volume=129&amp;issue=2&amp;publication_year=2005&amp;pages=178-88&amp;pmid=15813845&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="element-citation">Kitatani K, et al.  Acid beta-glucosidase 1 counteracts p38delta-dependent induction of interleukin-6: possible role for ceramide as an anti-inflammatory lipid. <span><span class="ref-journal">J Biol Chem. </span>2009;<span class="ref-vol">284</span>(19):12979–88.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2676030/">PMC free article</a>]</span> [<a href="/pubmed/19279008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=Acid+beta-glucosidase+1+counteracts+p38delta-dependent+induction+of+interleukin-6:+possible+role+for+ceramide+as+an+anti-inflammatory+lipid&amp;author=K+Kitatani&amp;volume=284&amp;issue=19&amp;publication_year=2009&amp;pages=12979-88&amp;pmid=19279008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="element-citation">Mistry PK, et al.  Glucocerebrosidase gene-deficient mouse recapitulates Gaucher disease displaying cellular and molecular dysregulation beyond the macrophage. <span><span class="ref-journal">Proceedings of the National Academy of Sciences of the United States of America. </span>2010;<span class="ref-vol">107</span>:19473–19478.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2984187/">PMC free article</a>]</span> [<a href="/pubmed/20962279" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings+of+the+National+Academy+of+Sciences+of+the+United+States+of+America&amp;title=Glucocerebrosidase+gene-deficient+mouse+recapitulates+Gaucher+disease+displaying+cellular+and+molecular+dysregulation+beyond+the+macrophage&amp;author=PK+Mistry&amp;volume=107&amp;publication_year=2010&amp;pages=19473-19478&amp;pmid=20962279&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="element-citation">Hruska KS, LaMarca ME, Sidransky E.  <span class="ref-journal">Gaucher disease: Molecular biology and genotype/phenotype correaltions In Gaucher disease Eds AH Futerman and A Zimran.</span> CRC Press;  <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Gaucher+disease:+Molecular+biology+and+genotype/phenotype+correaltions+In+Gaucher+disease+Eds+AH+Futerman+and+A+Zimran&amp;author=KS+Hruska&amp;author=ME+LaMarca&amp;author=E+Sidransky&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="element-citation">Grabowski GA. Gaucher disease: gene frequencies and genotype/phenotype correlations. <span><span class="ref-journal">Genet Test. </span>1997;<span class="ref-vol">1</span>(1):5–12.</span> [<a href="/pubmed/10464619" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Test&amp;title=Gaucher+disease:+gene+frequencies+and+genotype/phenotype+correlations&amp;author=GA+Grabowski&amp;volume=1&amp;issue=1&amp;publication_year=1997&amp;pages=5-12&amp;pmid=10464619&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="element-citation">Torralba MA, et al.  High prevalence of the 55-bp deletion (c.1263del55) in exon 9 of the glucocerebrosidase gene causing misdiagnosis (for homozygous N370S (c.1226A &gt; G) mutation) in Spanish Gaucher disease patients. <span><span class="ref-journal">Blood Cells Mol Dis. </span>2002;<span class="ref-vol">29</span>(1):35–40.</span> [<a href="/pubmed/12482401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood+Cells+Mol+Dis&amp;title=High+prevalence+of+the+55-bp+deletion+(c.1263del55)+in+exon+9+of+the+glucocerebrosidase+gene+causing+misdiagnosis+(for+homozygous+N370S+(c.1226A+>+G)+mutation)+in+Spanish+Gaucher+disease+patients&amp;author=MA+Torralba&amp;volume=29&amp;issue=1&amp;publication_year=2002&amp;pages=35-40&amp;pmid=12482401&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="element-citation">Taddei TH, et al.  The underrecognized progressive nature of N370S Gaucher disease and assessment of cancer risk in 403 patients. <span><span class="ref-journal">Am J Hematol. </span>2009;<span class="ref-vol">84</span>(4):208.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3008404/">PMC free article</a>]</span> [<a href="/pubmed/19260119" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hematol&amp;title=The+underrecognized+progressive+nature+of+N370S+Gaucher+disease+and+assessment+of+cancer+risk+in+403+patients&amp;author=TH+Taddei&amp;volume=84&amp;issue=4&amp;publication_year=2009&amp;pages=208&amp;pmid=19260119&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="element-citation">Mistry PK, et al.  A reappraisal of Gaucher disease-diagnosis and disease management algorithms. <span><span class="ref-journal">Am J Hematol. </span><span class="ref-vol">86</span>(1):110–5.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3058841/">PMC free article</a>]</span> [<a href="/pubmed/21080341" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hematol&amp;title=A+reappraisal+of+Gaucher+disease-diagnosis+and+disease+management+algorithms&amp;author=PK+Mistry&amp;volume=86&amp;issue=1&amp;pages=110-5&amp;pmid=21080341&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="element-citation">Beutler E, et al.  Gaucher disease: gene frequencies in the Ashkenazi Jewish population. <span><span class="ref-journal">Am J Hum Genet. </span>1993;<span class="ref-vol">52</span>(1):85–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1682129/">PMC free article</a>]</span> [<a href="/pubmed/8434610" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Gaucher+disease:+gene+frequencies+in+the+Ashkenazi+Jewish+population&amp;author=E+Beutler&amp;volume=52&amp;issue=1&amp;publication_year=1993&amp;pages=85-8&amp;pmid=8434610&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="element-citation">Goker-Alpan O, et al.  Divergent phenotypes in Gaucher disease implicate the role of modifiers. <span><span class="ref-journal">J Med Genet. </span>2005;<span class="ref-vol">42</span>(6):e37.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1736082/">PMC free article</a>]</span> [<a href="/pubmed/15937077" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Med+Genet&amp;title=Divergent+phenotypes+in+Gaucher+disease+implicate+the+role+of+modifiers&amp;author=O+Goker-Alpan&amp;volume=42&amp;issue=6&amp;publication_year=2005&amp;pages=e37&amp;pmid=15937077&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="element-citation">Aerts JM, et al.  Biomarkers in the diagnosis of lysosomal storage disorders: proteins, lipids, and inhibodies. <span><span class="ref-journal">J Inherit Metab Dis. </span><span class="ref-vol">34</span>(3):605–19.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3109260/">PMC free article</a>]</span> [<a href="/pubmed/21445610" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Inherit+Metab+Dis&amp;title=Biomarkers+in+the+diagnosis+of+lysosomal+storage+disorders:+proteins,+lipids,+and+inhibodies&amp;author=JM+Aerts&amp;volume=34&amp;issue=3&amp;pages=605-19&amp;pmid=21445610&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="element-citation">Amato D, et al.  Gaucher disease: variability in phenotype among siblings. <span><span class="ref-journal">J Inherit Metab Dis. </span>2004;<span class="ref-vol">27</span>(5):659–69.</span> [<a href="/pubmed/15669682" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Inherit+Metab+Dis&amp;title=Gaucher+disease:+variability+in+phenotype+among+siblings&amp;author=D+Amato&amp;volume=27&amp;issue=5&amp;publication_year=2004&amp;pages=659-69&amp;pmid=15669682&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="element-citation">Elstein D, et al.  Disease severity in sibling pairs with type 1 Gaucher disease. <span><span class="ref-journal">J Inherit Metab Dis. </span><span class="ref-vol">33</span>(1):79–83.</span> [<a href="/pubmed/20049528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Inherit+Metab+Dis&amp;title=Disease+severity+in+sibling+pairs+with+type+1+Gaucher+disease&amp;author=D+Elstein&amp;volume=33&amp;issue=1&amp;pages=79-83&amp;pmid=20049528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="element-citation">Lachmann RH, et al.  Twin pairs showing discordance of phenotype in adult Gaucher's disease. <span><span class="ref-journal">QJM. </span>2004;<span class="ref-vol">97</span>(4):199–204.</span> [<a href="/pubmed/15028849" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=QJM&amp;title=Twin+pairs+showing+discordance+of+phenotype+in+adult+Gaucher's+disease&amp;author=RH+Lachmann&amp;volume=97&amp;issue=4&amp;publication_year=2004&amp;pages=199-204&amp;pmid=15028849&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="element-citation">Biegstraaten M, et al.  A monozygotic twin pair with highly discordant Gaucher phenotypes. <span><span class="ref-journal">Blood Cells Mol Dis. </span><span class="ref-vol">46</span>(1):39–41.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3352676/">PMC free article</a>]</span> [<a href="/pubmed/21056933" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood+Cells+Mol+Dis&amp;title=A+monozygotic+twin+pair+with+highly+discordant+Gaucher+phenotypes&amp;author=M+Biegstraaten&amp;volume=46&amp;issue=1&amp;pages=39-41&amp;pmid=21056933&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="element-citation">van Weely S, et al.  Role of pH in determining the cell-type-specific residual activity of glucocerebrosidase in type 1 Gaucher disease. <span><span class="ref-journal">J Clin Invest. </span>1993;<span class="ref-vol">91</span>(3):1167–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC288073/">PMC free article</a>]</span> [<a href="/pubmed/8450045" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Role+of+pH+in+determining+the+cell-type-specific+residual+activity+of+glucocerebrosidase+in+type+1+Gaucher+disease&amp;author=S+van+Weely&amp;volume=91&amp;issue=3&amp;publication_year=1993&amp;pages=1167-75&amp;pmid=8450045&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="element-citation">Grabowski GA, Horowitz M. Gaucher's disease: molecular, genetic and enzymological aspects. <span><span class="ref-journal">Baillieres Clin Haematol. </span>1997 Dec;<span class="ref-vol">10</span>(4):635–56.</span> [<a href="/pubmed/9497856" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Baillieres+Clin+Haematol&amp;title=Gaucher's+disease:+molecular,+genetic+and+enzymological+aspects&amp;author=GA+Grabowski&amp;author=M+Horowitz&amp;volume=10&amp;issue=4&amp;publication_year=1997&amp;pages=635-56&amp;pmid=9497856&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="element-citation">Ron I, Horowitz M. ER retention and degradation as the molecular basis underlying Gaucher disease heterogeneity. <span><span class="ref-journal">Hum Mol Genet. </span>2005 Aug 15;<span class="ref-vol">14</span>(16):2387–98.</span> [<a href="/pubmed/16000318" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=ER+retention+and+degradation+as+the+molecular+basis+underlying+Gaucher+disease+heterogeneity&amp;author=I+Ron&amp;author=M+Horowitz&amp;volume=14&amp;issue=16&amp;publication_year=2005&amp;pages=2387-98&amp;pmid=16000318&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="element-citation">Kolodny EH, et al.  Phenotypic manifestations of Gaucher disease: clinical features in 48 biochemically verified type 1 patients and comment on type 2 patients. <span><span class="ref-journal">Prog Clin Biol Res. </span>1982;<span class="ref-vol">95</span>:33–65.</span> [<a href="/pubmed/6289358" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Prog+Clin+Biol+Res&amp;title=Phenotypic+manifestations+of+Gaucher+disease:+clinical+features+in+48+biochemically+verified+type+1+patients+and+comment+on+type+2+patients&amp;author=EH+Kolodny&amp;volume=95&amp;publication_year=1982&amp;pages=33-65&amp;pmid=6289358&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="element-citation">Diaz GA, et al.  Gaucher disease: the origins of the Ashkenazi Jewish N370S and 84GG acid beta-glucosidase mutations. <span><span class="ref-journal">Am J Hum Genet. </span>2000;<span class="ref-vol">66</span>(6):1821–32.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1378046/">PMC free article</a>]</span> [<a href="/pubmed/10777718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Gaucher+disease:+the+origins+of+the+Ashkenazi+Jewish+N370S+and+84GG+acid+beta-glucosidase+mutations&amp;author=GA+Diaz&amp;volume=66&amp;issue=6&amp;publication_year=2000&amp;pages=1821-32&amp;pmid=10777718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="element-citation">Beutler E, West C. Polymorphisms in glucosylceramide (glucocerebroside) synthase and the Gaucher disease phenotype. <span><span class="ref-journal">Isr Med Assoc J. </span>2002;<span class="ref-vol">4</span>(11):986–8.</span> [<a href="/pubmed/12489486" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Isr+Med+Assoc+J&amp;title=Polymorphisms+in+glucosylceramide+(glucocerebroside)+synthase+and+the+Gaucher+disease+phenotype&amp;author=E+Beutler&amp;author=C+West&amp;volume=4&amp;issue=11&amp;publication_year=2002&amp;pages=986-8&amp;pmid=12489486&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="element-citation">Dekker N, et al.  The cytosolic beta-glucosidase GBA3 does not influence type 1 Gaucher disease manifestation. <span><span class="ref-journal">Blood Cells Mol Dis. </span><span class="ref-vol">46</span>(1):19–26.</span> [<a href="/pubmed/20728381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood+Cells+Mol+Dis&amp;title=The+cytosolic+beta-glucosidase+GBA3+does+not+influence+type+1+Gaucher+disease+manifestation&amp;author=N+Dekker&amp;volume=46&amp;issue=1&amp;pages=19-26&amp;pmid=20728381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="element-citation">Altarescu G, et al.  The interleukin-6 promoter polymorphism in Gaucher disease: a new modifier gene? <span><span class="ref-journal">QJM. </span>2003;<span class="ref-vol">96</span>(8):575–8.</span> [<a href="/pubmed/12897342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=QJM&amp;title=The+interleukin-6+promoter+polymorphism+in+Gaucher+disease:+a+new+modifier+gene?&amp;author=G+Altarescu&amp;volume=96&amp;issue=8&amp;publication_year=2003&amp;pages=575-8&amp;pmid=12897342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="element-citation">Altarescu G, et al.  TNF-alpha levels and TNF-alpha gene polymorphism in type I Gaucher disease. <span><span class="ref-journal">Cytokine. </span>2005;<span class="ref-vol">31</span>(2):149–52.</span> [<a href="/pubmed/15919211" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine&amp;title=TNF-alpha+levels+and+TNF-alpha+gene+polymorphism+in+type+I+Gaucher+disease&amp;author=G+Altarescu&amp;volume=31&amp;issue=2&amp;publication_year=2005&amp;pages=149-52&amp;pmid=15919211&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="element-citation">Greenwood A, et al.  Vitamin D receptor (VDR) polymorphisms in the cardiac variant of Gaucher disease. <span><span class="ref-journal">Pediatr Cardiol. </span><span class="ref-vol">31</span>(1):30–2.</span> [<a href="/pubmed/19784695" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr+Cardiol&amp;title=Vitamin+D+receptor+(VDR)+polymorphisms+in+the+cardiac+variant+of+Gaucher+disease&amp;author=A+Greenwood&amp;volume=31&amp;issue=1&amp;pages=30-2&amp;pmid=19784695&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="element-citation">Maas M, et al.  Quantification of bone involvement in Gaucher disease: MR imaging bone marrow burden score as an alternative to Dixon quantitative chemical shift MR imaging--initial experience. <span><span class="ref-journal">Radiology. </span>2003;<span class="ref-vol">229</span>(2):554–61.</span> [<a href="/pubmed/14526090" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Radiology&amp;title=Quantification+of+bone+involvement+in+Gaucher+disease:+MR+imaging+bone+marrow+burden+score+as+an+alternative+to+Dixon+quantitative+chemical+shift+MR+imaging--initial+experience&amp;author=M+Maas&amp;volume=229&amp;issue=2&amp;publication_year=2003&amp;pages=554-61&amp;pmid=14526090&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="element-citation">Zimran A, et al.  Gaucher disease. Clinical, laboratory, radiologic, and genetic features of 53 patients. <span><span class="ref-journal">Medicine (Baltimore) </span>1992;<span class="ref-vol">71</span>(6):337–53.</span> [<a href="/pubmed/1435229" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Medicine+(Baltimore)&amp;title=Gaucher+disease.+Clinical,+laboratory,+radiologic,+and+genetic+features+of+53+patients&amp;author=A+Zimran&amp;volume=71&amp;issue=6&amp;publication_year=1992&amp;pages=337-53&amp;pmid=1435229&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="element-citation">Weinreb NJ, et al.  A validated disease severity scoring system for adults with type 1 Gaucher disease. <span><span class="ref-journal">Genet Med. </span><span class="ref-vol">12</span>(1):44–51.</span> [<a href="/pubmed/20027115" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Med&amp;title=A+validated+disease+severity+scoring+system+for+adults+with+type+1+Gaucher+disease&amp;author=NJ+Weinreb&amp;volume=12&amp;issue=1&amp;pages=44-51&amp;pmid=20027115&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>(3):559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;issue=3&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="element-citation">Choi Y, Wijsman EM, Weir BS. Case-control association testing in the presence of unknown relationships. <span><span class="ref-journal">Genet Epidemiol. </span>2009;<span class="ref-vol">33</span>(8):668–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2790016/">PMC free article</a>]</span> [<a href="/pubmed/19333967" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet+Epidemiol&amp;title=Case-control+association+testing+in+the+presence+of+unknown+relationships&amp;author=Y+Choi&amp;author=EM+Wijsman&amp;author=BS+Weir&amp;volume=33&amp;issue=8&amp;publication_year=2009&amp;pages=668-78&amp;pmid=19333967&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="element-citation">Thornton T, McPeek MS. Case-control association testing with related individuals: a more powerful quasi-likelihood score test. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>(2):321–37.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950805/">PMC free article</a>]</span> [<a href="/pubmed/17668381" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Case-control+association+testing+with+related+individuals:+a+more+powerful+quasi-likelihood+score+test&amp;author=T+Thornton&amp;author=MS+McPeek&amp;volume=81&amp;issue=2&amp;publication_year=2007&amp;pages=321-37&amp;pmid=17668381&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="element-citation">Barrett JC, et al.  Haploview: analysis and visualization of LD and haplotype maps. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21</span>(2):263–5.</span> [<a href="/pubmed/15297300" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Haploview:+analysis+and+visualization+of+LD+and+haplotype+maps&amp;author=JC+Barrett&amp;volume=21&amp;issue=2&amp;publication_year=2005&amp;pages=263-5&amp;pmid=15297300&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="element-citation">Johnson AD, et al.  SNAP: a web-based tool for identification and annotation of proxy SNPs using HapMap. <span><span class="ref-journal">Bioinformatics. </span>2008;<span class="ref-vol">24</span>(24):2938–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2720775/">PMC free article</a>]</span> [<a href="/pubmed/18974171" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SNAP:+a+web-based+tool+for+identification+and+annotation+of+proxy+SNPs+using+HapMap&amp;author=AD+Johnson&amp;volume=24&amp;issue=24&amp;publication_year=2008&amp;pages=2938-9&amp;pmid=18974171&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="element-citation">Veyrieras JB, et al.  High-resolution mapping of expression-QTLs yields insight into human gene regulation. <span><span class="ref-journal">PLoS Genetics. </span>2008 Oct;<span class="ref-vol">4</span>(10):e1000214.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2556086/">PMC free article</a>]</span> [<a href="/pubmed/18846210" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genetics&amp;title=High-resolution+mapping+of+expression-QTLs+yields+insight+into+human+gene+regulation&amp;author=JB+Veyrieras&amp;volume=4&amp;issue=10&amp;publication_year=2008&amp;pages=e1000214&amp;pmid=18846210&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">37a. <span class="element-citation">Sillence DJ, et al.  Glucosylceramide modulates membrane traffic along the endocytic pathway. <span><span class="ref-journal">J Lipid Res. </span>2002;<span class="ref-vol">43</span>(11):1837–45.</span> [<a href="/pubmed/12401882" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Lipid+Res&amp;title=Glucosylceramide+modulates+membrane+traffic+along+the+endocytic+pathway&amp;author=DJ+Sillence&amp;volume=43&amp;issue=11&amp;publication_year=2002&amp;pages=1837-45&amp;pmid=12401882&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">38. <span class="element-citation">Kerna I, et al.  Missense single nucleotide polymorphism of the ADAM12 gene is associated with radiographic knee osteoarthritis in middle-aged Estonian cohort. <span><span class="ref-journal">Osteoarthritis Cartilage. </span>2009;<span class="ref-vol">17</span>(8):1093–8.</span> [<a href="/pubmed/19268722" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Osteoarthritis+Cartilage&amp;title=Missense+single+nucleotide+polymorphism+of+the+ADAM12+gene+is+associated+with+radiographic+knee+osteoarthritis+in+middle-aged+Estonian+cohort&amp;author=I+Kerna&amp;volume=17&amp;issue=8&amp;publication_year=2009&amp;pages=1093-8&amp;pmid=19268722&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">39. <span class="element-citation">Bartholin L, et al.  FLRG, a new ADAM12-associated protein, modulates osteoclast differentiation. <span><span class="ref-journal">Biol Cell. </span>2005;<span class="ref-vol">97</span>(7):577–88.</span> [<a href="/pubmed/15574124" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol+Cell&amp;title=FLRG,+a+new+ADAM12-associated+protein,+modulates+osteoclast+differentiation&amp;author=L+Bartholin&amp;volume=97&amp;issue=7&amp;publication_year=2005&amp;pages=577-88&amp;pmid=15574124&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">40. <span class="element-citation">Elleder M. Glucosylceramide transfer from lysosomes--the missing link in molecular pathology of glucosylceramidase deficiency: a hypothesis based on existing data. <span><span class="ref-journal">J Inherit Metab Dis. </span>2006;<span class="ref-vol">29</span>(6):707–15.</span> [<a href="/pubmed/17080304" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Inherit+Metab+Dis&amp;title=Glucosylceramide+transfer+from+lysosomes--the+missing+link+in+molecular+pathology+of+glucosylceramidase+deficiency:+a+hypothesis+based+on+existing+data&amp;author=M+Elleder&amp;volume=29&amp;issue=6&amp;publication_year=2006&amp;pages=707-15&amp;pmid=17080304&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">41. <span class="element-citation">Zaidi M, et al.  ACTH protects against glucocorticoid-induced osteonecrosis of bone. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2010 May 11;<span class="ref-vol">107</span>(19):8782–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2889316/">PMC free article</a>]</span> [<a href="/pubmed/20421485" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=ACTH+protects+against+glucocorticoid-induced+osteonecrosis+of+bone&amp;author=M+Zaidi&amp;volume=107&amp;issue=19&amp;publication_year=2010&amp;pages=8782-7&amp;pmid=20421485&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">42. <span class="element-citation">Su AI, et al.  A gene atlas of the mouse and human protein-encoding transcriptomes. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2004 Apr 20;<span class="ref-vol">101</span>(16):6062–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC395923/">PMC free article</a>]</span> [<a href="/pubmed/15075390" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=A+gene+atlas+of+the+mouse+and+human+protein-encoding+transcriptomes&amp;author=AI+Su&amp;volume=101&amp;issue=16&amp;publication_year=2004&amp;pages=6062-7&amp;pmid=15075390&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">43. <span class="element-citation">Wu C, et al.  BioGPS: an extensible and customizable portal for querying and organizing gene annotation resources. <span><span class="ref-journal">Genome Biology. </span>2009;<span class="ref-vol">10</span>:R130.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3091323/">PMC free article</a>]</span> [<a href="/pubmed/19919682" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biology&amp;title=BioGPS:+an+extensible+and+customizable+portal+for+querying+and+organizing+gene+annotation+resources&amp;author=C+Wu&amp;volume=10&amp;publication_year=2009&amp;pages=R130&amp;pmid=19919682&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">44. <span class="element-citation">Ranta S, et al.  The neuronal ceroid lipofuscinoses in human EPMR and mnd mutant mice are associated with mutations in CLN8. <span><span class="ref-journal">Nat Genet. </span>1999 Oct;<span class="ref-vol">23</span>(2):233–6.</span> [<a href="/pubmed/10508524" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=The+neuronal+ceroid+lipofuscinoses+in+human+EPMR+and+mnd+mutant+mice+are+associated+with+mutations+in+CLN8&amp;author=S+Ranta&amp;volume=23&amp;issue=2&amp;publication_year=1999&amp;pages=233-6&amp;pmid=10508524&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">45. <span class="element-citation">Lonka L, et al.  The neuronal ceroid lipofuscinosis CLN8 membrane protein is a resident of the endoplasmic reticulum. <span><span class="ref-journal">Hum Mol Genet. </span>2000;<span class="ref-vol">9</span>(11):1691–7.</span> [<a href="/pubmed/10861296" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mol+Genet&amp;title=The+neuronal+ceroid+lipofuscinosis+CLN8+membrane+protein+is+a+resident+of+the+endoplasmic+reticulum&amp;author=L+Lonka&amp;volume=9&amp;issue=11&amp;publication_year=2000&amp;pages=1691-7&amp;pmid=10861296&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">46. <span class="element-citation">Lonka L, et al.  Localization of wild-type and mutant neuronal ceroid lipofuscinosis CLN8 proteins in non-neuronal and neuronal cells. <span><span class="ref-journal">J Neurosci Res. </span>2004;<span class="ref-vol">76</span>(6):862–71.</span> [<a href="/pubmed/15160397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurosci+Res&amp;title=Localization+of+wild-type+and+mutant+neuronal+ceroid+lipofuscinosis+CLN8+proteins+in+non-neuronal+and+neuronal+cells&amp;author=L+Lonka&amp;volume=76&amp;issue=6&amp;publication_year=2004&amp;pages=862-71&amp;pmid=15160397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">47. <span class="element-citation">Schorling S, et al.  Lag1p and Lac1p are essential for the Acyl-CoA-dependent ceramide synthase reaction in Saccharomyces cerevisae. <span><span class="ref-journal">Mol Biol Cell. </span>2001;<span class="ref-vol">12</span>(11):3417–27.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC60264/">PMC free article</a>]</span> [<a href="/pubmed/11694577" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Biol+Cell&amp;title=Lag1p+and+Lac1p+are+essential+for+the+Acyl-CoA-dependent+ceramide+synthase+reaction+in+Saccharomyces+cerevisae&amp;author=S+Schorling&amp;volume=12&amp;issue=11&amp;publication_year=2001&amp;pages=3417-27&amp;pmid=11694577&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">48. <span class="element-citation">Kousi M, Lehesjoki AE, Mole SE. Update of the mutation spectrum and clinical correlations of over 360 mutations in eight genes that underlie the neuronal ceroid lipofuscinoses. <span><span class="ref-journal">Hum Mutat. </span>2012 Jan;<span class="ref-vol">33</span>(1):42–63.</span> [<a href="/pubmed/21990111" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mutat&amp;title=Update+of+the+mutation+spectrum+and+clinical+correlations+of+over+360+mutations+in+eight+genes+that+underlie+the+neuronal+ceroid+lipofuscinoses&amp;author=M+Kousi&amp;author=AE+Lehesjoki&amp;author=SE+Mole&amp;volume=33&amp;issue=1&amp;publication_year=2012&amp;pages=42-63&amp;pmid=21990111&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">49. <span class="element-citation">Hermansson M, et al.  Mass spectrometric analysis reveals changes in phospholipid, neutral sphingolipid and sulfatide molecular species in progressive epilepsy with mental retardation, EPMR, brain: a case study. <span><span class="ref-journal">J Neurochem. </span>2005;<span class="ref-vol">95</span>(3):609–17.</span> [<a href="/pubmed/16086686" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Neurochem&amp;title=Mass+spectrometric+analysis+reveals+changes+in+phospholipid,+neutral+sphingolipid+and+sulfatide+molecular+species+in+progressive+epilepsy+with+mental+retardation,+EPMR,+brain:+a+case+study&amp;author=M+Hermansson&amp;volume=95&amp;issue=3&amp;publication_year=2005&amp;pages=609-17&amp;pmid=16086686&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">50. <span class="element-citation">Moraitou M, et al.  Plasmalogen levels in Gaucher disease. <span><span class="ref-journal">Blood Cells Mol Dis. </span>2008;<span class="ref-vol">41</span>(2):196–9.</span> [<a href="/pubmed/18501647" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood+Cells+Mol+Dis&amp;title=Plasmalogen+levels+in+Gaucher+disease&amp;author=M+Moraitou&amp;volume=41&amp;issue=2&amp;publication_year=2008&amp;pages=196-9&amp;pmid=18501647&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">51. <span class="element-citation">Hunter DJ, Kraft P. Drinking from the fire hose--statistical issues in genomewide association studies. <span><span class="ref-journal">N Engl J Med. </span>2007;<span class="ref-vol">357</span>(5):436–9.</span> [<a href="/pubmed/17634446" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N+Engl+J+Med&amp;title=Drinking+from+the+fire+hose--statistical+issues+in+genomewide+association+studies&amp;author=DJ+Hunter&amp;author=P+Kraft&amp;volume=357&amp;issue=5&amp;publication_year=2007&amp;pages=436-9&amp;pmid=17634446&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">52. <span class="element-citation">Weinreb N, et al.  A benchmark analysis of the achievement of therapeutic goals for type 1 Gaucher disease patients treated with imiglucerase. <span><span class="ref-journal">Am J Hematol. </span>2008;<span class="ref-vol">83</span>(12):890–5.</span> [<a href="/pubmed/18819093" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hematol&amp;title=A+benchmark+analysis+of+the+achievement+of+therapeutic+goals+for+type+1+Gaucher+disease+patients+treated+with+imiglucerase&amp;author=N+Weinreb&amp;volume=83&amp;issue=12&amp;publication_year=2008&amp;pages=890-5&amp;pmid=18819093&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">53. <span class="element-citation">Mistry PK, et al.  Timing of initiation of enzyme replacement therapy after diagnosis of type 1 Gaucher disease: effect on incidence of avascular necrosis. <span><span class="ref-journal">Br J Haematol. </span>2009;<span class="ref-vol">147</span>(4):561–70.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2774157/">PMC free article</a>]</span> [<a href="/pubmed/19732054" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br+J+Haematol&amp;title=Timing+of+initiation+of+enzyme+replacement+therapy+after+diagnosis+of+type+1+Gaucher+disease:+effect+on+incidence+of+avascular+necrosis&amp;author=PK+Mistry&amp;volume=147&amp;issue=4&amp;publication_year=2009&amp;pages=561-70&amp;pmid=19732054&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">54. <span class="element-citation">Enquist IB, et al.  Effective cell and gene therapy in a murine model of Gaucher disease. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2006 Sep 12;<span class="ref-vol">103</span>(37):13819–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1564262/">PMC free article</a>]</span> [<a href="/pubmed/16954197" target="pmc_ext" ref="reftype=pubmed&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Effective+cell+and+gene+therapy+in+a+murine+model+of+Gaucher+disease&amp;author=IB+Enquist&amp;volume=103&amp;issue=37&amp;publication_year=2006&amp;pages=13819-24&amp;pmid=16954197&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=3684025&amp;issue-id=224067&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140408489726608title">Abstract</a></li><li><a href="#S1title">Introduction</a></li><li><a href="#S2title">Methods</a></li><li><a href="#S8title">Results</a></li><li><a href="#S12title">Discussion</a></li><li><a href="#S13title">Supplementary Material</a></li><li><a href="#S14title">Acknowledgments</a></li><li><a href="#idm140408488501648title">References</a></li></ul></div></div><!--post-content--></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC3684025/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC3684025/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC3684025/pdf/nihms467231.pdf">PDF (2.0M)</a></li> | <li><a href="#" data-citationid="PMC3684025" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3684025%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3684025%2F&amp;text=Genome-wide%20Association%20Study%20of%20N370S%20Homozygous%20Gaucher%20Disease%20Reveals%20the%20Candidacy%20of%20CLN8%20gene%20as%20a%20Genetic%20Modifier%20Contributing%20to%20Extreme%20Phenotypic%20Variation" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3684025%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="22388998" data-aiid="3684025" data-aid="3684025" data-iid="224067" data-domainid="319" data-domain="nihpa" data-accid="PMC3684025" data-md5="2112b01cb0fc0811a0d19988147cdbc4"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T08:37:40-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A3183DD53C4D10000000003A80285&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3684025%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=nihpa&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC3684025/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>