<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="molmed"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4749491/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Genome-Wide Association Study of Late-Onset Myasthenia Gravis: Confirmation of TNFRSF11A and Identification of ZBTB10 and Three Distinct HLA Associations</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4749491/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Molecular Medicine"><meta name="citation_title" content="Genome-Wide Association Study of Late-Onset Myasthenia Gravis: Confirmation of TNFRSF11A and Identification of ZBTB10 and Three Distinct HLA Associations"><meta name="citation_authors" content="Michael F Seldin, Omar K Alkhairy, Annette T Lee, Janine A Lamb, Jon Sussman, Ritva Pirskanen-Matell, Fredrik Piehl, Jan J G M Verschuuren, Anna Kostera-Pruszczyk, Piotr Szczudlik, David McKee, Angelina H Maniaol, Hanne F Harbo, Benedicte A Lie, Arthur Melms, Henri-Jean Garchon, Nicholas Willcox, Peter K Gregersen, Lennart Hammarstrom"><meta name="citation_date" content="2015"><meta name="citation_issue" content="1"><meta name="citation_volume" content="21"><meta name="citation_firstpage" content="769"><meta name="citation_doi" content="10.2119/molmed.2015.00232"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4749491/?report=abstract"><meta name="citation_pmid" content="26562150"><meta name="DC.Title" content="Genome-Wide Association Study of Late-Onset Myasthenia Gravis: Confirmation of TNFRSF11A and Identification of ZBTB10 and Three Distinct HLA Associations"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="The Feinstein Institute for Medical Research"><meta name="DC.Contributor" content="Michael F Seldin"><meta name="DC.Contributor" content="Omar K Alkhairy"><meta name="DC.Contributor" content="Annette T Lee"><meta name="DC.Contributor" content="Janine A Lamb"><meta name="DC.Contributor" content="Jon Sussman"><meta name="DC.Contributor" content="Ritva Pirskanen-Matell"><meta name="DC.Contributor" content="Fredrik Piehl"><meta name="DC.Contributor" content="Jan J G M Verschuuren"><meta name="DC.Contributor" content="Anna Kostera-Pruszczyk"><meta name="DC.Contributor" content="Piotr Szczudlik"><meta name="DC.Contributor" content="David McKee"><meta name="DC.Contributor" content="Angelina H Maniaol"><meta name="DC.Contributor" content="Hanne F Harbo"><meta name="DC.Contributor" content="Benedicte A Lie"><meta name="DC.Contributor" content="Arthur Melms"><meta name="DC.Contributor" content="Henri-Jean Garchon"><meta name="DC.Contributor" content="Nicholas Willcox"><meta name="DC.Contributor" content="Peter K Gregersen"><meta name="DC.Contributor" content="Lennart Hammarstrom"><meta name="DC.Date" content="2015"><meta name="DC.Identifier" content="10.2119/molmed.2015.00232"><meta name="DC.Language" content="en"><meta property="og:title" content="Genome-Wide Association Study of Late-Onset Myasthenia Gravis: Confirmation of TNFRSF11A and Identification of ZBTB10 and Three Distinct HLA Associations"><meta property="og:type" content="article"><meta property="og:description" content="To investigate the genetics of late-onset myasthenia gravis (LOMG), we conducted a genome-wide association study imputation of >6 million single nucleotide polymorphisms (SNPs) in 532 LOMG cases (anti–acetylcholine receptor [AChR] antibody ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4749491/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-molmed.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4749491/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4749491/pdf/15_232_seldin.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE8A939DDD57F261000000000B8A06B1.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Bold.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Bold.otf?rev=2.6.1') format('opentype'); font-weight: bold}
@font-face {font-family: MathJax_Main; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Main-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Main-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Math; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Math-Italic.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Math-Italic.otf?rev=2.6.1') format('opentype'); font-style: italic}
@font-face {font-family: MathJax_Caligraphic; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Caligraphic-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Caligraphic-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size1; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size1-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size1-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size2; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size2-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size2-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size3; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size3-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size3-Regular.otf?rev=2.6.1') format('opentype')}
@font-face {font-family: MathJax_Size4; src: url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/woff/MathJax_Size4-Regular.woff?rev=2.6.1') format('woff'), url('https://www.ncbi.nlm.nih.gov/core/mathjax/2.6.1/fonts/HTML-CSS/TeX/otf/MathJax_Size4-Regular.otf?rev=2.6.1') format('opentype')}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4749491%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" class="last" data-ac_dict="pmc-search-autocomplete">PMC</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="archive"><a class="navlink" href="/pmc/journals/325/">Mol Med</a></li><li class="issue-page"><a class="navlink" href="/pmc/issues/253966/">v.21(1); 2015</a></li><li class="accid">PMC4749491</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-molmed.png" alt="Logo of molmed" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://molmed.biomedcentral.com/" target="pmc_ext" ref="reftype=publisher&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Mol Med</a></span>. 2015; 21(1): 769–781. </span></div><div><span class="fm-vol-iss-date">Published online 2015 Nov 10. </span><span class="fm-vol-iss-date">Prepublished online 2015 Nov 10. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.2119%2Fmolmed.2015.00232" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.2119/molmed.2015.00232</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4749491</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/26562150">26562150</a></div></div></div></div><h1 class="content-title">Genome-Wide Association Study of Late-Onset Myasthenia Gravis: Confirmation of <em>TNFRSF11A</em> and Identification of <em>ZBTB10</em> and Three Distinct HLA Associations</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Seldin%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705574983856" co-class="co-affbox">Michael F Seldin</a>,<sup>1</sup><sup></sup> <a href="/pubmed/?term=Alkhairy%20OK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705649578768" co-class="co-affbox">Omar K Alkhairy</a>,<sup>2</sup> <a href="/pubmed/?term=Lee%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705574981632" co-class="co-affbox">Annette T Lee</a>,<sup>3</sup> <a href="/pubmed/?term=Lamb%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705574979648" co-class="co-affbox">Janine A Lamb</a>,<sup>4</sup> <a href="/pubmed/?term=Sussman%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705578357392" co-class="co-affbox">Jon Sussman</a>,<sup>5</sup> <a href="/pubmed/?term=Pirskanen-Matell%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705578355408" co-class="co-affbox">Ritva Pirskanen-Matell</a>,<sup>6</sup> <a href="/pubmed/?term=Piehl%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705578353424" co-class="co-affbox">Fredrik Piehl</a>,<sup>6</sup> <a href="/pubmed/?term=Verschuuren%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705578351440" co-class="co-affbox">Jan J G M Verschuuren</a>,<sup>7</sup> <a href="/pubmed/?term=Kostera-Pruszczyk%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705578349456" co-class="co-affbox">Anna Kostera-Pruszczyk</a>,<sup>8</sup> <a href="/pubmed/?term=Szczudlik%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705578347472" co-class="co-affbox">Piotr Szczudlik</a>,<sup>8</sup> <a href="/pubmed/?term=McKee%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705576374624" co-class="co-affbox">David McKee</a>,<sup>5</sup> <a href="/pubmed/?term=Maniaol%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705576372640" co-class="co-affbox">Angelina H Maniaol</a>,<sup>9</sup> <a href="/pubmed/?term=Harbo%20HF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705576370656" co-class="co-affbox">Hanne F Harbo</a>,<sup>10</sup> <a href="/pubmed/?term=Lie%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705576368672" co-class="co-affbox">Benedicte A Lie</a>,<sup>11</sup> <a href="/pubmed/?term=Melms%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705576366688" co-class="co-affbox">Arthur Melms</a>,<sup>12</sup> <a href="/pubmed/?term=Garchon%20HJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705576364704" co-class="co-affbox">Henri-Jean Garchon</a>,<sup>13</sup> <a href="/pubmed/?term=Willcox%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705576362720" co-class="co-affbox">Nicholas Willcox</a>,<sup>14</sup> <a href="/pubmed/?term=Gregersen%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705577395840" co-class="co-affbox">Peter K Gregersen</a>,<sup>3</sup> and  <a href="/pubmed/?term=Hammarstrom%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150" class="affpopup" co-rid="_co_idm140705577393856" co-class="co-affbox">Lennart Hammarstrom</a><sup>2</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140705574983856"><h3 class="no_margin">Michael F Seldin</h3><p><sup>1</sup>Department of Biochemistry and Molecular Medicine, and Department of Medicine, University of California, Davis, California, United States of America</p><div>Find articles by <a href="/pubmed/?term=Seldin%20MF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Michael F Seldin</a></div></div><div id="_co_idm140705649578768"><h3 class="no_margin">Omar K Alkhairy</h3><p><sup>2</sup>Division of Clinical Immunology, Karolinska Institutet at Karolinska University Hospital Huddinge, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Alkhairy%20OK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Omar K Alkhairy</a></div></div><div id="_co_idm140705574981632"><h3 class="no_margin">Annette T Lee</h3><p><sup>3</sup>The Robert S. Boas Center for Genomics and Human Genetics, Feinstein Institute for Medical Research, North Shore-LIJ Health System, Manhasset, New York, United States of America</p><div>Find articles by <a href="/pubmed/?term=Lee%20AT%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Annette T Lee</a></div></div><div id="_co_idm140705574979648"><h3 class="no_margin">Janine A Lamb</h3><p><sup>4</sup>Centre for Integrated Genomic Medical Research, Manchester Academic Health Science Centre, University of Manchester, Manchester, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Lamb%20JA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Janine A Lamb</a></div></div><div id="_co_idm140705578357392"><h3 class="no_margin">Jon Sussman</h3><p><sup>5</sup>Department of Neurology, Greater Manchester Neuroscience Centre, Manchester, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Sussman%20J%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Jon Sussman</a></div></div><div id="_co_idm140705578355408"><h3 class="no_margin">Ritva Pirskanen-Matell</h3><p><sup>6</sup>Department of Neurology, Karolinska University Hospital Solna, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Pirskanen-Matell%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Ritva Pirskanen-Matell</a></div></div><div id="_co_idm140705578353424"><h3 class="no_margin">Fredrik Piehl</h3><p><sup>6</sup>Department of Neurology, Karolinska University Hospital Solna, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Piehl%20F%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Fredrik Piehl</a></div></div><div id="_co_idm140705578351440"><h3 class="no_margin">Jan J G M Verschuuren</h3><p><sup>7</sup>Department of Neurology, Leiden University Medical Center, Leiden, the Netherlands</p><div>Find articles by <a href="/pubmed/?term=Verschuuren%20JJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Jan J G M Verschuuren</a></div></div><div id="_co_idm140705578349456"><h3 class="no_margin">Anna Kostera-Pruszczyk</h3><p><sup>8</sup>Department of Neurology, Medical University of Warsaw, Warsaw, Poland</p><div>Find articles by <a href="/pubmed/?term=Kostera-Pruszczyk%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Anna Kostera-Pruszczyk</a></div></div><div id="_co_idm140705578347472"><h3 class="no_margin">Piotr Szczudlik</h3><p><sup>8</sup>Department of Neurology, Medical University of Warsaw, Warsaw, Poland</p><div>Find articles by <a href="/pubmed/?term=Szczudlik%20P%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Piotr Szczudlik</a></div></div><div id="_co_idm140705576374624"><h3 class="no_margin">David McKee</h3><p><sup>5</sup>Department of Neurology, Greater Manchester Neuroscience Centre, Manchester, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=McKee%20D%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">David McKee</a></div></div><div id="_co_idm140705576372640"><h3 class="no_margin">Angelina H Maniaol</h3><p><sup>9</sup>Department of Neurology, Oslo University Hospital, Ullevål, Oslo, Norway</p><div>Find articles by <a href="/pubmed/?term=Maniaol%20AH%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Angelina H Maniaol</a></div></div><div id="_co_idm140705576370656"><h3 class="no_margin">Hanne F Harbo</h3><p><sup>10</sup>Department of Neurology, Oslo University Hospital and University of Oslo, Oslo, Norway</p><div>Find articles by <a href="/pubmed/?term=Harbo%20HF%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Hanne F Harbo</a></div></div><div id="_co_idm140705576368672"><h3 class="no_margin">Benedicte A Lie</h3><p><sup>11</sup>Department of Medical Genetics, University of Oslo and Oslo University Hospital, Oslo, Norway</p><div>Find articles by <a href="/pubmed/?term=Lie%20BA%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Benedicte A Lie</a></div></div><div id="_co_idm140705576366688"><h3 class="no_margin">Arthur Melms</h3><p><sup>12</sup>Department of Neurology, Tübingen University Medical Center, Tübingen, Germany, and Neurologische Klinik, Universitätsklinikum Erlangen, Erlangen, Germany</p><div>Find articles by <a href="/pubmed/?term=Melms%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Arthur Melms</a></div></div><div id="_co_idm140705576364704"><h3 class="no_margin">Henri-Jean Garchon</h3><p><sup>13</sup>INSERM U1173, University of Versailles, Campus Paris-Saclay, France</p><div>Find articles by <a href="/pubmed/?term=Garchon%20HJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Henri-Jean Garchon</a></div></div><div id="_co_idm140705576362720"><h3 class="no_margin">Nicholas Willcox</h3><p><sup>14</sup>Nuffield Department of Clinical Neurosciences, Weatherall Institute for Molecular Medicine, University of Oxford, Oxford, United Kingdom</p><div>Find articles by <a href="/pubmed/?term=Willcox%20N%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Nicholas Willcox</a></div></div><div id="_co_idm140705577395840"><h3 class="no_margin">Peter K Gregersen</h3><p><sup>3</sup>The Robert S. Boas Center for Genomics and Human Genetics, Feinstein Institute for Medical Research, North Shore-LIJ Health System, Manhasset, New York, United States of America</p><div>Find articles by <a href="/pubmed/?term=Gregersen%20PK%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Peter K Gregersen</a></div></div><div id="_co_idm140705577393856"><h3 class="no_margin">Lennart Hammarstrom</h3><p><sup>2</sup>Division of Clinical Immunology, Karolinska Institutet at Karolinska University Hospital Huddinge, Stockholm, Sweden</p><div>Find articles by <a href="/pubmed/?term=Hammarstrom%20L%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=26562150">Lennart Hammarstrom</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140705613293040_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140705613293040_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140705613293040_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140705613293040_ai" style="display:none"><div class="fm-affl" id="af1-15_232_seldin" lang="en"><sup>1</sup>Department of Biochemistry and Molecular Medicine, and Department of Medicine, University of California, Davis, California, United States of America</div><div class="fm-affl" id="af2-15_232_seldin" lang="en"><sup>2</sup>Division of Clinical Immunology, Karolinska Institutet at Karolinska University Hospital Huddinge, Stockholm, Sweden</div><div class="fm-affl" id="af3-15_232_seldin" lang="en"><sup>3</sup>The Robert S. Boas Center for Genomics and Human Genetics, Feinstein Institute for Medical Research, North Shore-LIJ Health System, Manhasset, New York, United States of America</div><div class="fm-affl" id="af4-15_232_seldin" lang="en"><sup>4</sup>Centre for Integrated Genomic Medical Research, Manchester Academic Health Science Centre, University of Manchester, Manchester, United Kingdom</div><div class="fm-affl" id="af5-15_232_seldin" lang="en"><sup>5</sup>Department of Neurology, Greater Manchester Neuroscience Centre, Manchester, United Kingdom</div><div class="fm-affl" id="af6-15_232_seldin" lang="en"><sup>6</sup>Department of Neurology, Karolinska University Hospital Solna, Stockholm, Sweden</div><div class="fm-affl" id="af7-15_232_seldin" lang="en"><sup>7</sup>Department of Neurology, Leiden University Medical Center, Leiden, the Netherlands</div><div class="fm-affl" id="af8-15_232_seldin" lang="en"><sup>8</sup>Department of Neurology, Medical University of Warsaw, Warsaw, Poland</div><div class="fm-affl" id="af9-15_232_seldin" lang="en"><sup>9</sup>Department of Neurology, Oslo University Hospital, Ullevål, Oslo, Norway</div><div class="fm-affl" id="af10-15_232_seldin" lang="en"><sup>10</sup>Department of Neurology, Oslo University Hospital and University of Oslo, Oslo, Norway</div><div class="fm-affl" id="af11-15_232_seldin" lang="en"><sup>11</sup>Department of Medical Genetics, University of Oslo and Oslo University Hospital, Oslo, Norway</div><div class="fm-affl" id="af12-15_232_seldin" lang="en"><sup>12</sup>Department of Neurology, Tübingen University Medical Center, Tübingen, Germany, and Neurologische Klinik, Universitätsklinikum Erlangen, Erlangen, Germany</div><div class="fm-affl" id="af13-15_232_seldin" lang="en"><sup>13</sup>INSERM U1173, University of Versailles, Campus Paris-Saclay, France</div><div class="fm-affl" id="af14-15_232_seldin" lang="en"><sup>14</sup>Nuffield Department of Clinical Neurosciences, Weatherall Institute for Molecular Medicine, University of Oxford, Oxford, United Kingdom</div><div id="c1-15_232_seldin">Address correspondence to: Michael F Seldin, Department of Biochemistry and Molecular Medicine, and Department of Medicine, University of California, Davis, CA 95616. Phone: 530-754-6016; Fax: 530-754-6015; E-mail: <a href="mailto:dev@null" data-email="ude.sivadcu@nidlesfm" class="oemail">ude.sivadcu@nidlesfm</a>.</div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140705613293040_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2015 Nov 4; Accepted 2015 Nov 9.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140705613293040_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  2015, The Feinstein Institute for Medical Research</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4749491/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC4749491/bin/15_232_Seldin_Suppl.pdf" data-ga-action="click_feat_suppl">15_232_Seldin_Suppl.pdf</a><span style="color:gray"> (1.2M)</span></div><div class="small guid">GUID:&nbsp;F157F03E-B052-4BA5-89CF-E77ACDC6EE46</div></div></dd></dl></div><div id="idm140705577889472" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140705577889472title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">To investigate the genetics of late-onset myasthenia gravis (LOMG), we conducted a genome-wide association study imputation of &gt;6 million single nucleotide polymorphisms (SNPs) in 532 LOMG cases (anti–acetylcholine receptor [AChR] antibody positive; onset age ≥50 years) and 2,128 controls matched for sex and population substructure. The data confirm reported <em>TNFRSF11A</em> associations (rs4574025, <em>P</em> = 3.9 × 10<sup>−7</sup>, odds ratio [OR] 1.42) and identify a novel candidate gene, <em>ZBTB10</em>, achieving genome-wide significance (rs6998967, <em>P</em> = 8.9 × 10<sup>−10</sup>, OR 0.53). Several other SNPs showed suggestive significance including rs2476601 (<em>P</em> = 6.5 × 10<sup>−6</sup>, OR 1.62) encoding the PTPN22 R620W variant noted in early-onset myasthenia gravis (EOMG) and other autoimmune diseases. In contrast, EOMG-associated SNPs in <em>TNIP1</em> showed no association in LOMG, nor did other loci suggested for EOMG. Many SNPs within the major histocompatibility complex (MHC) region showed strong associations in LOMG, but with smaller effect sizes than in EOMG (highest OR ~2 versus ~6 in EOMG). Moreover, the strongest associations were in opposite directions from EOMG, including an OR of 0.54 for <em>DQA1*05:01</em> in LOMG (<em>P</em> = 5.9 × 10<sup>−12</sup>) versus 2.82 in EOMG (<em>P</em> = 3.86 × 10<sup>−45</sup>). Association and conditioning studies for the MHC region showed three distinct and largely independent association peaks for LOMG corresponding to (a) MHC class II (highest attenuation when conditioning on <em>DQA1</em>), (b) <em>HLA-A</em> and (c) MHC class III SNPs. Conditioning studies of human leukocyte antigen (HLA) amino acid residues also suggest potential functional correlates. Together, these findings emphasize the value of subgrouping myasthenia gravis patients for clinical and basic investigations and imply distinct predisposing mechanisms in LOMG.</p></div></div><div id="__sec1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">INTRODUCTION</h2><p id="__p2" class="p p-first">Myasthenia gravis (MG) is an uncommon idiopathic autoimmune disease characterized by impaired neuromuscular transmission and fatigable muscle weakness. Its overall incidence is ~3/100,000, and its prevalence is ~2 per 10,000. Both incidence and prevalence have apparently increased in recent decades, especially in subjects aged &gt;50 years at onset (<a href="#b1-15_232_seldin" rid="b1-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532937">1</a>–<a href="#b3-15_232_seldin" rid="b3-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532980">3</a>). Over 80% of patients with generalized MG have antibodies (Abs) to the skeletal muscle acetylcholine receptor (AChR), which are generally accepted to be pathogenic (<a href="#b2-15_232_seldin" rid="b2-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532989">2</a>,<a href="#b4-15_232_seldin" rid="b4-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532967">4</a>). About 10% of them have thymomas, which mostly generate (and export) abundant T cells. The neoplastic thymic epithelial cells usually fail to express both human leukocyte antigen (HLA) class II and the autoimmune regulator gene (AIRE). The lack of expression of these genes presumably impairs self-tolerance induction in T cells developing in thymomas (which has been implicated in these patients’ distinctive clinical and autoantibody profiles) (<a href="/pmc/articles/PMC4749491/table/t1-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t1-15_232_seldin" rid-ob="ob-t1-15_232_seldin" co-legend-rid=""><span>Table 1</span></a>) (<a href="#b2-15_232_seldin" rid="b2-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532951">2</a>,<a href="#b5-15_232_seldin" rid="b5-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532927">5</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t1-15_232_seldin"><h3>Table 1</h3><!--caption a7--><div class="caption"><p id="__p63">Subgrouping of MG patients in previous studies.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140705646659312" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th rowspan="1" colspan="1" valign="bottom" align="center">EOMG</th><th rowspan="1" colspan="1" valign="bottom" align="center">LOMG</th><th rowspan="1" colspan="1" valign="bottom" align="center">Thymoma-MG</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">MG onset age (years)</td><td rowspan="1" colspan="1" valign="top" align="left">10–40, 50 or 60 (puberty to menopause)</td><td rowspan="1" colspan="1" valign="top" align="left">&gt;40, &gt;50 or &gt;60</td><td rowspan="1" colspan="1" valign="top" align="left">&gt;10</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Female:male</td><td rowspan="1" colspan="1" valign="top" align="left">≥ 3:1</td><td rowspan="1" colspan="1" valign="top" align="left">1:1.7</td><td rowspan="1" colspan="1" valign="top" align="left">1:1</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">Thymic histology</td><td rowspan="1" colspan="1" valign="top" align="left">Medullary lymph node-like infiltrates (&gt;80%)</td><td rowspan="1" colspan="1" valign="top" align="left">Atrophy/normal-for-age</td><td rowspan="1" colspan="1" valign="top" align="left">Epithelial neoplasm</td></tr><tr><td colspan="4" rowspan="1" valign="top" align="left">Autoantibodies (at diagnosis)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> AChR</td><td rowspan="1" colspan="1" valign="top" align="left">100% by definition</td><td rowspan="1" colspan="1" valign="top" align="left">100% by definition</td><td rowspan="1" colspan="1" valign="top" align="left">≤100%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"> Titin</td><td rowspan="1" colspan="1" valign="top" align="left">&lt;5%</td><td rowspan="1" colspan="1" valign="top" align="left">&gt;70%</td><td rowspan="1" colspan="1" valign="top" align="left">&gt;95%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">IFN-α and/or IL12</td><td rowspan="1" colspan="1" valign="top" align="left">&lt;5%</td><td rowspan="1" colspan="1" valign="top" align="left">~60%</td><td rowspan="1" colspan="1" valign="top" align="left">&gt;70%</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">HLA associations (references)</td><td rowspan="1" colspan="1" valign="top" align="left">DQ2–DR3–B8–A1: especially in females (<a href="#b10-15_232_seldin" rid="b10-15_232_seldin" class=" bibr popnode">10</a>,<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode">15</a>)</td><td rowspan="1" colspan="1" valign="top" align="left">DR15 (<a href="#b6-15_232_seldin" rid="b6-15_232_seldin" class=" bibr popnode">6</a>); DR2/B7 in males (<a href="#b10-15_232_seldin" rid="b10-15_232_seldin" class=" bibr popnode">10</a>); DR7 if titin-Ab<sup>+</sup> (<a href="#b12-15_232_seldin" rid="b12-15_232_seldin" class=" bibr popnode">12</a>)</td><td rowspan="1" colspan="1" valign="top" align="left">None confirmed</td></tr></tbody></table></div><div id="largeobj_idm140705646659312" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4749491/table/t1-15_232_seldin/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tfn1-15_232_seldin"><p id="__p64" class="p p-first-last">The summary is based on studies of European patients. The specific autoantibodies are rare in other diseases, apart from those against IFN-α values, which are found in ≤100% of patients with autoimmune polyendocrine syndrome type 1 (<a href="#b5-15_232_seldin" rid="b5-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532917">5</a>).</p></div></div></div><p id="__p3">Patients with generalized AChR-Ab–positive MG without thymoma are generally sub-grouped into those with early-onset myasthenia gravis (EOMG) or late-onset myasthenia gravis (LOMG) (<a href="/pmc/articles/PMC4749491/table/t1-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t1-15_232_seldin" rid-ob="ob-t1-15_232_seldin" co-legend-rid=""><span>Table 1</span></a>), with a cutoff at ages 40, 45, 50 or even 60 years in different studies (6–10; rev. in 5). As summarized in <a href="/pmc/articles/PMC4749491/table/t1-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t1-15_232_seldin" rid-ob="ob-t1-15_232_seldin" co-legend-rid=""><span>Table 1</span></a>, these subgroups consistently show contrasting sex ratios, thymic histology and HLA associations (<a href="#b6-15_232_seldin" rid="b6-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532924">6</a>,<a href="#b10-15_232_seldin" rid="b10-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532925">10</a>–<a href="#b13-15_232_seldin" rid="b13-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532959">13</a>). Moreover, several autoantibodies are almost uniquely shared by patients with LOMG or thymomatous MG but not EOMG, which remains unexplained.</p><p id="__p4">We have previously reported a genome-wide association study (GWAS) in 649 EOMG patients using rigid entry criteria (onset age &lt;40 or &lt;45 years if the thymus showed hyperplasia). We confirmed the association with <em>PTPN22</em> (<a href="#b14-15_232_seldin" rid="b14-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532952">14</a>) and identified a novel association with a <em>TNIP1</em> coding single nucleotide polymorphism (SNP) and defined HLA associations (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532914">15</a>). The study mapped the strongest HLA association to <em>HLA-B*08</em> rather than class II SNPs in linkage disequilibrium (LD) with this class I allele (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532942">15</a>).</p><p id="__p5" class="p p-last">Recently, another group reported a GWAS in EOMG (n = 235) and LOMG patients (n = 737), both separately and combined (<a href="#b16-15_232_seldin" rid="b16-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550442222">16</a>). Although not supporting all of the previously reported associations (see Discussion), this study provided strong evidence for an association with <em>TNFRSF11A</em> (alias RANK) in the LOMG subset defined by age of onset &gt;40 years (<a href="#b16-15_232_seldin" rid="b16-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550442223">16</a>). The current study was undertaken to further define genetic associations in stringently defined LOMG patients (onset ≥50 and ≥60 years), again focusing on Northern Europeans, using population substructure matching of publicly available control genotypes, and imputation methods extending to specific HLA variants and amino acids (AAs). The results suggest a biological basis for defining a cutoff for age of onset between EOMG and LOMG and again imply distinct genetic and environmental factors in predisposition in these groups.</p></div><div id="__sec2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec2title">MATERIALS AND METHODS</h2><div id="__sec3" class="sec sec-first"><p></p><h3 id="__sec3title" class="inline">Study Subjects </h3><p id="__p6" class="p p-first">All the LOMG patients included in these studies were European and met the following criteria: (a) clinical diagnostic criteria for MG; (b) positive for anti-AChR Ab; (c) no radiological evidence of thymoma; and (d) onset age ≥ 50 years. Cases were collected from Stockholm, Sweden; Oslo, Norway; Manchester and Oxford, England; Paris, France; Leiden, the Netherlands; Tübingen, Germany; and Warsaw, Poland. Samples for all participants of this study were collected under institutionally approved inform consent (at the respective institutes) and complied with the Helsinki Declaration as revised in 1983.</p><p id="__p7" class="p p-last">Of a combined total of 557 LOMG samples collected, 25 were excluded because they failed checks for quality control (QC) (&lt;90% complete genotyping data), cryptic relationship (<span class="MathJax_Preview"><math><mover accent="true"><mi>p</mi><mo>̂</mo></mover></math></span><span class="MathJax MathJax_Processed" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-1" role="math"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 121%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-2"><span class="mover" id="MathJax-Span-3"><span style="display: inline-block; position: relative; width: 0.585em; height: 0px;"><span style="position: absolute; clip: rect(3.38em, 1000.5em, 4.326em, -1000em); top: -3.977em; left: 0em;"><span class="mi" id="MathJax-Span-4" style="font-family: MathJax_Math; font-style: italic;">p</span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span><span style="position: absolute; clip: rect(3.128em, 1000.39em, 3.601em, -1000em); top: -4.017em; left: 0.085em;"><span class="mo" id="MathJax-Span-5" style=""><span style="font-family: MathJax_Main;">ˆ</span></span><span style="display: inline-block; width: 0px; height: 3.977em;"></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-1"><math><mover accent="true"><mi>p</mi><mo>̂</mo></mover></math></script> &gt;0.15, using PLINK [<a href="#b17-15_232_seldin" rid="b17-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532997">17</a>]), ancestry and/or SNP heterozygosity (&gt;3 standard deviation above or below mean). The remaining 532 patients were matched 1:4 with controls available from these same populations plus others from European–American populations (<a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S1</a>).</p></div><div id="__sec4" class="sec"><p></p><h3 id="__sec4title" class="inline">Quality Control </h3><p id="__p8" class="p p-first">In addition to the QC for individual samples, the SNP data were carefully reviewed, and exclusion criteria were applied to minimize potential batch effects as described below. The application of these criteria was particularly important given the derivation of genotypes from multiple platforms typed in different laboratories. We included only SNPs with &lt;5% missing data, Hardy-Weinberg equilibrium <em>P</em> values &gt;1 × 10<sup>−4</sup> in controls and &gt;1 × 10<sup>−5</sup> in combined cases and controls, and minor allele frequencies &gt;0.02. These procedures were applied in a stepwise approach separately for each data set and after combining the data sets. Thus, for each of the separately derived control genotyping sets, SNPs were excluded if they failed the above criteria within the individual control set (platform and laboratory) or in combination with any of the other control groups, or in the complete data set. The Hardy-Weinberg criteria were applied after exclusion of non-European individuals. Finally, after selection of the control data set, SNPs were excluded if the allele frequencies differed by &gt;10% between different control groups.</p><p id="__p9" class="p p-last">We used two different genotyping platforms: the 370K or compatible Illumina platforms for set 1 (n = 214 cases and 856 controls) and the 2.5 million Illumina platform for set 2 (n = 318 cases and 1,272 controls (<a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S1</a>). For sets 1 and 2, a total of 280,929 and 1,406,133 SNPs, respectively, passed all data filters. Because only ~150,000 SNPs were shared between platforms, imputation (see below) was performed separately for each platform, and the data were combined and resubjected to the same QC metrics described above.</p></div><div id="__sec5" class="sec"><p></p><h3 id="__sec5title" class="inline">Ancestry and Case-Control Matching </h3><p id="__p10" class="p p-first">European ancestry in both cases and controls was determined using a panel of 2,440 SNPs (intermarker <em>r</em><sup>2</sup> &lt;0.05, minor allele frequency &gt;0.05) and analyzed using STRUCTURE v2.3.4 (<a href="#b18-15_232_seldin" rid="b18-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532940">18</a>,<a href="#b19-15_232_seldin" rid="b19-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532938">19</a>) and subjects of known European, Amerindian, East Asian, and West African origin as previously described (<a href="#b20-15_232_seldin" rid="b20-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532920">20</a>). We used STRUCTURE to exclude non-European and admixed study participants, because this Bayesian clustering method allows exclusion/inclusion criteria to be set using reference populations. Unlike principal component analysis (PCA), the clustering algorithm can be run under conditions that are only marginally affected by the inclusion of the unknown samples. Briefly, analyses were performed using &gt;100,000 resamplings and &gt;50,000 burn-in cycles under the admixture model and using the prior population information for the reference populations. Runs were performed under the λ = 1 option, where λ estimates the prior probability of the allele frequency and is based on the Dirichlet distribution of allele frequencies. Subjects with &gt;10% non-European ancestry were excluded from further analyses.</p><p id="__p11">Each case was then matched with four controls based on PCAs performed using 25,914 selected SNPs shared across platforms (both sets 1 and 2) using EIGEN-SOFT v5 (<a href="#b21-15_232_seldin" rid="b21-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532981">21</a>). SNPs were chosen from overlapping SNPs with minimal inter-marker LD (<em>r</em><sup>2</sup> &lt; 0.05) in Windows of 2 Mb after exclusion of the major histocompatibility complex (MHC) and genomic regions with large inversions.</p><p id="__p12">Matching (4 controls:1 case) was performed separately for males and females and also for set 1 (370K Illumina platform, 87 female and 127 male cases) and set 2 (HumanOmni 2.5M-8v1-1 chip Illumina platform, 113 female and 205 male cases) using procedures previously described (EOMG) (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532972">15</a>). A total of 3,190 female and 2,159 male European control genotypes for set 1 and 5,392 female and 3,940 male European control genotypes for set 2 were available for matching after QC (<a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S1</a>). Matching used the first five principal components (PCs) (which accounted for &gt;99% of EIGENVALUE variance in Tracy-Widom significant PCs). As shown in <a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S1</a>, there were a total of 856 controls for set 1 (4× the number of set 1 cases) and 1,272 controls for set 2 (4× the number of set 2 cases).</p><p id="__p13" class="p p-last">After matching, the first 7 PCs showed significant Tracy-Widom statistics, but just the first 3 PCs explained over 99% of their total EIGENVALUE variance for these PCs. After matching, λ<sub>gc</sub> was 1.039 without correction and decreased to 1.029 after controlling for these three PCs. PCA and scree plots are shown in <a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Figures S1 and S2</a>.</p></div><div id="__sec6" class="sec"><p></p><h3 id="__sec6title" class="inline">Genome-Wide Imputation of SNPs </h3><p id="__p14" class="p p-first-last">Imputation was performed by using a 1000 Genomes Phase 1 V3 reference set and the University of Michigan Imputation Server (<a href="https://imputationserver.sph.umich.edu/start.html#!run/minimac" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://imputationserver.sph.umich.edu/start.html#!run/minimac</a>). This method uses a computationally efficient implementation of the Markov chain-based haplotyper (MaCH) algorithm with prephasing using SHAPEIT (<a href="#b22-15_232_seldin" rid="b22-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512533000">22</a>) as previously described (<a href="#b23-15_232_seldin" rid="b23-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532950">23</a>). The program outputs the most likely genotype and provides <em>r</em><sup>2</sup> metrics for quality measurement. We performed imputation separately for set 1 and set 2 genotypes. A total of 6,175,472 genotypes showed <em>r</em><sup>2</sup> &gt;0.8 common to both set 1 and set 2, and all QC criteria as described above that were applied to these joined set 1 and set 2 genotypes. (Note: testing of strongly suggestive associated SNPs (<em>P</em> values &lt;10<sup>−7</sup>) with genotypes that met lower <em>r</em><sup>2</sup> thresholds in one or both sets showed <em>P</em> values &gt;5 × 10<sup>−5</sup> after regional imputation and testing. We therefore restricted further analyses to genotypes with <em>r</em><sup>2</sup> &gt; 0.8 in both sets 1 and 2.) The combined set 1 and 2 case (532) and control (2,128) imputed genotypes (6,175,472) were used as the data set for initial analyses.</p></div><div id="__sec7" class="sec"><p></p><h3 id="__sec7title" class="inline">Regional Imputation of SNPs and Insertion/Deletion Polymorphisms </h3><p id="__p15" class="p p-first-last">For any SNP that showed suggestive association (<em>P</em> &lt; 10<sup>−5</sup>) in the screening imputation (described above), we performed regional imputation using Impute2 (IMPUTE v2.3.1) (<a href="https://mathgen.stats.ox.ac.uk/impute/impute_v2.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/impute/impute_v2.html</a>) (<a href="#b24-15_232_seldin" rid="b24-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532986">24</a>) under recommended settings and the afore-noted 1,000-genome phase 1v3 reference set. This software provides genotype probabilities (rather than base assignments), enabling a more accurate assessment of statistical support. Overall errors compared favorably with other approaches (<a href="#b24-15_232_seldin" rid="b24-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532982">24</a>). The intervals examined (excluding the MHC) included at least a three order of magnitude fall-off of significance from the association peak and a minimum of 250 kb. For the MHC, we examined a 4-Mb interval (chromosome 6, ~29–33 Mb, HG19/HG38). We applied the algorithm with 250-kb buffers (in addition to the interval under examination) and without prephasing (standard IMPUTE2 MCMC algorithm) as recommended by the developers for more accurate, although less computationally efficient, imputation.</p></div><div id="__sec8" class="sec"><p></p><h3 id="__sec8title" class="inline">Imputation of HLA Antigens </h3><p id="__p16" class="p p-first-last">To impute HLA antigens, we used a separate reference data set collected by the Type 1 Diabetes Genetics Consortium and the SNP2HLA software, as described by its developers (<a href="#b25-15_232_seldin" rid="b25-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532966">25</a>). The Beagle software package (<a href="#b26-15_232_seldin" rid="b26-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532934">26</a>) was used for imputation in this data set (compatible with this software). Here, we also used genotype probabilities for association testing; our final data consisted of only antigens, AAs and SNPs with information scores &gt;0.8 in SNPTEST analyses.</p></div><div id="__sec9" class="sec"><p></p><h3 id="__sec9title" class="inline">Association Testing Statistics </h3><p id="__p17" class="p p-first-last">For all association results, analyses included covariates for sex, genotype set and the first three PCs (as discussed above). All association tests were done by using the combined imputed SNP data set as described above. For autosomal data, we used SNPTEST V2.3 (<a href="https://mathgen.stats.ox.ac.uk/genet-ics_software/snptest/snptest.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mathgen.stats.ox.ac.uk/genet-ics_software/snptest/snptest.html</a>) for the primary association analyses for the imputed genotypes, as well as for conditional analyses. For the X chromosome analyses we also used SNPTEST V2.5. Analyses included an X inactivation model and a model that allows for heterogeneity (stratify_on option). With the exception of the screening imputed genotypes, genotype probabilities (and a minimum information score of 0.8) were used in all analyses.</p></div><div id="__sec10" class="sec"><p></p><h3 id="__sec10title" class="inline">Epigenetic Markers </h3><p id="__p18" class="p p-first-last">To assess possible regulatory function, we used RegulomeDB (<a href="#b27-15_232_seldin" rid="b27-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532928">27</a>). This database (search engine) annotates SNPs for predicted and known regulatory region for DNase hypersensitivity, transcription factor binding sites and promoter regions. It includes the current Encyclopedia of DNA Elements (ENCODE) releases and chromatin states from the Roadmap Epigenome Consortium and uses multiple resources that include DNase footprinting, position weight matrices and DNA methylation information (for further description of RegulomeDB see <a href="http://regulomedb.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://regulomedb.org</a>). For this assessment, we examined all SNPs that were in at least weak LD (<em>r</em><sup>2</sup> &gt; 0.25) to the strongest associated SNP and had annotations that showed possible regulatory function. Conditional analyses were then performed, and the SNP with the strongest likelihood of function was presented in Results. We also tested combinations of putative functional SNPs. However, conditional analyses on combinations of SNPs did not enhance the signal attenuation compared with the SNP presented in Results for <em>ZBTB10</em>, <em>TNFRSF11A</em> or the class III MHC region.</p></div><div id="__sec11" class="sec sec-last"><p></p><h3 id="__sec11title" class="inline">Other Software </h3><p id="__p19" class="p p-first">For graphical presentation of association results, we used qqman for quantile–quantile (Q–Q) and Manhattan plots (<a href="http://cran.r-project.org/web/packages/qqman/qqman.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://cran.r-project.org/web/packages/qqman/qqman.pdf</a>) and LocusZoom for chromosome region association plots (<a href="#b28-15_232_seldin" rid="b28-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532916">28</a>).</p><p id="__p20" class="p p-last">All <a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">supplementary materials</a> are available online at <a href="http://www.molmed.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.molmed.org</a>.</p></div></div><div id="__sec12" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec12title">RESULTS</h2><div id="__sec13" class="sec sec-first"><p></p><h3 id="__sec13title" class="inline">Screening for Association with LOMG </h3><p id="__p21" class="p p-first">To screen for putative candidate loci, we selected AChR-Ab–positive LOMG cases with no radiologic evidence of thymomas (see Materials and Methods and <a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S2</a>). We used two age of onset cutoffs (≥50 and ≥60 years) based both on previous studies (see Introduction) and the bimodal distribution of MG with age of onset (<a href="/pmc/articles/PMC4749491/figure/f1-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f1-15_232_seldin" rid-ob="ob-f1-15_232_seldin" co-legend-rid="lgnd_f1-15_232_seldin"><span>Figure 1</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="f1-15_232_seldin" co-legend-rid="lgnd_f1-15_232_seldin"><a href="/pmc/articles/PMC4749491/figure/f1-15_232_seldin/" target="figure" rid-figpopup="f1-15_232_seldin" rid-ob="ob-f1-15_232_seldin"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140705578218848" class="figure"><a href="/pmc/articles/PMC4749491/figure/f1-15_232_seldin/" target="figure" rid-figpopup="f1-15_232_seldin" rid-ob="ob-f1-15_232_seldin"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4749491_15_232_seldinf1.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 15_232_seldinf1.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4749491/bin/15_232_seldinf1.jpg"></a></div><div id="largeobj_idm140705578218848" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4749491/figure/f1-15_232_seldin/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f1-15_232_seldin"><div><a class="figpopup" href="/pmc/articles/PMC4749491/figure/f1-15_232_seldin/" target="figure" rid-figpopup="f1-15_232_seldin" rid-ob="ob-f1-15_232_seldin">Figure 1</a></div><!--caption a7--><div class="caption"><p id="__p58">Relationship of sex and age of onset for AChR Ab + MG. The histogram shows the number of female (blue) and male (red) MG cases for 10-year age of onset intervals starting at decade 2 (age of onset 10–19 years). The cases include EOMG (onset age by definition &lt;40 years if no histology or &lt;45 years if histology shows thymic hyperplasia, n = 649) (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532999">15</a>), unclassified (onset age 40 to &lt; 50 years, n = 23) and LOMG (onset age ≥50 years, n = 532) (current study).</p></div></div></div><p id="__p22">Since two disparate SNP GWAS chipsets were used for genotyping, we melded the data using a genome-wide imputation strategy (see Materials and Methods for additional details). We tested for associations with variants imputed with high confidence (<em>r</em><sup>2</sup> ≥ 0.8, required for each imputed variant from both GWAS panels); the analyses included over 6 million variants with minor allele frequencies ≥0.02. The association tests included covariates to control for residual population substructure, and sex.</p><p id="__p23">As expected in any autoimmune disease, we found major contributions from the MHC region on chromosome 6 in patients with cutoff ages ≥50 and ≥60 years, using Q–Q analyses and Manhattan plots (<a href="/pmc/articles/PMC4749491/figure/f2-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-15_232_seldin" rid-ob="ob-f2-15_232_seldin" co-legend-rid="lgnd_f2-15_232_seldin"><span>Figure 2</span></a>). HLA associations were further refined with more precise imputation methods, as detailed later.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="f2-15_232_seldin" co-legend-rid="lgnd_f2-15_232_seldin"><a href="/pmc/articles/PMC4749491/figure/f2-15_232_seldin/" target="figure" rid-figpopup="f2-15_232_seldin" rid-ob="ob-f2-15_232_seldin"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140705578215456" class="figure"><a href="/pmc/articles/PMC4749491/figure/f2-15_232_seldin/" target="figure" rid-figpopup="f2-15_232_seldin" rid-ob="ob-f2-15_232_seldin"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4749491_15_232_seldinf2.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 15_232_seldinf2.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4749491/bin/15_232_seldinf2.jpg"></a></div><div id="largeobj_idm140705578215456" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4749491/figure/f2-15_232_seldin/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f2-15_232_seldin"><div><a class="figpopup" href="/pmc/articles/PMC4749491/figure/f2-15_232_seldin/" target="figure" rid-figpopup="f2-15_232_seldin" rid-ob="ob-f2-15_232_seldin">Figure 2</a></div><!--caption a7--><div class="caption"><p id="__p59">Quantile–quantile (Q–Q) and Manhattan plots for LOMG GWAS results. (A) Results are from screening study (for Minimac imputation, see Materials and Methods). (B) Age of onset ≥50 years (n = 532). (C, D) Age of onset ≥60 years (n = 449). (A, C) The observed deviation above the diagonal (expected Q–Q distribution) indicates evidence for genetic association. (B, D) Blue and orange show alternate chromosomes.</p></div></div></div><p id="__p24" class="p p-last">The results also suggest other loci on chromosomes 1, 5, 8, 12, 18 and 21 (<a href="/pmc/articles/PMC4749491/figure/f2-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-15_232_seldin" rid-ob="ob-f2-15_232_seldin" co-legend-rid="lgnd_f2-15_232_seldin"><span>Figure 2</span></a>, <a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S3</a>). Even after removal of the extended MHC region, the Q–Q plots still show deviations from the expected <em>P</em> value distribution (<a href="/pmc/articles/PMC4749491/figure/f2-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-15_232_seldin" rid-ob="ob-f2-15_232_seldin" co-legend-rid="lgnd_f2-15_232_seldin"><span>Figures 2A, C</span></a>), especially in the stronger-powered subgroup with onset at ≥50 years of age. We found suggestive evidence for association at the <em>PTPN22</em> locus, as previously noted in EOMG (<a href="#b14-15_232_seldin" rid="b14-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532949">14</a>,<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532996">15</a>) (<a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S3</a>). In contrast, there was only marginal evidence for any association signal for <em>TNIP1</em> (all <em>P</em> values &gt;0.001 within 100 kb flanking this gene or in <em>r</em><sup>2</sup> &gt; 0.25), the strongest non-HLA signal we observed in EOMG (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532935">15</a>).</p></div><div id="__sec14" class="sec"><p></p><h3 id="__sec14title" class="inline">More Precise Imputation of Potentially Associated Loci </h3><p id="__p25" class="p p-first">To further define the loci thus imputed (<em>P</em> &lt; 10<sup>−5</sup>), we next applied a more accurate but computationally less efficient imputation method without prephasing or applying pruning procedures (see Materials and Methods). Several of the above candidates maintained either suggestive or significant associations (<a href="/pmc/articles/PMC4749491/table/t2-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t2-15_232_seldin" rid-ob="ob-t2-15_232_seldin" co-legend-rid=""><span>Table 2</span></a>, <a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Tables S4 and S5</a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t2-15_232_seldin"><h3>Table 2</h3><!--caption a7--><div class="caption"><p id="__p65">Strongest association signals for LOMG.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140705577022608" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Variant</th><th rowspan="1" colspan="1" valign="bottom" align="center">Chromosome</th><th rowspan="1" colspan="1" valign="bottom" align="left">Position (HG19)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Minor Allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Major Allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Case MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">Control MAF</th><th rowspan="1" colspan="1" valign="bottom" align="center">Additive <em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">OR age ≥60 years</th><th rowspan="1" colspan="1" valign="bottom" align="center">Gene/location</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">rs111945767</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">32517045</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.46</td><td rowspan="1" colspan="1" valign="top" align="center">0.62</td><td rowspan="1" colspan="1" valign="top" align="center">3.13 × 10<sup>−17</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.52 (0.44–0.61)</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="left">MHC region</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6998967</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="left">81364205</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.17</td><td rowspan="1" colspan="1" valign="top" align="center">8.86 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.53 (0.42–0.65)</td><td rowspan="1" colspan="1" valign="top" align="center">0.53</td><td rowspan="1" colspan="1" valign="top" align="left">ZBTB10/34 kb 5′</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4574025</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="left">60009814</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.55</td><td rowspan="1" colspan="1" valign="top" align="center">0.47</td><td rowspan="1" colspan="1" valign="top" align="center">3.91 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.42 (1.24–1.63)</td><td rowspan="1" colspan="1" valign="top" align="center">1.31</td><td rowspan="1" colspan="1" valign="top" align="left">TNFRSF11/intron</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs9963862</td><td rowspan="1" colspan="1" valign="top" align="center">18</td><td rowspan="1" colspan="1" valign="top" align="left">20587911</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">4.19 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.44 (1.25–1.66)</td><td rowspan="1" colspan="1" valign="top" align="center">1.47</td><td rowspan="1" colspan="1" valign="top" align="left">RBBP8/intron</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs12653117</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="left">9386585</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">5.34 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.76 (1.42–2.17)</td><td rowspan="1" colspan="1" valign="top" align="center">1.75</td><td rowspan="1" colspan="1" valign="top" align="left">SEMA5A/intron</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs6914704</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">1759880</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.42</td><td rowspan="1" colspan="1" valign="top" align="center">0.49</td><td rowspan="1" colspan="1" valign="top" align="center">2.32 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.72 (0.63–0.83)</td><td rowspan="1" colspan="1" valign="top" align="center">0.74</td><td rowspan="1" colspan="1" valign="top" align="left">GMDS/intron</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4128527</td><td rowspan="1" colspan="1" valign="top" align="center">8</td><td rowspan="1" colspan="1" valign="top" align="left">72682693</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">C</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">3.68 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.62 (0.50–0.77)</td><td rowspan="1" colspan="1" valign="top" align="center">0.62</td><td rowspan="1" colspan="1" valign="top" align="left">MSC/71 kb 5′</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">5:31213962:D</td><td rowspan="1" colspan="1" valign="top" align="center">5</td><td rowspan="1" colspan="1" valign="top" align="left">31213962</td><td rowspan="1" colspan="1" valign="top" align="center">TC</td><td rowspan="1" colspan="1" valign="top" align="center">T</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">3.84 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.84 (1.44–2.37)</td><td rowspan="1" colspan="1" valign="top" align="center">1.68</td><td rowspan="1" colspan="1" valign="top" align="left">CDH6/intron</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs4518467</td><td rowspan="1" colspan="1" valign="top" align="center">6</td><td rowspan="1" colspan="1" valign="top" align="left">92196398</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">4.31 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.77 (1.40–2.23)</td><td rowspan="1" colspan="1" valign="top" align="center">1.94</td><td rowspan="1" colspan="1" valign="top" align="left">MIR4643/35 kb 5′</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs2476601</td><td rowspan="1" colspan="1" valign="top" align="center">1</td><td rowspan="1" colspan="1" valign="top" align="left">1143777568</td><td rowspan="1" colspan="1" valign="top" align="center">A</td><td rowspan="1" colspan="1" valign="top" align="center">G</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">6.50 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.62 (1.31–1.99)</td><td rowspan="1" colspan="1" valign="top" align="center">1.66</td><td rowspan="1" colspan="1" valign="top" align="left">PTPN22/coding</td></tr></tbody></table></div><div id="largeobj_idm140705577022608" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4749491/table/t2-15_232_seldin/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tfn2-15_232_seldin"><p id="__p66" class="p p-first-last">Data are shown for onset age ≥50 years, except for the OR for onset age ≥60 years as comparison. The strongest <em>P</em> values (if <em>P</em> &lt; 10<sup>−5</sup>) for putative associations of each region are shown after re-analyses using IMPUTE2 including relevant covariates (see Materials and Methods). Additional SNPs for each region are shown in <a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S4</a>. The OR and 95% CI are shown for the minor allele. The minor allele and minor allele frequency (MAF) was determined based on the entire data set (controls matched 4:1 with cases).</p></div></div></div><p id="__p26" class="p p-last">Outside the MHC region, only the chromosome 8 SNP, rs6998967, 5′ of <em>ZBTB10</em>, reached conservative genome-wide criteria (5 × 10<sup>−8</sup>) for association using either cutoff age. We also found suggestive evidence for associations in eight disparate regions (<a href="/pmc/articles/PMC4749491/table/t2-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t2-15_232_seldin" rid-ob="ob-t2-15_232_seldin" co-legend-rid=""><span>Table 2</span></a>); they include the same R620W variant of PTPN22 noted previously, and one for the tumor necrosis factor receptor superfamily, member 11a, NFKB activator (<em>TNFRSF11A</em>, <em>alias RANK</em> and its particular SNP, rs4574025), reported recently in LOMG (<a href="#b16-15_232_seldin" rid="b16-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550442220">16</a>). For each of the loci achieving at least suggestive significance (<em>P</em> &lt; 10<sup>−5</sup>), odds ratios (ORs) were similar for the more stringent cutoff age (≥60 years) (<a href="/pmc/articles/PMC4749491/table/t2-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t2-15_232_seldin" rid-ob="ob-t2-15_232_seldin" co-legend-rid=""><span>Table 2</span></a> and <a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S4</a>).</p></div><div id="__sec15" class="sec"><p></p><h3 id="__sec15title" class="inline">Conditional Analyses of the <em>ZBTB10</em> and <em>TNFRSF11A</em> Locus SNPs </h3><p id="__p27" class="p p-first">The details of the <em>ZBTB10</em> association are shown in a LocusZoom plot (<a href="#b28-15_232_seldin" rid="b28-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532933">28</a>) graph (<a href="/pmc/articles/PMC4749491/figure/f2-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f2-15_232_seldin" rid-ob="ob-f2-15_232_seldin" co-legend-rid="lgnd_f2-15_232_seldin"><span>Figure 2</span></a>). The rs6998967 SNP is located ~34 kb 5′ of the zinc finger– and BTB domain–containing 10 (<em>ZBTB10</em>) gene, and there are multiple SNPs in strong LD showing suggestive associations (<a href="/pmc/articles/PMC4749491/figure/f3-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3-15_232_seldin" rid-ob="ob-f3-15_232_seldin" co-legend-rid="lgnd_f3-15_232_seldin"><span>Figure 3A</span></a>). Conditioning for the rs6998967 SNP implicated a single haplotype marked by this variant (<a href="/pmc/articles/PMC4749491/figure/f3-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3-15_232_seldin" rid-ob="ob-f3-15_232_seldin" co-legend-rid="lgnd_f3-15_232_seldin"><span>Figure 3B</span></a>). Conditioning on rs1065238, a potential strong regulatory SNP (RegulomeDB score = 1f, eQTL + transcription factor [TF] binding site/DNase peak; see Materials and Methods) (<a href="#b27-15_232_seldin" rid="b27-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532948">27</a>), also attenuated most of the signal from this region (<a href="/pmc/articles/PMC4749491/figure/f3-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f3-15_232_seldin" rid-ob="ob-f3-15_232_seldin" co-legend-rid="lgnd_f3-15_232_seldin"><span>Figure 3C</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="f3-15_232_seldin" co-legend-rid="lgnd_f3-15_232_seldin"><a href="/pmc/articles/PMC4749491/figure/f3-15_232_seldin/" target="figure" rid-figpopup="f3-15_232_seldin" rid-ob="ob-f3-15_232_seldin"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140705578210656" class="figure"><a href="/pmc/articles/PMC4749491/figure/f3-15_232_seldin/" target="figure" rid-figpopup="f3-15_232_seldin" rid-ob="ob-f3-15_232_seldin"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4749491_15_232_seldinf3.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 15_232_seldinf3.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4749491/bin/15_232_seldinf3.jpg"></a></div><div id="largeobj_idm140705578210656" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4749491/figure/f3-15_232_seldin/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f3-15_232_seldin"><div><a class="figpopup" href="/pmc/articles/PMC4749491/figure/f3-15_232_seldin/" target="figure" rid-figpopup="f3-15_232_seldin" rid-ob="ob-f3-15_232_seldin">Figure 3</a></div><!--caption a7--><div class="caption"><p id="__p60"><em>ZBTB10</em> association signals in LOMG (age of onset ≥50 years). The ordinate shows the strengths of the association signals, with the position on chromosome 8 shown in megabases (HG19 map) along the abscissa. The <em>P</em> value for each SNP is shown before (A) and after (B) conditioning with rs6998967, the strongest associated SNP, and rs1065238 (C), a SNP in strong LD, with probable regulatory effects (RegulomeDB score = 1f). The color-coded symbols correspond to the strength of linkage disequilibrium (based on HG19/1000 genome, November 2014, European population) with the most significantly associated SNP (purple filled circle) in each panel.</p></div></div></div><p id="__p28" class="p p-last">Similarly, the <em>TNFRSF11A</em> association was primarily due to a single haplotype (<a href="/pmc/articles/PMC4749491/figure/f4-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f4-15_232_seldin" rid-ob="ob-f4-15_232_seldin" co-legend-rid="lgnd_f4-15_232_seldin"><span>Figure 4</span></a>). It includes rs8086340, also with potential regulatory function (RegulomeDB score = 4, possible regulatory site), in LD with the strongest associated SNP, rs4574025.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="f4-15_232_seldin" co-legend-rid="lgnd_f4-15_232_seldin"><a href="/pmc/articles/PMC4749491/figure/f4-15_232_seldin/" target="figure" rid-figpopup="f4-15_232_seldin" rid-ob="ob-f4-15_232_seldin"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140705646671584" class="figure"><a href="/pmc/articles/PMC4749491/figure/f4-15_232_seldin/" target="figure" rid-figpopup="f4-15_232_seldin" rid-ob="ob-f4-15_232_seldin"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4749491_15_232_seldinf4.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 15_232_seldinf4.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4749491/bin/15_232_seldinf4.jpg"></a></div><div id="largeobj_idm140705646671584" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4749491/figure/f4-15_232_seldin/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f4-15_232_seldin"><div><a class="figpopup" href="/pmc/articles/PMC4749491/figure/f4-15_232_seldin/" target="figure" rid-figpopup="f4-15_232_seldin" rid-ob="ob-f4-15_232_seldin">Figure 4</a></div><!--caption a7--><div class="caption"><p id="__p61"><em>TNFRSF11A</em> association signals in LOMG. The ordinate shows the strength of the association signals, with the position on chromosome 18 shown in megabases (HG19 map) along the abscissa. The <em>P</em> value for each SNP is shown before (A) and after (B) conditioning with rs4574025, the strongest associated SNP, and rs8086340 (C), a SNP with possible regulatory effects (RegulomeDB score = 4). The color-coded symbols correspond to the strength of linkage disequilibrium, with the most significantly associated SNP (purple-filled circle) in each panel.</p></div></div></div></div><div id="__sec16" class="sec"><p></p><h3 id="__sec16title" class="inline">HLA Associations </h3><p id="__p29" class="p p-first">To further examine associations within the MHC region, we also imputed both classical <em>HLA</em> alleles and AA variants in them. Three peaks of association were observed in the MHC region (<a href="/pmc/articles/PMC4749491/figure/f5-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5-15_232_seldin" rid-ob="ob-f5-15_232_seldin" co-legend-rid="lgnd_f5-15_232_seldin"><span>Figure 5A</span></a>). These corresponded to the <em>HLA-A</em>, MHC class III and MHC class II genes.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="f5-15_232_seldin" co-legend-rid="lgnd_f5-15_232_seldin"><a href="/pmc/articles/PMC4749491/figure/f5-15_232_seldin/" target="figure" rid-figpopup="f5-15_232_seldin" rid-ob="ob-f5-15_232_seldin"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140705646662240" class="figure"><a href="/pmc/articles/PMC4749491/figure/f5-15_232_seldin/" target="figure" rid-figpopup="f5-15_232_seldin" rid-ob="ob-f5-15_232_seldin"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4749491_15_232_seldinf5.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is 15_232_seldinf5.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4749491/bin/15_232_seldinf5.jpg"></a></div><div id="largeobj_idm140705646662240" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4749491/figure/f5-15_232_seldin/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_f5-15_232_seldin"><div><a class="figpopup" href="/pmc/articles/PMC4749491/figure/f5-15_232_seldin/" target="figure" rid-figpopup="f5-15_232_seldin" rid-ob="ob-f5-15_232_seldin">Figure 5</a></div><!--caption a7--><div class="caption"><p id="__p62">Analysis of the HLA region association signals in LOMG. In each panel, the symbols show the strength of the association signal (ordinate) for this region of chromosome 6 shown in Mb (HG19) along the abscissa; the SNPs, HLA antigens and HLA AAs are given color-coded symbols that correspond to the strength of LD with the most significantly associated SNP (purple filled circle). The <em>P</em> values for each variant are plotted: (A) no conditioning; (B) conditioning on <em>HLA DQA1</em>; (C) <em>HLA DQA1</em> and <em>HLA-A</em>; and (D) <em>HLA DQA1</em>, <em>HLA-A</em> and class III SN<em>P</em> (rs2071596). Additional conditioning results are summarized in <a href="/pmc/articles/PMC4749491/table/t4-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t4-15_232_seldin" rid-ob="ob-t4-15_232_seldin" co-legend-rid=""><span>Table 4</span></a>.</p></div></div></div><p id="__p30">For classical HLA class II antigens, multiple genes showed evidence for associations (<a href="/pmc/articles/PMC4749491/table/t3-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t3-15_232_seldin" rid-ob="ob-t3-15_232_seldin" co-legend-rid=""><span>Table 3</span></a> and <a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S6</a>). However, most of these show protective ORs &lt;1, in stark contrast with those observed in EOMG. Furthermore, the LOMG risk (ORs &gt;1) associations were weaker, and none of them achieved genome-wide significance in the series as a whole. The strongest association we found in LOMG was with <em>DQA1*05:01</em> (<em>P</em> = 5.9 × 10<sup>−12</sup>, OR 0.54). Similarly, <em>DRB1*03:01</em>, which is in strong LD, showed an OR of 0.5 that is consistent with a previous study showing negative association in anti-titin-Ab–positive LOMG patients (<a href="#b12-15_232_seldin" rid="b12-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532987">12</a>). These alleles belong to the B8.1 haplotype that predisposes strongly to EOMG (OR 2.82); <em>HLA-B*08</em> (OR 6.41) also belongs, which gave a contrasting OR of 0.69 in LOMG. Evidently, these alleles favor early onset of MG in susceptible subjects, rather than protecting against it at all ages.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t3-15_232_seldin"><h3>Table 3</h3><!--caption a7--><div class="caption"><p id="__p67">Classic HLA associations in LOMG and comparison with EOMG.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140705575463296" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th colspan="4" rowspan="1" valign="bottom" align="center">LOMG</th><th colspan="4" rowspan="1" valign="bottom" align="center">EOMG</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th colspan="4" rowspan="1" valign="bottom" align="left">
<hr></th><th colspan="4" rowspan="1" valign="bottom" align="left">
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Allele</th><th rowspan="1" colspan="1" valign="bottom" align="center">Case frequency</th><th rowspan="1" colspan="1" valign="bottom" align="center">Control frequency</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% CI)</th><th rowspan="1" colspan="1" valign="bottom" align="center">Case frequency</th><th rowspan="1" colspan="1" valign="bottom" align="center">Control frequency</th><th rowspan="1" colspan="1" valign="bottom" align="center"><em>P</em></th><th rowspan="1" colspan="1" valign="bottom" align="center">OR (95% CI)</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-A*01:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">1.09 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.85 (0.70–1.04)</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">0.16</td><td rowspan="1" colspan="1" valign="top" align="center">1.99 × 10<sup>−44</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.99 (2.57–3.49)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-A*03:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">9.20 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.42 (1.19–1.69)</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">5.92 × 10<sup>−9</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.56 (0.46–0.68)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-C*07:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">1.80 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.67 (0.54–0.83)</td><td rowspan="1" colspan="1" valign="top" align="center">0.52</td><td rowspan="1" colspan="1" valign="top" align="center">0.34</td><td rowspan="1" colspan="1" valign="top" align="center">5.67 × 10<sup>−31</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.18 (1.91–2.48)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-C*05:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.05</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">1.88 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.64 (0.47–0.86)</td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">7.84 × 10<sup>−13</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.44 (0.35–0.55)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-B*07:02</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">1.20 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.26 (1.05–1.50)</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">3.54 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.53 (0.44–0.64)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-B*08:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">0.11</td><td rowspan="1" colspan="1" valign="top" align="center">1.75 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.69 (0.54–0.88)</td><td rowspan="1" colspan="1" valign="top" align="center">0.41</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">2.87 × 10<sup>−113</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6.41 (5.46–7.53)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-B*44:02</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">2.01 × 10<sup>−2</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.73 (0.56–0.96)</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.09</td><td rowspan="1" colspan="1" valign="top" align="center">3.27 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.44 (0.35–0.56)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-DRB1*03:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">6.70 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.50 (0.39–0.66)</td><td rowspan="1" colspan="1" valign="top" align="center">0.35</td><td rowspan="1" colspan="1" valign="top" align="center">0.13</td><td rowspan="1" colspan="1" valign="top" align="center">9.69 × 10<sup>−74</sup></td><td rowspan="1" colspan="1" valign="top" align="center">4.70 (3.98–5.56)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-DRB1*07:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">2.43 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.45 (1.19–1.76)</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">3.74 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.42 (0.34–0.52)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-DRB1*13:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.03</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">1.90 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.39 (0.26–0.58)</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">3.18 × 10<sup>−04</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.60 (0.46–0.79)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-DRB1*15:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.18</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">3.10 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.31 (1.10–1.57)</td><td rowspan="1" colspan="1" valign="top" align="center">0.10</td><td rowspan="1" colspan="1" valign="top" align="center">0.14</td><td rowspan="1" colspan="1" valign="top" align="center">8.92 × 10<sup>−06</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.66 (0.55–0.79)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-DQA1*05:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">5.92 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.54 (0.45–0.65)</td><td rowspan="1" colspan="1" valign="top" align="center">0.43</td><td rowspan="1" colspan="1" valign="top" align="center">0.24</td><td rowspan="1" colspan="1" valign="top" align="center">3.63 × 10<sup>−45</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.82 (2.44–3.26)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-DQA1*02:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.15</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">2.96 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.44 (1.19–1.75)</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">3.19 × 10<sup>−16</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.42 (0.35–0.52)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-DQA1*01:03</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">4.31 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.48 (0.34–0.68)</td><td rowspan="1" colspan="1" valign="top" align="center">0.04</td><td rowspan="1" colspan="1" valign="top" align="center">0.07</td><td rowspan="1" colspan="1" valign="top" align="center">7.34 × 10<sup>−04</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.64 (0.49–0.83)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-DQB1*02</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.19</td><td rowspan="1" colspan="1" valign="top" align="center">0.20</td><td rowspan="1" colspan="1" valign="top" align="center">5.37 × 10<sup>−1</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.95 (0.80–1.12)</td><td rowspan="1" colspan="1" valign="top" align="center">0.37</td><td rowspan="1" colspan="1" valign="top" align="center">0.22</td><td rowspan="1" colspan="1" valign="top" align="center">1.63 × 10<sup>−33</sup></td><td rowspan="1" colspan="1" valign="top" align="center">2.50 (2.16–2.90)</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-DQB1*02:01</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.06</td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">1.23 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="center">0.52 (0.40–0.67)</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-DQB1*02:02</em></td><td rowspan="1" colspan="1" valign="top" align="center">0.12</td><td rowspan="1" colspan="1" valign="top" align="center">0.08</td><td rowspan="1" colspan="1" valign="top" align="center">8.74 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="center">1.66 (1.33–2.06)</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td><td rowspan="1" colspan="1" valign="top" align="center">NA</td></tr></tbody></table></div><div id="largeobj_idm140705575463296" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4749491/table/t3-15_232_seldin/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tfn3-15_232_seldin"><p id="__p68" class="p p-first-last">Results of LOMG with onset age ≥50 years are compared with results from our previous study of EOMG (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532965">15</a>). Case and control antigen frequencies are shown. The ORs and 95% CIs are shown for each HLA antigen except where not available (NA). Data were selected to show the 10 most significant associations in the MG subgroups and also included <em>DRB1*15:01</em> and <em>HLA-B*07:01</em> for comparison with previous studies (see <a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S6</a> for complete HLA analyses).</p></div></div></div><p id="__p31" class="p p-last">Although none of the <em>HLA-A</em> antigens reached genome-wide significance, the <sup>70</sup>Q present in HLA-A*68:01, *03:01 and *11:01 (and other rare alleles) showed a significant risk association (OR 1.5, <em>P</em> = 2.6 × 10<sup>−8</sup>). For MHC class III, the minor alleles of multiple SNPs showed a mixed pattern of significant positive and negative associations (<a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S5</a>).</p></div><div id="__sec17" class="sec"><p></p><h3 id="__sec17title" class="inline">Conditional Analyses of the MHC Region </h3><p id="__p32" class="p p-first">To localize the key variants more precisely, we performed extensive conditional analyses (<a href="/pmc/articles/PMC4749491/figure/f5-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5-15_232_seldin" rid-ob="ob-f5-15_232_seldin" co-legend-rid="lgnd_f5-15_232_seldin"><span>Figure 5</span></a> and <a href="/pmc/articles/PMC4749491/table/t4-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t4-15_232_seldin" rid-ob="ob-t4-15_232_seldin" co-legend-rid=""><span>Table 4</span></a>). Conditioning on all nominally significant classical <em>HLA-DQA1</em> antigen genes (<em>P</em> &lt; 0.05) almost completely eliminated the strongest peak of association, which was located in the MHC class II region (<a href="/pmc/articles/PMC4749491/figure/f5-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5-15_232_seldin" rid-ob="ob-f5-15_232_seldin" co-legend-rid="lgnd_f5-15_232_seldin"><span>Figure 5B</span></a> and <a href="/pmc/articles/PMC4749491/table/t4-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t4-15_232_seldin" rid-ob="ob-t4-15_232_seldin" co-legend-rid=""><span>Table 4</span></a>). Conditioning instead on <em>DRB1</em> antigens had similar but less complete effects on the MHC class II region (residual <em>P</em> = 5.58 × 10<sup>−5</sup> versus <em>P</em> = 1.11 × 10<sup>−3</sup>; <a href="/pmc/articles/PMC4749491/table/t4-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t4-15_232_seldin" rid-ob="ob-t4-15_232_seldin" co-legend-rid=""><span>Table 4</span></a>).</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="t4-15_232_seldin"><h3>Table 4</h3><!--caption a7--><div class="caption"><p id="__p69">Highlights of conditional analyses of the MHC region in LOMG.</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140705578825616" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th colspan="3" rowspan="1" valign="bottom" align="center"><em>HLA-A</em>: 29510630-30465978</th><th colspan="3" rowspan="1" valign="bottom" align="center">Class III: 30466879-31763744</th><th colspan="3" rowspan="1" valign="bottom" align="center">Class II: 31765860-33005932</th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left"></th><th colspan="3" rowspan="1" valign="bottom" align="left">
<hr></th><th colspan="3" rowspan="1" valign="bottom" align="left">
<hr></th><th colspan="3" rowspan="1" valign="bottom" align="left">
<hr></th></tr><tr><th rowspan="1" colspan="1" valign="bottom" align="left">Conditioning</th><th rowspan="1" colspan="1" valign="bottom" align="center">Lowest residual <em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="center">Variant</th><th rowspan="1" colspan="1" valign="bottom" align="center">Position</th><th rowspan="1" colspan="1" valign="bottom" align="center">Lowest residual <em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="center">Variant</th><th rowspan="1" colspan="1" valign="bottom" align="center">Position</th><th rowspan="1" colspan="1" valign="bottom" align="center">Lowest residual <em>P</em> value</th><th rowspan="1" colspan="1" valign="bottom" align="center">Variant</th><th rowspan="1" colspan="1" valign="bottom" align="center">Position</th></tr></thead><tbody><tr><td rowspan="1" colspan="1" valign="top" align="left">None</td><td rowspan="1" colspan="1" valign="top" align="center">1.00 × 10<sup>−11</sup></td><td rowspan="1" colspan="1" valign="top" align="center">rs150881176</td><td rowspan="1" colspan="1" valign="top" align="center">29947740</td><td rowspan="1" colspan="1" valign="top" align="center">4.11 × 10<sup>−12</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs2071591</td><td rowspan="1" colspan="1" valign="top" align="center">31515799</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>3.13 × 10</strong><strong><sup>−17</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="left">rs111945767</td><td rowspan="1" colspan="1" valign="top" align="center">32517045</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">rs111945767</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>2.24 × 10</strong><strong><sup>−8</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs150881176</td><td rowspan="1" colspan="1" valign="top" align="center">29947740</td><td rowspan="1" colspan="1" valign="top" align="center">1.17 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">6:31534010:D</td><td rowspan="1" colspan="1" valign="top" align="center">31534010</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">7.91 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs204887</td><td rowspan="1" colspan="1" valign="top" align="center">32029226</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>DRB1</em></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>3.26 × 10</strong><strong><sup>−8</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs150881176</td><td rowspan="1" colspan="1" valign="top" align="center">29947740</td><td rowspan="1" colspan="1" valign="top" align="center">8.33 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs2071596</td><td rowspan="1" colspan="1" valign="top" align="center">31506691</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">5.58 × 10<sup>−5</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs9501624</td><td rowspan="1" colspan="1" valign="top" align="center">32399286</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>DQA1</em></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>1.78 × 10</strong><strong><sup>−8</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs150881176</td><td rowspan="1" colspan="1" valign="top" align="center">29947740</td><td rowspan="1" colspan="1" valign="top" align="center">1.77 × 10<sup>−7</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs2256974</td><td rowspan="1" colspan="1" valign="top" align="center">31555392</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">1.11 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs147008997</td><td rowspan="1" colspan="1" valign="top" align="center">32447598</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>DQB1</em></td><td rowspan="1" colspan="1" valign="top" align="center"><strong>1.84 × 10</strong><strong><sup>−9</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs115207654</td><td rowspan="1" colspan="1" valign="top" align="center">29942293</td><td rowspan="1" colspan="1" valign="top" align="center">7.19 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs2071591</td><td rowspan="1" colspan="1" valign="top" align="center">31515799</td><td rowspan="1" colspan="1" valign="top" align="center">9.52 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">DQA1_AA107_I</td><td rowspan="1" colspan="1" valign="top" align="center">32609806</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-A</em></td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">1.16 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="center">rs1632958</td><td rowspan="1" colspan="1" valign="top" align="center">29688643</td><td rowspan="1" colspan="1" valign="top" align="center">2.40 × 10<sup>−10</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs2071591</td><td rowspan="1" colspan="1" valign="top" align="center">31515799</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>7.02 × 10</strong><strong><sup>−16</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="left">rs111945767</td><td rowspan="1" colspan="1" valign="top" align="center">32517045</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-B</em></td><td rowspan="1" colspan="1" valign="top" align="center">2.82 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">rs150881176</td><td rowspan="1" colspan="1" valign="top" align="center">29947740</td><td rowspan="1" colspan="1" valign="top" align="center">3.17 × 10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs116706902</td><td rowspan="1" colspan="1" valign="top" align="center">31529335</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>6.85 × 10</strong><strong><sup>−13</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="left">rs111938555</td><td rowspan="1" colspan="1" valign="top" align="center">32509707</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>HLA-C</em></td><td rowspan="1" colspan="1" valign="top" align="center">6.99 × −10<sup>−8</sup></td><td rowspan="1" colspan="1" valign="top" align="center">rs150881176</td><td rowspan="1" colspan="1" valign="top" align="center">29947740</td><td rowspan="1" colspan="1" valign="top" align="center">2.68 × 10<sup>−6</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs2071591</td><td rowspan="1" colspan="1" valign="top" align="center">31515799</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>3.75 × 10</strong><strong><sup>−12</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="left">rs112780983</td><td rowspan="1" colspan="1" valign="top" align="center">32511348</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>DRB1</em>, <em>HLA-A</em>, Class III SNP</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">2.55 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">rs1632958</td><td rowspan="1" colspan="1" valign="top" align="center">29688643</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center"><strong>4.15 × 10</strong><strong><sup>−4</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="left">rs138758805</td><td rowspan="1" colspan="1" valign="top" align="center">31533722</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>1.57 × 10</strong><strong><sup>−5</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="left">rs9501624</td><td rowspan="1" colspan="1" valign="top" align="center">32399286</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>DQA1</em>, <em>HLA-A</em>, Class III SNP</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">2.18 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6:30014294:I</td><td rowspan="1" colspan="1" valign="top" align="center">30014294</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center"><strong>2.16 × 10</strong><strong><sup>−3</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="left">rs12665489</td><td rowspan="1" colspan="1" valign="top" align="center">31507987</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">2.96 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">chr6:31834138</td><td rowspan="1" colspan="1" valign="top" align="center">31834138</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left"><em>DQA1</em>, <em>HLA-A</em>, <em>HLA-C</em></td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">2.52 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="center">6:30014294:I</td><td rowspan="1" colspan="1" valign="top" align="center">30014294</td><td rowspan="1" colspan="1" valign="top" align="center"><strong>4.20 × 10</strong><strong><sup>−5</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="left">rs2071596</td><td rowspan="1" colspan="1" valign="top" align="center">31506691</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">4.08 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs28732173</td><td rowspan="1" colspan="1" valign="top" align="center">32071010</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DRB1_AA11, A_AA70, Class III SNP</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center"><strong>2.34 × 10</strong><strong><sup>−4</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs150881176</td><td rowspan="1" colspan="1" valign="top" align="center">29947740</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">7.05 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs12665489</td><td rowspan="1" colspan="1" valign="top" align="center">31507987</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">2.19 × 10<sup>−4</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs114053271</td><td rowspan="1" colspan="1" valign="top" align="center">32535948</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DQA1_AA75, DQA1_AA130, A_AA70, Class III SNP</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center"><strong>5.42 × 10</strong><strong><sup>−4</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs114705891</td><td rowspan="1" colspan="1" valign="top" align="center">29990708</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">1.57 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs12665489</td><td rowspan="1" colspan="1" valign="top" align="center">31507987</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">3.12 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs6907185</td><td rowspan="1" colspan="1" valign="top" align="center">31844684</td></tr><tr><td rowspan="1" colspan="1" valign="top" align="left">DQA1 AA75, DQA1 AA130, A_AA70, A_AA152, Class III SNP</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center"><strong>1.95 × 10</strong><strong><sup>−3</sup></strong></td><td rowspan="1" colspan="1" valign="top" align="center">rs150881176</td><td rowspan="1" colspan="1" valign="top" align="center">29947740</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">2.06 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs12665489</td><td rowspan="1" colspan="1" valign="top" align="center">31507987</td><td style="background-color:#C7C8CA" rowspan="1" colspan="1" valign="top" align="center">3.40 × 10<sup>−3</sup></td><td rowspan="1" colspan="1" valign="top" align="left">rs143832718</td><td rowspan="1" colspan="1" valign="top" align="center">32287072</td></tr></tbody></table></div><div id="largeobj_idm140705578825616" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4749491/table/t4-15_232_seldin/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="tfn4-15_232_seldin"><p id="__p70" class="p p-first-last">The most significant <em>P</em> value for each condition is shown for each of the three regions with signal peaks (<a href="/pmc/articles/PMC4749491/figure/f5-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5-15_232_seldin" rid-ob="ob-f5-15_232_seldin" co-legend-rid="lgnd_f5-15_232_seldin"><span>Figure 5</span></a>). The most significant <em>P</em> value for each condition (row) is shown in bold. Gray shading indicates where the signal has been attenuated above <em>P</em> = 5 × 10<sup>−5</sup>. The class III SNP included in the conditioning is rs2071596, but nearly equivalent results are obtained with rs6929796 or rs2256974. For some of the AAs indicated in this table, others gave nearly identical results: <sup>130</sup>A was in complete LD with <sup>41</sup>K, and <sup>75</sup>I was in nearly complete LD with AAs at multiple positions (40, 47, 50, 51, 53, 107, 156, 161, 163, and 175). For these studies, conditioning was performed using all HLA antigens for each gene that had nominally significant <em>P</em> values (<em>P</em> &lt; 0.05). For AAs, we individually conditioned on each AA (in DQA1, DRB1 and HLA-A) that had nominal <em>P</em> values &lt;0.001. Results are for those AAs with the strongest attenuation of the association signals.</p></div></div></div><p id="__p33">Nearly all the signal in the HLA-A region resisted conditioning on MHC class II genes or the strongest associated SNP there, rs111945767 (which attenuated the MHC class II signal nearly as well as <em>HLA-DQA1</em>) (<a href="/pmc/articles/PMC4749491/table/t4-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t4-15_232_seldin" rid-ob="ob-t4-15_232_seldin" co-legend-rid=""><span>Table 4</span></a>). Conversely, conditioning on all classic <em>HLA-A</em> antigen genes (<em>P</em> &lt; 0.05) left strong residual association signals in the MHC class II region (<em>P</em> value still &lt;10<sup>−15</sup>; <a href="/pmc/articles/PMC4749491/table/t4-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t4-15_232_seldin" rid-ob="ob-t4-15_232_seldin" co-legend-rid=""><span>Table 4</span></a>).</p><p id="__p34">The MHC class III region signals were largely unaffected, even by conditioning on the combination of <em>HLA-A</em> and <em>DQA1</em> (<a href="/pmc/articles/PMC4749491/figure/f5-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5-15_232_seldin" rid-ob="ob-f5-15_232_seldin" co-legend-rid="lgnd_f5-15_232_seldin"><span>Figure 5C</span></a>), although this step almost completely attenuated the <em>HLA-A</em> and the MHC class II region peaks. Conditioning on <em>HLA-A</em>, <em>DQA1</em> and <em>HLA-C</em> also left residual signals in class III (<a href="/pmc/articles/PMC4749491/table/t4-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t4-15_232_seldin" rid-ob="ob-t4-15_232_seldin" co-legend-rid=""><span>Table 4</span></a>). We also note that the combination of <em>HLA-A</em>, <em>DRB1</em> and <em>HLA-C</em>, and the combination of <em>HLA-A</em>, <em>DQB1</em> and <em>HLA-C</em>, also did not attenuate the class III signal (data not shown). Finally, conditioning on <em>HLA-A</em>, <em>DQA1</em> and an MHC class III SNP effectively eliminated almost all association signals in the entire region (residual <em>P</em> &gt; 0.002) (<a href="/pmc/articles/PMC4749491/figure/f5-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5-15_232_seldin" rid-ob="ob-f5-15_232_seldin" co-legend-rid="lgnd_f5-15_232_seldin"><span>Figure 5D</span></a>). We cannot locate the precise locus in MHC class III because of the strong LD within this region (<a href="/pmc/articles/PMC4749491/figure/f5-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f5-15_232_seldin" rid-ob="ob-f5-15_232_seldin" co-legend-rid="lgnd_f5-15_232_seldin"><span>Figure 5C</span></a>); three of its SNPs each showed nearly equivalent effects in combination with the <em>HLA-A</em> and <em>DQA1</em> conditioning: rs2256974, <em>P</em> = 0.0022; rs6929796, <em>P</em> = 0.0018; and rs2071596, <em>P</em> = 0.0015.</p><p id="__p35" class="p p-last">The results were similar whether we conditioned on HLA alleles or selected AAs within them (<a href="/pmc/articles/PMC4749491/table/t4-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t4-15_232_seldin" rid-ob="ob-t4-15_232_seldin" co-legend-rid=""><span>Table 4</span></a>). Conditioning just on the <sup>11</sup>S (present in HLA-DR1*03:01 and most DRB1*11, *13 and *14 subtypes) plus <sup>70</sup>Q in HLA-A (in HLA-A*68:01, 03:01, 11:01), plus the rs2256974 class III SNP, attenuated most of the association signal (residual <em>P</em> = 2.3 × 10<sup>−4</sup>). Conditioning on <sup>130</sup>A and <sup>75</sup>I in DQA1 (present in DQA1*01:03 and DQA1*05:01, respectively) plus the same HLA-A AA and class III SNPs showed marginally better attenuation (residual <em>P</em> = 5.4 × 10<sup>−4</sup>), which became almost complete after adding <sup>152</sup>A (includes HLA-A*11:01 and 01:01) and/or <sup>152</sup>E (includes HLA-A*03:01, 25:01 and 26:01) in HLA-A (residual <em>P</em> = 1.95 × 10<sup>−3</sup>).</p></div><div id="__sec18" class="sec"><p></p><h3 id="__sec18title" class="inline">Comparing Associations in Males and Females </h3><p id="__p36" class="p p-first-last">Because we found significantly stronger associations and effect sizes with <em>HLA-B8</em> in EOMG females than in males (<a href="#b10-15_232_seldin" rid="b10-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532961">10</a>,<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532991">15</a>), we checked for analogous differences in LOMG at all the strongest associated loci within and outside HLA (<a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S7</a>). Although none of these differences were significant (based on <em>Z</em> score calculations), effect sizes (whether risk or protective alleles) were larger in the males for each MHC sub-region. Among the other loci, only the <em>PTPN22</em> SNP showed any significant gender difference (unlike in EOMG), the association being confined to males (OR 1.83, 95% confidence interval [CI] 1.4–2.4), with no significant association in females (OR 0.75, 95% CI 0.54–1.04).</p></div><div id="__sec19" class="sec sec-last"><p></p><h3 id="__sec19title" class="inline">Comparing Associations in Generalized and Ocular LOMG </h3><p id="__p37" class="p p-first-last">Lastly, we evaluated whether stratifying AChR-Ab–positive LOMG for clinical features of generalized versus ocular disease would affect our results. The ORs were similar among the entire LOMG study grou<em>p</em> (n = 532) and those classified as generalized (n = 408) for both MHC (for example, <em>HLA-DQA1*05:01</em>, OR 0.52 versus OR 0.51) and non-MHC loci (<a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S8</a>). The ocular group showed more fluctuation in the ORs, as expected from the small sample size (n = 78), but the pattern was broadly consistent with that observed for the generalized subgroup (<a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S8</a>).</p></div></div><div id="__sec20" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec20title">DISCUSSION</h2><p id="__p38" class="p p-first">This genetic study included only patients strictly defined by MG with age of onset ≥50 years. Importantly, our GWAS findings show sharp contrasts between EOMG and LOMG (whether with onset ≥60 years or even ≥50 years) and implicate regions both within and outside the MHC complex; only <em>PTPN22</em> showed similar associations in both groups. By providing a more biological basis for the cutoff between EOMG and LOMG, our findings should provoke new hypotheses about the distinct genetic and environmental factors influencing susceptibility.</p><p id="__p39">For non-MHC variants, only a single locus with a peak at rs6998967, 34 kb proximal to <em>ZBTB10</em>, met conventional GWAS criteria for a significant association. It is in LD with a variant, rs1065238, that could account for most of the signal and is located in a DNase-sensitive quantitative trait locus; in fact, it is located within a binding site for POU2F1, Zfp187 (ZSCAN26), POU3F1 and POU4F3 (<a href="#b29-15_232_seldin" rid="b29-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532939">29</a>). This gene previously met genome-wide association criteria in studies of sex hormone–binding globulin levels (linked to sex steroid regulation) (<a href="#b30-15_232_seldin" rid="b30-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532955">30</a>) and asthma with hay fever (<a href="#b31-15_232_seldin" rid="b31-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532941">31</a>). It has also shown suggestive levels of association in hypothyroidism (<a href="#b32-15_232_seldin" rid="b32-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532964">32</a>), atopic dermatitis (<a href="#b33-15_232_seldin" rid="b33-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532960">33</a>) and self-reported allergy (<a href="#b34-15_232_seldin" rid="b34-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532921">34</a>). ZBTB10 is thought to regulate specificity proteins Sp1, Sp3 and Sp4 (<a href="#b35-15_232_seldin" rid="b35-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532931">35</a>,<a href="#b36-15_232_seldin" rid="b36-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532946">36</a>); in cell culture studies, it is suppressed by ROS- microRNA27a, thereby enhancing estrogen receptor α expression and mediating estrogen effects (<a href="#b36-15_232_seldin" rid="b36-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532978">36</a>). Potential immunoregulation could either be through Sp1 (critical for interleukin [IL]-10 and CD40 responses) (<a href="#b37-15_232_seldin" rid="b37-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532922">37</a>,<a href="#b38-15_232_seldin" rid="b38-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532974">38</a>) or indirectly via estrogen effects on autoimmunity (<a href="#b39-15_232_seldin" rid="b39-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532919">39</a>,<a href="#b40-15_232_seldin" rid="b40-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532954">40</a>).</p><p id="__p40">Non-MHC loci meeting suggestive criteria for association notably include <em>PTPN22</em> and <em>TNFRSF11A. PTPN22</em> was reported in several studies in MG (<a href="#b14-15_232_seldin" rid="b14-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532953">14</a>,<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532968">15</a>,<a href="#b41-15_232_seldin" rid="b41-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532956">41</a>), reaching GWAS criteria in EOMG (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532985">15</a>). This association is with the same coding SNP, rs2476601, as in EOMG, with similar effect sizes (ORs 1.62–1.71), and in multiple other autoimmune diseases including type 1 diabetes mellitus, rheumatoid arthritis (RA) and systemic lupus erythematosus (<a href="#b42-15_232_seldin" rid="b42-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532998">42</a>–<a href="#b44-15_232_seldin" rid="b44-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532945">44</a>). The predisposing <sup>620</sup>W variant is thought to restrain T-cell responses less than its <sup>620</sup>R counterpart, as well as increase B-cell activation on antigen binding (<a href="#b45-15_232_seldin" rid="b45-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_810609531">45</a>). The higher prevalence of this variant in our males together with the lack of association with <em>TNIP1</em> in LOMG and the starkly contrasting HLA associations (discussed below) argue strongly that the <em>PTPN22</em> association is not due to EOMG contaminants in our LOMG series.</p><p id="__p41">Our findings with <em>TNFRSF11A</em> confirm a recent report where one of the same SNPs (rs4574025) showed one of the strongest associations, again only in MG patients with onset after age 40 years (<a href="#b16-15_232_seldin" rid="b16-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550442221">16</a>). This result might reflect the potential regulatory function of rs8086340 SNP, which is in such strong LD with the rs4574025 that conditioning on either SNP gave almost identical residual <em>P</em> values (&lt;0.004). Variants of <em>TNFRSF11A</em> (RANK) underlie a familial form of Paget disease of the bone (<a href="#b46-15_232_seldin" rid="b46-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532932">46</a>) and a recessive form of osteopetrosis with hypogammaglobulinemia (<a href="#b47-15_232_seldin" rid="b47-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532979">47</a>). <em>TNFRSF11A</em> variants are also implicated in GWAS both in other forms of Paget disease and of bone mineral density (<a href="#b48-15_232_seldin" rid="b48-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532977">48</a>,<a href="#b49-15_232_seldin" rid="b49-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532975">49</a>). With respect to immunologic functions, <em>TNFRSF11A</em> appears to be important in lymph node development and thymic selection (<a href="#b50-15_232_seldin" rid="b50-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532994">50</a>), and its expression on dendritic cells may also be critical to specific interactions with T cells (<a href="#b51-15_232_seldin" rid="b51-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532947">51</a>,<a href="#b52-15_232_seldin" rid="b52-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532963">52</a>).</p><p id="__p42">In the MHC, we found strong associations with multiple different loci. For HLA alleles, the strongest association was with <em>HLA-DQA1*05:01</em> (<em>P</em> = 5.3 × 10<sup>−12</sup>, OR 0.54) and minimal with <em>HLA-B*08:01</em> (<em>P</em> = 1.8 × 10<sup>−3</sup>, OR 0.69), unlike in EOMG, where <em>HLA-B*08</em> predisposes very strongly (OR ~6) (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532993">15</a>) and evidently biases toward earlier onset of MG. In addition, the sharp contrast between these risk associations in EOMG (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532923">15</a>) and the protective associations in LOMG (and their much greater strength for this -DQ allele than for -B8 or -A1 [<a href="/pmc/articles/PMC4749491/table/t3-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t3-15_232_seldin" rid-ob="ob-t3-15_232_seldin" co-legend-rid=""><span>Table 3</span></a>]) implies an extra protective role for DQA1*0501 or linked variants (for example, <em>DRB1*03:01</em>) in LOMG. In both subgroups, the findings also implicate each variant’s function in presenting antigens to specific T cells (see below). We postulate that the striking differences in MHC region associations between LOMG and EOMG may in turn be due to differences in the environmental stimulus that presumably induces the break in tolerance to AChR.</p><p id="__p43">In some of our LOMG patients, we previously noted risk associations with <em>DRB1*15:01</em> (<em>P</em> = 7.4 × 10<sup>−5</sup>, OR 2.38) in Norwegians (<a href="#b6-15_232_seldin" rid="b6-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532918">6</a>) or with -DR2 and/or -B7 in British males (<a href="#b10-15_232_seldin" rid="b10-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532995">10</a>). Although still evident in our larger present series, these associations are substantially weaker (for example, <em>DRB1*15:01</em>, <em>P</em> = 3.1 × 10<sup>−3</sup>, OR 1.32), even just in the ~200 combined Scandinavian and British males (<em>P</em> = 8.9 × 10<sup>−3</sup>, OR 1.44). This difference may reflect underlying heterogeneity within LOMG and might best be clarified in further studies with larger patient numbers with onset before or after 1990, focusing separately on males and females with or without anti-titin or other autoantibodies listed in <a href="/pmc/articles/PMC4749491/table/t1-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t1-15_232_seldin" rid-ob="ob-t1-15_232_seldin" co-legend-rid=""><span>Table 1</span></a> and with onset before or after ages 40, 50, or 60 years, or even higher.</p><p id="__p44">In addition to the <em>HLA-DQA1</em>/<em>DRB1</em> associations, there were independent signals from the HLA-A and the MHC class III region. The HLA-A signal was due to two risk-associated AAs as discussed below. In class III, the associated SNPs are located within introns of three genes: the read-through transcript of the Dead box polypeptide 39B and V-type proton ATPase subunit G2 (<em>DDX39B-ATP6V1G2</em>); the nuclear factor of κ light polypeptide gene enhancer in B-cells inhibitor-like 1 (<em>NFKBIL1</em>); and leukocyte-specific transcript 1 (<em>LST1</em>). <em>NFKBIL1</em> and <em>LST1</em> are potential modulators of immune and auto-immune responses. Moreover, the 50-kb interval between these SNPs also contains both tumor necrosis factor (<em>TNF</em>) and lymphotoxin α (<em>LTA</em>), each of which are strong candidate genes. Notably, differences in macrophage expression of these class III genes after stimulation reportedly associate with different <em>LST1</em> haplotypes (<a href="#b53-15_232_seldin" rid="b53-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532944">53</a>). Of the three SNPs, only rs2071596 has a high predicted regulatory function (RegulomeDB score = 2b indicating TF binding + any TF binding motif + DNase Footprint + DNase peak). While another SNP (rs45457097; RegulomeDB score = 2b) located in the 3’UTR of <em>NFKBIL1</em> may be another functional candidate, it was not imputed with high confidence and could not be evaluated by conditioning studies.</p><p id="__p45">Previous studies have shown several independent signals within the MHC region in multiple autoimmune diseases, including RA and systemic lupus erythematosus (<a href="#b54-15_232_seldin" rid="b54-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532984">54</a>–<a href="#b57-15_232_seldin" rid="b57-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532971">57</a>), and specifically with <em>HLA-A</em> in type 1 diabetes (<a href="#b54-15_232_seldin" rid="b54-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532973">54</a>). The data are less clear for MHC class III because of the strong LD there (<a href="/pmc/articles/PMC4749491/figure/f4-15_232_seldin/" target="figure" class="fig-table-link figpopup" rid-figpopup="f4-15_232_seldin" rid-ob="ob-f4-15_232_seldin" co-legend-rid="lgnd_f4-15_232_seldin"><span>Figure 4C</span></a>). One study has suggested a possible independent role for <em>DDX39B</em> (formerly <em>BAT1</em>) in RA (<a href="#b58-15_232_seldin" rid="b58-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532958">58</a>).</p><p id="__p46">Conditional association tests using specific HLA AA variants may also provide insights into possible functional correlates. For <em>DQA1</em>, <sup>130</sup>A (DQA1*05:01) and <sup>75</sup>I (DQA1*01:03) gave equivalent results to conditioning on all <em>DQA1</em> antigen genes. Structural modeling studies suggested a critical role for AA 130 in DQA1/DM interactions in peptide loading (<a href="#b59-15_232_seldin" rid="b59-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532976">59</a>). Because <sup>75</sup>I protrudes into the peptide-binding groove, it might affect recognition of the bound peptides. Although the data modestly favor <em>DQA1</em> over <em>DRB1</em>, we also noted a strong effect at AA position 11 in <em>DRB1</em> (<a href="/pmc/articles/PMC4749491/table/t4-15_232_seldin/" target="table" class="fig-table-link figpopup" rid-figpopup="t4-15_232_seldin" rid-ob="ob-t4-15_232_seldin" co-legend-rid=""><span>Table 4</span></a>), which is also critical in peptide binding.</p><p id="__p47">For <em>HLA-A</em>, <sup>70</sup>Q (present in HLA-A*68:01, *03:01 and *11:01 and other lower-frequency HLA-A AAs) and <sup>152</sup>A or <sup>152</sup>E (present in HLA-A*01:01, *03:01, *11:01, *25:01 and *26:01) could account for the signal and were risk variants. Interestingly, these AAs have been implicated in cytotoxic T-cell recognition of influenza A epitopes (<a href="#b60-15_232_seldin" rid="b60-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532990">60</a>) and may thus implicate environmental provoking factors. The high prevalence in LOMG of oligoclonal expansions in both CD8<sup>+</sup> and CD4<sup>+</sup> T cells (<a href="#b61-15_232_seldin" rid="b61-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532988">61</a>) is another tantalizing hint that viruses might be involved. Changes in exposure could be relevant to the recent increase in incidence of LOMG. In this regard, we did not find any significant difference in association signals when we subdivided LOMG cases with onset before or after 1990 (data not shown).</p><p id="__p48">When we compared associations in males and females, none of the differences proved significant (<em>Z</em> scores with <em>P</em> values &gt;0.4), but the effect sizes for the HLA associations were larger in the LOMG males, whereas they were significantly higher in the females with EOMG (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532962">15</a>). Outside of the MHC, we observed a significant association with <em>PTPN22</em> only in males with LOMG. By contrast, effect sizes for the <em>ZBTB10</em>-linked SNP were almost identical in males and females. Confirmation of these potential differences might imply stronger genetic and/or hormonal influences on MG susceptibility in younger females and older males. Sexual dimorphism and the potential role of X chromosome genes have been emphasized in other studies of autoimmunity (<a href="#b40-15_232_seldin" rid="b40-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532929">40</a>,<a href="#b62-15_232_seldin" rid="b62-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532970">62</a>).</p><p id="__p49">In our patients with generalized and pure ocular AChR-Ab–positive LOMG, we also found broadly similar ORs for the associated SNPs, in both <em>HLA</em> and other regions (<a href="#s1-15_232_seldin" rid="s1-15_232_seldin" class=" supplementary-material">Supplementary Table S8</a>). This finding may not be surprising, since neurotransmission is impaired in peripheral muscles in nearly all patients with AChR-Ab–positive pure ocular LOMG when tested by sensitive single-fiber electromyography, just as in generalized AChR-Ab–positive LOMG, but unlike in AChR-Ab–negative ocular MG (<a href="#b63-15_232_seldin" rid="b63-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532957">63</a>–<a href="#b65-15_232_seldin" rid="b65-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532943">65</a>).</p><p id="__p50">Finally, this study has not detected associations with several genes previously implicated in EOMG and MG. These include <em>TNIP1</em> and such possible candidates as <em>STAT4</em>, <em>IKZF1</em>, <em>IRF5</em>, <em>NKX2-3</em>, <em>ORMDL3</em>, <em>CD226</em> and <em>PPG1</em> (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532915">15</a>). We also found no firm evidence for the <em>CTLA4</em> associations previously reported in MG (<a href="#b5-15_232_seldin" rid="b5-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532930">5</a>,<a href="#b16-15_232_seldin" rid="b16-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_550442219">16</a>). Similarly, for the LOMG-associated genes, <em>ZBTB10</em> and <em>TNFRSF11A</em>, there was no suggestion of association in our previous EOMG study (<a href="#b15-15_232_seldin" rid="b15-15_232_seldin" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_512532913">15</a>).</p><p id="__p51" class="p p-last">Our study has several limitations. First, the sample size limited power to identify loci with relatively modest effect sizes. This finding was also partially compounded by the incomplete ability to impute all variants in the 1,000-genome resource at a high confidence (<em>r</em><sup>2</sup> &gt; 0.8). Second, control genotypes were obtained from disparate sources. This result was mitigated by our assessment and matching strategy based on PCA as well as including appropriate covariates to account for differences in population substructure and genotyping platform. Finally, an independent replication was precluded by the small numbers of these uncommon patients.</p></div><div id="__sec21" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec21title">CONCLUSION</h2><p id="__p52" class="p p-first-last">The data presented here provide compelling evidence that LOMG has a different genetic basis from EOMG. The bimodal distribution of onset ages of MG, and the similar results with cutoffs ≥50 and ≥60 years for LOMG (except for diminished power), together with previous studies, suggest that a cutoff age of onset of ≥50 years is a useful criterion for future investigative and clinical studies. The study confirms that a variant of <em>TNFRSF11A</em> is a risk factor for LOMG and defines independent associations with specific MHC class 1, class 2 and class 3 polymorphisms. Additional replication studies are warranted to further support the novel identification of <em>ZBTB10</em> as an LOMG risk factor, and larger sample sets using strict phenotypic definitions, as well as directed functional studies, will be necessary to further unravel the complex genetics of different forms of MG.</p></div><div id="__sec22" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec22title">Supplemental Data</h2><!--/article/body/sec/--><div class="sec suppmat" id="s1-15_232_seldin"><div class="sup-box half_rhythm" id="d36e1534"><a href="/pmc/articles/PMC4749491/bin/15_232_Seldin_Suppl.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(1.2M, pdf)</sup></div></div></div><div id="idm140705612488656" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140705612488656title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="__p53">This work was supported by the National Institutes of Health (NIH/NIAID RO1-AI-68759 to P K Gregersen); by grants from the Palle Ferb Foundation and the Swedish Research Council (to OK Alkhairy and L Hammarstrom); by program grants from the UK Medical Research Council (to N Willcox); by the Prinses Beatrix Fonds (to JJGM Verschuuren); and by grants from the South-Eastern Norwegian Regional Health Authority and the Norwegian Association for Patients with Muscle Diseases (to AH Maniaol). We thank the Norwegian Bone Marrow Registry for control DNAs. We thank the CIDR for providing access to the Health And Retirement Study (HRS) through dbGaP Study Accession: phs000428.v1.p1. We also thank E Bakker in the Department of Human Genetics, Leiden University Medical Center, the Medical Research Council and Myasthenia Gravis Association in the UK, and the many patients and their physicians in all the centers who generously participated in this study.</p></div></div><div id="idm140705612486672" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140705612486672title">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn1-15_232_seldin"></p><p id="__p54" class="p p-first-last">Online address: <a href="http://www.molmed.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.molmed.org</a></p><p></p><p class="fn sec" id="fn2-15_232_seldin"></p><p id="__p55" class="p p-first"><strong>DISCLOSURE</strong></p><p id="__p56" class="p p-last">The authors declare that they have no competing interests as defined by <em>Molecular Medicine</em>, or other interests that might be perceived to influence the results and discussion reported in this paper.</p><p></p><p class="fn sec" id="fn3-15_232_seldin"></p><p id="__p57" class="p p-first-last">Cite this article as: Seldin MF, <em>et al</em>. (2015) Genome-wide association study of late-onset myasthenia gravis: confirmation of <em>TNFRSF11A</em> and identification of <em>ZBTB10</em> and three distinct HLA associations. <em>Mol. Med</em>. 21:769–81.</p><p></p></div></div><div id="idm140705612478208" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140705612478208title">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="b1-15_232_seldin">1. <span class="element-citation">McGrogan A, Sneddon S, de Vries CS. The incidence of myasthenia gravis: a systematic literature review. <span><span class="ref-journal">Neuroepidemiology. </span>2010;<span class="ref-vol">34</span>:171–83.</span> [<a href="/pubmed/20130418" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuroepidemiology&amp;title=The+incidence+of+myasthenia+gravis:+a+systematic+literature+review&amp;author=A+McGrogan&amp;author=S+Sneddon&amp;author=CS+de+Vries&amp;volume=34&amp;publication_year=2010&amp;pages=171-83&amp;pmid=20130418&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b2-15_232_seldin">2. <span class="element-citation">Meriggioli MN, Sanders DB. Autoimmune myasthenia gravis: emerging clinical and biological heterogeneity. <span><span class="ref-journal">Lancet. </span>2009;<span class="ref-vol">8</span>:475–90.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2730933/">PMC free article</a>]</span> [<a href="/pubmed/19375665" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Autoimmune+myasthenia+gravis:+emerging+clinical+and+biological+heterogeneity&amp;author=MN+Meriggioli&amp;author=DB+Sanders&amp;volume=8&amp;publication_year=2009&amp;pages=475-90&amp;pmid=19375665&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b3-15_232_seldin">3. <span class="element-citation">Carr AS, Cardwell CR, McCarron PO, McConville J. A systematic review of population based epidemiological studies in Myasthenia Gravis. <span><span class="ref-journal">BMC Neurol. </span>2010;<span class="ref-vol">10</span>:46.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2905354/">PMC free article</a>]</span> [<a href="/pubmed/20565885" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Neurol&amp;title=A+systematic+review+of+population+based+epidemiological+studies+in+Myasthenia+Gravis&amp;author=AS+Carr&amp;author=CR+Cardwell&amp;author=PO+McCarron&amp;author=J+McConville&amp;volume=10&amp;publication_year=2010&amp;pages=46&amp;pmid=20565885&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b4-15_232_seldin">4. <span class="element-citation">Lindstrom JM, Seybold ME, Lennon VA, Whittingham S, Duane DD. Antibody to acetylcholine receptor in myasthenia gravis. Prevalence, clinical correlates, and diagnostic value. <span><span class="ref-journal">Neurology. </span>1976;<span class="ref-vol">26</span>:1054–9.</span> [<a href="/pubmed/988512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Antibody+to+acetylcholine+receptor+in+myasthenia+gravis.+Prevalence,+clinical+correlates,+and+diagnostic+value&amp;author=JM+Lindstrom&amp;author=ME+Seybold&amp;author=VA+Lennon&amp;author=S+Whittingham&amp;author=DD+Duane&amp;volume=26&amp;publication_year=1976&amp;pages=1054-9&amp;pmid=988512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b5-15_232_seldin">5. <span class="element-citation">Chuang WY, et al.  Late-onset myasthenia gravis-CTLA4(low) genotype association and low-for-age thymic output of naive T cells. <span><span class="ref-journal">J Autoimmun. </span>2014;<span class="ref-vol">52</span>:122–9.</span> [<a href="/pubmed/24373506" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autoimmun&amp;title=Late-onset+myasthenia+gravis-CTLA4(low)+genotype+association+and+low-for-age+thymic+output+of+naive+T+cells&amp;author=WY+Chuang&amp;volume=52&amp;publication_year=2014&amp;pages=122-9&amp;pmid=24373506&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b6-15_232_seldin">6. <span class="element-citation">Maniaol AH, et al.  Late onset myasthenia gravis is associated with HLA DRB1*15:01 in the Norwegian population. <span><span class="ref-journal">PLoS One. </span>2012;<span class="ref-vol">7</span>:e36603.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3348874/">PMC free article</a>]</span> [<a href="/pubmed/22590574" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Late+onset+myasthenia+gravis+is+associated+with+HLA+DRB1*15:01+in+the+Norwegian+population&amp;author=AH+Maniaol&amp;volume=7&amp;publication_year=2012&amp;pages=e36603&amp;pmid=22590574&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b7-15_232_seldin">7. <span class="element-citation">Compston DA, Vincent A, Newsom-Davis J, Batchelor JR. Clinical, pathological, HLA antigen and immunological evidence for disease heterogeneity in myasthenia gravis. <span><span class="ref-journal">Brain. </span>1980;<span class="ref-vol">103</span>:579–601.</span> [<a href="/pubmed/6968236" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Clinical,+pathological,+HLA+antigen+and+immunological+evidence+for+disease+heterogeneity+in+myasthenia+gravis&amp;author=DA+Compston&amp;author=A+Vincent&amp;author=J+Newsom-Davis&amp;author=JR+Batchelor&amp;volume=103&amp;publication_year=1980&amp;pages=579-601&amp;pmid=6968236&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b8-15_232_seldin">8. <span class="element-citation">Evoli A, Batocchi AP, Minisci C, Di Schino C, Tonali P. Clinical characteristics and prognosis of myasthenia gravis in older people. <span><span class="ref-journal">J Am Geriatr Soc. </span>2000;<span class="ref-vol">48</span>:1442–8.</span> [<a href="/pubmed/11083321" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Am+Geriatr+Soc&amp;title=Clinical+characteristics+and+prognosis+of+myasthenia+gravis+in+older+people&amp;author=A+Evoli&amp;author=AP+Batocchi&amp;author=C+Minisci&amp;author=C+Di+Schino&amp;author=P+Tonali&amp;volume=48&amp;publication_year=2000&amp;pages=1442-8&amp;pmid=11083321&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b9-15_232_seldin">9. <span class="element-citation">Akaishi T, et al.  Insights into the classification of myasthenia gravis. <span><span class="ref-journal">PLoS One. </span>2014;<span class="ref-vol">9</span>:e106757.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4156422/">PMC free article</a>]</span> [<a href="/pubmed/25192469" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Insights+into+the+classification+of+myasthenia+gravis&amp;author=T+Akaishi&amp;volume=9&amp;publication_year=2014&amp;pages=e106757&amp;pmid=25192469&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b10-15_232_seldin">10. <span class="element-citation">Janer M, et al.  A susceptibility region for myasthenia gravis extending into the HLA-class I sector telomeric to HLA-C. <span><span class="ref-journal">Hum Immunol. </span>1999;<span class="ref-vol">60</span>:909–17.</span> [<a href="/pubmed/10527401" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Immunol&amp;title=A+susceptibility+region+for+myasthenia+gravis+extending+into+the+HLA-class+I+sector+telomeric+to+HLA-C&amp;author=M+Janer&amp;volume=60&amp;publication_year=1999&amp;pages=909-17&amp;pmid=10527401&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b11-15_232_seldin">11. <span class="element-citation">Avidan N, Le Panse R, Berrih-Aknin S, Miller A. Genetic basis of myasthenia gravis: a comprehensive review. <span><span class="ref-journal">J Autoimmun. </span>2014;<span class="ref-vol">52</span>:146–53.</span> [<a href="/pubmed/24361103" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Autoimmun&amp;title=Genetic+basis+of+myasthenia+gravis:+a+comprehensive+review&amp;author=N+Avidan&amp;author=R+Le+Panse&amp;author=S+Berrih-Aknin&amp;author=A+Miller&amp;volume=52&amp;publication_year=2014&amp;pages=146-53&amp;pmid=24361103&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b12-15_232_seldin">12. <span class="element-citation">Giraud M, et al.  Linkage of HLA to myasthenia gravis and genetic heterogeneity depending on anti-titin antibodies. <span><span class="ref-journal">Neurology. </span>2001;<span class="ref-vol">57</span>:1555–60.</span> [<a href="/pubmed/11706089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurology&amp;title=Linkage+of+HLA+to+myasthenia+gravis+and+genetic+heterogeneity+depending+on+anti-titin+antibodies&amp;author=M+Giraud&amp;volume=57&amp;publication_year=2001&amp;pages=1555-60&amp;pmid=11706089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b13-15_232_seldin">13. <span class="element-citation">Grob D, Brunner N, Namba T, Pagala M. Lifetime course of myasthenia gravis. <span><span class="ref-journal">Muscle Nerve. </span>2008;<span class="ref-vol">37</span>:141–9.</span> [<a href="/pubmed/18059039" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Muscle+Nerve&amp;title=Lifetime+course+of+myasthenia+gravis&amp;author=D+Grob&amp;author=N+Brunner&amp;author=T+Namba&amp;author=M+Pagala&amp;volume=37&amp;publication_year=2008&amp;pages=141-9&amp;pmid=18059039&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b14-15_232_seldin">14. <span class="element-citation">Vandiedonck C, et al.  Association of the PTPN22*R620W polymorphism with autoimmune myasthenia gravis. <span><span class="ref-journal">Ann Neurol. </span>2006;<span class="ref-vol">59</span>:404–7.</span> [<a href="/pubmed/16437561" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Association+of+the+PTPN22*R620W+polymorphism+with+autoimmune+myasthenia+gravis&amp;author=C+Vandiedonck&amp;volume=59&amp;publication_year=2006&amp;pages=404-7&amp;pmid=16437561&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b15-15_232_seldin">15. <span class="element-citation">Gregersen PK, et al.  Risk for myasthenia gravis maps to a (151) Pro—&gt;Ala change in TNIP1 and to human leukocyte antigen-B*08. <span><span class="ref-journal">Ann Neurol. </span>2012;<span class="ref-vol">72</span>:927–935.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3535539/">PMC free article</a>]</span> [<a href="/pubmed/23055271" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Neurol&amp;title=Risk+for+myasthenia+gravis+maps+to+a+(151)+Pro—>Ala+change+in+TNIP1+and+to+human+leukocyte+antigen-B*08&amp;author=PK+Gregersen&amp;volume=72&amp;publication_year=2012&amp;pages=927-935&amp;pmid=23055271&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b16-15_232_seldin">16. <span class="element-citation">Renton AE, et al.  A genome-wide association study of myasthenia gravis. <span><span class="ref-journal">JAMA Neurol. </span>2015;<span class="ref-vol">72</span>:396–404.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4856525/">PMC free article</a>]</span> [<a href="/pubmed/25643325" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA+Neurol&amp;title=A+genome-wide+association+study+of+myasthenia+gravis&amp;author=AE+Renton&amp;volume=72&amp;publication_year=2015&amp;pages=396-404&amp;pmid=25643325&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b17-15_232_seldin">17. <span class="element-citation">Purcell S, et al.  PLINK: a tool set for whole-genome association and population-based linkage analyses. <span><span class="ref-journal">Am J Hum Genet. </span>2007;<span class="ref-vol">81</span>:559–75.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1950838/">PMC free article</a>]</span> [<a href="/pubmed/17701901" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=PLINK:+a+tool+set+for+whole-genome+association+and+population-based+linkage+analyses&amp;author=S+Purcell&amp;volume=81&amp;publication_year=2007&amp;pages=559-75&amp;pmid=17701901&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b18-15_232_seldin">18. <span class="element-citation">Pritchard JK, Stephens M, Donnelly P. Inference of population structure using multilocus genotype data. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">155</span>:945–59.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1461096/">PMC free article</a>]</span> [<a href="/pubmed/10835412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-59&amp;pmid=10835412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b19-15_232_seldin">19. <span class="element-citation">Falush D, Stephens M, Pritchard JK. Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies. <span><span class="ref-journal">Genetics. </span>2003;<span class="ref-vol">164</span>:1567–87.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1462648/">PMC free article</a>]</span> [<a href="/pubmed/12930761" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference+of+population+structure+using+multilocus+genotype+data:+linked+loci+and+correlated+allele+frequencies&amp;author=D+Falush&amp;author=M+Stephens&amp;author=JK+Pritchard&amp;volume=164&amp;publication_year=2003&amp;pages=1567-87&amp;pmid=12930761&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b20-15_232_seldin">20. <span class="element-citation">Kosoy R, et al.  Ancestry informative marker sets for determining continental origin and admixture proportions in common populations in America. <span><span class="ref-journal">Hum Mutat. </span>2009;<span class="ref-vol">30</span>:69–78.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3073397/">PMC free article</a>]</span> [<a href="/pubmed/18683858" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum+Mutat&amp;title=Ancestry+informative+marker+sets+for+determining+continental+origin+and+admixture+proportions+in+common+populations+in+America&amp;author=R+Kosoy&amp;volume=30&amp;publication_year=2009&amp;pages=69-78&amp;pmid=18683858&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b21-15_232_seldin">21. <span class="element-citation">Price AL, et al.  Principal components analysis corrects for stratification in genome-wide association studies. <span><span class="ref-journal">Nat Genet. </span>2006;<span class="ref-vol">38</span>:904–9.</span> [<a href="/pubmed/16862161" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Principal+components+analysis+corrects+for+stratification+in+genome-wide+association+studies&amp;author=AL+Price&amp;volume=38&amp;publication_year=2006&amp;pages=904-9&amp;pmid=16862161&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b22-15_232_seldin">22. <span class="element-citation">O’Connell J, et al.  A general approach for haplotype phasing across the full spectrum of relatedness. <span><span class="ref-journal">PLoS Genet. </span>2014;<span class="ref-vol">10</span>:e1004234.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3990520/">PMC free article</a>]</span> [<a href="/pubmed/24743097" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+general+approach+for+haplotype+phasing+across+the+full+spectrum+of+relatedness&amp;author=J+O’Connell&amp;volume=10&amp;publication_year=2014&amp;pages=e1004234&amp;pmid=24743097&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b23-15_232_seldin">23. <span class="element-citation">Fuchsberger C, Abecasis GR, Hinds DA. minimac2: faster genotype imputation. <span><span class="ref-journal">Bioinformatics. </span>2014;<span class="ref-vol">31</span>:782–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4341061/">PMC free article</a>]</span> [<a href="/pubmed/25338720" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=minimac2:+faster+genotype+imputation&amp;author=C+Fuchsberger&amp;author=GR+Abecasis&amp;author=DA+Hinds&amp;volume=31&amp;publication_year=2014&amp;pages=782-4&amp;pmid=25338720&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b24-15_232_seldin">24. <span class="element-citation">Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. <span><span class="ref-journal">PLoS Genet. </span>2009;<span class="ref-vol">5</span>:e1000529.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689936/">PMC free article</a>]</span> [<a href="/pubmed/19543373" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+flexible+and+accurate+genotype+imputation+method+for+the+next+generation+of+genome-wide+association+studies&amp;author=BN+Howie&amp;author=P+Donnelly&amp;author=J+Marchini&amp;volume=5&amp;publication_year=2009&amp;pages=e1000529&amp;pmid=19543373&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b25-15_232_seldin">25. <span class="element-citation">Jia X, et al.  Imputing amino acid polymorphisms in human leukocyte antigens. <span><span class="ref-journal">PLoS One. </span>2013;<span class="ref-vol">8</span>:e64683.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3675122/">PMC free article</a>]</span> [<a href="/pubmed/23762245" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Imputing+amino+acid+polymorphisms+in+human+leukocyte+antigens&amp;author=X+Jia&amp;volume=8&amp;publication_year=2013&amp;pages=e64683&amp;pmid=23762245&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b26-15_232_seldin">26. <span class="element-citation">Browning BL, Browning SR. A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals. <span><span class="ref-journal">Am J Hum Genet. </span>2009;<span class="ref-vol">84</span>:210–23.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2668004/">PMC free article</a>]</span> [<a href="/pubmed/19200528" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+unified+approach+to+genotype+imputation+and+haplotype-phase+inference+for+large+data+sets+of+trios+and+unrelated+individuals&amp;author=BL+Browning&amp;author=SR+Browning&amp;volume=84&amp;publication_year=2009&amp;pages=210-23&amp;pmid=19200528&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b27-15_232_seldin">27. <span class="element-citation">Boyle AP, et al.  Annotation of functional variation in personal genomes using RegulomeDB. <span><span class="ref-journal">Genome Res. </span>2012;<span class="ref-vol">22</span>:1790–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3431494/">PMC free article</a>]</span> [<a href="/pubmed/22955989" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Annotation+of+functional+variation+in+personal+genomes+using+RegulomeDB&amp;author=AP+Boyle&amp;volume=22&amp;publication_year=2012&amp;pages=1790-7&amp;pmid=22955989&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b28-15_232_seldin">28. <span class="element-citation">Pruim RJ, et al.  LocusZoom: regional visualization of genome-wide association scan results. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>:2336–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2935401/">PMC free article</a>]</span> [<a href="/pubmed/20634204" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LocusZoom:+regional+visualization+of+genome-wide+association+scan+results&amp;author=RJ+Pruim&amp;volume=26&amp;publication_year=2010&amp;pages=2336-7&amp;pmid=20634204&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b29-15_232_seldin">29. <span class="element-citation">Degner JF, et al.  DNase I sensitivity QTLs are a major determinant of human expression variation. <span><span class="ref-journal">Nature. </span>2012;<span class="ref-vol">482</span>:390–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3501342/">PMC free article</a>]</span> [<a href="/pubmed/22307276" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=DNase+I+sensitivity+QTLs+are+a+major+determinant+of+human+expression+variation&amp;author=JF+Degner&amp;volume=482&amp;publication_year=2012&amp;pages=390-4&amp;pmid=22307276&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b30-15_232_seldin">30. <span class="element-citation">Coviello AD, et al.  A genome-wide association meta-analysis of circulating sex hormone-binding globulin reveals multiple Loci implicated in sex steroid hormone regulation. <span><span class="ref-journal">PLoS Genet. </span>2012;<span class="ref-vol">8</span>:e1002805.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3400553/">PMC free article</a>]</span> [<a href="/pubmed/22829776" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet&amp;title=A+genome-wide+association+meta-analysis+of+circulating+sex+hormone-binding+globulin+reveals+multiple+Loci+implicated+in+sex+steroid+hormone+regulation&amp;author=AD+Coviello&amp;volume=8&amp;publication_year=2012&amp;pages=e1002805&amp;pmid=22829776&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b31-15_232_seldin">31. <span class="element-citation">Ferreira MA, et al.  Genome-wide association analysis identifies 11 risk variants associated with the asthma with hay fever phenotype. <span><span class="ref-journal">J Allergy Clin Immunol. </span>2014;<span class="ref-vol">133</span>:1564–71.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4280183/">PMC free article</a>]</span> [<a href="/pubmed/24388013" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Allergy+Clin+Immunol&amp;title=Genome-wide+association+analysis+identifies+11+risk+variants+associated+with+the+asthma+with+hay+fever+phenotype&amp;author=MA+Ferreira&amp;volume=133&amp;publication_year=2014&amp;pages=1564-71&amp;pmid=24388013&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b32-15_232_seldin">32. <span class="element-citation">Eriksson N, et al.  Novel associations for hypothyroidism include known autoimmune risk loci. <span><span class="ref-journal">PLoS One. </span>2012;<span class="ref-vol">7</span>:e34442.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3321023/">PMC free article</a>]</span> [<a href="/pubmed/22493691" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Novel+associations+for+hypothyroidism+include+known+autoimmune+risk+loci&amp;author=N+Eriksson&amp;volume=7&amp;publication_year=2012&amp;pages=e34442&amp;pmid=22493691&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b33-15_232_seldin">33. <span class="element-citation">Paternoster L, et al.  Meta-analysis of genome-wide association studies identifies three new risk loci for atopic dermatitis. <span><span class="ref-journal">Nat Genet. </span>2011;<span class="ref-vol">44</span>:187–92.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3272375/">PMC free article</a>]</span> [<a href="/pubmed/22197932" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Meta-analysis+of+genome-wide+association+studies+identifies+three+new+risk+loci+for+atopic+dermatitis&amp;author=L+Paternoster&amp;volume=44&amp;publication_year=2011&amp;pages=187-92&amp;pmid=22197932&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b34-15_232_seldin">34. <span class="element-citation">Hinds DA, et al.  A genome-wide association meta-analysis of self-reported allergy identifies shared and allergy-specific susceptibility loci. <span><span class="ref-journal">Nat Genet. </span>2013;<span class="ref-vol">45</span>:907–11.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3753407/">PMC free article</a>]</span> [<a href="/pubmed/23817569" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+genome-wide+association+meta-analysis+of+self-reported+allergy+identifies+shared+and+allergy-specific+susceptibility+loci&amp;author=DA+Hinds&amp;volume=45&amp;publication_year=2013&amp;pages=907-11&amp;pmid=23817569&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b35-15_232_seldin">35. <span class="element-citation">Tillotson LG. RIN ZF, a novel zinc finger gene, encodes proteins that bind to the CACC element of the gastrin promoter. <span><span class="ref-journal">J Biol Chem. </span>1999;<span class="ref-vol">274</span>:8123–8.</span> [<a href="/pubmed/10075714" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=RIN+ZF,+a+novel+zinc+finger+gene,+encodes+proteins+that+bind+to+the+CACC+element+of+the+gastrin+promoter&amp;author=LG+Tillotson&amp;volume=274&amp;publication_year=1999&amp;pages=8123-8&amp;pmid=10075714&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b36-15_232_seldin">36. <span class="element-citation">Li X, et al.  MicroRNA-27a indirectly regulates estrogen receptor α expression and hormone responsiveness in MCF-7 breast cancer cells. <span><span class="ref-journal">Endocrinology. </span>2010;<span class="ref-vol">151</span>:2462–73.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2875816/">PMC free article</a>]</span> [<a href="/pubmed/20382698" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Endocrinology&amp;title=MicroRNA-27a+indirectly+regulates+estrogen+receptor+α+expression+and+hormone+responsiveness+in+MCF-7+breast+cancer+cells&amp;author=X+Li&amp;volume=151&amp;publication_year=2010&amp;pages=2462-73&amp;pmid=20382698&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b37-15_232_seldin">37. <span class="element-citation">Tone M, Powell MJ, Tone Y, Thompson SA, Waldmann H. IL-10 gene expression is controlled by the transcription factors Sp1 and Sp3. <span><span class="ref-journal">J Immunol. </span>2000;<span class="ref-vol">165</span>:286–91.</span> [<a href="/pubmed/10861063" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Immunol&amp;title=IL-10+gene+expression+is+controlled+by+the+transcription+factors+Sp1+and+Sp3&amp;author=M+Tone&amp;author=MJ+Powell&amp;author=Y+Tone&amp;author=SA+Thompson&amp;author=H+Waldmann&amp;volume=165&amp;publication_year=2000&amp;pages=286-91&amp;pmid=10861063&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b38-15_232_seldin">38. <span class="element-citation">Tone M, Tone Y, Babik JM, Lin CY, Waldmann H. The role of Sp1 and NF-kappa B in regulating CD40 gene expression. <span><span class="ref-journal">J Biol Chem. </span>2002;<span class="ref-vol">277</span>:8890–7.</span> [<a href="/pubmed/11751910" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Biol+Chem&amp;title=The+role+of+Sp1+and+NF-kappa+B+in+regulating+CD40+gene+expression&amp;author=M+Tone&amp;author=Y+Tone&amp;author=JM+Babik&amp;author=CY+Lin&amp;author=H+Waldmann&amp;volume=277&amp;publication_year=2002&amp;pages=8890-7&amp;pmid=11751910&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b39-15_232_seldin">39. <span class="element-citation">Bynote KK, et al.  Estrogen receptor-alpha deficiency attenuates autoimmune disease in (NZB x NZW)F1 mice. <span><span class="ref-journal">Genes Immun. </span>2008;<span class="ref-vol">9</span>:137–52.</span> [<a href="/pubmed/18200028" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=Estrogen+receptor-alpha+deficiency+attenuates+autoimmune+disease+in+(NZB+x+NZW)F1+mice&amp;author=KK+Bynote&amp;volume=9&amp;publication_year=2008&amp;pages=137-52&amp;pmid=18200028&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b40-15_232_seldin">40. <span class="element-citation">Rubtsova K, Marrack P, Rubtsov AV. Sexual dimorphism in autoimmunity. <span><span class="ref-journal">J Clin Invest. </span>2015;<span class="ref-vol">125</span>:2187–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4497744/">PMC free article</a>]</span> [<a href="/pubmed/25915581" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Clin+Invest&amp;title=Sexual+dimorphism+in+autoimmunity&amp;author=K+Rubtsova&amp;author=P+Marrack&amp;author=AV+Rubtsov&amp;volume=125&amp;publication_year=2015&amp;pages=2187-93&amp;pmid=25915581&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b41-15_232_seldin">41. <span class="element-citation">Kaya GA, et al.  The association of PTPN22 R620W polymorphism is stronger with late-onset AChR-myasthenia gravis in Turkey. <span><span class="ref-journal">PLoS One. </span>2014;<span class="ref-vol">9</span>:e104760.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4132102/">PMC free article</a>]</span> [<a href="/pubmed/25119822" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=The+association+of+PTPN22+R620W+polymorphism+is+stronger+with+late-onset+AChR-myasthenia+gravis+in+Turkey&amp;author=GA+Kaya&amp;volume=9&amp;publication_year=2014&amp;pages=e104760&amp;pmid=25119822&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b42-15_232_seldin">42. <span class="element-citation">Bottini N, et al.  A functional variant of lymphoid tyrosine phosphatase is associated with type I diabetes. <span><span class="ref-journal">Nat Genet. </span>2004;<span class="ref-vol">36</span>:337–8.</span> [<a href="/pubmed/15004560" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=A+functional+variant+of+lymphoid+tyrosine+phosphatase+is+associated+with+type+I+diabetes&amp;author=N+Bottini&amp;volume=36&amp;publication_year=2004&amp;pages=337-8&amp;pmid=15004560&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b43-15_232_seldin">43. <span class="element-citation">Begovich AB, et al.  A missense single- nucleotide polymorphism in a gene encoding a protein tyrosine phosphatase (PTPN22) is associated with rheumatoid arthritis. <span><span class="ref-journal">Am J Hum Genet. </span>2004;<span class="ref-vol">75</span>:330–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1216068/">PMC free article</a>]</span> [<a href="/pubmed/15208781" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=A+missense+single-+nucleotide+polymorphism+in+a+gene+encoding+a+protein+tyrosine+phosphatase+(PTPN22)+is+associated+with+rheumatoid+arthritis&amp;author=AB+Begovich&amp;volume=75&amp;publication_year=2004&amp;pages=330-7&amp;pmid=15208781&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b44-15_232_seldin">44. <span class="element-citation">Kyogoku C, et al.  Genetic association of the R620W polymorphism of protein tyrosine phosphatase PTPN22 with human SLE. <span><span class="ref-journal">Am J Hum Genet. </span>2004;<span class="ref-vol">75</span>:504–7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1182029/">PMC free article</a>]</span> [<a href="/pubmed/15273934" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Genetic+association+of+the+R620W+polymorphism+of+protein+tyrosine+phosphatase+PTPN22+with+human+SLE&amp;author=C+Kyogoku&amp;volume=75&amp;publication_year=2004&amp;pages=504-7&amp;pmid=15273934&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b45-15_232_seldin">45. <span class="element-citation">Bottini N, Peterson EJ. Tyrosine phosphatase PTPN22: multifunctional regulator of immune signaling, development, and disease. <span><span class="ref-journal">Ann Rev Immunol. </span>2014;<span class="ref-vol">32</span>:83–119.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6402334/">PMC free article</a>]</span> [<a href="/pubmed/24364806" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann+Rev+Immunol&amp;title=Tyrosine+phosphatase+PTPN22:+multifunctional+regulator+of+immune+signaling,+development,+and+disease&amp;author=N+Bottini&amp;author=EJ+Peterson&amp;volume=32&amp;publication_year=2014&amp;pages=83-119&amp;pmid=24364806&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b46-15_232_seldin">46. <span class="element-citation">Hughes AE, et al.  Mutations in TNFRSF11A, affecting the signal peptide of RANK, cause familial expansile osteolysis. <span><span class="ref-journal">Nat Genet. </span>2000;<span class="ref-vol">24</span>:45–8.</span> [<a href="/pubmed/10615125" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Mutations+in+TNFRSF11A,+affecting+the+signal+peptide+of+RANK,+cause+familial+expansile+osteolysis&amp;author=AE+Hughes&amp;volume=24&amp;publication_year=2000&amp;pages=45-8&amp;pmid=10615125&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b47-15_232_seldin">47. <span class="element-citation">Guerrini MM, et al.  Human osteoclast-poor osteopetrosis with hypogammaglobulinemia due to TNFRSF11A (RANK) mutations. <span><span class="ref-journal">Am J Hum Genet. </span>2008;<span class="ref-vol">83</span>:64–76.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2443850/">PMC free article</a>]</span> [<a href="/pubmed/18606301" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Human+osteoclast-poor+osteopetrosis+with+hypogammaglobulinemia+due+to+TNFRSF11A+(RANK)+mutations&amp;author=MM+Guerrini&amp;volume=83&amp;publication_year=2008&amp;pages=64-76&amp;pmid=18606301&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b48-15_232_seldin">48. <span class="element-citation">Albagha OM, et al.  Genome-wide association study identifies variants at CSF1, OPTN and TNFRSF11A as genetic risk factors for Paget’s disease of bone. <span><span class="ref-journal">Nat Genet. </span>2010;<span class="ref-vol">42</span>:520–4.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3217192/">PMC free article</a>]</span> [<a href="/pubmed/20436471" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+association+study+identifies+variants+at+CSF1,+OPTN+and+TNFRSF11A+as+genetic+risk+factors+for+Paget’s+disease+of+bone&amp;author=OM+Albagha&amp;volume=42&amp;publication_year=2010&amp;pages=520-4&amp;pmid=20436471&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b49-15_232_seldin">49. <span class="element-citation">Estrada K, et al.  Genome-wide meta-analysis identifies 56 bone mineral density loci and reveals 14 loci associated with risk of fracture. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:491–501.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3338864/">PMC free article</a>]</span> [<a href="/pubmed/22504420" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Genome-wide+meta-analysis+identifies+56+bone+mineral+density+loci+and+reveals+14+loci+associated+with+risk+of+fracture&amp;author=K+Estrada&amp;volume=44&amp;publication_year=2012&amp;pages=491-501&amp;pmid=22504420&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b50-15_232_seldin">50. <span class="element-citation">Roberts NA, et al.  Rank signaling links the development of invariant gammadelta T cell progenitors and Aire(+) medullary epithelium. <span><span class="ref-journal">Immunity. </span>2012;<span class="ref-vol">36</span>:427–37.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3368267/">PMC free article</a>]</span> [<a href="/pubmed/22425250" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Rank+signaling+links+the+development+of+invariant+gammadelta+T+cell+progenitors+and+Aire(+)+medullary+epithelium&amp;author=NA+Roberts&amp;volume=36&amp;publication_year=2012&amp;pages=427-37&amp;pmid=22425250&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b51-15_232_seldin">51. <span class="element-citation">Anderson DM, et al.  A homologue of the TNF receptor and its ligand enhance T-cell growth and dendritic-cell function. <span><span class="ref-journal">Nature. </span>1997;<span class="ref-vol">390</span>:175–9.</span> [<a href="/pubmed/9367155" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+homologue+of+the+TNF+receptor+and+its+ligand+enhance+T-cell+growth+and+dendritic-cell+function&amp;author=DM+Anderson&amp;volume=390&amp;publication_year=1997&amp;pages=175-9&amp;pmid=9367155&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b52-15_232_seldin">52. <span class="element-citation">Dougall WC, et al.  RANK is essential for osteoclast and lymph node development. <span><span class="ref-journal">Genes Dev. </span>1999;<span class="ref-vol">13</span>:2412–24.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC317030/">PMC free article</a>]</span> [<a href="/pubmed/10500098" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Dev&amp;title=RANK+is+essential+for+osteoclast+and+lymph+node+development&amp;author=WC+Dougall&amp;volume=13&amp;publication_year=1999&amp;pages=2412-24&amp;pmid=10500098&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b53-15_232_seldin">53. <span class="element-citation">Mewar D, et al.  Haplotype-specific gene expression profiles in a telomeric major histocompatibility complex gene cluster and susceptibility to autoimmune diseases. <span><span class="ref-journal">Genes Immun. </span>2006;<span class="ref-vol">7</span>:625–31.</span> [<a href="/pubmed/16971954" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genes+Immun&amp;title=Haplotype-specific+gene+expression+profiles+in+a+telomeric+major+histocompatibility+complex+gene+cluster+and+susceptibility+to+autoimmune+diseases&amp;author=D+Mewar&amp;volume=7&amp;publication_year=2006&amp;pages=625-31&amp;pmid=16971954&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b54-15_232_seldin">54. <span class="element-citation">Noble JA, et al.  HLA class I and genetic susceptibility to type 1 diabetes: results from the type 1 Diabetes Genetics Consortium. <span><span class="ref-journal">Diabetes. </span>2010;<span class="ref-vol">59</span>:2972–9.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2963558/">PMC free article</a>]</span> [<a href="/pubmed/20798335" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=HLA+class+I+and+genetic+susceptibility+to+type+1+diabetes:+results+from+the+type+1+Diabetes+Genetics+Consortium&amp;author=JA+Noble&amp;volume=59&amp;publication_year=2010&amp;pages=2972-9&amp;pmid=20798335&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b55-15_232_seldin">55. <span class="element-citation">Raychaudhuri S, et al.  Five amino acids in three HLA proteins explain most of the association between MHC and seropositive rheumatoid arthritis. <span><span class="ref-journal">Nat Genet. </span>2012;<span class="ref-vol">44</span>:291–6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3288335/">PMC free article</a>]</span> [<a href="/pubmed/22286218" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Five+amino+acids+in+three+HLA+proteins+explain+most+of+the+association+between+MHC+and+seropositive+rheumatoid+arthritis&amp;author=S+Raychaudhuri&amp;volume=44&amp;publication_year=2012&amp;pages=291-6&amp;pmid=22286218&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b56-15_232_seldin">56. <span class="element-citation">Jawaheer D, et al.  Dissecting the genetic complexity of the association between human leukocyte antigens and rheumatoid arthritis. <span><span class="ref-journal">Am J Hum Genet. </span>2002;<span class="ref-vol">71</span>:585–94.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC449696/">PMC free article</a>]</span> [<a href="/pubmed/12181776" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Dissecting+the+genetic+complexity+of+the+association+between+human+leukocyte+antigens+and+rheumatoid+arthritis&amp;author=D+Jawaheer&amp;volume=71&amp;publication_year=2002&amp;pages=585-94&amp;pmid=12181776&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b57-15_232_seldin">57. <span class="element-citation">Morris DL, et al.  Unraveling multiple MHC gene associations with systemic lupus erythematosus: model choice indicates a role for HLA alleles and non-HLA genes in Europeans. <span><span class="ref-journal">Am J Hum Genet. </span>2012;<span class="ref-vol">91</span>:778–93.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3487133/">PMC free article</a>]</span> [<a href="/pubmed/23084292" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am+J+Hum+Genet&amp;title=Unraveling+multiple+MHC+gene+associations+with+systemic+lupus+erythematosus:+model+choice+indicates+a+role+for+HLA+alleles+and+non-HLA+genes+in+Europeans&amp;author=DL+Morris&amp;volume=91&amp;publication_year=2012&amp;pages=778-93&amp;pmid=23084292&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b58-15_232_seldin">58. <span class="element-citation">Quinones-Lombrana A, et al.  BAT1 promoter polymorphism is associated with rheumatoid arthritis susceptibility. <span><span class="ref-journal">J Rheumatol. </span>2008;<span class="ref-vol">35</span>:741–4.</span> [<a href="/pubmed/18381799" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J+Rheumatol&amp;title=BAT1+promoter+polymorphism+is+associated+with+rheumatoid+arthritis+susceptibility&amp;author=A+Quinones-Lombrana&amp;volume=35&amp;publication_year=2008&amp;pages=741-4&amp;pmid=18381799&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b59-15_232_seldin">59. <span class="element-citation">Gockel I, et al.  Common variants in the HLA-DQ region confer susceptibility to idiopathic achalasia. <span><span class="ref-journal">Nat Genet. </span>2014;<span class="ref-vol">46</span>:901–4.</span> [<a href="/pubmed/24997987" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat+Genet&amp;title=Common+variants+in+the+HLA-DQ+region+confer+susceptibility+to+idiopathic+achalasia&amp;author=I+Gockel&amp;volume=46&amp;publication_year=2014&amp;pages=901-4&amp;pmid=24997987&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b60-15_232_seldin">60. <span class="element-citation">McMichael AJ, Gotch FM, Santos-Aguado J, Strominger JL. Effect of mutations and variations of HLA-A2 on recognition of a virus peptide epitope by cytotoxic T lymphocytes. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>1988;<span class="ref-vol">85</span>:9194–8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC282705/">PMC free article</a>]</span> [<a href="/pubmed/2461564" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Effect+of+mutations+and+variations+of+HLA-A2+on+recognition+of+a+virus+peptide+epitope+by+cytotoxic+T+lymphocytes&amp;author=AJ+McMichael&amp;author=FM+Gotch&amp;author=J+Santos-Aguado&amp;author=JL+Strominger&amp;volume=85&amp;publication_year=1988&amp;pages=9194-8&amp;pmid=2461564&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b61-15_232_seldin">61. <span class="element-citation">Tackenberg B, et al.  Clonal expansions of CD4+ B helper T cells in autoimmune myasthenia gravis. <span><span class="ref-journal">Eur J Immunol. </span>2007;<span class="ref-vol">37</span>:849–63.</span> [<a href="/pubmed/17323412" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur+J+Immunol&amp;title=Clonal+expansions+of+CD4++B+helper+T+cells+in+autoimmune+myasthenia+gravis&amp;author=B+Tackenberg&amp;volume=37&amp;publication_year=2007&amp;pages=849-63&amp;pmid=17323412&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b62-15_232_seldin">62. <span class="element-citation">Selmi C, Brunetta E, Raimondo MG, Meroni PL. The X chromosome and the sex ratio of autoimmunity. <span><span class="ref-journal">Autoimmun Rev. </span>2012;<span class="ref-vol">11</span>:A531–7.</span> [<a href="/pubmed/22155196" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Autoimmun+Rev&amp;title=The+X+chromosome+and+the+sex+ratio+of+autoimmunity&amp;author=C+Selmi&amp;author=E+Brunetta&amp;author=MG+Raimondo&amp;author=PL+Meroni&amp;volume=11&amp;publication_year=2012&amp;pages=A531-7&amp;pmid=22155196&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b63-15_232_seldin">63. <span class="element-citation">Padua L, et al.  SFEMG in ocular myasthenia gravis diagnosis. <span><span class="ref-journal">Clin Neurophysiol. </span>2000;<span class="ref-vol">111</span>:1203–7.</span> [<a href="/pubmed/10880794" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Neurophysiol&amp;title=SFEMG+in+ocular+myasthenia+gravis+diagnosis&amp;author=L+Padua&amp;volume=111&amp;publication_year=2000&amp;pages=1203-7&amp;pmid=10880794&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b64-15_232_seldin">64. <span class="element-citation">Farrugia ME, Jacob S, Sarrigiannis PG, Kennett RP. Correlating extent of neuromuscular instability with acetylcholine receptor antibodies. <span><span class="ref-journal">Muscle Nerve. </span>2009;<span class="ref-vol">39</span>:489–93.</span> [<a href="/pubmed/19260064" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Muscle+Nerve&amp;title=Correlating+extent+of+neuromuscular+instability+with+acetylcholine+receptor+antibodies&amp;author=ME+Farrugia&amp;author=S+Jacob&amp;author=PG+Sarrigiannis&amp;author=RP+Kennett&amp;volume=39&amp;publication_year=2009&amp;pages=489-93&amp;pmid=19260064&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="b65-15_232_seldin">65. <span class="element-citation">Witoonpanich R, Dejthevaporn C, Sriphrapradang A, Pulkes T. Electrophysiological and immunological study in myasthenia gravis: diagnostic sensitivity and correlation. <span><span class="ref-journal">Clin Neurophysiol. </span>2011;<span class="ref-vol">122</span>:1873–7.</span> [<a href="/pubmed/21419697" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin+Neurophysiol&amp;title=Electrophysiological+and+immunological+study+in+myasthenia+gravis:+diagnostic+sensitivity+and+correlation&amp;author=R+Witoonpanich&amp;author=C+Dejthevaporn&amp;author=A+Sriphrapradang&amp;author=T+Pulkes&amp;volume=122&amp;publication_year=2011&amp;pages=1873-7&amp;pmid=21419697&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4749491&amp;issue-id=253966&amp;journal-id=325&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140705577889472title">Abstract</a></li><li><a href="#__sec1title">INTRODUCTION</a></li><li><a href="#__sec2title">MATERIALS AND METHODS</a></li><li><a href="#__sec12title">RESULTS</a></li><li><a href="#__sec20title">DISCUSSION</a></li><li><a href="#__sec21title">CONCLUSION</a></li><li><a href="#__sec22title">Supplemental Data</a></li><li><a href="#idm140705612488656title">ACKNOWLEDGMENTS</a></li><li><a href="#idm140705612486672title">Footnotes</a></li><li><a href="#idm140705612478208title">REFERENCES</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Molecular Medicine</span> are provided here courtesy of <strong>The Feinstein Institute for Medical Research at North Shore LIJ</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4749491/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4749491/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4749491/pdf/15_232_seldin.pdf">PDF (3.7M)</a></li> | <li><a href="#" data-citationid="PMC4749491" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4749491%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4749491%2F&amp;text=Genome-Wide%20Association%20Study%20of%20Late-Onset%20Myasthenia%20Gravis%3A%20Confirmation%20of%20TNFRSF11A%20and%20Identification%20of%20ZBTB10%20and%20Three%20Distinct%20HLA%20Associations" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4749491%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="26562150" data-aiid="4749491" data-aid="4749491" data-iid="253966" data-domainid="325" data-domain="molmed" data-accid="PMC4749491" data-md5="e0f21b04c9816416c2e63e4018cd29ed"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T14:16:22-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal103&amp;ncbi_phid=CE8A939DDD57F261000000000B8A06B1&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4749491%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=molmed&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4749491/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal103 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Main, sans-serif;"></div></div></body></html>