<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8">
        <!-- AppResources meta begin -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        <!-- AppResources meta end -->
        
        <!-- TemplateResources meta begin -->
        <meta name="paf_template" content="">

        <!-- TemplateResources meta end -->
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="pmc"><meta name="ncbi_pdid" content="article"><meta name="ncbi_acc" content=""><meta name="ncbi_domain" content="hmg"><meta name="ncbi_report" content="record"><meta name="ncbi_type" content="fulltext"><meta name="ncbi_objectid" content=""><meta name="ncbi_pcid" content="/articles/PMC4380064/"><meta name="ncbi_app" content="pmc">
        <!-- Logger end -->
        
        <title>Evidence for several independent genetic variants affecting lipoprotein (a) cholesterol levels</title>
        
        <!-- AppResources external_resources begin -->
        <link rel="stylesheet" href="/core/jig/1.14.8/css/jig.min.css"><script type="text/javascript" src="/core/jig/1.14.8/js/jig.min.js"></script>

        <!-- AppResources external_resources end -->
        
        <!-- Page meta begin -->
        <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4380064/"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Human Molecular Genetics"><meta name="citation_title" content="Evidence for several independent genetic variants affecting lipoprotein (a) cholesterol levels"><meta name="citation_authors" content="Wensheng Lu, Yu-Ching Cheng, Keping Chen, Hong Wang, Glenn S. Gerhard, Christopher D. Still, Xin Chu, Rongze Yang, Ankita Parihar, Jeffrey R. O'Connell, Toni I. Pollin, Eduardo Angles-Cano, Michael J. Quon, Braxton D. Mitchell, Alan R. Shuldiner, Mao Fu"><meta name="citation_date" content="15 April 2015"><meta name="citation_issue" content="8"><meta name="citation_volume" content="24"><meta name="citation_firstpage" content="2390"><meta name="citation_doi" content="10.1093/hmg/ddu731"><meta name="citation_abstract_html_url" content="/pmc/articles/PMC4380064/?report=abstract"><meta name="citation_pmid" content="25575512"><meta name="DC.Title" content="Evidence for several independent genetic variants affecting lipoprotein (a) cholesterol levels"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Oxford University Press"><meta name="DC.Contributor" content="Wensheng Lu"><meta name="DC.Contributor" content="Yu-Ching Cheng"><meta name="DC.Contributor" content="Keping Chen"><meta name="DC.Contributor" content="Hong Wang"><meta name="DC.Contributor" content="Glenn S. Gerhard"><meta name="DC.Contributor" content="Christopher D. Still"><meta name="DC.Contributor" content="Xin Chu"><meta name="DC.Contributor" content="Rongze Yang"><meta name="DC.Contributor" content="Ankita Parihar"><meta name="DC.Contributor" content="Jeffrey R. O'Connell"><meta name="DC.Contributor" content="Toni I. Pollin"><meta name="DC.Contributor" content="Eduardo Angles-Cano"><meta name="DC.Contributor" content="Michael J. Quon"><meta name="DC.Contributor" content="Braxton D. Mitchell"><meta name="DC.Contributor" content="Alan R. Shuldiner"><meta name="DC.Contributor" content="Mao Fu"><meta name="DC.Date" content="2015 Apr 15"><meta name="DC.Identifier" content="10.1093/hmg/ddu731"><meta name="DC.Language" content="en"><meta property="og:title" content="Evidence for several independent genetic variants affecting lipoprotein (a) cholesterol levels"><meta property="og:type" content="article"><meta property="og:description" content="Lipoprotein (a) [Lp(a)] is an independent risk factor for atherosclerosis-related events that is under strong genetic control (heritability = 0.68–0.98). However, causal mutations and functional validation of biological pathways modulating Lp(a) ..."><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4380064/"><meta property="og:site_name" content="PubMed Central (PMC)"><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-logo-share.png"><meta name="twitter:card" content="summary"><meta name="twitter:site" content="@ncbi"><meta name="ncbi_feature" content="associated_data"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc.min.css" type="text/css"><link rel="stylesheet" href="/corehtml/pmc/css/3.14.1/pmc_extras_prnt.min.css" type="text/css" media="print"><script type="text/javascript" src="/corehtml/pmc/js/common.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'"><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css"><script type="text/javascript" src="/core/mathjax/2.6.1/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-hmg.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style><style type="text/css">
        div.pmc_para_cit li.highlight,
        div.pmc_para_cit li.highlight .one_line_source
        { background: #E0E0E0; }
        a.bibr.highlight { background: #E0E0E0; } 
      </style><meta name="cited_in_systematic_reviews" content=""><link rel="alternate" type="application/epub+zip" href="/pmc/articles/PMC4380064/epub/"><link rel="alternate" type="application/pdf" href="/pmc/articles/PMC4380064/pdf/ddu731.pdf">

        <!-- Page meta end -->
    <link rel="shortcut icon" href="//www.ncbi.nlm.nih.gov/favicon.ico"><meta name="ncbi_phid" content="CE894FC6DD57FF6100000000015400BD.m_8">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3852956/3985586/3808861/4121862/3974050/3917732/251717/4048120/3846471/14534/45193/4113719/3849091/3984811/3751656/4033350/3840896/3577051/3852958/3881636/3579733/4062871/12930/3964959/3855473/4047625/3854974/3854955/4076335/4128070/9685/3549676/3609192/3609193/3609213/3395586/4143404.css"><link type="text/css" rel="stylesheet" href="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/css/3411343/3882866/4157116.css" media="print"><script async="1" src="https://www.google-analytics.com/analytics.js"></script><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style></head>
    <body class="article" id="ui-ncbiexternallink-4"><div id="MathJax_Message" style="display: none;"></div>
        <div class="grid">
            <div class="col twelve_col nomargin shadow">
                <!-- System messages like service outage or JS required; this is handled by the TemplateResources portlet -->
                <div class="sysmessages">
                    <noscript>
	<p class="nojs">
	<strong>Warning:</strong>
	The NCBI web site requires JavaScript to function. 
	<a href="/guide/browsers/#enablejs" title="Learn how to enable JavaScript" target="_blank">more...</a>
	</p>
	</noscript>
                </div>
                <!--/.sysmessage-->
                <div class="wrap">
                    <div class="page">
                        <div class="top">
                            <div class="universal_header" id="universal_header"><ul class="inline_list jig-ncbimenu ui-ncbimenu resources_list ui-ncbibasicmenu orientation_hort" id="navcontent" role="menubar"><li class="ui-ncbimenu-item-leaf ui-ncbimenu-item-first ui-helper-reset ui-ncbimenu-item-no-hlt" role="presentation"><a class="ui-ncbimenu-link-first" href="/" role="menuitem" title="NCBI Home" id="ncbihome" accesskey="1"><span class="offscreen_noflow">NCBI</span><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/28977" class="ncbi_logo" title="NCBI" alt="NCBI Logo"></a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#maincontent" title="Skip to the content" tabindex="0" accesskey="3">Skip to main
                        content</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="#navcontent" title="Skip to the navigation" tabindex="0" accesskey="4">Skip to
                        navigation</a></li><li id="resource-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first ui-helper-reset" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#resources" tabindex="-1" role="menuitem">Resources</a></li><li id="all-howtos-menu" class="topmenu ui-helper-reset ui-ncbimenu-item-first" role="presentation"><a class="ui-ncbimenu-first-link-has-submenu ui-ncbimenu-link-first topanchor expandDown" href="/static/header_footer_ajax/submenu/#howto" tabindex="-1" role="menuitem">How To</a></li><li class="offscreen_noflow ui-ncbimenu-item-skip access"><a href="/guide/browsers/#accesskeys" title="About My NCBI Accesskeys" tabindex="0" accesskey="0">About NCBI Accesskeys</a></li></ul><div class="myncbi" style="top: -0.4em;"><iframe src="https://eutils.ncbi.nlm.nih.gov/entrez/eutils/emyncbi.fcgi?cmd=htmlportal&amp;currenturl=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4380064%2F" style="border:none; height:2.5em; width:40em;" scrolling="no"></iframe></div></div>
                            <div class="header">
    <div class="res_logo">
  <h1 class="img_logo"><a href="/pmc/" class="pmc_logo offscreen">PMC</a></h1>
  <div class="NLMLogo">
    <a href="https://www.nlm.nih.gov/" title="US National Library of Medicine">US National Library of Medicine</a>
    <br>
    <a href="https://www.nih.gov/" title="National Institutes of Health">National Institutes of Health</a>
  </div>
</div>
    <div class="search"><form method="get" action="/pmc/"><div class="search_form"><label for="database" class="offscreen_noflow">Search database</label><select id="database"><optgroup label="Recent"><option value="pmc" selected="selected" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="gene">Gene</option><option value="pubmed" class="last">PubMed</option></optgroup><optgroup label="All"><option value="gquery">All Databases</option><option value="assembly">Assembly</option><option value="biocollections">Biocollections</option><option value="bioproject">BioProject</option><option value="biosample">BioSample</option><option value="biosystems">BioSystems</option><option value="books">Books</option><option value="clinvar">ClinVar</option><option value="cdd">Conserved Domains</option><option value="gap">dbGaP</option><option value="dbvar">dbVar</option><option value="gene">Gene</option><option value="genome">Genome</option><option value="gds">GEO DataSets</option><option value="geoprofiles">GEO Profiles</option><option value="gtr">GTR</option><option value="homologene">HomoloGene</option><option value="ipg">Identical Protein Groups</option><option value="medgen">MedGen</option><option value="mesh">MeSH</option><option value="ncbisearch">NCBI Web Site</option><option value="nlmcatalog">NLM Catalog</option><option value="nuccore">Nucleotide</option><option value="omim">OMIM</option><option value="pmc" data-ac_dict="pmc-search-autocomplete">PMC</option><option value="popset">PopSet</option><option value="probe">Probe</option><option value="protein">Protein</option><option value="proteinclusters">Protein Clusters</option><option value="pcassay">PubChem BioAssay</option><option value="pccompound">PubChem Compound</option><option value="pcsubstance">PubChem Substance</option><option value="pubmed">PubMed</option><option value="snp">SNP</option><option value="sparcle">Sparcle</option><option value="sra">SRA</option><option value="structure">Structure</option><option value="taxonomy">Taxonomy</option><option value="toolkit">ToolKit</option><option value="toolkitall">ToolKitAll</option><option value="toolkitbookgh">ToolKitBookgh</option></optgroup></select><div class="nowrap"><label for="term" class="offscreen_noflow" accesskey="/">Search term</label><div class="nowrap"><div class="jig-ncbiclearbutton-wrap ui-ncbiclearbutton-wrap box-shadow"><input name="term" id="term" title="Search PMC. Use up and down arrows to choose an item from the autocomplete." value="" class="jig-ncbiclearbutton jig-ncbiautocomplete" data-jigconfig="dictionary:'pmc-search-autocomplete',disableUrl:'NcbiSearchBarAutoComplCtrl'" autocomplete="off" data-sbconfig="ds:'no',pjs:'no',afs:'yes'" aria-haspopup="true" aria-autocomplete="list" role="textbox" type="text"><a class="reset" href="#" style="visibility: hidden;"><img src="/core/jig/1.14.8/img/clear.png" alt="Clear input"></a></div></div><button id="search" type="submit" class="button_search nowrap" cmd="go">Search</button></div></div></form><ul class="searchlinks inline_list"><li>
                        <a href="/pmc/advanced/">Advanced</a>
                    </li><li>
                        <a href="/pmc/journals/">Journal list</a>
                    </li><li class="help">
                        <a target="_blank" href="/books/NBK3825/">Help</a>
                    </li></ul></div>
</div>

                            
                            
                        <!--<component id="Page" label="headcontent"/>-->
                            
                        </div>
                        <div class="content">
                            <!-- site messages -->
                            <div class="container">
    <div id="maincontent" class="content eight_col col">
        <div class="navlink-box">
            <ul class="page-breadcrumbs inline_list small"><li class="journal-list"><a href="/pmc/journals/" class="navlink">Journal List</a></li><li class="article-entrez-filter"><a href="/pmc/?term=%22Hum Mol Genet%22[journal]" class="navlink">Hum Mol Genet</a></li><li class="accid">PMC4380064</li></ul>
        </div>

        <!-- Journal banner -->
        <div class="pmc-page-banner whole_rhythm"><div><img src="/corehtml/pmc/pmcgifs/logo-hmg.png" alt="Logo of hmg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://academic.oup.com/hmg" target="pmc_ext" ref="reftype=publisher&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A"></map></div> </div>
        
        <!--component id='MainPortlet' label='search-reference'/-->
        
        <!-- Book content -->
        <div class="">
            
        
            
            <div class="hide-overflow article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden'" id="ui-ncbiinpagenav-1"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-citation half_rhythm no_top_margin clearfix"><div class="inline_block eight_col va_top"><div><div><span class="cit"><span role="menubar"><a href="#" role="menuitem" aria-expanded="false" aria-haspopup="true">Hum Mol Genet</a></span>. 2015 Apr 15; 24(8): 2390–2400. </span></div><div><span class="fm-vol-iss-date">Published online 2015 Jan 9. </span>  <span class="doi">doi:&nbsp;<a href="//dx.doi.org/10.1093%2Fhmg%2Fddu731" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1093/hmg/ddu731</a></span></div></div></div><div class="inline_block four_col va_top show-overflow align_right"><div class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC4380064</span></div><div class="fm-citation-pmid">PMID: <a href="/pubmed/25575512">25575512</a></div></div></div></div><h1 class="content-title">Evidence for several independent genetic variants affecting lipoprotein (a) cholesterol levels</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="/pubmed/?term=Lu%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237471598096" co-class="co-affbox">Wensheng Lu</a>,<sup>1,</sup><sup>2</sup> <a href="/pubmed/?term=Cheng%20YC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237401006656" co-class="co-affbox">Yu-Ching Cheng</a>,<sup>1,</sup><sup>3</sup> <a href="/pubmed/?term=Chen%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237400463440" co-class="co-affbox">Keping Chen</a>,<sup>1</sup> <a href="/pubmed/?term=Wang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237401641120" co-class="co-affbox">Hong Wang</a>,<sup>1</sup> <a href="/pubmed/?term=Gerhard%20GS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237401639136" co-class="co-affbox">Glenn S. Gerhard</a>,<sup>4,</sup><sup>5</sup> <a href="/pubmed/?term=Still%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237399357616" co-class="co-affbox">Christopher D. Still</a>,<sup>4</sup> <a href="/pubmed/?term=Chu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237399355632" co-class="co-affbox">Xin Chu</a>,<sup>4</sup> <a href="/pubmed/?term=Yang%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237392705280" co-class="co-affbox">Rongze Yang</a>,<sup>1</sup> <a href="/pubmed/?term=Parihar%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237392703296" co-class="co-affbox">Ankita Parihar</a>,<sup>1</sup> <a href="/pubmed/?term=O'Connell%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237388837440" co-class="co-affbox">Jeffrey R. O'Connell</a>,<sup>1</sup> <a href="/pubmed/?term=Pollin%20TI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237388835456" co-class="co-affbox">Toni I. Pollin</a>,<sup>1</sup> <a href="/pubmed/?term=Angles-Cano%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237397348032" co-class="co-affbox">Eduardo Angles-Cano</a>,<sup>6,</sup><sup>7</sup> <a href="/pubmed/?term=Quon%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237397345312" co-class="co-affbox">Michael J. Quon</a>,<sup>1</sup> <a href="/pubmed/?term=Mitchell%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237399922688" co-class="co-affbox">Braxton D. Mitchell</a>,<sup>1</sup> <a href="/pubmed/?term=Shuldiner%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237399920704" co-class="co-affbox">Alan R. Shuldiner</a>,<sup>1,</sup><sup>3</sup> and  <a href="/pubmed/?term=Fu%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512" class="affpopup" co-rid="_co_idm140237436748848" co-class="co-affbox">Mao Fu</a><sup>1,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140237471598096"><h3 class="no_margin">Wensheng Lu</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><p><sup>2</sup>Department of Endocrinology, People's Hospital of Guangxi Zhuang Autonomous Region, Nanning, Guangxi 530021, China,</p><div>Find articles by <a href="/pubmed/?term=Lu%20W%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Wensheng Lu</a></div></div><div id="_co_idm140237401006656"><h3 class="no_margin">Yu-Ching Cheng</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><p><sup>3</sup>Veterans Affairs Maryland Health Care System, Baltimore, MD 21201, USA,</p><div>Find articles by <a href="/pubmed/?term=Cheng%20YC%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Yu-Ching Cheng</a></div></div><div id="_co_idm140237400463440"><h3 class="no_margin">Keping Chen</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><div>Find articles by <a href="/pubmed/?term=Chen%20K%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Keping Chen</a></div></div><div id="_co_idm140237401641120"><h3 class="no_margin">Hong Wang</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><div>Find articles by <a href="/pubmed/?term=Wang%20H%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Hong Wang</a></div></div><div id="_co_idm140237401639136"><h3 class="no_margin">Glenn S. Gerhard</h3><p><sup>4</sup>Geisinger Obesity Institute, Geisinger Clinic, Danville, PA 17822, USA,</p><p><sup>5</sup>Penn State Institute for Personalized Medicine, Penn State College of Medicine, Hershey, PA 17033, USA,</p><div>Find articles by <a href="/pubmed/?term=Gerhard%20GS%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Glenn S. Gerhard</a></div></div><div id="_co_idm140237399357616"><h3 class="no_margin">Christopher D. Still</h3><p><sup>4</sup>Geisinger Obesity Institute, Geisinger Clinic, Danville, PA 17822, USA,</p><div>Find articles by <a href="/pubmed/?term=Still%20CD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Christopher D. Still</a></div></div><div id="_co_idm140237399355632"><h3 class="no_margin">Xin Chu</h3><p><sup>4</sup>Geisinger Obesity Institute, Geisinger Clinic, Danville, PA 17822, USA,</p><div>Find articles by <a href="/pubmed/?term=Chu%20X%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Xin Chu</a></div></div><div id="_co_idm140237392705280"><h3 class="no_margin">Rongze Yang</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><div>Find articles by <a href="/pubmed/?term=Yang%20R%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Rongze Yang</a></div></div><div id="_co_idm140237392703296"><h3 class="no_margin">Ankita Parihar</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><div>Find articles by <a href="/pubmed/?term=Parihar%20A%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Ankita Parihar</a></div></div><div id="_co_idm140237388837440"><h3 class="no_margin">Jeffrey R. O'Connell</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><div>Find articles by <a href="/pubmed/?term=O'Connell%20JR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Jeffrey R. O'Connell</a></div></div><div id="_co_idm140237388835456"><h3 class="no_margin">Toni I. Pollin</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><div>Find articles by <a href="/pubmed/?term=Pollin%20TI%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Toni I. Pollin</a></div></div><div id="_co_idm140237397348032"><h3 class="no_margin">Eduardo Angles-Cano</h3><p><sup>6</sup>Inserm U1140, Institut National de la Santé et de la Recherche Médicale, Paris, France,  and</p><p><sup>7</sup>Faculty of Pharmaceutical and Biological Sciences, University Paris Descartes, Paris F-75006, France</p><div>Find articles by <a href="/pubmed/?term=Angles-Cano%20E%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Eduardo Angles-Cano</a></div></div><div id="_co_idm140237397345312"><h3 class="no_margin">Michael J. Quon</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><div>Find articles by <a href="/pubmed/?term=Quon%20MJ%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Michael J. Quon</a></div></div><div id="_co_idm140237399922688"><h3 class="no_margin">Braxton D. Mitchell</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><div>Find articles by <a href="/pubmed/?term=Mitchell%20BD%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Braxton D. Mitchell</a></div></div><div id="_co_idm140237399920704"><h3 class="no_margin">Alan R. Shuldiner</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><p><sup>3</sup>Veterans Affairs Maryland Health Care System, Baltimore, MD 21201, USA,</p><div>Find articles by <a href="/pubmed/?term=Shuldiner%20AR%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Alan R. Shuldiner</a></div></div><div id="_co_idm140237436748848"><h3 class="no_margin">Mao Fu</h3><p><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</p><div>Find articles by <a href="/pubmed/?term=Fu%20M%5BAuthor%5D&amp;cauthor=true&amp;cauthor_uid=25575512">Mao Fu</a></div></div></div></div><div class="fm-panel half_rhythm"><div class="togglers"><a href="#" class="pmctoggle" rid="idm140237402107200_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140237402107200_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140237402107200_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/">Disclaimer</a></div><div class="fm-authors-info fm-panel hide half_rhythm" id="idm140237402107200_ai" style="display:none"><div class="fm-affl" id="af1" lang="en"><sup>1</sup>Division of Endocrinology, Diabetes and Nutrition, University of Maryland School of Medicine, Baltimore, MD 21201, USA,</div><div class="fm-affl" id="af2" lang="en"><sup>2</sup>Department of Endocrinology, People's Hospital of Guangxi Zhuang Autonomous Region, Nanning, Guangxi 530021, China,</div><div class="fm-affl" id="af3" lang="en"><sup>3</sup>Veterans Affairs Maryland Health Care System, Baltimore, MD 21201, USA,</div><div class="fm-affl" id="af4" lang="en"><sup>4</sup>Geisinger Obesity Institute, Geisinger Clinic, Danville, PA 17822, USA,</div><div class="fm-affl" id="af5" lang="en"><sup>5</sup>Penn State Institute for Personalized Medicine, Penn State College of Medicine, Hershey, PA 17033, USA,</div><div class="fm-affl" id="af6" lang="en"><sup>6</sup>Inserm U1140, Institut National de la Santé et de la Recherche Médicale, Paris, France,  and</div><div class="fm-affl" id="af7" lang="en"><sup>7</sup>Faculty of Pharmaceutical and Biological Sciences, University Paris Descartes, Paris F-75006, France</div><div id="cor1"><sup>*</sup>To whom correspondence should be addressed at: University of Maryland School of Medicine, 660 West Redwood Street, HH492, Baltimore, MD 21201, USA. Tel: +1 4107061065;&nbsp;Fax: +1 4107061622; Email: <a href="mailto:dev@null" data-email="ude.dnalyramu.enicidem@ufm" class="oemail">ude.dnalyramu.enicidem@ufm</a></div></div><div class="fm-article-notes fm-panel hide half_rhythm" id="idm140237402107200_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2014 Aug 22; Revised 2014 Dec 5; Accepted 2014 Dec 22.</div></div><div class="permissions fm-panel half_rhythm hide" id="idm140237402107200_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> © The Author 2015. Published by Oxford University Press. All rights reserved. For Permissions, please email: journals.permissions@oup.com</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm"><div class="fm-panel"><div>This article has been <a href="/pmc/articles/PMC4380064/citedby/">cited by</a> other articles in PMC.</div></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><div class="goto jig-ncbiinpagenav-goto-container"><span role="menubar"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="menuitem" aria-expanded="false" aria-haspopup="true">Go to:</a></span></div><h2 class="ui-helper-clearfix" id="ui-ncbiinpagenav-heading-3">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin">
Supplementary Data
</div>

<div><a href="/pmc/articles/PMC4380064/bin/supp_24_8_2390__index.html" data-ga-action="click_feat_suppl">supp_24_8_2390__index.html</a><span style="color:gray"> (1.0K)</span></div><div class="small guid">GUID:&nbsp;F1AF81EF-D0C6-480D-85BA-D2C2A5F84067</div>
<div><a href="/pmc/articles/PMC4380064/bin/supp_ddu731_ddu731supp.docx" data-ga-action="click_feat_suppl">supp_ddu731_ddu731supp.docx</a><span style="color:gray"> (1.2M)</span></div><div class="small guid">GUID:&nbsp;EFA6B973-ED1E-4B0C-961A-275C15A2205F</div>
</div></dd></dl></div><div id="idm140237399124768" class="tsec sec" lang="en"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140237399124768title">Abstract</h2><!--article-meta--><div><p id="__p1" class="p p-first-last">Lipoprotein (a) [Lp(a)] is an independent risk factor for atherosclerosis-related events that is under strong genetic control (heritability = 0.68–0.98). However, causal mutations and functional validation of biological pathways modulating Lp(a) metabolism are lacking. We performed a genome-wide association scan to identify genetic variants associated with Lp(a)-cholesterol levels in the Old Order Amish. We confirmed a previously known locus on chromosome 6q25-26 and found Lp(a) levels also to be significantly associated with a SNP near the <em>APOA5–APOA4–APOC3–APOA1</em> gene cluster on chromosome 11q23 linked in the Amish to the <em>APOC3</em> R19X null mutation. On 6q locus, we detected associations of Lp(a)-cholesterol with 118 common variants (<em>P</em> = 5 × 10<sup>−8</sup> to 3.91 × 10<sup>−19</sup>) spanning a ∼5.3 Mb region that included the <em>LPA</em> gene. To further elucidate variation within <em>LPA</em>, we sequenced <em>LPA</em> and identified two variants most strongly associated with Lp(a)-cholesterol, rs3798220 (<em>P</em> = 1.07 × 10<sup>−14</sup>) and rs10455872 (<em>P</em> = 1.85 × 10<sup>−12</sup>). We also measured copy numbers of kringle IV-2 (KIV-2) in <em>LPA</em> using qPCR. KIV-2 numbers were significantly associated with Lp(a)-cholesterol (<em>P</em> = 2.28 × 10<sup>−9</sup>). Conditional analyses revealed that rs3798220 and rs10455872 were associated with Lp(a)-cholesterol levels independent of each other and KIV-2 copy number. Furthermore, we determined for the first time that levels of <em>LPA</em> mRNA were higher in the carriers than non-carriers of rs10455872 (<em>P</em> = 0.0001) and were not different between carriers and non-carriers of rs3798220. Protein levels of apo(a) were higher in the carriers than non-carriers of both rs10455872 and rs3798220. In summary, we identified multiple independent genetic determinants for Lp(a)-cholesterol. These findings provide new insights into Lp(a) regulation.</p></div></div><div id="s1" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s1title">Introduction</h2><p id="__p2" class="p p-first">Cardiovascular disease (CVD) is a leading cause of human morbidity and mortality throughout the world. Dyslipidemia is an important independent risk factor for CVD (<a href="#DDU731C1" rid="DDU731C1" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662710">1</a>). Recently, Lp(a) has been identified as a causal role in atherosclerotic cardiovascular disease (<a href="#DDU731C2" rid="DDU731C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662691">2</a>–<a href="#DDU731C4" rid="DDU731C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662692">4</a>). Lp(a), which consists of a cholesterol-laden LDL particle and a plasminogen-like glycoprotein [apo(a), encoded by the <em>LPA</em> gene], may contribute to the processes of both atherosclerosis (<a href="#DDU731C5" rid="DDU731C5" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662664">5</a>–<a href="#DDU731C8" rid="DDU731C8" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662707">8</a>) and thrombosis (<a href="#DDU731C9" rid="DDU731C9" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662674">9</a>–<a href="#DDU731C11" rid="DDU731C11" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662686">11</a>). Lp(a) concentrations are under strict genetic control that are minimally influenced by age, weight and diet (<a href="#DDU731C4" rid="DDU731C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662669">4</a>). The heritability of Lp(a) is estimated to range from 0.68 to 0.98 (<a href="#DDU731C12" rid="DDU731C12" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662722">12</a>–<a href="#DDU731C14" rid="DDU731C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662673">14</a>). The pattern of inheritance indicates that a major gene, as well as polygenic factors likely, contribute to plasma Lp(a) concentrations (<a href="#DDU731C15" rid="DDU731C15" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662689">15</a>–<a href="#DDU731C17" rid="DDU731C17" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662705">17</a>). Candidate gene studies indicate that the copy number of kringle IV-2 repeats and single nucleotide polymorphisms (SNPs) in the <em>LPA</em> gene may be determinants of Lp(a) levels (<a href="#DDU731C2" rid="DDU731C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662670">2</a>,<a href="#DDU731C14" rid="DDU731C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662675">14</a>,<a href="#DDU731C18" rid="DDU731C18" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662667">18</a>,<a href="#DDU731C19" rid="DDU731C19" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662714">19</a>). Recently, Lim et al. (<a href="#DDU731C20" rid="DDU731C20" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662663">20</a>) identified two splice variants (loss of function) in the <em>LPA</em> gene that were associated with lower plasma Lp(a) levels and protection from cardiovascular disease. Genome-wide linkage analyses strongly suggests that the <em>LPA</em> locus on chromosome 6q25-27 is a major determinant of Lp(a) levels (LOD score from 9 to 108) and also suggests other new genetic determinants on chromosomes 1, 2, 11, 12, 13, 14, 15 and 19 (<a href="#DDU731C21" rid="DDU731C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662706">21</a>,<a href="#DDU731C22" rid="DDU731C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662702">22</a>). Ober et al. (<a href="#DDU731C23" rid="DDU731C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662676">23</a>) performed a genome-wide association study (GWAS) of plasma Lp(a) levels in 386 Hutterite subjects and identified SNPs in at least six genes on chromosome 6q in addition to <em>LPA</em> that are significantly associated with Lp(a) levels independent of each other and of the number of KIV-2 repeats. However, causal mutations and functional validation of biological pathways modulating Lp(a) metabolism are lacking. A greater understanding of Lp(a) regulation and metabolism may lead to novel targets for pharmacologic lowering of Lp(a) levels for treatment and/or prevention of CVD.</p><p id="__p3" class="p p-last">To better define the genetic architecture of Lp(a), we performed a GWAS for Lp(a)-cholesterol in the Old Order Amish (OOA), a population relatively homogenous in genetic ancestry and lifestyle characteristics. We then sequenced the 40 exons, all intron–exon boundaries and 2 kb of the promoter region of the <em>LPA</em> gene in 24 Amish subjects. To further cover the <em>LPA</em> gene region, we also genotyped five tag SNPs from sequencing results and investigated the previously described KIV-2 repeat polymorphism in <em>LPA</em>. Finally, we investigated whether genetic variants significantly associated with variation in Lp(a)-cholesterol levels were also associated with Lp(a) mRNA and protein levels in human liver.</p></div><div id="s2" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s2title">Results</h2><div id="s2a" class="sec sec-first"><h3 id="s2atitle">Clinical characteristics of OOA subjects</h3><p id="__p4" class="p p-first-last">OOA subjects who were phenotyped for Lp(a)-cholesterol and related traits were recruited in this study. The study population of 1376 OOA represented a group of relatively healthy adults. The mean age was 49.8 years with women slightly older than men (<em>P</em> = 0.025) (Table&nbsp;<a href="/pmc/articles/PMC4380064/table/DDU731TB1/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU731TB1" rid-ob="ob-DDU731TB1" co-legend-rid=""><span>1</span></a>). The mean level of Lp(a)-cholesterol in this sample was 7.13 mg/dl, similar to the white population in the US. The heritability for Lp(a)-cholesterol in the OOA was 0.68 (<em>P</em> &lt; 0.0001). Mean levels of BMI, triglycerides (TG), high density lipoprotein- (HDL) and Lp(a)-cholesterol were significantly higher in women than in men (age adjusted <em>P</em> &lt; 0.01).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU731TB1"><h3>Table&nbsp;1.</h3><!--caption a7--><div class="caption"><p id="__p5">Clinical characteristics of the OOA subjects</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237399062336" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1"></th><th rowspan="1" colspan="1" align="left">Female</th><th rowspan="1" colspan="1" align="left">Male</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">Number</td><td rowspan="1" colspan="1">676</td><td rowspan="1" colspan="1">700</td></tr><tr><td rowspan="1" colspan="1">Age (years)</td><td rowspan="1" colspan="1">50.9 ± 16.6</td><td rowspan="1" colspan="1">48.8 ± 17.4</td></tr><tr><td rowspan="1" colspan="1">BMI (kg/m<sup>2</sup>)</td><td rowspan="1" colspan="1">27.8 ± 5.5</td><td rowspan="1" colspan="1">26.1 ± 3.6</td></tr><tr><td rowspan="1" colspan="1">TG (mg/dl)</td><td rowspan="1" colspan="1">83.6 ± 58.5</td><td rowspan="1" colspan="1">73.8 ± 45.5</td></tr><tr><td rowspan="1" colspan="1">HDL (mg/dl)</td><td rowspan="1" colspan="1">59.6 ± 15.4</td><td rowspan="1" colspan="1">51.8 ± 13.0</td></tr><tr><td rowspan="1" colspan="1">LDL (mg/dl)</td><td rowspan="1" colspan="1">143.1 ± 47.6</td><td rowspan="1" colspan="1">138.5 ± 40.6</td></tr><tr><td rowspan="1" colspan="1">Lp(a) (mg/dl)</td><td rowspan="1" colspan="1">7.7 ± 4.0</td><td rowspan="1" colspan="1">6.6 ± 3.5</td></tr></tbody></table></div><div id="largeobj_idm140237399062336" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4380064/table/DDU731TB1/?report=objectonly">Open in a separate window</a></div></div></div><div id="s2b" class="sec"><h3 id="s2btitle">Genome-wide association analysis for Lp(a)-cholesterol</h3><p id="__p6" class="p p-first">We completed a GWAS of Lp(a)-cholesterol in the OOA using the Affymetrix 500 K SNP array after imputing all autosomal SNPs on HapMap by MACH (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu731/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S1</a>). We identified two loci significantly associated with Lp(a)-cholesterol on chromosome 6q25-q26 and 11q23 (<em>P</em> &lt; 5 × 10<sup>−8</sup>) (Fig.&nbsp;<a href="/pmc/articles/PMC4380064/figure/DDU731F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU731F1" rid-ob="ob-DDU731F1" co-legend-rid="lgnd_DDU731F1"><span>1</span></a>, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu731/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S1</a>). The SNPs most strongly associated with Lp(a)-cholesterol are shown in <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu731/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Table S2</a>. An imputed SNP rs2305695 on chromosome 16 also reached GWAS significance [<em>P</em> = 2.11 × 10<sup>−9</sup>, minor allele frequency (MAF) = 0.005]; however, when genotyped, this SNP was found not to be polymorphic in Amish samples, suggesting an imputation error that may be due to low MAF. Previous observations have shown that imputation error rate increases as the MAF decreases (<a href="#DDU731C24" rid="DDU731C24" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662698">24</a>,<a href="#DDU731C25" rid="DDU731C25" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662685">25</a>).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU731F1" co-legend-rid="lgnd_DDU731F1"><a href="/pmc/articles/PMC4380064/figure/DDU731F1/" target="figure" rid-figpopup="DDU731F1" rid-ob="ob-DDU731F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237397695824" class="figure"><img class="fig-image" alt="An external file that holds a picture, illustration, etc.
Object name is ddu73101.jpg" title="An external file that holds a picture, illustration, etc.
Object name is ddu73101.jpg" src="/pmc/articles/PMC4380064/bin/ddu73101.jpg"></div></a><div id="largeobj_idm140237397695824" class="largeobj-link align_right" style="display: none"><a href="/pmc/articles/PMC4380064/figure/DDU731F1/" target="figure" rid-figpopup="DDU731F1" rid-ob="ob-DDU731F1"></a><a target="object" href="/pmc/articles/PMC4380064/figure/DDU731F1/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU731F1"><div><a class="figpopup" href="/pmc/articles/PMC4380064/figure/DDU731F1/" target="figure" rid-figpopup="DDU731F1" rid-ob="ob-DDU731F1">Figure&nbsp;1.</a></div><!--caption a7--><div class="caption"><p id="__p7">Association results of Lp(a)-cholesterol across all autosomal chromosomes. SNPs are plotted on the <em>x</em>-axis according to their position on each chromosome against association with Lp(a)-cholesterol on the <em>t</em>-axis (shown as –log<sub>10</sub>
<em>P</em>-value).</p></div></div></div><p id="__p8">We identified an imputed SNP near the <em>APOA5–APOA4–APOC3–APOA1</em> gene cluster on chromosome 11 to be significantly associated with increased Lp(a)-cholesterol levels (<em>P</em> = 8 × 10<sup>−8</sup>) (Fig.&nbsp;<a href="/pmc/articles/PMC4380064/figure/DDU731F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU731F2" rid-ob="ob-DDU731F2" co-legend-rid="lgnd_DDU731F2"><span>2</span></a>). This SNP, rs12787909, is located in intron 3 of <em>DSCAML1</em> and has a MAF of 0.045 in the Amish and a MAF of 0.07 in HapMap-CEU population, but is not polymorphic in HapMap-HCB, JPT or YRI samples. SNP rs12787909 was also associated with decreased TG (<em>P</em> = 2 × 10<sup>−7</sup>), increased HDL-C (<em>P</em> = 4 × 10<sup>−9</sup>) and decreased LDL-C (<em>P</em> = 0.0001) in the OOA. Since the OOA, due to a founder effect, exhibit a high frequency of the rare null R19X mutation in nearby <em>APOC3</em> (rs76353203, <em>r</em><sup>2</sup> in Amish with rs12787909 = 0.75) that is also associated with increased Lp(a), decreased TG, increased HDL-C and decreased LDL-C, a conditional analysis was performed. The association of <em>APOC3</em> rs76353203 not only explained the signal at rs12787909, but was also more significant (Table&nbsp;<a href="/pmc/articles/PMC4380064/table/DDU731TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU731TB2" rid-ob="ob-DDU731TB2" co-legend-rid=""><span>2</span></a>) thus, indicating rs12787909 is a within-Amish association (i.e. linkage) rather than a population association.

</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU731TB2"><h3>Table&nbsp;2.</h3><!--caption a7--><div class="caption"><p id="__p9">Associations of the <em>DSCAML1</em> SNPs andAPOC3 null variant with Lp(a) and Lipid Traits</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237388638000" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Trait</th><th rowspan="2" colspan="1" align="left"><em>N</em></th><th colspan="2" rowspan="1" align="center">APOC3 R19X<hr></th><th colspan="2" rowspan="1" align="center">DSCAML1 irs12787909 (imputation quality = 0.38)</th><th rowspan="1" colspan="1" align="left">DSCAML1 irs12787909 adjusted for APOC3 R19X</th><th rowspan="1" colspan="1" align="left">DSCAML1 irs12787909 adjusted for lnTG</th></tr><tr><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">LnLp(a)</td><td rowspan="1" colspan="1">1098<sup>a</sup></td><td rowspan="1" colspan="1">0.419</td><td rowspan="1" colspan="1">3 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.459</td><td rowspan="1" colspan="1">8 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.28</td><td rowspan="1" colspan="1">0.014</td></tr><tr><td rowspan="1" colspan="1">Ln[Lp(a)+1]</td><td rowspan="1" colspan="1">1113</td><td rowspan="1" colspan="1">0.396</td><td rowspan="1" colspan="1">4 × 10<sup>−8</sup></td><td rowspan="1" colspan="1">0.430</td><td rowspan="1" colspan="1">2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.32</td><td rowspan="1" colspan="1">0.037</td></tr><tr><td rowspan="1" colspan="1">TG</td><td rowspan="1" colspan="1">1113</td><td rowspan="1" colspan="1">−43.340</td><td rowspan="1" colspan="1">3 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">−40.681</td><td rowspan="1" colspan="1">2 × 10<sup>−7</sup></td><td rowspan="1" colspan="1">0.62</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">lnTG</td><td rowspan="1" colspan="1">1113</td><td rowspan="1" colspan="1">−0.637</td><td rowspan="1" colspan="1">4 × 10<sup>−18</sup></td><td rowspan="1" colspan="1">−0.645</td><td rowspan="1" colspan="1">1 × 10<sup>−14</sup></td><td rowspan="1" colspan="1">0.66</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">HDL-C</td><td rowspan="1" colspan="1">1113</td><td rowspan="1" colspan="1">13.545</td><td rowspan="1" colspan="1">6 × 10<sup>−10</sup></td><td rowspan="1" colspan="1">14.607</td><td rowspan="1" colspan="1">4 × 10<sup>−9</sup></td><td rowspan="1" colspan="1">0.29</td><td rowspan="1" colspan="1"></td></tr><tr><td rowspan="1" colspan="1">LDL-C</td><td rowspan="1" colspan="1">1113</td><td rowspan="1" colspan="1">−22.391</td><td rowspan="1" colspan="1">0.0004</td><td rowspan="1" colspan="1">−27.571</td><td rowspan="1" colspan="1">0.0001</td><td rowspan="1" colspan="1">0.13</td><td rowspan="1" colspan="1"></td></tr></tbody></table></div><div id="largeobj_idm140237388638000" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4380064/table/DDU731TB2/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN1"><p id="__p10" class="p p-first-last"><sup>a</sup>Fifteen subjects had Lp(a) = 0.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU731F2" co-legend-rid="lgnd_DDU731F2"><a href="/pmc/articles/PMC4380064/figure/DDU731F2/" target="figure" rid-figpopup="DDU731F2" rid-ob="ob-DDU731F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237401280192" class="figure"><a href="/pmc/articles/PMC4380064/figure/DDU731F2/" target="figure" rid-figpopup="DDU731F2" rid-ob="ob-DDU731F2"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4380064_ddu73102.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu73102.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4380064/bin/ddu73102.jpg"></a></div><div id="largeobj_idm140237401280192" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4380064/figure/DDU731F2/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU731F2"><div><a class="figpopup" href="/pmc/articles/PMC4380064/figure/DDU731F2/" target="figure" rid-figpopup="DDU731F2" rid-ob="ob-DDU731F2">Figure&nbsp;2.</a></div><!--caption a7--><div class="caption"><p id="__p11">Association of SNPs on chromosome 11 with Lp(a)-cholesterol levels. SNPs are plotted by position on chromosome 11q23 against association with Lp(a)-cholesterol (–log<sub>10</sub>
<em>P</em>-value). The SNP name shown on the plot was the most significant SNP. Estimated recombination rates (from HapMap) are plotted in cyan to reflect the local LD structure. The SNPs surrounding the most significant SNP are color-coded to reflect their LD with this SNP (taken from pairwise <em>r</em><sup>2</sup> values from the HapMap-CEU database, <a href="http://www.hapmap.org">www.hapmap.org</a>). Genes, position of exons, and direction of transcription from UCSC genome browser (<a href="http://genome.ucsc.edu">http://genome.ucsc.edu</a>) are noted. Regional plots were generated using LocusZoom (<a href="http://csg.sph.umich.edu/locuszoom">http://csg.sph.umich.edu/locuszoom</a>).</p></div></div></div><p id="__p12" class="p p-last">We identified 118 common variants significantly associated with Lp(a)-cholesterol levels (<em>P</em> = 5 × 10<sup>−8</sup> to 3.91 × 10<sup>−19</sup>) that spanned ∼5.3 Mb on chromosome 6q25-26 in a region encompassing 26 genes, including <em>LPA</em>. Common variants within six genes (<em>SNX9</em>, <em>SERAC3, TAGAP</em>, <em>FNDC1</em>, <em>PLG</em> and <em>AGPAD4</em>) were significantly associated with Lp(a)-cholesterol levels (Fig.&nbsp;<a href="/pmc/articles/PMC4380064/figure/DDU731F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU731F3" rid-ob="ob-DDU731F3" co-legend-rid="lgnd_DDU731F3"><span>3</span></a>A).
</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU731F3" co-legend-rid="lgnd_DDU731F3"><a href="/pmc/articles/PMC4380064/figure/DDU731F3/" target="figure" rid-figpopup="DDU731F3" rid-ob="ob-DDU731F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237399111200" class="figure"><a href="/pmc/articles/PMC4380064/figure/DDU731F3/" target="figure" rid-figpopup="DDU731F3" rid-ob="ob-DDU731F3"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4380064_ddu73103.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu73103.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4380064/bin/ddu73103.jpg"></a></div><div id="largeobj_idm140237399111200" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4380064/figure/DDU731F3/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU731F3"><div><a class="figpopup" href="/pmc/articles/PMC4380064/figure/DDU731F3/" target="figure" rid-figpopup="DDU731F3" rid-ob="ob-DDU731F3">Figure&nbsp;3.</a></div><!--caption a7--><div class="caption"><p id="__p13">Association of SNPs on chromosome 6 with Lp(a)-cholesterol levels. SNPs are plotted by position on chromosome 6q25-26 against association with Lp(a)-cholesterol (–log<sub>10</sub>
<em>P</em>-value). (<strong>A</strong>) Highlights the most significant SNP in association analysis adjusted age, age2 and sex; (<strong>B</strong>) The most significant SNP after conditional analysis where included age, age<sup>2</sup> sex and KIV-2 numbers as a covariate; (<strong>C</strong>) The most significant SNP after conditional analysis where included age, age<sup>2</sup> sex, KIV-2 numbers, rs3798220 and rs10455872 as a covariate. (Arrow marked LPA location.)</p></div></div></div></div><div id="s2c" class="sec"><h3 id="s2ctitle">Rare variants in <em>LPA</em> are associated with Lp(a)-cholesterol levels</h3><p id="__p14" class="p p-first-last">Variants in <em>LPA</em> have previously been associated with Lp(a) levels (<a href="#DDU731C14" rid="DDU731C14" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662693">14</a>). The common variants located in <em>LPA</em> gene in our GWAS were only modestly associated with Lp(a)-cholesterol levels with the lowest <em>P</em>-value = 9.70 × 10<sup>−5</sup> (for imputed SNP rs7770685). To further evaluate DNA variation within <em>LPA</em>, we sequenced the 40 exons, all intron–exon boundaries and 2 kb of the <em>LPA</em> promoter region in 24 Amish subjects who were in the extreme high or low tail of the Lp(a)-cholesterol distribution, or who had different genotypes than the most significant association SNP with Lp(a)-cholesterol in GWAS. We identified 23 variants, including 6 missense variants falling within 5 exons: exon 26 (rs7765803 L1358V, rs7765781 L1372V), exon 32 (rs1801693 M1679T), exon 37(rs3798220 I1891M), exon 39 (rs41267809 L1961P) and exon 40 (rs3124784 R2016C) (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu731/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S2</a> and <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu731/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Table S3</a>). Selected based on the LD structure, we genotyped five SNPs in the <em>LPA</em> gene region that had not been well covered by our GWAS. Two of these variants (rs3798220 I1891M in exon 37 and rs10455872 A&gt;G in intron 25) were significantly associated with Lp(a)-cholesterol levels (<em>P</em> values: 1.07 × 10<sup>−14</sup> and 1.85 × 10<sup>−12</sup>, respectively) (Table&nbsp;<a href="/pmc/articles/PMC4380064/table/DDU731TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU731TB3" rid-ob="ob-DDU731TB3" co-legend-rid=""><span>3</span></a>). Neither of these SNPs was significantly associated with LDL-C, HDL-C or triglycerides (Table&nbsp;<a href="/pmc/articles/PMC4380064/table/DDU731TB3/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU731TB3" rid-ob="ob-DDU731TB3" co-legend-rid=""><span>3</span></a>).
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU731TB3"><h3>Table&nbsp;3.</h3><!--caption a7--><div class="caption"><p id="__p15">The association of top variants with lipid traits</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237392778576" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="2" colspan="1" align="left">Variants</th><th rowspan="2" colspan="1" align="left">CHR<sup>a</sup></th><th rowspan="2" colspan="1" align="left">Gene</th><th rowspan="2" colspan="1" align="left">Location</th><th rowspan="2" colspan="1" align="left">MAF</th><th colspan="2" rowspan="1" align="center">LnLP(a)<hr></th><th colspan="2" rowspan="1" align="center">TG<hr></th><th colspan="2" rowspan="1" align="center">HDL-C<hr></th><th colspan="2" rowspan="1" align="center">LDL-C<hr></th></tr><tr><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs76353203</td><td rowspan="1" colspan="1">11</td><td rowspan="1" colspan="1"><em>APOC3</em></td><td rowspan="1" colspan="1">Exon2</td><td rowspan="1" colspan="1">0.045</td><td rowspan="1" colspan="1">0.419</td><td rowspan="1" colspan="1">3.00E−08</td><td rowspan="1" colspan="1">−43.340</td><td rowspan="1" colspan="1">3.00E−10</td><td rowspan="1" colspan="1">13.545</td><td rowspan="1" colspan="1">6.00E−10</td><td rowspan="1" colspan="1">−22.391</td><td rowspan="1" colspan="1">4.00E−03</td></tr><tr><td rowspan="1" colspan="1">rs3798220</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1"><em>LPA</em></td><td rowspan="1" colspan="1">Exon 37</td><td rowspan="1" colspan="1">0.009</td><td rowspan="1" colspan="1">0.843</td><td rowspan="1" colspan="1">1.07E−14</td><td rowspan="1" colspan="1">−10.194</td><td rowspan="1" colspan="1">2.94E−01</td><td rowspan="1" colspan="1">−2.233</td><td rowspan="1" colspan="1">4.60E−01</td><td rowspan="1" colspan="1">9.887</td><td rowspan="1" colspan="1">2.64E−01</td></tr><tr><td rowspan="1" colspan="1">rs10455872</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1"><em>LPA</em></td><td rowspan="1" colspan="1">intron25</td><td rowspan="1" colspan="1">0.022</td><td rowspan="1" colspan="1">0.556</td><td rowspan="1" colspan="1">1.85E−12</td><td rowspan="1" colspan="1">−7.630</td><td rowspan="1" colspan="1">2.69E−01</td><td rowspan="1" colspan="1">2.501</td><td rowspan="1" colspan="1">2.47E−01</td><td rowspan="1" colspan="1">−4.600</td><td rowspan="1" colspan="1">4.71E−01</td></tr><tr><td rowspan="1" colspan="1">KIV-2</td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1"><em>LPA</em></td><td rowspan="1" colspan="1">Exons</td><td rowspan="1" colspan="1">NA<sup>a</sup></td><td rowspan="1" colspan="1">−0.168</td><td rowspan="1" colspan="1">2.28E−09</td><td rowspan="1" colspan="1">−1.378</td><td rowspan="1" colspan="1">5.75E−01</td><td rowspan="1" colspan="1">−0.575</td><td rowspan="1" colspan="1">4.54E−01</td><td rowspan="1" colspan="1">−2.553</td><td rowspan="1" colspan="1">2.45E−01</td></tr></tbody></table></div><div id="largeobj_idm140237392778576" class="largeobj-link align_right" style=""><a target="object" href="/pmc/articles/PMC4380064/table/DDU731TB3/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN2"><p id="__p16" class="p p-first-last"><sup>a</sup>CHR, chromosome; NA, no data available.</p></div></div></div></div><div id="s2d" class="sec"><h3 id="s2dtitle">Determination of kringle IV-2 number in the <em>LPA</em> gene</h3><p id="__p17" class="p p-first-last">KIV-2 copy number variation in <em>LPA</em> is strongly associated with plasma Lp(a) levels and risk of CVD (<a href="#DDU731C26" rid="DDU731C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662703">26</a>,<a href="#DDU731C27" rid="DDU731C27" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662666">27</a>). Therefore, we measured KIV-2 copy number using quantitative real-time polymerase chain reaction (qPCR) (<a href="#DDU731C28" rid="DDU731C28" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662721">28</a>). Real-time PCR signals of the two probe sets used for qPCR targeting exons four and five, respectively, were strongly correlated (<em>r</em><sup>2</sup> = 0.76, <em>P</em> = 2.98 × 10<sup>−259</sup>, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu731/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S3</a>). Numbers of KIV-2 were inversely correlated with plasma Lp(a)-cholesterol levels (<em>r</em><sup>2</sup> = 0.06, <em>P</em> = 9.44 × 10<sup>−17</sup>, <a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu731/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S4</a>). The KIV-2 copy numbers were significantly associated with Lp(a)-cholesterol levels (<em>P</em> = 2.28 × 10<sup>−9</sup>), although not with LDL-C, HDL-C or triglycerides (Table&nbsp;<a href="/pmc/articles/PMC4380064/table/DDU731TB2/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU731TB2" rid-ob="ob-DDU731TB2" co-legend-rid=""><span>2</span></a>). The rare alleles of both the rs10455872 and rs3798220 SNPs were each correlated with lower KIV-2 copy numbers (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu731/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S5</a>).</p></div><div id="s2e" class="sec"><h3 id="s2etitle">Conditional analysis</h3><p id="__p18" class="p p-first-last">We performed linkage disequilibrium (LD) analysis for chromosome 6q25-26 SNPs associated with Lp(a)-cholesterol using Haploview. The significantly associated SNPs fell into different blocks suggesting that multiple variants/genes in this region may regulate Lp(a)-cholesterol levels (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu731/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S6</a>). To further assess the number of independent associations on chromosome 6q25-26, we performed conditional association analysis using the most significantly associated SNPs as covariates. We first estimated independence between rs3798220 and rs10455872 in the <em>LPA</em> gene. We performed association analysis of one SNP with Lp(a)-cholesterol including the other SNP as a covariate. In this analysis, the effect sizes (i.e. betas) and statistical evidence for association (i.e. <em>P</em>-values) for each SNP were only minimally changed, suggesting that these two SNPs influenced variation in Lp(a)-cholesterol levels independently from each other (Table&nbsp;<a href="/pmc/articles/PMC4380064/table/DDU731TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU731TB4" rid-ob="ob-DDU731TB4" co-legend-rid=""><span>4</span></a>). Next, we assessed independence of KIV-2 numbers and these two SNPs (rs3798220, rs10455872). We tested the association of one variant with Lp(a)-cholesterol levels while adjusting for the other two variants simultaneously. These results indicated that both SNPs remained significantly associated with Lp(a)-cholesterol levels independently of KIV-2 numbers. KIV-2 numbers also remained significantly associated with Lp(a)-cholesterol levels independently of both SNPs, although the magnitude of the association was reduced due to partial correlation of KIV-2 numbers with the two SNPs (Table&nbsp;<a href="/pmc/articles/PMC4380064/table/DDU731TB4/" target="table" class="fig-table-link figpopup" rid-figpopup="DDU731TB4" rid-ob="ob-DDU731TB4" co-legend-rid=""><span>4</span></a>). We also reciprocally examined the association of SNPs on chromosome 6q25-q26 by adjusting for KIV-2 numbers. These SNPs remained highly associated with Lp(a)-cholesterol levels with only a modest reduction in strength of association (the best <em>P</em> value from 3.91 × 10<sup>−19</sup> to 1.49 × 10<sup>−17</sup>) (Fig.&nbsp;<a href="/pmc/articles/PMC4380064/figure/DDU731F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU731F3" rid-ob="ob-DDU731F3" co-legend-rid="lgnd_DDU731F3"><span>3</span></a>B). Finally, we performed conditional analysis of SNPs on chromosome 6q25-26 adjusted for all three <em>LPA</em> variants (KIV-2 CNV, rs3798220 and rs10455872). The magnitude of the association of SNPs on 6q25-q26 with Lp(a)-cholesterol levels was largely reduced (the best <em>P</em> = 3.01 × 10<sup>−6</sup>) (Fig.&nbsp;<a href="/pmc/articles/PMC4380064/figure/DDU731F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU731F3" rid-ob="ob-DDU731F3" co-legend-rid="lgnd_DDU731F3"><span>3</span></a>C). These three variants in <em>LPA</em> gene explained 12.0% of the variation in Lp(a)-cholesterol levels in the Old Order Amish.
</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap table anchored whole_rhythm" id="DDU731TB4"><h3>Table&nbsp;4.</h3><!--caption a7--><div class="caption"><p id="__p19">Conditional analyses adjusted by age, age<sup>2</sup>, sex and variants</p></div><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237432950912" class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1" align="left">Variants</th><th rowspan="1" colspan="1" align="left">Chromosome</th><th rowspan="1" colspan="1" align="left">Gene</th><th rowspan="1" colspan="1" align="left">Location</th><th rowspan="1" colspan="1" align="left">Beta</th><th rowspan="1" colspan="1" align="left">SE</th><th rowspan="1" colspan="1" align="left"><em>P</em>-value</th></tr></thead><tbody><tr><td rowspan="1" colspan="1">rs3798220<sup>a</sup></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">LPA</td><td rowspan="1" colspan="1">Exon37</td><td rowspan="1" colspan="1">8.59E−01</td><td rowspan="1" colspan="1">1.07E−01</td><td rowspan="1" colspan="1">2.02E−15</td></tr><tr><td rowspan="1" colspan="1">rs10455872<sup>a</sup></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">LPA</td><td rowspan="1" colspan="1">intron25</td><td rowspan="1" colspan="1">5.77E−01</td><td rowspan="1" colspan="1">7.65E−02</td><td rowspan="1" colspan="1">9.67E−14</td></tr><tr><td rowspan="1" colspan="1">rs3798220<sup>b</sup></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">LPA</td><td rowspan="1" colspan="1">Exon37</td><td rowspan="1" colspan="1">8.23E−01</td><td rowspan="1" colspan="1">1.07E−01</td><td rowspan="1" colspan="1">3.62E−14</td></tr><tr><td rowspan="1" colspan="1">rs10455872<sup>b</sup></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">LPA</td><td rowspan="1" colspan="1">intron25</td><td rowspan="1" colspan="1">5.10E−01</td><td rowspan="1" colspan="1">7.85E−02</td><td rowspan="1" colspan="1">1.19E−10</td></tr><tr><td rowspan="1" colspan="1">KIV-2<sup>b</sup></td><td rowspan="1" colspan="1">6</td><td rowspan="1" colspan="1">LPA</td><td rowspan="1" colspan="1">Exons</td><td rowspan="1" colspan="1">−1.35E−01</td><td rowspan="1" colspan="1">2.78E−02</td><td rowspan="1" colspan="1">1.31E−06</td></tr></tbody></table></div><div id="largeobj_idm140237432950912" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4380064/table/DDU731TB4/?report=objectonly">Open in a separate window</a></div><div class="tblwrap-foot"><div id="TN3"><p id="__p20" class="p p-first-last"><sup>a</sup>The effect of rs3798220 is adjusted for the effect of rs10455872; the effect of rs10455872 is adjusted for the effect of 3798220.</p></div><div id="TN4"><p id="__p21" class="p p-first-last"><sup>b</sup>The effect of rs3798220 is adjusted for KIV-2 and rs10455872; the effect of rs10455872 is adjusted for rs3798220 and KIV-2; the effect of KIV-2 is adjusted for the effect of rs3798220 and rs10455872.</p></div></div></div></div><div id="s2f" class="sec sec-last"><h3 id="s2ftitle">Regulation of <em>LPA</em> gene expression</h3><p id="__p22" class="p p-first-last">To gain more insights into the associations between <em>LPA</em> SNPs rs3798220 (I1891M) or rs10455872 (A&gt;G in intron 25) and Lp(a)-cholesterol levels, we tested whether these were also associated with <em>LPA</em> gene expression. Since apo(a) is synthesized predominantly in the liver, we obtained liver samples from 78 white subjects undergoing bariatric weight loss procedures for extreme obesity or related comorbid medical conditions (<a href="#DDU731C29" rid="DDU731C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662657">29</a>,<a href="#DDU731C30" rid="DDU731C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662661">30</a>). We selected samples according to patient's genotype at rs3798220 (<em>N</em> = 20 subjects each with the CT and TT genotypes) and rs10455872 (<em>N</em> = 3, 20 and 20 subjects with the GG, AG and AA genotypes, respectively), matching patients across genotypes by age and gender. We measured <em>LPA</em> gene expression by quantitative PCR (qPCR) and found that <em>LPA</em> mRNA levels were significantly higher in carriers (GG and AG genotypes) than non-carriers (AA genotype) of rs10455872 (<em>P</em> = 0.0001, Fig.&nbsp;<a href="/pmc/articles/PMC4380064/figure/DDU731F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU731F4" rid-ob="ob-DDU731F4" co-legend-rid="lgnd_DDU731F4"><span>4</span></a>A)). In contrast, there was no difference in <em>LPA</em> mRNA levels when rs3798220 carriers and non-carriers were compared (<em>P</em> &gt; 0.05, Fig.&nbsp;<a href="/pmc/articles/PMC4380064/figure/DDU731F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU731F4" rid-ob="ob-DDU731F4" co-legend-rid="lgnd_DDU731F4"><span>4</span></a>B). We measure protein levels of Apo(a) in these liver samples (rs3798220. <em>N</em> = 18 subjects each for the CT and TT genotypes; rs10455872, <em>N</em> = 12 subjects each for the AG and AA genotypes). Apo(a) protein levels are higher in carriers than non-carriers of either rs10455872 and rs3798220 (<em>P</em> = 0.004, Fig.&nbsp;<a href="/pmc/articles/PMC4380064/figure/DDU731F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU731F5" rid-ob="ob-DDU731F5" co-legend-rid="lgnd_DDU731F5"><span>5</span></a>A and <em>P</em> = 0.048, Fig.&nbsp;<a href="/pmc/articles/PMC4380064/figure/DDU731F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="DDU731F5" rid-ob="ob-DDU731F5" co-legend-rid="lgnd_DDU731F5"><span>5</span></a>B respectively). These findings suggest that rs10455872 within <em>LPA</em> may play a key role in influencing <em>LPA</em> transcription levels or mRNA stability, while rs3798220 may influence Lp(a) levels through other mechanisms including effects on translation or post-translation.

</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU731F4" co-legend-rid="lgnd_DDU731F4"><a href="/pmc/articles/PMC4380064/figure/DDU731F4/" target="figure" rid-figpopup="DDU731F4" rid-ob="ob-DDU731F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237398288096" class="figure"><a href="/pmc/articles/PMC4380064/figure/DDU731F4/" target="figure" rid-figpopup="DDU731F4" rid-ob="ob-DDU731F4"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4380064_ddu73104.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu73104.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4380064/bin/ddu73104.jpg"></a></div><div id="largeobj_idm140237398288096" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4380064/figure/DDU731F4/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU731F4"><div><a class="figpopup" href="/pmc/articles/PMC4380064/figure/DDU731F4/" target="figure" rid-figpopup="DDU731F4" rid-ob="ob-DDU731F4">Figure&nbsp;4.</a></div><!--caption a7--><div class="caption"><p id="__p23">mRNA levels for <em>LPA</em> were assessed by qPCR in liver samples with different genotypes. (<strong>A</strong>) <em>LPA</em> expression was increased in the A&gt;G variant of rs10455872; (<strong>B</strong>) The T&gt;C variant of rs3798220 did not affect <em>LPA</em> mRNA expression. *<em>P</em> = 0.70054; **<em>P</em> = 0.000182; ***<em>P</em> = 0.000111.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm clearfix" id="DDU731F5" co-legend-rid="lgnd_DDU731F5"><a href="/pmc/articles/PMC4380064/figure/DDU731F5/" target="figure" rid-figpopup="DDU731F5" rid-ob="ob-DDU731F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--></a><div data-largeobj="" data-largeobj-link-rid="largeobj_idm140237386426272" class="figure"><a href="/pmc/articles/PMC4380064/figure/DDU731F5/" target="figure" rid-figpopup="DDU731F5" rid-ob="ob-DDU731F5"></a><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=4380064_ddu73105.jpg" target="tileshopwindow"><div class="ts_bar small" title="Click on image to zoom"></div><img alt="An external file that holds a picture, illustration, etc.
Object name is ddu73105.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC4380064/bin/ddu73105.jpg"></a></div><div id="largeobj_idm140237386426272" class="largeobj-link align_right" style="display: none"><a target="object" href="/pmc/articles/PMC4380064/figure/DDU731F5/?report=objectonly">Open in a separate window</a></div><div class="icnblk_cntnt" id="lgnd_DDU731F5"><div><a class="figpopup" href="/pmc/articles/PMC4380064/figure/DDU731F5/" target="figure" rid-figpopup="DDU731F5" rid-ob="ob-DDU731F5">Figure&nbsp;5.</a></div><!--caption a7--><div class="caption"><p id="__p24">Western blot analysis of Lp(a) expression in liver samples with different genotypes. Representative western blots were shown for Lp(a) expression in rs10455872 (<strong>A</strong>; <em>N</em> = 12: 12 for AG: AA genotypes) and in rs3798220 (<strong>B</strong>; <em>N</em> = 18: 18 for CT: TT genotypes). Densitometric analysis of Lp(a) versus vinculin was performed for each genotype. A mixture of recombinant human Lp(a) isoforms (A10, A14, A22 and A34; provided by Dr Eduardo Angles-Cano) and HiMark™ Pre-stained Protein Standard (Life Technologies, Grand Island, NY) were applied in western blot. Data are presented as means ± SEM. A.U., arbitrary units. *<em>P</em> = 0.048; **<em>P</em> = 0.004.</p></div></div></div></div></div><div id="s3" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s3title">Discussion</h2><p id="__p25" class="p p-first">Recent evidence has convincingly shown a causal role for Lp(a) in atherosclerotic cardiovascular disease (<a href="#DDU731C2" rid="DDU731C2" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662678">2</a>–<a href="#DDU731C4" rid="DDU731C4" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662671">4</a>). Lp(a) levels in human plasma are under primarily genetic rather than environmental control. Here we present a systematic search for genetic variants with Lp(a)-cholesterol levels and find two significant association signals located on chromosome 6q25-q26 and chromosome 11q23. We further identify two SNPs (rs3798220, rs10455872) and KIV-2 in the <em>LPA</em> gene that are strongly associated with Lp(a)-cholesterol levels. Conditional analyses indicate that these variants act largely independently of one another to influence Lp(a)-cholesterol levels, although the mechanisms by which they do so are different, e.g. intronic variant rs10455872 is associated with reduced <em>LPA</em> mRNA levels while exonic SNP rs3798220 acts through other mechanisms.</p><p id="__p26">We conducted a GWAS in the OOA, a culturally and genetically homogenous population originating from a small founder population. We identified a SNP on chromosome 11q23 that was significantly associated with increased Lp(a)-cholesterol and also with decreased TG, increased HDL-C and decreased LDL-C levels in the OOA. This association is mostly likely attributable to the <em>APOC3</em> R19X variant, which exists at high frequency in the Amish due to a founder effect and is linked in the Amish to (but not in LD in the outbred population with) rs12787909. The <em>APOC3</em> R19X is a null allele that reduces apoC-III production in heterozygotes by 50% and is associated with reduced triglycerides and increased HDL-C, LDL-C and cardioprotection as measured by less coronary calcification (<a href="#DDU731C31" rid="DDU731C31" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662711">31</a>) and a lower rate of myocardial infarction (<a href="#DDU731C32" rid="DDU731C32" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662680">32</a>,<a href="#DDU731C33" rid="DDU731C33" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662688">33</a>). While it may seem counterintuitive at first glance that an otherwise cardioprotective variant would be associated with increased Lp(a) levels, a strong inverse correlation between Lp(a) levels and TG levels has been observed by us (<em>r</em> = −0.38, <em>P</em> = 6 × 10<sup>−27</sup>) and others (<em>r</em> = −0.22, <em>P</em> = 0.001) (<a href="#DDU731C34" rid="DDU731C34" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662684">34</a>). Since the <em>APOC3</em> variant is associated with TG and Lp(a) levels but the <em>LPA</em> variants are associated with Lp(a) but not TG levels, this finding provides strong evidence that the Lp(a)/TG correlation is driven by a causal effect of TG on Lp(a). That Lp(a) can be both a risk factor for CVD at very high levels and correlated with cardioprotective factors speaks to the limited understanding of the underlying biology of Lp(a) that our research is designed to address.</p><p id="__p27">We confirmed a strong association of Lp(a)-cholesterol levels with the <em>LPA</em> locus on chromosome 6q25-26. Previous genome-wide linkage analyses have identified linkage of this region with Lp(a) levels with robust LOD scores ranging from 9 to 108 (<a href="#DDU731C21" rid="DDU731C21" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662720">21</a>,<a href="#DDU731C22" rid="DDU731C22" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662699">22</a>). The significant variants span ∼5.3 Mb region on chromosome 6q25-26 region that are within or flanking 26 genes including <em>LPA</em>. The <em>LPA</em> gene is an obvious and important candidate gene for influencing Lp(a) levels (<a href="#DDU731C26" rid="DDU731C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662662">26</a>). Although the initial GWAS identified variants within <em>LPA</em> that were modestly associated with Lp(a)-cholesterol levels, sequence analysis identified two rare SNPs (rs3798220, rs10455872) in <em>LPA</em> that were strongly associated with Lp(a)-cholesterol levels. These SNPs (rs3798220, rs10455872) have been associated with Lp(a) levels and with cardiovascular diseases in previous studies used the HumanCVD BeadChip or direct genotyping (<a href="#DDU731C3" rid="DDU731C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662659">3</a>,<a href="#DDU731C35" rid="DDU731C35" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662668">35</a>,<a href="#DDU731C36" rid="DDU731C36" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662694">36</a>). These rare SNPs were not tagged well by SNPs on the GWAS panel nor were they tagged by imputed SNPs. Systematically capturing the underlying causal variants within GWAS signals is an important approach for moving beyond GWAS.</p><p id="__p28">The copy number variation of kringle IV-2 in <em>LPA</em> is highly polymorphic and a predictor of Lp(a) levels (<a href="#DDU731C26" rid="DDU731C26" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662709">26</a>). We measured KIV-2 copy numbers using qPCR and found that the KIV-2 numbers were significantly associated with Lp(a)-cholesterol levels. As reported by others (<a href="#DDU731C37" rid="DDU731C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662713">37</a>), smaller copy numbers of KIV-2 were associated with higher plasma Lp(a)-cholesterol levels. The number of KIV-2 repeats is variable ranging from 2 to &gt;50 in each allele. Most individuals have two alleles that contain different numbers of KIV-2 repeats (<a href="#DDU731C38" rid="DDU731C38" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662717">38</a>). The qPCR assay we employed measures the average number from both alleles. Our KIV-2 number represents the sum of KIV-2 number from both alleles that may be potentially limiting. This is because studies have demonstrated variable expressions of different KIV-2 numbers (<a href="#DDU731C39" rid="DDU731C39" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662656">39</a>). Despite this limitation, we detected strong association between the sum of KIV-2 repeats from both alleles and Lp(a)-cholesterol levels consistent with earlier observations (<a href="#DDU731C3" rid="DDU731C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662701">3</a>).</p><p id="__p29">Our studies confirmed that multiple genetic variants on chromosome 6q25-26 are associated with Lp(a)-cholesterol levels. We found SNP rs3798220 was not in linkage disequilibrium with SNP rs10455872 (<em>r</em><sup>2</sup> = 0). Conditional analysis also revealed that these two SNPs influenced variation in Lp(a)-cholesterol levels independently from each other. We measured KIV-2 copy numbers with qPCR. A weakness of this approach is that we cannot assess LD between genotyped SNPs with KIV-2 copy numbers. We found that the carriers of both rs10455872 and rs3798220 had a lower KIV-2 copy numbers than non-carriers. A previous study has shown similar results in that rare alleles of both rs10455872 and rs3798220 are each correlated with a smaller apolipoprotein(a) isoform (as assessed by immunoblotting) and a lower copy number (as assessed by quantitative PCR assay) (<a href="#DDU731C3" rid="DDU731C3" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662658">3</a>). However, conditional analysis revealed that the most significant SNPs on chromosome 6q25-26 remained strongly associated with Lp(a)-cholesterol levels after adjusting for KIV-2 numbers, suggesting these SNPs were significantly associated with Lp(a)-cholesterol levels independently of KIV-2 copy numbers. Previous studies indicated that KIV-2 copy numbers are not in strong LD with any biallelic markers in <em>LPA</em> (<a href="#DDU731C23" rid="DDU731C23" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662665">23</a>,<a href="#DDU731C40" rid="DDU731C40" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662677">40</a>). Therefore, two variants (rs3798220, rs10455872) were independently associated with Lp(a)-cholesterol levels. Our conditional analysis revealed that KIV-2 copy numbers were significantly associated with levels independently of both SNPs, although the magnitude of the association was reduced. We also found some SNPs were significantly associated with Lp(a)-cholesterol levels adjusted for KIV-2 numbers, rs3798220 and rs10455872. Additional rare variants in the <em>LPA</em> non-coding region or other gene region remain to be identified.</p><p id="__p30">We further investigated the effect of rs3798220 and rs10455872 on <em>LPA</em> gene expression. To our knowledge, the higher transcription level of <em>LPA</em> in carriers of rs10455872 has not been previously reported. We also found that a few carriers of rs10455872 have lower <em>LPA</em> gene expression and a few non-carriers have higher <em>LPA</em> gene expression (<a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu731/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material, Fig. S7</a>). The SNP rs10455872 may thus be a surrogate for, or ‘tags,’ the underlying causal variant within a linkage block to influence gene expression. Deep sequencing will be necessary to discover functional variants in this region. There are no differences in <em>LPA</em> transcription levels between carriers and non-carriers of rs3798220. However, the protein levels of Apo(a) in liver samples are higher in carrier than non-carrier of either rs10455872 or rs3798220. The substitution from isoleucine to methionine in rs3798220 may affect apo(a) and Lp(a) catabolism to increase Lp(a) levels (<a href="#DDU731C41" rid="DDU731C41" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662696">41</a>). We used SIFT (<a href="#DDU731C42" rid="DDU731C42" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662700">42</a>) and PolyPhen-2_HDIV (<a href="#DDU731C43" rid="DDU731C43" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662672">43</a>) to predict whether this substitution may alter the function of apo(a). The substitution of isoleucine with methionine is predicted to be a functionally deleterious substitution (SIFT: 0.01 and PolyPen-2_HDIV: 0.998).</p><p id="__p31" class="p p-last">In summary, we performed a systematic analysis of genetic variants with Lp(a)-cholesterol levels and identified multiple genetic determinants. We confirmed at least three sequence variants on chromosome 6q25-26 that were associated with Lp(a)-cholesterol levels and appear to act largely independent from one another. We also found the <em>APOC3</em> R19X null mutation on chromosome 11q23 to be associated with increased Lp(a)-cholesterol levels. Our results suggest rs10455872 within <em>LPA</em> may influence mRNA levels of <em>LPA</em> (transcription or stability), while rs3798220 may influence Lp(a) levels through effects on translation or protein stability. Future work will be required to precisely elucidate the complex genetic architecture and regulation of Lp(a) and the contribution of genetic variation to risk factors for cardiovascular diseases.</p></div><div id="s4" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s4title">Materials and Methods</h2><div id="s4a" class="sec sec-first"><h3 id="s4atitle">Subjects</h3><p id="__p32" class="p p-first-last">The OOA subjects of Lancaster Pennsylvania are relatively homogenous in terms of both genetic ancestry and lifestyle characteristics (<a href="#DDU731C44" rid="DDU731C44" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662690">44</a>). Subjects (<em>n</em> = 1376) included in this study were relatively healthy adults aged 20 years or older, who participated in the Heredity and Phenotype Intervention (HAPI) Heart Study (<a href="#DDU731C45" rid="DDU731C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662716">45</a>) and the Amish Family Longevity Study (AFLS) (<a href="#DDU731C46" rid="DDU731C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662679">46</a>). Details of the study design, recruitment, and phenotyping have been described previously (<a href="#DDU731C45" rid="DDU731C45" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662708">45</a>,<a href="#DDU731C46" rid="DDU731C46" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662682">46</a>). For Lp(a)-cholesterol specifically, quantitative measurements were made with Vertical Auto Profile-II (VAP-II) method (Atherotech, Birmingham, AL, USA) (<a href="#DDU731C47" rid="DDU731C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662704">47</a>). The VAP method measures cholesterol concentration of Lp(a) particles instead of apo(a) or Lp(a) particle concentration that will be a advantage of VAP. Measuring cholesterol concentration enables VAP testing of Lp(a) without influence of apo(a) size, which is a major problem influencing the accuracy of almost all immunoassay-based methods (<a href="#DDU731C47" rid="DDU731C47" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662719">47</a>,<a href="#DDU731C48" rid="DDU731C48" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662723">48</a>). The VAP method provides the continuous enzymatic analysis of cholesterol using a continuous flow VAP analyzer in lipoprotein classes separated by single vertical spin density-gradient ultracentrifugations that may be the potential for misclassification between different categories of LDL-particles. Informed consent was obtained from all participants and the study protocol was approved by the Institutional Review Board at the University of Maryland School of Medicine.</p></div><div id="s4b" class="sec"><h3 id="s4btitle">Genotyping and sequence</h3><p id="__p33" class="p p-first">GWAS samples were genotyped using the Affymetrix GeneChip<sup>®</sup> Human Mapping 500 K array set including a total of 500 568 SNPs (Santa Clara, California). The GeneChip Genotyping Analysis Software (GTYPE 4.0) was used for automated genotype calling as part of the GeneChip Operating Software (GCOS) platform. The GTYPE-generated chip files were re-analyzed using the BRLMM genotype calling algorithm which provided improved call accuracy compared with the DM algorithm (<a href="http://www.affymetrix.com/support/technical">http://www.affymetrix.com/support/technical</a>). A confidence threshold, a measure of the call quality, of 0.33 was used for this analysis. As an initial quality control (QC) measure, BRLMM-generated chip files with call rates &lt;90% for both enzymes were excluded from further analysis. Following quality control, Mendel and HWE checks, 382 935 informative SNPs were included in our analyses. We then used the 382 935 genotyped SNPs to impute all autosomal SNPs on HapMap (∼2.5 million SNPs) by MACH (version 1.0.15), using the publicly available phased haplotypes from HapMap (release 22, build 36, CEU population) as a reference panel (<a href="#DDU731C49" rid="DDU731C49" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662683">49</a>). Follow-up genotyping was performed using TaqMan Allelic Discrimination Assay (Applied Biosystems).</p><p id="__p34" class="p p-last">Direct sequencing (<a href="#DDU731C50" rid="DDU731C50" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662715">50</a>) was used to screen the 40 exons, exon–intron boundaries, and 2 kb of the promoter region of <em>LPA</em> for genetic variation in 24 Amish participants. We selected subjects according to their genotype at rs3805766 (top SNP from GWAS) or their Lp(a)-cholesterol levels. Both strands were sequenced on an ABI 3730xl DNA sequencer (Applied Biosystems) and analyzed using Sequencher software (GeneCodes).</p></div><div id="s4c" class="sec"><h3 id="s4ctitle">KIV-2 copy number genotyping</h3><p id="__p35" class="p p-first-last"><em>LPA</em> KIV-2 copy number was determined by quantitative real-time polymerase chain reaction (qPCR) as previously described (<a href="#DDU731C37" rid="DDU731C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662697">37</a>). Briefly, a multiplexed qPCR was carried out using TaqMan: probes for <em>LPA</em> KIV-2 and an endogenous single-copy control gene in the Applied Biosystems 7900HT Fast Real-Time PCR system. Experiments were performed using probes targeted to exons 4 and 5 of LPA, each of which is found once within every KIV-2 repeat at the genomic level (<a href="#DDU731C37" rid="DDU731C37" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662695">37</a>). TaqMan<sup>®</sup> TERT control reagent was used as single-copy reference gene (part number 44 01 633). Reaction volumes contained: 5 μl of Applied Biosystems Taqman Genotyping Master Mix (part number 43 71 357), 0.5 μl of 20× TaqMan<sup>®</sup>: primer/probe mix for <em>LPA</em>, 0.5 μl of 20× TaqMan<sup>®</sup>: primer/probe mix for TERT, 2 μl of water and 2 μl of genomic DNA at a concentration of 3–5 ng/μl. The average of ΔCT4 and ΔCT5 was then used for further analysis as the relative kringle repeat number.</p></div><div id="s4d" class="sec"><h3 id="s4dtitle"><em>LPA</em> gene expression analyses</h3><div id="s4d1" class="sec sec-first"><p></p><h4 id="s4d1title" class="inline">Liver samples </h4><p id="__p36" class="p p-first-last">Seventy-eight wedge biopsy liver samples based on the carrier and non-carrier status for rs3798220 (<em>N</em> = 20: 20 for CT: TT genotypes) and rs10455872 (<em>N</em> = 3: 20: 20 for GG: AG: AA genotypes) matched by age, sex and BMI were obtained from white patients undergoing open or laparoscopic Roux-en-Y gastric bypass operations or laparoscopic adjustable gastric banding procedures for extreme obesity or its comorbid medical problems at Geisinger Medical Center, Danville, Pennsylvania (<a href="#DDU731C29" rid="DDU731C29" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662655">29</a>,<a href="#DDU731C30" rid="DDU731C30" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662687">30</a>).</p></div><div id="s4d2" class="sec"><p></p><h4 id="s4d2title" class="inline">RT-qPCR </h4><p id="__p37" class="p p-first-last">Liver total RNA was extracted by TRIzol<sup>®</sup> reagent (Invitrogen) according to the manufacturer's instruction. The resulting RNAs were subjected to DNAse digestion using RNase-free DNase Set (Qiagen) and purification using RNeasy<sup>®</sup> MinElute<sup>®</sup> Cleanup Kit (Qiagen). The RNA quantity and quality were determined using a nanodrop spectrophotometer. cDNA was produced using Transcriptor First Strand cDNA Synthesis Kit (Roche). Real-time PCR was performed using TaqMan gene expression assay primers and probes [assay ID: Hs00916691-m1 for human <em>LPA</em>, Hs99999903_m1 for human Beta Actin (ACTB) as the endogenous control] and LightCycler 480 Probes Master (Roche) on LightCycler 480 Real-Time PCR system (Roche) using standard conditions and cycle times/temperatures. Relative expression of mRNAs was determined after normalization with reference gene levels using LightCycler 480 software 1.5. We normalized expression of mRNAs to a sample that had been tested in both SNPs.</p></div><div id="s4d3" class="sec sec-last"><p></p><h4 id="s4d3title" class="inline">Immunoblotting </h4><p id="__p38" class="p p-first-last">Liver samples were homogenized in radioimmunoprecipitation buffer (Teknova, Hollister, CA) containing 150 mM sodium chloride, 1% Triton X-100, 1% deoxycholic acid-sodium salt, 0.1% sodium dodecyl sulfate, 50 mM Tris–HCL (pH 7.5), 2 mM EDTA, 50 mM NaF, 100 µM Na<sub>3</sub>VO<sub>4</sub>, and a pill of complete inhibitor (Roche Diagnostics, Indianapolis, Indiana). Liver tissue lysates were centrifuged at 16 000<em>g</em> for 30 min at 4°C and the supernatant was saved for subsequent immunoblotting. Protein concentrations were measured with the Pierce BCA kit (Thermo Scientific, Rockford, IL). Of note, 66 µg of liver protein was subjected to electrophoresis on NuPage 3–8% Tris–Acetate Gel (Life Technologies, Grand Island, NY), samples were transferred to nitrocellulose membrane, and then immunoblotted with Lp(a) sheep polyclonal antibody (provided by Dr Eduardo Angles-Cano, Institut national de la santé et de la recherche médicale, France) at a dilution of 1:20 000 for 2 h at room temperature (<a href="#DDU731C51" rid="DDU731C51" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662712">51</a>). Then the blots were incubated with a donkey anti-sheep IgG secondary antibody conjugated with peroxidase (Santa Cruz Biotechnology, Dallas, Texas) at a dilution of 1:20 000 for 1 h at room temperature. The blots were subsequently incubated with monoclonal anti-vinculin mouse antibody (Sigma-Aldrich, St. Louis, MO) used as an internal control. The immunoblot signals were visualized with the Pierce ECL Western Blotting Substrate (Thermo Scientific, Rockford, IL). Image quantitative analysis was performed using ImageJ Software (<a href="#DDU731C52" rid="DDU731C52" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_641515728">52</a>).</p></div></div><div id="s4e" class="sec sec-last"><h3 id="s4etitle">Statistical analysis</h3><p id="__p39" class="p p-first">The natural log of plasma Lp(a)-cholesterol was used for association analyses. Association analyses were performed using a variance component model that includes a polygenic component to account for phenotypic correlation due to relatedness. The polygenic component was modeled using the relationship matrix derived from the complete 14-generation Amish pedigree structure (<a href="#DDU731C53" rid="DDU731C53" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662718">53</a>,<a href="#DDU731C54" rid="DDU731C54" class=" bibr popnode tag_hotlink tag_tooltip" id="__tag_439662660">54</a>). A one-degree of freedom likelihood ratio test was used to assess significance of the measured genotype under an additive model. We included age, age<sup>2</sup> and sex as covariates in the association models. Pairwise linkage-disequilibrium (LD) statistics (<em>D</em>′ and <em>r</em><sup>2</sup>) were calculated with Haploview version 4.2 (<a href="http://www.broadinstitute.org/haploview/">http://www.broadinstitute.org/haploview/</a>).</p><p id="__p40" class="p p-last"><em>LPA</em> gene expression data were normalized to the quantity of beta-actin mRNA present in each sample. This allowed us to account for any variability in the initial template concentration as well as the conversion efficiency of the reverse transcription reaction. Student's <em>t</em> test was used to compare expression levels between carriers and non-carriers.</p></div></div><div id="s5" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s5title">Supplementary Material</h2><p id="__p41" class="p p-first-last"><a href="http://hmg.oxfordjournals.org/lookup/suppl/doi:10.1093/hmg/ddu731/-/DC1" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">Supplementary Material is available at <em>HMG</em> online</a>.</p></div><div id="s6" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="s6title">Funding</h2><p id="__p42" class="p p-first-last">This work was supported in part by American Heart Association grant (<span id="cn1">10SDG2690004</span> to M.F.); American Diabetes Association grant (<span id="cn2">7-12-CT-26</span> to M.F.); National Institutes of Health (<span id="cn3">U01 HL072515)</span>;, (<span id="cn4">U01 HL84756), (R01 AG18728), (R01 DK088231)</span> the Mid-Atlantic Nutrition Obesity Research Center (grant <span>P30 DK72488)</span>; the VA BLR&amp;D Career Development award (Y.-C.C.).</p></div><div id="__sec1" class="tsec sec"><a id="supplementary-material-sec"></a><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="__sec1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="PMC_1">

<!--caption a9--><strong>Supplementary Data:</strong>

<div class="sup-box half_rhythm" id="idm140237394122864"><a href="/pmc/articles/PMC4380064/bin/supp_24_8_2390__index.html" data-ga-action="click_feat_suppl">Click here to view.</a></div>

</div></div><div id="idm140237396560560" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140237396560560title">Acknowledgements</h2><div class="sec"><p id="__p43">We thank the Amish Research Clinic Staff for their energetic efforts in study subject recruitment and characterization as well as all the volunteers for their participation in these studies. This study would not have been possible without the outstanding cooperation of the Amish community. We thank Alexis Gorden for helping to get clinic information from Geisinger Clinic bariatric surgery patients.</p><p id="__p44"><em>Conflict of Interest statement</em>. None declared.</p></div></div><div id="idm140237396558272" class="tsec sec"><div class="goto jig-ncbiinpagenav-goto-container"><a class="tgt_dark page-toc-label jig-ncbiinpagenav-goto-heading" href="#" title="Go to other sections in this page" role="button" aria-expanded="false" aria-haspopup="true">Go to:</a></div><h2 class="head no_bottom_margin ui-helper-clearfix" id="idm140237396558272title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="DDU731C1">1. <span class="mixed-citation">(2001) <span class="ref-title">Executive summary of the third report of the National Cholesterol Education Program (NCEP) expert panel on detection, evaluation, and treatment of high blood cholesterol in adults (Adult Treatment Panel III)</span>. <span class="ref-journal">JAMA</span>, <span class="ref-vol">285</span>, 2486–2497. [<a href="/pubmed/11368702" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Executive+summary+of+the+third+report+of+the+National+Cholesterol+Education+Program+(NCEP)+expert+panel+on+detection,+evaluation,+and+treatment+of+high+blood+cholesterol+in+adults+(Adult+Treatment+Panel+III)&amp;volume=285&amp;publication_year=2001&amp;pages=2486-2497&amp;pmid=11368702&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C2">2. <span class="mixed-citation">Kamstrup P.R., Tybjaerg-Hansen A., Steffensen R., Nordestgaard B.G. (2009) <span class="ref-title">Genetically elevated lipoprotein(a) and increased risk of myocardial infarction</span>. <span class="ref-journal">JAMA</span>, <span class="ref-vol">301</span>, 2331–2339. [<a href="/pubmed/19509380" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=JAMA&amp;title=Genetically+elevated+lipoprotein(a)+and+increased+risk+of+myocardial+infarction&amp;author=P.R.+Kamstrup&amp;author=A.+Tybjaerg-Hansen&amp;author=R.+Steffensen&amp;author=B.G.+Nordestgaard&amp;volume=301&amp;publication_year=2009&amp;pages=2331-2339&amp;pmid=19509380&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C3">3. <span class="mixed-citation">Clarke R., Peden J.F., Hopewell J.C., Kyriakou T., Goel A., Heath S.C., Parish S., Barlera S., Franzosi M.G., Rust S., et al.  (2009) <span class="ref-title">Genetic variants associated with Lp(a) lipoprotein level and coronary disease</span>. <span class="ref-journal">N. Engl. J. Med.</span>, <span class="ref-vol">361</span>, 2518–2528. [<a href="/pubmed/20032323" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Genetic+variants+associated+with+Lp(a)+lipoprotein+level+and+coronary+disease&amp;author=R.+Clarke&amp;author=J.F.+Peden&amp;author=J.C.+Hopewell&amp;author=T.+Kyriakou&amp;author=A.+Goel&amp;volume=361&amp;publication_year=2009&amp;pages=2518-2528&amp;pmid=20032323&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C4">4. <span class="mixed-citation">Tregouet D.A., Konig I.R., Erdmann J., Munteanu A., Braund P.S., Hall A.S., Grosshennig A., Linsel-Nitschke P., Perret C., DeSuremain M., et al.  (2009) <span class="ref-title">Genome-wide haplotype association study identifies the SLC22A3-LPAL2-LPA gene cluster as a risk locus for coronary artery disease</span>. <span class="ref-journal">Nat. Genet.</span>, <span class="ref-vol">41</span>, 283–285. [<a href="/pubmed/19198611" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Genome-wide+haplotype+association+study+identifies+the+SLC22A3-LPAL2-LPA+gene+cluster+as+a+risk+locus+for+coronary+artery+disease&amp;author=D.A.+Tregouet&amp;author=I.R.+Konig&amp;author=J.+Erdmann&amp;author=A.+Munteanu&amp;author=P.S.+Braund&amp;volume=41&amp;publication_year=2009&amp;pages=283-285&amp;pmid=19198611&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C5">5. <span class="mixed-citation">Haberland M.E., Fless G.M., Scanu A.M., Fogelman A.M. (1992) <span class="ref-title">Malondialdehyde modification of lipoprotein(a) produces avid uptake by human monocyte-macrophages</span>. <span class="ref-journal">J. Biol. Chem.</span>, <span class="ref-vol">267</span>, 4143–4151. [<a href="/pubmed/1531481" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biol.+Chem.&amp;title=Malondialdehyde+modification+of+lipoprotein(a)+produces+avid+uptake+by+human+monocyte-macrophages&amp;author=M.E.+Haberland&amp;author=G.M.+Fless&amp;author=A.M.+Scanu&amp;author=A.M.+Fogelman&amp;volume=267&amp;publication_year=1992&amp;pages=4143-4151&amp;pmid=1531481&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C6">6. <span class="mixed-citation">Longenecker J.C., Klag M.J., Marcovina S.M., Powe N.R., Fink N.E., Giaculli F., Coresh J. (2002) <span class="ref-title">Small apolipoprotein(a) size predicts mortality in end-stage renal disease: The CHOICE study</span>. <span class="ref-journal">Circulation</span>, <span class="ref-vol">106</span>, 2812–2818. [<a href="/pubmed/12451008" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circulation&amp;title=Small+apolipoprotein(a)+size+predicts+mortality+in+end-stage+renal+disease:+The+CHOICE+study&amp;author=J.C.+Longenecker&amp;author=M.J.+Klag&amp;author=S.M.+Marcovina&amp;author=N.R.+Powe&amp;author=N.E.+Fink&amp;volume=106&amp;publication_year=2002&amp;pages=2812-2818&amp;pmid=12451008&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C7">7. <span class="mixed-citation">Grainger D.J., Kirschenlohr H.L., Metcalfe J.C., Weissberg P.L., Wade D.P., Lawn R.M. (1993) <span class="ref-title">Proliferation of human smooth muscle cells promoted by lipoprotein(a)</span>. <span class="ref-journal">Science</span>, <span class="ref-vol">260</span>, 1655–1658. [<a href="/pubmed/8503012" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Proliferation+of+human+smooth+muscle+cells+promoted+by+lipoprotein(a)&amp;author=D.J.+Grainger&amp;author=H.L.+Kirschenlohr&amp;author=J.C.+Metcalfe&amp;author=P.L.+Weissberg&amp;author=D.P.+Wade&amp;volume=260&amp;publication_year=1993&amp;pages=1655-1658&amp;pmid=8503012&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C8">8. <span class="mixed-citation">Enas E.A., Chacko V., Senthilkumar A., Puthumana N., Mohan V. (2006) <span class="ref-title">Elevated lipoprotein(a)—a genetic risk factor for premature vascular disease in people with and without standard risk factors: a review</span>. <span class="ref-journal">Dis. Mon.</span>, <span class="ref-vol">52</span>, 5–50. [<a href="/pubmed/16549089" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Dis.+Mon.&amp;title=Elevated+lipoprotein(a)—a+genetic+risk+factor+for+premature+vascular+disease+in+people+with+and+without+standard+risk+factors:+a+review&amp;author=E.A.+Enas&amp;author=V.+Chacko&amp;author=A.+Senthilkumar&amp;author=N.+Puthumana&amp;author=V.+Mohan&amp;volume=52&amp;publication_year=2006&amp;pages=5-50&amp;pmid=16549089&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C9">9. <span class="mixed-citation">Caplice N.M., Panetta C., Peterson T.E., Kleppe L.S., Mueske C.S., Kostner G.M., Broze G.J., Jr., Simari R.D. (2001) <span class="ref-title">Lipoprotein (a) binds and inactivates tissue factor pathway inhibitor: a novel link between lipoproteins and thrombosis</span>. <span class="ref-journal">Blood</span>, <span class="ref-vol">98</span>, 2980–2987. [<a href="/pubmed/11698280" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Lipoprotein+(a)+binds+and+inactivates+tissue+factor+pathway+inhibitor:+a+novel+link+between+lipoproteins+and+thrombosis&amp;author=N.M.+Caplice&amp;author=C.+Panetta&amp;author=T.E.+Peterson&amp;author=L.S.+Kleppe&amp;author=C.S.+Mueske&amp;volume=98&amp;publication_year=2001&amp;pages=2980-2987&amp;pmid=11698280&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C10">10. <span class="mixed-citation">Marcovina S.M., Koschinsky M.L. (2003) <span class="ref-title">Evaluation of lipoprotein(a) as a prothrombotic factor: progress from bench to bedside</span>. <span class="ref-journal">Curr. Opin. Lipidol.</span>, <span class="ref-vol">14</span>, 361–366. [<a href="/pubmed/12865733" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Lipidol.&amp;title=Evaluation+of+lipoprotein(a)+as+a+prothrombotic+factor:+progress+from+bench+to+bedside&amp;author=S.M.+Marcovina&amp;author=M.L.+Koschinsky&amp;volume=14&amp;publication_year=2003&amp;pages=361-366&amp;pmid=12865733&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C11">11. <span class="mixed-citation">McLean J.W., Tomlinson J.E., Kuang W.J., Eaton D.L., Chen E.Y., Fless G.M., Scanu A.M., Lawn R.M. (1987) <span class="ref-title">cDNA sequence of human apolipoprotein(a) is homologous to plasminogen</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">330</span>, 132–137. [<a href="/pubmed/3670400" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=cDNA+sequence+of+human+apolipoprotein(a)+is+homologous+to+plasminogen&amp;author=J.W.+McLean&amp;author=J.E.+Tomlinson&amp;author=W.J.+Kuang&amp;author=D.L.+Eaton&amp;author=E.Y.+Chen&amp;volume=330&amp;publication_year=1987&amp;pages=132-137&amp;pmid=3670400&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C12">12. <span class="mixed-citation">Ober C., Abney M., McPeek M.S. (2001) <span class="ref-title">The genetic dissection of complex traits in a founder population</span>. <span class="ref-journal">Am. J. Hum. Genet.</span>, <span class="ref-vol">69</span>, 1068–1079. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1274354/">PMC free article</a>]</span> [<a href="/pubmed/11590547" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Hum.+Genet.&amp;title=The+genetic+dissection+of+complex+traits+in+a+founder+population&amp;author=C.+Ober&amp;author=M.+Abney&amp;author=M.S.+McPeek&amp;volume=69&amp;publication_year=2001&amp;pages=1068-1079&amp;pmid=11590547&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C13">13. <span class="mixed-citation">Rainwater D.L., Kammerer C.M., VandeBerg J.L., Hixson J.E. (1997) <span class="ref-title">Characterization of the genetic elements controlling lipoprotein(a) concentrations in Mexican Americans. Evidence for at least three controlling elements linked to LPA, the locus encoding apolipoprotein(a)</span>. <span class="ref-journal">Atherosclerosis</span>, <span class="ref-vol">128</span>, 223–233. [<a href="/pubmed/9050779" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Characterization+of+the+genetic+elements+controlling+lipoprotein(a)+concentrations+in+Mexican+Americans.+Evidence+for+at+least+three+controlling+elements+linked+to+LPA,+the+locus+encoding+apolipoprotein(a)&amp;author=D.L.+Rainwater&amp;author=C.M.+Kammerer&amp;author=J.L.+VandeBerg&amp;author=J.E.+Hixson&amp;volume=128&amp;publication_year=1997&amp;pages=223-233&amp;pmid=9050779&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C14">14. <span class="mixed-citation">Kraft H.G., Kochl S., Menzel H.J., Sandholzer C., Utermann G. (1992) <span class="ref-title">The apolipoprotein (a) gene: a transcribed hypervariable locus controlling plasma lipoprotein (a) concentration</span>. <span class="ref-journal">Hum. Genet.</span>, <span class="ref-vol">90</span>, 220–230. [<a href="/pubmed/1336760" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Genet.&amp;title=The+apolipoprotein+(a)+gene:+a+transcribed+hypervariable+locus+controlling+plasma+lipoprotein+(a)+concentration&amp;author=H.G.+Kraft&amp;author=S.+Kochl&amp;author=H.J.+Menzel&amp;author=C.+Sandholzer&amp;author=G.+Utermann&amp;volume=90&amp;publication_year=1992&amp;pages=220-230&amp;pmid=1336760&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C15">15. <span class="mixed-citation">Hasstedt S.J., Wilson D.E., Edwards C.Q., Cannon W.N., Carmelli D., Williams R.R. (1983) <span class="ref-title">The genetics of quantitative plasma Lp(a): analysis of a large pedigree</span>. <span class="ref-journal">Am. J. Med. Genet.</span>, <span class="ref-vol">16</span>, 179–188. [<a href="/pubmed/6228142" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.&amp;title=The+genetics+of+quantitative+plasma+Lp(a):+analysis+of+a+large+pedigree&amp;author=S.J.+Hasstedt&amp;author=D.E.+Wilson&amp;author=C.Q.+Edwards&amp;author=W.N.+Cannon&amp;author=D.+Carmelli&amp;volume=16&amp;publication_year=1983&amp;pages=179-188&amp;pmid=6228142&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C16">16. <span class="mixed-citation">Morton N.E., Berg K., Dahlen G., Ferrell R.E., Rhoads G.G. (1985) <span class="ref-title">Genetics of the Lp lipoprotein in Japanese-Americans</span>. <span class="ref-journal">Genet. Epidemiol.</span>, <span class="ref-vol">2</span>, 113–121. [<a href="/pubmed/2932366" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Genetics+of+the+Lp+lipoprotein+in+Japanese-Americans&amp;author=N.E.+Morton&amp;author=K.+Berg&amp;author=G.+Dahlen&amp;author=R.E.+Ferrell&amp;author=G.G.+Rhoads&amp;volume=2&amp;publication_year=1985&amp;pages=113-121&amp;pmid=2932366&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C17">17. <span class="mixed-citation">Hasstedt S.J., Williams R.R. (1986) <span class="ref-title">Three alleles for quantitative Lp(a)</span>. <span class="ref-journal">Genet. Epidemiol.</span>, <span class="ref-vol">3</span>, 53–55. [<a href="/pubmed/3957004" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genet.+Epidemiol.&amp;title=Three+alleles+for+quantitative+Lp(a)&amp;author=S.J.+Hasstedt&amp;author=R.R.+Williams&amp;volume=3&amp;publication_year=1986&amp;pages=53-55&amp;pmid=3957004&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C18">18. <span class="mixed-citation">Rosby O., Berg K. (2000) <span class="ref-title">LPA gene: interaction between the apolipoprotein(a) size (‘kringle IV’ repeat) polymorphism and a pentanucleotide repeat polymorphism influences Lp(a) lipoprotein level</span>. <span class="ref-journal">J. Intern. Med.</span>, <span class="ref-vol">247</span>, 139–152. [<a href="/pubmed/10672142" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Intern.+Med.&amp;title=LPA+gene:+interaction+between+the+apolipoprotein(a)+size+(‘kringle+IV’+repeat)+polymorphism+and+a+pentanucleotide+repeat+polymorphism+influences+Lp(a)+lipoprotein+level&amp;author=O.+Rosby&amp;author=K.+Berg&amp;volume=247&amp;publication_year=2000&amp;pages=139-152&amp;pmid=10672142&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C19">19. <span class="mixed-citation">Ogorelkova M., Kraft H.G., Ehnholm C., Utermann G. (2001) <span class="ref-title">Single nucleotide polymorphisms in exons of the apo(a) kringles IV types 6 to 10 domain affect Lp(a) plasma concentrations and have different patterns in Africans and Caucasians</span>. <span class="ref-journal">Hum. Mol. Genet.</span>, <span class="ref-vol">10</span>, 815–824. [<a href="/pubmed/11285247" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Mol.+Genet.&amp;title=Single+nucleotide+polymorphisms+in+exons+of+the+apo(a)+kringles+IV+types+6+to+10+domain+affect+Lp(a)+plasma+concentrations+and+have+different+patterns+in+Africans+and+Caucasians&amp;author=M.+Ogorelkova&amp;author=H.G.+Kraft&amp;author=C.+Ehnholm&amp;author=G.+Utermann&amp;volume=10&amp;publication_year=2001&amp;pages=815-824&amp;pmid=11285247&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C20">20. <span class="mixed-citation">Lim E.T., Wurtz P., Havulinna A.S., Palta P., Tukiainen T., Rehnstrom K., Esko T., Magi R., Inouye M., Lappalainen T., et al.  (2014) <span class="ref-title">Distribution and medical impact of loss-of-function variants in the Finnish founder population</span>. <span class="ref-journal">PLoS Genet.</span>, <span class="ref-vol">10</span>, e1004494. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4117444/">PMC free article</a>]</span> [<a href="/pubmed/25078778" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=Distribution+and+medical+impact+of+loss-of-function+variants+in+the+Finnish+founder+population&amp;author=E.T.+Lim&amp;author=P.+Wurtz&amp;author=A.S.+Havulinna&amp;author=P.+Palta&amp;author=T.+Tukiainen&amp;volume=10&amp;publication_year=2014&amp;pages=e1004494&amp;pmid=25078778&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C21">21. <span class="mixed-citation">Lopez S., Buil A., Ordonez J., Souto J.C., Almasy L., Lathrop M., Blangero J., Blanco-Vaca F., Fontcuberta J., Soria J.M. (2008) <span class="ref-title">Genome-wide linkage analysis for identifying quantitative trait loci involved in the regulation of lipoprotein a (Lpa) levels</span>. <span class="ref-journal">Eur. J. Hum. Genet.</span>, <span class="ref-vol">16</span>, 1372–1379. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2749722/">PMC free article</a>]</span> [<a href="/pubmed/18560444" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=Genome-wide+linkage+analysis+for+identifying+quantitative+trait+loci+involved+in+the+regulation+of+lipoprotein+a+(Lpa)+levels&amp;author=S.+Lopez&amp;author=A.+Buil&amp;author=J.+Ordonez&amp;author=J.C.+Souto&amp;author=L.+Almasy&amp;volume=16&amp;publication_year=2008&amp;pages=1372-1379&amp;pmid=18560444&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C22">22. <span class="mixed-citation">Barlera S., Specchia C., Farrall M., Chiodini B.D., Franzosi M.G., Rust S., Green F., Nicolis E.B., Peden J., Assmann G., et al.  (2007) <span class="ref-title">Multiple QTL influence the serum Lp(a) concentration: a genome-wide linkage screen in the PROCARDIS study</span>. <span class="ref-journal">Eur. J. Hum. Genet.</span>, <span class="ref-vol">15</span>, 221–227. [<a href="/pubmed/17133260" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+J.+Hum.+Genet.&amp;title=Multiple+QTL+influence+the+serum+Lp(a)+concentration:+a+genome-wide+linkage+screen+in+the+PROCARDIS+study&amp;author=S.+Barlera&amp;author=C.+Specchia&amp;author=M.+Farrall&amp;author=B.D.+Chiodini&amp;author=M.G.+Franzosi&amp;volume=15&amp;publication_year=2007&amp;pages=221-227&amp;pmid=17133260&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C23">23. <span class="mixed-citation">Ober C., Nord A.S., Thompson E.E., Pan L., Tan Z., Cusanovich D., Sun Y., Nicolae R., Edelstein C., Schneider D.H., et al.  (2009) <span class="ref-title">Genome-wide association study of plasma lipoprotein(a) levels identifies multiple genes on chromosome 6q</span>. <span class="ref-journal">J. Lipid. Res.</span>, <span class="ref-vol">50</span>, 798–806. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2666166/">PMC free article</a>]</span> [<a href="/pubmed/19124843" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid.+Res.&amp;title=Genome-wide+association+study+of+plasma+lipoprotein(a)+levels+identifies+multiple+genes+on+chromosome+6q&amp;author=C.+Ober&amp;author=A.S.+Nord&amp;author=E.E.+Thompson&amp;author=L.+Pan&amp;author=Z.+Tan&amp;volume=50&amp;publication_year=2009&amp;pages=798-806&amp;pmid=19124843&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C24">24. <span class="mixed-citation">Frazer K.A., Ballinger D.G., Cox D.R., Hinds D.A., Stuve L.L., Gibbs R.A., Belmont J.W., Boudreau A., Hardenbol P., Leal S.M., et al.  (2007) <span class="ref-title">A second generation human haplotype map of over 3.1 million SNPs</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">449</span>, 851–861. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2689609/">PMC free article</a>]</span> [<a href="/pubmed/17943122" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A+second+generation+human+haplotype+map+of+over+3.1+million+SNPs&amp;author=K.A.+Frazer&amp;author=D.G.+Ballinger&amp;author=D.R.+Cox&amp;author=D.A.+Hinds&amp;author=L.L.+Stuve&amp;volume=449&amp;publication_year=2007&amp;pages=851-861&amp;pmid=17943122&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C25">25. <span class="mixed-citation">Marchini J., Howie B. (2010) <span class="ref-title">Genotype imputation for genome-wide association studies</span>. <span class="ref-journal">Nat. Rev. Genet.</span>, <span class="ref-vol">11</span>, 499–511. [<a href="/pubmed/20517342" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Genotype+imputation+for+genome-wide+association+studies&amp;author=J.+Marchini&amp;author=B.+Howie&amp;volume=11&amp;publication_year=2010&amp;pages=499-511&amp;pmid=20517342&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C26">26. <span class="mixed-citation">Kronenberg F., Utermann G. (2013) <span class="ref-title">Lipoprotein(a): resurrected by genetics</span>. <span class="ref-journal">J. Intern. Med.</span>, <span class="ref-vol">273</span>, 6–30. [<a href="/pubmed/22998429" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Intern.+Med.&amp;title=Lipoprotein(a):+resurrected+by+genetics&amp;author=F.+Kronenberg&amp;author=G.+Utermann&amp;volume=273&amp;publication_year=2013&amp;pages=6-30&amp;pmid=22998429&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C27">27. <span class="mixed-citation">Erqou S., Thompson A., Di Angelantonio E., Saleheen D., Kaptoge S., Marcovina S., Danesh J. (2010) <span class="ref-title">Apolipoprotein(a) isoforms and the risk of vascular disease: systematic review of 40 studies involving 58,000 participants</span>. <span class="ref-journal">J. Am. Coll. Cardiol.</span>, <span class="ref-vol">55</span>, 2160–2167. [<a href="/pubmed/20447543" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Am.+Coll.+Cardiol.&amp;title=Apolipoprotein(a)+isoforms+and+the+risk+of+vascular+disease:+systematic+review+of+40+studies+involving+58,000+participants&amp;author=S.+Erqou&amp;author=A.+Thompson&amp;author=E.+Di+Angelantonio&amp;author=D.+Saleheen&amp;author=S.+Kaptoge&amp;volume=55&amp;publication_year=2010&amp;pages=2160-2167&amp;pmid=20447543&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C28">28. <span class="mixed-citation">Lanktree M.B., Anand S.S., Yusuf S., Hegele R.A. (2009) <span class="ref-title">Replication of genetic associations with plasma lipoprotein traits in a multiethnic sample</span>. <span class="ref-journal">J. Lipid Res.</span>, <span class="ref-vol">50</span>, 1487–1496. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2694347/">PMC free article</a>]</span> [<a href="/pubmed/19299407" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid+Res.&amp;title=Replication+of+genetic+associations+with+plasma+lipoprotein+traits+in+a+multiethnic+sample&amp;author=M.B.+Lanktree&amp;author=S.S.+Anand&amp;author=S.+Yusuf&amp;author=R.A.+Hegele&amp;volume=50&amp;publication_year=2009&amp;pages=1487-1496&amp;pmid=19299407&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C29">29. <span class="mixed-citation">Still C.D., Wood G.C., Chu X., Erdman R., Manney C.H., Benotti P.N., Petrick A.T., Strodel W.E., Mirshahi U.L., Mirshahi T., et al.  (2011) <span class="ref-title">High allelic burden of four obesity SNPs is associated with poorer weight loss outcomes following gastric bypass surgery</span>. <span class="ref-journal">Obesity (Silver Spring)</span>, <span class="ref-vol">19</span>, 1676–1683. [<a href="/pubmed/21311511" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Obesity+(Silver+Spring)&amp;title=High+allelic+burden+of+four+obesity+SNPs+is+associated+with+poorer+weight+loss+outcomes+following+gastric+bypass+surgery&amp;author=C.D.+Still&amp;author=G.C.+Wood&amp;author=X.+Chu&amp;author=R.+Erdman&amp;author=C.H.+Manney&amp;volume=19&amp;publication_year=2011&amp;pages=1676-1683&amp;pmid=21311511&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C30">30. <span class="mixed-citation">Kumashiro N., Erion D.M., Zhang D., Kahn M., Beddow S.A., Chu X., Still C.D., Gerhard G.S., Han X., Dziura J., et al.  (2011) <span class="ref-title">Cellular mechanism of insulin resistance in nonalcoholic fatty liver disease</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. USA</span>, <span class="ref-vol">108</span>, 16381–16385. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3182681/">PMC free article</a>]</span> [<a href="/pubmed/21930939" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Cellular+mechanism+of+insulin+resistance+in+nonalcoholic+fatty+liver+disease&amp;author=N.+Kumashiro&amp;author=D.M.+Erion&amp;author=D.+Zhang&amp;author=M.+Kahn&amp;author=S.A.+Beddow&amp;volume=108&amp;publication_year=2011&amp;pages=16381-16385&amp;pmid=21930939&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C31">31. <span class="mixed-citation">Pollin T.I., Damcott C.M., Shen H., Ott S.H., Shelton J., Horenstein R.B., Post W., McLenithan J.C., Bielak L.F., Peyser P.A., et al.  (2008) <span class="ref-title">A null mutation in human APOC3 confers a favorable plasma lipids profile and apparent cardioprotection</span>. <span class="ref-journal">Science</span>, <span class="ref-vol">322</span>, 1702–1705. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2673993/">PMC free article</a>]</span> [<a href="/pubmed/19074352" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+null+mutation+in+human+APOC3+confers+a+favorable+plasma+lipids+profile+and+apparent+cardioprotection&amp;author=T.I.+Pollin&amp;author=C.M.+Damcott&amp;author=H.+Shen&amp;author=S.H.+Ott&amp;author=J.+Shelton&amp;volume=322&amp;publication_year=2008&amp;pages=1702-1705&amp;pmid=19074352&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C32">32. <span class="mixed-citation">TG and HDL Working Group of the Exome Sequencing Project, National Heart, Lung, and Blood Institute, Crosby J., Peloso G.M., Auer P.L., Crosslin D.R., Stitziel N.O., Lange L.A., Lu Y., Tang Z.Z., Zhang H., Hindy G., et al.  (2014) <span class="ref-title">Loss-of-function mutations in APOC3, triglycerides, and coronary disease</span>. <span class="ref-journal">N. Engl. J. Med.</span>, <span class="ref-vol">371</span>, 22–31. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4180269/">PMC free article</a>]</span> [<a href="/pubmed/24941081" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Loss-of-function+mutations+in+APOC3,+triglycerides,+and+coronary+disease&amp;author=J.+Crosby&amp;author=G.M.+Peloso&amp;author=P.L.+Auer&amp;author=D.R.+Crosslin&amp;author=N.O.+Stitziel&amp;volume=371&amp;publication_year=2014&amp;pages=22-31&amp;pmid=24941081&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C33">33. <span class="mixed-citation">Jørgensen A.B., Frikke-Schidt R., Nordestgaard B.G., Tybjærg-Hansen A. (2014) <span class="ref-title">Loss-of-function mutations in APOC3 and risk of ischemic vascular disease</span>. <span class="ref-journal">N. Engl. J. Med.</span>, <span class="ref-vol">371</span>, 32–41. [<a href="/pubmed/24941082" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=N.+Engl.+J.+Med.&amp;title=Loss-of-function+mutations+in+APOC3+and+risk+of+ischemic+vascular+disease&amp;author=A.B.+Jørgensen&amp;author=R.+Frikke-Schidt&amp;author=B.G.+Nordestgaard&amp;author=A.+Tybjærg-Hansen&amp;volume=371&amp;publication_year=2014&amp;pages=32-41&amp;pmid=24941082&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C34">34. <span class="mixed-citation">Rainwater D.L. (1996) <span class="ref-title">Lp(a) concentrations are related to plasma lipid concentrations</span>. <span class="ref-journal">Atherosclerosis</span>, <span class="ref-vol">127</span>, 13–18. [<a href="/pubmed/9006799" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Lp(a)+concentrations+are+related+to+plasma+lipid+concentrations&amp;author=D.L.+Rainwater&amp;volume=127&amp;publication_year=1996&amp;pages=13-18&amp;pmid=9006799&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C35">35. <span class="mixed-citation">Koch W., Mueller J.C., Schrempf M., Wolferstetter H., Kirchhofer J., Schomig A., Kastrati A. (2013) <span class="ref-title">Two rare variants explain association with acute myocardial infarction in an extended genomic region including the apolipoprotein(A) gene</span>. <span class="ref-journal">Ann. Hum. Genet.</span>, <span class="ref-vol">77</span>, 47–55. [<a href="/pubmed/23278389" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+Hum.+Genet.&amp;title=Two+rare+variants+explain+association+with+acute+myocardial+infarction+in+an+extended+genomic+region+including+the+apolipoprotein(A)+gene&amp;author=W.+Koch&amp;author=J.C.+Mueller&amp;author=M.+Schrempf&amp;author=H.+Wolferstetter&amp;author=J.+Kirchhofer&amp;volume=77&amp;publication_year=2013&amp;pages=47-55&amp;pmid=23278389&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C36">36. <span class="mixed-citation">Zabaneh D., Kumari M., Sandhu M., Wareham N., Wainwright N., Papamarkou T., Hopewell J., Clarke R., Li K., Palmen J., et al.  (2011) <span class="ref-title">Meta analysis of candidate gene variants outside the LPA locus with Lp(a) plasma levels in 14,500 participants of six White European cohorts</span>. <span class="ref-journal">Atherosclerosis</span>, <span class="ref-vol">217</span>, 447–451. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3972487/">PMC free article</a>]</span> [<a href="/pubmed/21592478" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Atherosclerosis&amp;title=Meta+analysis+of+candidate+gene+variants+outside+the+LPA+locus+with+Lp(a)+plasma+levels+in+14,500+participants+of+six+White+European+cohorts&amp;author=D.+Zabaneh&amp;author=M.+Kumari&amp;author=M.+Sandhu&amp;author=N.+Wareham&amp;author=N.+Wainwright&amp;volume=217&amp;publication_year=2011&amp;pages=447-451&amp;pmid=21592478&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C37">37. <span class="mixed-citation">Lanktree M.B., Rajakumar C., Brunt J.H., Koschinsky M.L., Connelly P.W., Hegele R.A. (2009) <span class="ref-title">Determination of lipoprotein(a) kringle repeat number from genomic DNA: copy number variation genotyping using qPCR</span>. <span class="ref-journal">J. Lipid Res.</span>, <span class="ref-vol">50</span>, 768–772. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2656671/">PMC free article</a>]</span> [<a href="/pubmed/19060253" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid+Res.&amp;title=Determination+of+lipoprotein(a)+kringle+repeat+number+from+genomic+DNA:+copy+number+variation+genotyping+using+qPCR&amp;author=M.B.+Lanktree&amp;author=C.+Rajakumar&amp;author=J.H.+Brunt&amp;author=M.L.+Koschinsky&amp;author=P.W.+Connelly&amp;volume=50&amp;publication_year=2009&amp;pages=768-772&amp;pmid=19060253&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C38">38. <span class="mixed-citation">Marcovina S.M., Hobbs H.H., Albers J.J. (1996) <span class="ref-title">Relation between number of apolipoprotein(a) kringle 4 repeats and mobility of isoforms in agarose gel: basis for a standardized isoform nomenclature</span>. <span class="ref-journal">Clin. Chem.</span>, <span class="ref-vol">42</span>, 436–439. [<a href="/pubmed/8598109" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Chem.&amp;title=Relation+between+number+of+apolipoprotein(a)+kringle+4+repeats+and+mobility+of+isoforms+in+agarose+gel:+basis+for+a+standardized+isoform+nomenclature&amp;author=S.M.+Marcovina&amp;author=H.H.+Hobbs&amp;author=J.J.+Albers&amp;volume=42&amp;publication_year=1996&amp;pages=436-439&amp;pmid=8598109&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C39">39. <span class="mixed-citation">Kraft H.G., Lingenhel A., Kochl S., Hoppichler F., Kronenberg F., Abe A., Muhlberger V., Schonitzer D., Utermann G. (1996) <span class="ref-title">Apolipoprotein(a) kringle IV repeat number predicts risk for coronary heart disease</span>. <span class="ref-journal">Arterioscler. Thromb. Vasc. Biol.</span>, <span class="ref-vol">16</span>, 713–719. [<a href="/pubmed/8640397" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler.+Thromb.+Vasc.+Biol.&amp;title=Apolipoprotein(a)+kringle+IV+repeat+number+predicts+risk+for+coronary+heart+disease&amp;author=H.G.+Kraft&amp;author=A.+Lingenhel&amp;author=S.+Kochl&amp;author=F.+Hoppichler&amp;author=F.+Kronenberg&amp;volume=16&amp;publication_year=1996&amp;pages=713-719&amp;pmid=8640397&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C40">40. <span class="mixed-citation">Crawford D.C., Peng Z., Cheng J.F., Boffelli D., Ahearn M., Nguyen D., Shaffer T., Yi Q., Livingston R.J., Rieder M.J., et al.  (2008) <span class="ref-title">LPA and PLG sequence variation and kringle IV-2 copy number in two populations</span>. <span class="ref-journal">Hum. Hered.</span>, <span class="ref-vol">66</span>, 199–209. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2861532/">PMC free article</a>]</span> [<a href="/pubmed/18612205" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hum.+Hered.&amp;title=LPA+and+PLG+sequence+variation+and+kringle+IV-2+copy+number+in+two+populations&amp;author=D.C.+Crawford&amp;author=Z.+Peng&amp;author=J.F.+Cheng&amp;author=D.+Boffelli&amp;author=M.+Ahearn&amp;volume=66&amp;publication_year=2008&amp;pages=199-209&amp;pmid=18612205&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C41">41. <span class="mixed-citation">Luke M.M., Kane J.P., Liu D.M., Rowland C.M., Shiffman D., Cassano J., Catanese J.J., Pullinger C.R., Leong D.U., Arellano A.R., et al.  (2007) <span class="ref-title">A polymorphism in the protease-like domain of apolipoprotein(a) is associated with severe coronary artery disease</span>. <span class="ref-journal">Arterioscler. Thromb. Vasc. Biol.</span>, <span class="ref-vol">27</span>, 2030–2036. [<a href="/pubmed/17569884" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler.+Thromb.+Vasc.+Biol.&amp;title=A+polymorphism+in+the+protease-like+domain+of+apolipoprotein(a)+is+associated+with+severe+coronary+artery+disease&amp;author=M.M.+Luke&amp;author=J.P.+Kane&amp;author=D.M.+Liu&amp;author=C.M.+Rowland&amp;author=D.+Shiffman&amp;volume=27&amp;publication_year=2007&amp;pages=2030-2036&amp;pmid=17569884&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C42">42. <span class="mixed-citation">Ng P.C., Henikoff S. (2003) <span class="ref-title">SIFT: predicting amino acid changes that affect protein function</span>. <span class="ref-journal">Nucleic Acids Res.</span>, <span class="ref-vol">31</span>, 3812–3814. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC168916/">PMC free article</a>]</span> [<a href="/pubmed/12824425" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=SIFT:+predicting+amino+acid+changes+that+affect+protein+function&amp;author=P.C.+Ng&amp;author=S.+Henikoff&amp;volume=31&amp;publication_year=2003&amp;pages=3812-3814&amp;pmid=12824425&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C43">43. <span class="mixed-citation">Adzhubei I.A., Schmidt S., Peshkin L., Ramensky V.E., Gerasimova A., Bork P., Kondrashov A.S., Sunyaev S.R. (2010) <span class="ref-title">A method and server for predicting damaging missense mutations</span>. <span class="ref-journal">Nat. Methods</span>, <span class="ref-vol">7</span>, 248–249. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2855889/">PMC free article</a>]</span> [<a href="/pubmed/20354512" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=A+method+and+server+for+predicting+damaging+missense+mutations&amp;author=I.A.+Adzhubei&amp;author=S.+Schmidt&amp;author=L.+Peshkin&amp;author=V.E.+Ramensky&amp;author=A.+Gerasimova&amp;volume=7&amp;publication_year=2010&amp;pages=248-249&amp;pmid=20354512&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C44">44. <span class="mixed-citation">Cross H.E. (1976) <span class="ref-title">Population studies and the Old Order Amish</span>. <span class="ref-journal">Nature</span>, <span class="ref-vol">262</span>, 17–20. [<a href="/pubmed/934323" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Population+studies+and+the+Old+Order+Amish&amp;author=H.E.+Cross&amp;volume=262&amp;publication_year=1976&amp;pages=17-20&amp;pmid=934323&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C45">45. <span class="mixed-citation">Mitchell B.D., McArdle P.F., Shen H., Rampersaud E., Pollin T.I., Bielak L.F., Jaquish C., Douglas J.A., Roy-Gagnon M.H., Sack P., et al.  (2008) <span class="ref-title">The genetic response to short-term interventions affecting cardiovascular function: rationale and design of the Heredity and Phenotype Intervention (HAPI) Heart Study</span>. <span class="ref-journal">Am. Heart J.</span>, <span class="ref-vol">155</span>, 823–828. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2443415/">PMC free article</a>]</span> [<a href="/pubmed/18440328" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+Heart+J.&amp;title=The+genetic+response+to+short-term+interventions+affecting+cardiovascular+function:+rationale+and+design+of+the+Heredity+and+Phenotype+Intervention+(HAPI)+Heart+Study&amp;author=B.D.+Mitchell&amp;author=P.F.+McArdle&amp;author=H.+Shen&amp;author=E.+Rampersaud&amp;author=T.I.+Pollin&amp;volume=155&amp;publication_year=2008&amp;pages=823-828&amp;pmid=18440328&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C46">46. <span class="mixed-citation">Sorkin J., Post W., Pollin T.I., O'Connell J.R., Mitchell B.D., Shuldiner A.R. (2005) <span class="ref-title">Exploring the genetics of longevity in the Old Order Amish</span>. <span class="ref-journal">Mech. Ageing Dev.</span>, <span class="ref-vol">126</span>, 347–350. [<a href="/pubmed/15621217" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mech.+Ageing+Dev.&amp;title=Exploring+the+genetics+of+longevity+in+the+Old+Order+Amish&amp;author=J.+Sorkin&amp;author=W.+Post&amp;author=T.I.+Pollin&amp;author=J.R.+O'Connell&amp;author=B.D.+Mitchell&amp;volume=126&amp;publication_year=2005&amp;pages=347-350&amp;pmid=15621217&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C47">47. <span class="mixed-citation">Kulkarni K.R., Garber D.W., Marcovina S.M., Segrest J.P. (1994) <span class="ref-title">Quantification of cholesterol in all lipoprotein classes by the VAP-II method</span>. <span class="ref-journal">J. Lipid Res.</span>, <span class="ref-vol">35</span>, 159–168. [<a href="/pubmed/8138718" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid+Res.&amp;title=Quantification+of+cholesterol+in+all+lipoprotein+classes+by+the+VAP-II+method&amp;author=K.R.+Kulkarni&amp;author=D.W.+Garber&amp;author=S.M.+Marcovina&amp;author=J.P.+Segrest&amp;volume=35&amp;publication_year=1994&amp;pages=159-168&amp;pmid=8138718&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C48">48. <span class="mixed-citation">Kulkarni K.R. (2006) <span class="ref-title">Cholesterol profile measurement by vertical auto profile method</span>. <span class="ref-journal">Clin. Lab. Med.</span>, <span class="ref-vol">26</span>, 787–802. [<a href="/pubmed/17110240" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Lab.+Med.&amp;title=Cholesterol+profile+measurement+by+vertical+auto+profile+method&amp;author=K.R.+Kulkarni&amp;volume=26&amp;publication_year=2006&amp;pages=787-802&amp;pmid=17110240&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C49">49. <span class="mixed-citation">Heard-Costa N.L., Zillikens M.C., Monda K.L., Johansson A., Harris T.B., Fu M., Haritunians T., Feitosa M.F., Aspelund T., Eiriksdottir G., et al.  (2009) <span class="ref-title">NRXN3 is a novel locus for waist circumference: a genome-wide association study from the CHARGE Consortium</span>. <span class="ref-journal">PLoS Genet.</span>, <span class="ref-vol">5</span>, e1000539. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2695005/">PMC free article</a>]</span> [<a href="/pubmed/19557197" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Genet.&amp;title=NRXN3+is+a+novel+locus+for+waist+circumference:+a+genome-wide+association+study+from+the+CHARGE+Consortium&amp;author=N.L.+Heard-Costa&amp;author=M.C.+Zillikens&amp;author=K.L.+Monda&amp;author=A.+Johansson&amp;author=T.B.+Harris&amp;volume=5&amp;publication_year=2009&amp;pages=e1000539&amp;pmid=19557197&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C50">50. <span class="mixed-citation">Fu M., Sabra M.M., Damcott C., Pollin T.I., Ma L., Ott S., Shelton J.C., Shi X., Reinhart L., O'Connell J., et al.  (2007) <span class="ref-title">Evidence that Rho guanine nucleotide exchange factor 11 (ARHGEF11) on 1q21 is a type 2 diabetes susceptibility gene in the Old Order Amish</span>. <span class="ref-journal">Diabetes</span>, <span class="ref-vol">56</span>, 1363–1368. [<a href="/pubmed/17369523" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Diabetes&amp;title=Evidence+that+Rho+guanine+nucleotide+exchange+factor+11+(ARHGEF11)+on+1q21+is+a+type+2+diabetes+susceptibility+gene+in+the+Old+Order+Amish&amp;author=M.+Fu&amp;author=M.M.+Sabra&amp;author=C.+Damcott&amp;author=T.I.+Pollin&amp;author=L.+Ma&amp;volume=56&amp;publication_year=2007&amp;pages=1363-1368&amp;pmid=17369523&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C51">51. <span class="mixed-citation">Angles-Cano E., Loyau S., Cardoso-Saldana G., Couderc R., Gillery P. (1999) <span class="ref-title">A novel kringle-4 number-based recombinant apo[a] standard for human apo[a] phenotyping</span>. <span class="ref-journal">J. Lipid Res.</span>, <span class="ref-vol">40</span>, 354–359. [<a href="/pubmed/9925666" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Lipid+Res.&amp;title=A+novel+kringle-4+number-based+recombinant+apo[a]+standard+for+human+apo[a]+phenotyping&amp;author=E.+Angles-Cano&amp;author=S.+Loyau&amp;author=G.+Cardoso-Saldana&amp;author=R.+Couderc&amp;author=P.+Gillery&amp;volume=40&amp;publication_year=1999&amp;pages=354-359&amp;pmid=9925666&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C52">52. <span class="mixed-citation">Schneider C.A., Rasband W.S., Eliceiri K.W. (2012) <span class="ref-title">NIH Image to ImageJ: 25 years of image analysis</span>. <span class="ref-journal">Nat. Methods</span>, <span class="ref-vol">9</span>, 671–675. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5554542/">PMC free article</a>]</span> [<a href="/pubmed/22930834" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=NIH+Image+to+ImageJ:+25+years+of+image+analysis&amp;author=C.A.+Schneider&amp;author=W.S.+Rasband&amp;author=K.W.+Eliceiri&amp;volume=9&amp;publication_year=2012&amp;pages=671-675&amp;pmid=22930834&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C53">53. <span class="mixed-citation">Agarwala R., Biesecker L.G., Hopkins K.A., Francomano C.A., Schaffer A.A. (1998) <span class="ref-title">Software for constructing and verifying pedigrees within large genealogies and an application to the Old Order Amish of Lancaster County</span>. <span class="ref-journal">Genome Res.</span>, <span class="ref-vol">8</span>, 211–221. [<a href="/pubmed/9521925" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res.&amp;title=Software+for+constructing+and+verifying+pedigrees+within+large+genealogies+and+an+application+to+the+Old+Order+Amish+of+Lancaster+County&amp;author=R.+Agarwala&amp;author=L.G.+Biesecker&amp;author=K.A.+Hopkins&amp;author=C.A.+Francomano&amp;author=A.A.+Schaffer&amp;volume=8&amp;publication_year=1998&amp;pages=211-221&amp;pmid=9521925&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="DDU731C54">54. <span class="mixed-citation">Agarwala R., Biesecker L.G., Tomlin J.F., Schaffer A.A. (1999) <span class="ref-title">Towards a complete North American Anabaptist genealogy: A systematic approach to merging partially overlapping genealogy resources</span>. <span class="ref-journal">Am. J. Med. Genet.</span>, <span class="ref-vol">86</span>, 156–161. [<a href="/pubmed/10449652" target="pmc_ext" ref="reftype=pubmed&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Med.+Genet.&amp;title=Towards+a+complete+North+American+Anabaptist+genealogy:+A+systematic+approach+to+merging+partially+overlapping+genealogy+resources&amp;author=R.+Agarwala&amp;author=L.G.+Biesecker&amp;author=J.F.+Tomlin&amp;author=A.A.+Schaffer&amp;volume=86&amp;publication_year=1999&amp;pages=156-161&amp;pmid=10449652&amp;" target="pmc_ext" ref="reftype=other&amp;article-id=4380064&amp;issue-id=251075&amp;journal-id=740&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar" role="button" aria-expanded="false" aria-haspopup="true">Google Scholar</a>]</span></span></div></div></div><div style="display: none; width: 200px; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbilinksmenu"><ul id="ui-ncbiinpagenav-2"><li><a href="#ui-ncbiinpagenav-heading-3">Associated Data</a></li><li><a href="#idm140237399124768title">Abstract</a></li><li><a href="#s1title">Introduction</a></li><li><a href="#s2title">Results</a></li><li><a href="#s3title">Discussion</a></li><li><a href="#s4title">Materials and Methods</a></li><li><a href="#s5title">Supplementary Material</a></li><li><a href="#s6title">Funding</a></li><li><a href="#__sec1title">Supplementary Material</a></li><li><a href="#idm140237396560560title">Acknowledgements</a></li><li><a href="#idm140237396558272title">References</a></li></ul></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr>Articles from <span class="acknowledgment-journal-title">Human Molecular Genetics</span> are provided here courtesy of <strong>Oxford University Press</strong></div></div>
            
            
        
            
        </div>
        <!-- Book content -->
    </div>
    
    <div id="rightcolumn" class="four_col col last">
        <!-- Custom content above discovery portlets -->
        <div class="col6">
            
        </div>
        
        <div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"><div class="format-menu"><h2>Formats:</h2><ul><li class="selected">Article</li> | <li><a href="/pmc/articles/PMC4380064/?report=reader">PubReader</a></li> | <li class="epub-link"><a href="/pmc/articles/PMC4380064/epub/">ePub (beta)</a></li> | <li><a href="/pmc/articles/PMC4380064/pdf/ddu731.pdf">PDF (670K)</a></li> | <li><a href="#" data-citationid="PMC4380064" class="citationexporter ctxp" role="button" aria-expanded="false" aria-haspopup="true">Citation</a></li></ul></div></div><div xmlns:np="http://ncbi.gov/portal/XSLT/namespace" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" class="share-buttons"><h2>Share</h2><ul><li class="facebook"><a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4380064%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047626" alt="Share on Facebook">
                             Facebook
                        </a></li><li class="twitter"><a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4380064%2F&amp;text=Evidence%20for%20several%20independent%20genetic%20variants%20affecting%20lipoprotein%20(a)%20cholesterol%20levels" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047627" alt="Share on Twitter">
                             Twitter
                        </a></li><li class="gplus"><a href="https://plus.google.com/share?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4380064%2F" role="button" aria-expanded="false" aria-haspopup="true" target="_blank"><img src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/img/4047628" alt="Share on Google Plus">
                             Google+
                        </a></li></ul></div>
        
        <div id="ajax-portlets" data-pmid="25575512" data-aiid="4380064" data-aid="4380064" data-iid="251075" data-domainid="740" data-domain="hmg" data-accid="PMC4380064" data-md5="d557650dd8dc13d42ee7d836614e1845"><img src="/portal/portal3rc.fcgi/4018218/img/3631476" alt="Loading..." align="absmiddle"> Loading portlets...</div>
                
        <!-- Custom content below discovery portlets -->
        <div class="col7">
            
        </div>
    </div>
</div>

<!-- Custom content after all -->
<div class="col8">
    
</div>
<div class="col9">
    
</div>

<script src="/corehtml/pmc/js/jquery.scrollTo-1.4.2.js"></script>
<script>
    (function($){
        $('.skiplink').each(function(i, item){
            var href = $($(item).attr('href'));
            href.attr('tabindex', '-1').addClass('skiptarget'); // ensure the target can receive focus
            $(item).on('click', function(event){
                event.preventDefault();
                $.scrollTo(href, 0, {
                    onAfter: function(){
                        href.focus();
                    }
                });
            });
        });
    })(jQuery);
</script>



<div id="body-link-poppers"></div>
                        </div>
                        <div class="bottom">
                            
                            <div id="NCBIFooter_dynamic">
    <a id="help-desk-link" class="help_desk" href="" target="_blank">Support Center</a>
    <a id="help-desk-link" class="help_desk" href="https://support.ncbi.nlm.nih.gov/ics/support/KBList.asp?Time=2019-11-20T13:12:25-05:00&amp;Snapshot=%2Fprojects%2FPMC%2FPMCViewer@4.46&amp;Host=portal102&amp;ncbi_phid=CE894FC6DD57FF6100000000015400BD&amp;ncbi_session=CE8C6010DD4F8B31_0740SID&amp;from=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4380064%2F&amp;Db=pmc&amp;folderID=132&amp;Ncbi_App=pmc&amp;Page=literature&amp;style=classic&amp;deptID=28049" target="_blank">Support Center</a>
    
</div>

                            <div class="footer" id="footer">
    
    <div class="subfooter"> </div><script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/preloaderWidget.js"> </script>
    <div id="external-disclaimer" class="offscreen_noflow">
        External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.
    </div>    
    <div id="ncbifooter" class="contact_info">      
        <div id="footer-contents-right">
            <div id="nlm_thumb_logo">
                <a href="https://www.nlm.nih.gov" title="NLM">NLM</a>
            </div>
            <div id="nih_thumb_logo">
                <a href="https://www.nih.gov" title="NIH">NIH</a>
            </div>
            <div id="hhs_thumb_logo">
                <a href="https://www.hhs.gov" title="DHHS">DHHS</a>
            </div>
            <div id="usagov_thumb_logo">
                <a href="https://www.usa.gov" title="USA.gov">USA.gov</a>
            </div>         
        </div>
        
        <div id="footer-contents-left">
            <p class="address vcard">
                <span class="url">
                    <a class="fn url newdomain" href="https://www.ncbi.nlm.nih.gov">National Center for
                        Biotechnology Information</a>,
                </span> <span class="org url newdomain"><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></span>
                <span class="adr">
                    <span class="street-address">8600 Rockville Pike</span>, <span class="locality">Bethesda</span>
                    <span class="region">MD</span>, <span class="postal-code">20894</span>
                    <span class="country-name">USA</span>
                </span>
            </p>
            
            <a href="/home/about/policies.shtml">Policies and Guidelines</a> | <a href="/home/about/contact.shtml">Contact</a>
        </div>
    </div>
    <script type="text/javascript" src="/portal/portal3rc.fcgi/rlib/js/InstrumentOmnitureBaseJS/InstrumentNCBIConfigJS/InstrumentNCBIBaseJS/InstrumentPageStarterJS.js?v=1"> </script>    
    <script type="text/javascript" src="/portal/portal3rc.fcgi/static/js/hfjs2.js"> </script>
</div>
                        </div>
                    </div>
                    <!--/.page-->
                </div>
                <!--/.wrap-->
            </div><!-- /.twelve_col -->
        <div class="ui-ncbiautocomplete-holder shadow ui-ncbiautocomplete-holder-clearfix" aria-live="assertive" style="display: none; top: 68.1px; left: 377.2px; width: 670px; z-index: 1001;"><ul class="ui-ncbiautocomplete-options" role="listbox" aria-activedescendant="term"></ul><div class="ui-ncbiautocomplete-actions shadow" style="display: block;"><a class="ui-ncbiautocomplete-link-pref" style="display: none;">Preferences</a><a href="#" class="ui-ncbiautocomplete-link-off ui-ncbiautocomplete-link-pref-right" style="display: block;">Turn off</a></div></div><div style="display: none; top: -100px; left: -100px;" aria-live="assertive" aria-hidden="true" class="ui-helper-reset ui-ncbipopper-wrapper ui-ncbipopper-basic">External link. Please review our <a href="https://www.nlm.nih.gov/privacy.html">privacy policy</a>.</div></div>
        <!-- /.grid -->

        <span class="PAFAppResources"></span>
        
        <!-- BESelector tab -->
        
        
        
        <noscript><img alt="statistics" src="/stat?jsdisabled=true&amp;ncbi_db=pmc&amp;ncbi_pdid=article&amp;ncbi_acc=&amp;ncbi_domain=hmg&amp;ncbi_report=record&amp;ncbi_type=fulltext&amp;ncbi_objectid=&amp;ncbi_pcid=/articles/PMC4380064/&amp;ncbi_app=pmc" /></noscript>
        
        
        <!-- usually for JS scripts at page bottom -->
        <!--<component id="PageFixtures" label="styles"></component>-->
    

<!-- CE8C6010DD4F8B31_0740SID /projects/PMC/PMCViewer@4.46 portal102 v4.1.r585844 Mon, May 06 2019 02:53:16 -->

<script type="text/javascript" src="//static.pubmed.gov/portal/portal3rc.fcgi/4160049/js/3879255/4121861/3818874/4168176/3821238/4117325/4087685/4072593/4076480/3921943/4105668/4065628.js" snapshot="pmc"></script>
<div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="epubDialog" aria-labelledby="ui-id-1"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-1" class="ui-dialog-title">Making articles easier to read in PMC</span><button type="button" class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" title="Close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">Close</span></button></div><div id="epubDialog" style="display: block;" class="ui-dialog-content ui-widget-content">  <p>We are experimenting with display styles that make it easier to read articles     in PMC.     Our first effort uses eBook readers, which have several "ease of reading"     features already built in.</p>  <p>These PMC articles are best viewed in the <em>iBooks     reader</em>. You may notice problems with the display of certain parts of an article     in other eReaders.</p>  <div class="ui-dialog-buttonpane ui-widget-content ui-helper-clearfix"><button id="cancelEpub" class="ui-state-default ui-corner-all">Cancel</button><button id="downloadEpub" style="float: left" class="ui-state-default ui-corner-all">Download article</button></div>          </div></div></body></html>